
<html lang="en"     class="pb-page"  data-request-id="d5ef0afb-b117-46a1-9630-781a12e35947"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2019.62.issue-15;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.9b00757"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Novel Human Aminopeptidase N Inhibitors: Discovery and Optimization of Subsite Binding Interactions" /></meta><meta name="dc.Creator" content="Jisook  Lee" /></meta><meta name="dc.Creator" content="Natalie B.  Vinh" /></meta><meta name="dc.Creator" content="Nyssa  Drinkwater" /></meta><meta name="dc.Creator" content="Wei  Yang" /></meta><meta name="dc.Creator" content="Komagal  Kannan Sivaraman" /></meta><meta name="dc.Creator" content="Luke S.  Schembri" /></meta><meta name="dc.Creator" content="Michelle  Gazdik" /></meta><meta name="dc.Creator" content="Peter M.  Grin" /></meta><meta name="dc.Creator" content="Georgina S.  Butler" /></meta><meta name="dc.Creator" content="Christopher M.  Overall" /></meta><meta name="dc.Creator" content="Susan A.  Charman" /></meta><meta name="dc.Creator" content="Sheena  McGowan" /></meta><meta name="dc.Creator" content="Peter J.  Scammells" /></meta><meta name="dc.Description" content="Aminopeptidase N (APN/CD13) is a zinc-dependent M1 aminopeptidase that contributes to cancer progression by promoting angiogenesis, metastasis, and tumor invasion. We have previously identified hyd..." /></meta><meta name="Description" content="Aminopeptidase N (APN/CD13) is a zinc-dependent M1 aminopeptidase that contributes to cancer progression by promoting angiogenesis, metastasis, and tumor invasion. We have previously identified hyd..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 28, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00757" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00757" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00757" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00757" /></link>
        
    
    

<title>Novel Human Aminopeptidase N Inhibitors: Discovery and Optimization of Subsite Binding Interactions | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00757" /></meta><meta property="og:title" content="Novel Human Aminopeptidase N Inhibitors: Discovery and Optimization of Subsite Binding Interactions" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0011.jpeg" /></meta><meta property="og:description" content="Aminopeptidase N (APN/CD13) is a zinc-dependent M1 aminopeptidase that contributes to cancer progression by promoting angiogenesis, metastasis, and tumor invasion. We have previously identified hydroxamic acid-containing analogues that are potent inhibitors of the APN homologue from the malarial parasite Plasmodium falciparum M1 aminopeptidase (PfA-M1). Herein, we describe the rationale that underpins the repurposing of PfA-M1 inhibitors as novel APN inhibitors. A series of novel hydroxamic acid analogues were developed using a structure-based design approach and evaluated their inhibition activities against APN. N-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)-4-(methylsulfonamido)benzamide (6ad) proved to be an extremely potent inhibitor of APN activity in vitro, selective against other zinc-dependent enzymes such as matrix metalloproteases, and possessed limited cytotoxicity against Ad293 cells and favorable physicochemical and metabolic stability properties. The combined results indicate that compound 6ad may be a useful lead for the development of anticancer agents." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00757"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00757">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00757&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00757&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00757&amp;href=/doi/10.1021/acs.jmedchem.9b00757" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 15</span><span class="cit-fg-pageRange">, 7185-7209</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/15" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00735" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00809" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Novel Human Aminopeptidase N Inhibitors: Discovery and Optimization of Subsite Binding Interactions</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Jisook Lee</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jisook Lee</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, Parkville Campus, Parkville, VIC 3052, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jisook++Lee">Jisook Lee</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Natalie B. Vinh</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Natalie B. Vinh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, Parkville Campus, Parkville, VIC 3052, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Natalie+B.++Vinh">Natalie B. Vinh</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nyssa Drinkwater</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nyssa Drinkwater</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Infection and Immunity Program, Biomedicine Discovery Institute and Department of Microbiology, Monash University, Clayton Campus, Clayton, VIC 3800, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nyssa++Drinkwater">Nyssa Drinkwater</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wei Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wei Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Infection and Immunity Program, Biomedicine Discovery Institute and Department of Microbiology, Monash University, Clayton Campus, Clayton, VIC 3800, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wei++Yang">Wei Yang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Komagal Kannan Sivaraman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Komagal Kannan Sivaraman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Infection and Immunity Program, Biomedicine Discovery Institute and Department of Microbiology, Monash University, Clayton Campus, Clayton, VIC 3800, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Komagal++Kannan+Sivaraman">Komagal Kannan Sivaraman</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Luke S. Schembri</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Luke S. Schembri</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, Parkville Campus, Parkville, VIC 3052, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Luke+S.++Schembri">Luke S. Schembri</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5588-1077" title="Orcid link">http://orcid.org/0000-0001-5588-1077</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michelle Gazdik</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michelle Gazdik</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, Parkville Campus, Parkville, VIC 3052, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michelle++Gazdik">Michelle Gazdik</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter M. Grin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter M. Grin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry & Molecular Biology,  Centre for Blood Research, Life Sciences Institute, University of British Columbia, Vancouver, British Columbia V6T 1Z3, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter+M.++Grin">Peter M. Grin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Georgina S. Butler</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Georgina S. Butler</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry & Molecular Biology,  Department of Oral Biological & Medical Sciences,  Centre for Blood Research, Life Sciences Institute, University of British Columbia, Vancouver, British Columbia V6T 1Z3, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Georgina+S.++Butler">Georgina S. Butler</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christopher M. Overall</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christopher M. Overall</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry & Molecular Biology,  Department of Oral Biological & Medical Sciences,  Centre for Blood Research, Life Sciences Institute, University of British Columbia, Vancouver, British Columbia V6T 1Z3, Canada</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christopher+M.++Overall">Christopher M. Overall</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5844-2731" title="Orcid link">http://orcid.org/0000-0001-5844-2731</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Susan A. Charman</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Susan A. Charman</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Centre for Drug Candidate Optimisation, Monash Institute of Pharmaceutical Sciences, Monash University, Parkville Campus, Parkville, VIC 3052, Australia</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Susan+A.++Charman">Susan A. Charman</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sheena McGowan</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sheena McGowan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Infection and Immunity Program, Biomedicine Discovery Institute and Department of Microbiology, Monash University, Clayton Campus, Clayton, VIC 3800, Australia</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#6a39020f0f040b4427092d051d0b042a0705040b1902440f0e1f"><span class="__cf_email__" data-cfemail="5e0d363b3b303f70133d1931293f301e3331303f2d36703b3a2b">[email protected]</span></a>. Phone: +61 (0)3 9902 9309 (S.M.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sheena++McGowan">Sheena McGowan</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6863-1106" title="Orcid link">http://orcid.org/0000-0001-6863-1106</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Peter J. Scammells</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter J. Scammells</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Monash Institute of Pharmaceutical Sciences, Monash University, Parkville Campus, Parkville, VIC 3052, Australia</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#09596c7d6c7b275a6a6864646c65657a49646667687a61276c6d7c"><span class="__cf_email__" data-cfemail="acfcc9d8c9de82ffcfcdc1c1c9c0c0dfecc1c3c2cddfc482c9c8d9">[email protected]</span></a>. Phone: +61 (0)3 9903 9542 (P.J.S.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter+J.++Scammells">Peter J. Scammells</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2930-895X" title="Orcid link">http://orcid.org/0000-0003-2930-895X</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00757&amp;href=/doi/10.1021%2Facs.jmedchem.9b00757" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 15</span><span class="cit-pageRange">, 7185–7209</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 28, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>8 May 2019</li><li><span class="item_label"><b>Published</b> online</span>28 June 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 8 August 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00757" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00757</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7185%26pageCount%3D25%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJisook%2BLee%252C%2BNatalie%2BB.%2BVinh%252C%2BNyssa%2BDrinkwater%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D15%26contentID%3Dacs.jmedchem.9b00757%26title%3DNovel%2BHuman%2BAminopeptidase%2BN%2BInhibitors%253A%2BDiscovery%2Band%2BOptimization%2Bof%2BSubsite%2BBinding%2BInteractions%26numPages%3D25%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7209%26publicationDate%3DAugust%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00757"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1272</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">11</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00757" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Novel Human Aminopeptidase N Inhibitors: Discovery and Optimization of Subsite Binding Interactions&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jisook&quot;,&quot;last_name&quot;:&quot;Lee&quot;},{&quot;first_name&quot;:&quot;Natalie&quot;,&quot;last_name&quot;:&quot;B. Vinh&quot;},{&quot;first_name&quot;:&quot;Nyssa&quot;,&quot;last_name&quot;:&quot;Drinkwater&quot;},{&quot;first_name&quot;:&quot;Wei&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Komagal&quot;,&quot;last_name&quot;:&quot;Kannan Sivaraman&quot;},{&quot;first_name&quot;:&quot;Luke&quot;,&quot;last_name&quot;:&quot;S. Schembri&quot;},{&quot;first_name&quot;:&quot;Michelle&quot;,&quot;last_name&quot;:&quot;Gazdik&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;M. Grin&quot;},{&quot;first_name&quot;:&quot;Georgina&quot;,&quot;last_name&quot;:&quot;S. Butler&quot;},{&quot;first_name&quot;:&quot;Christopher&quot;,&quot;last_name&quot;:&quot;M. Overall&quot;},{&quot;first_name&quot;:&quot;Susan&quot;,&quot;last_name&quot;:&quot;A. Charman&quot;},{&quot;first_name&quot;:&quot;Sheena&quot;,&quot;last_name&quot;:&quot;McGowan&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;J. Scammells&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;28&quot;,&quot;issue&quot;:&quot;15&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;7185-7209&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00757&quot;},&quot;abstract&quot;:&quot;Aminopeptidase N (APN/CD13) is a zinc-dependent M1 aminopeptidase that contributes to cancer progression by promoting angiogenesis, metastasis, and tumor invasion. We have previously identified hydroxamic acid-containing analogues that are potent inhibitors of the APN homologue from the malarial parasite Plasmodium falciparum M1 aminopeptidase (PfA-M1). Herein, we describe the rationale that underpins the repurposing of PfA-M1 inhibitors as novel APN inhibitors. A series of novel hydroxamic acid analogues were developed using a structure-based design approach and evaluated their inhibition activities against APN. N-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)-4-(methylsulfonamido)benzamide (6ad) proved to be an extremely potent inhibitor of APN activity in vitro, selective against other zinc-dependent enzymes such as matrix metalloproteases, and possessed limited cytotoxicity against Ad293 cells and favorable physicochemical and metabolic stability properties. The combined results&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00757&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00757" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00757&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00757" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00757&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00757" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00757&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00757&amp;href=/doi/10.1021/acs.jmedchem.9b00757" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00757" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00757" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00757&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00757%26sid%3Dliteratum%253Aachs%26pmid%3D31251594%26genre%3Darticle%26aulast%3DLee%26date%3D2019%26atitle%3DNovel%2BHuman%2BAminopeptidase%2BN%2BInhibitors%253A%2BDiscovery%2Band%2BOptimization%2Bof%2BSubsite%2BBinding%2BInteractions%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D15%26spage%3D7185%26epage%3D7209%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (5)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291984" title="Molecular mechanics">Molecular mechanics</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292529" title="Aromatic compounds">Aromatic compounds</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/15" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/jmcmar.2019.62.issue-15/20190808/jmcmar.2019.62.issue-15.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/medium/jm-2019-007578_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00757&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Aminopeptidase N (APN/CD13) is a zinc-dependent M1 aminopeptidase that contributes to cancer progression by promoting angiogenesis, metastasis, and tumor invasion. We have previously identified hydroxamic acid-containing analogues that are potent inhibitors of the APN homologue from the malarial parasite <i>Plasmodium falciparum</i> M1 aminopeptidase (<i>Pf</i>A-M1). Herein, we describe the rationale that underpins the repurposing of <i>Pf</i>A-M1 inhibitors as novel APN inhibitors. A series of novel hydroxamic acid analogues were developed using a structure-based design approach and evaluated their inhibition activities against APN. <i>N</i>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)-4-(methylsulfonamido)benzamide (<b>6ad</b>) proved to be an extremely potent inhibitor of APN activity in vitro, selective against other zinc-dependent enzymes such as matrix metalloproteases, and possessed limited cytotoxicity against Ad293 cells and favorable physicochemical and metabolic stability properties. The combined results indicate that compound <b>6ad</b> may be a useful lead for the development of anticancer agents.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec fnGroup"><div class="fnGroupItem" id="fn9999a"><span><h4></h4></span><span><p class="last">May account for two carbons, as there are many overlapping quaternary carbons in this region.</p></span></div></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12200" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12200" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Aminopeptidase N (APN/CD13; EC 3.4.11.2) is a ubiquitous transmembrane ectoenzyme that is widely present in different types of cells, including renal, intestinal, fibroblast, endothelial, and tumor cells.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> APN is described as a “moonlighting” enzyme due to its multifunctional roles: an enzyme to cleave peptide substrates, a receptor, and a signaling molecule.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The enzyme cleaves hydrophobic and basic amino acid residues from the N-terminus of polypeptides with broad substrate specificity.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> For example, APN catalyzes the metabolism of angiotensin III to generate angiotensin IV to regulate the renin–angiotensin system<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> and levels of neuropeptides such as enkephalins.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> APN also acts as a viral receptor for mammalian coronavirus, and is a signaling molecule in phagocytosis, angiogenesis, and cell adhesion.<a onclick="showRef(event, 'ref8 ref9 ref10 ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11 ref12 ref13">(8−13)</a></div><div class="NLM_p">APN is a member of the zinc-dependent M1 aminopeptidase superfamily of enzymes (protease clan MA) that can be found in all kingdoms of life except viruses.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> M1 aminopeptidases are characterized by a thermolysin fold and two consensus sequence motifs: an HEXXHX<sub>18</sub>E zinc-binding motif and a GXMEN substrate-guiding motif.<a onclick="showRef(event, 'ref4 ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref4 ref14 ref15 ref16">(4,14−16)</a> Wong et al. reported the X-ray crystal structure of human APN, as well as the structures of APN bound to generic inhibitors bestatin and amastatin, and an endogeneous peptide substrate, Angiotensin IV.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Human APN consists of a short intracellular tail, a transmembrane anchor, a small serine-/threonine-rich extracellular stalk, and a large ectodomain, composed of four domains (I–IV) characteristic of M1 aminopeptidase superfamily.<a onclick="showRef(event, 'ref4 ref14'); return false;" href="javascript:void(0);" class="ref ref4 ref14">(4,14)</a> The catalytic domain II contains the consensus motifs <sup>352</sup>GXMEN<sup>356</sup> and <sup>388</sup>HEXXHX<sub>18</sub>E<sup>411</sup>, the latter of which includes the catalytic triad His<sup>388</sup>, His<sup>392</sup>, and Glu<sup>411</sup> that coordinates the essential zinc ion.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">APN has been extensively studied due to its significant role in the regulation of metastasis and angiogenesis.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> A significant body of evidence supports the rationale that APN is an effective therapeutic target for malignancies.<a onclick="showRef(event, 'ref17 ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19 ref20">(17−20)</a> Dysregulated activity of APN has been found to develop into a wide spectrum of human malignancies.<a onclick="showRef(event, 'ref21 ref22 ref23 ref24 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23 ref24 ref25 ref26 ref27">(21−27)</a> Metastasis is a complex multistep process of cell migration, cell invasion, and angiogenesis, and it is the major cause of cancer-related deaths worldwide.<a onclick="showRef(event, 'ref28 ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30 ref31">(28−31)</a> Multiple studies have shown that APN activity is involved in extracellular matrix (ECM) degradation, an essential step for metastasis, which was later determined to increase tumor cell migration and invasion by stimulating mitogen-activated protein kinases/phosphoinositide 3-kinase signaling cascade.<a onclick="showRef(event, 'ref24 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref24 ref32 ref33">(24,32,33)</a> Thus, there is an ongoing interest to develop potent APN inhibitors as effective anticancer drug candidates.</div><div class="NLM_p">A variety of APN inhibitors have been developed as potential anticancer candidates.<a onclick="showRef(event, 'ref18 ref19 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref34 ref35">(18,19,34,35)</a> Among them, a natural peptidomimetic, bestatin, is the most widely studied competitive APN inhibitor. Originally isolated from <i>Streptomyces olivoreticuli</i> as an immunomodulating agent, bestatin was found to have antitumor activity<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36,37)</a> as well as clinical efficacy against acute myeloid leukemia and lung cancer in clinical trials.<a onclick="showRef(event, 'ref38 ref39 ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref38 ref39 ref40 ref41">(38−41)</a> Another APN inhibitor Tosedostat (CHR2797) is an orally bioavailable prodrug that is converted to a pharmacologically active drug (CHR79888) inside cells.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Tosedostat demonstrated significant antileukemic activity in phase II clinical trials in elderly or relapsing patients with acute myeloid leukemia.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div><div class="NLM_p">Previous work by our group generated a series of hydroxamic acid-containing compounds that were inhibitors of the <i>Plasmodium falciparum</i> M1 aminopeptidase, <i>Pf</i>A-M1.<a onclick="showRef(event, 'ref44 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref44 ref45 ref46">(44−46)</a> We described <i>N</i>-(2-(hydroxyamino)-2-oxo-1-[3′,4′,5′-trifluoro(1,1′-biphenyl)-4-yl]ethyl)pivalamide (<b>1</b>) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) as a potent inhibitor of <i>Pf</i>A-M1, exhibiting an inhibitory constant (<i>K</i><sub>i</sub><sup>(app)</sup>) in the nanomolar range.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Here, we show that <b>1</b> is also active toward APN and is more potent than both bestatin and Tosedostat. We have repurposed compound <b>1</b> as a novel APN inhibitor and developed a new series of analogues with improved potency against APN through structure-based approaches.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/medium/jm-2019-007578_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0001.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structure of <i>N</i>-(2-(hydroxyamino)-2-oxo-1-[3′,4′,5′-trifluoro(1,1′-biphenyl)-4-yl]ethyl)pivalamide (<b>1</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00757&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20186" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20186" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Compound <b>1</b> Can Inhibit Recombinant Human APN</h3><div class="NLM_p"><i>Pf</i>A-M1 is a homologue of APN found in <i>P. falciparum</i>, which is one of the parasites that causes malaria. Being part of the same M1 aminopeptidase family, <i>Pf</i>A-M1 shares a number of structural similarities with APN, particularly within the catalytic domain II. Overall, <i>Pf</i>A-M1 and APN share 19% sequence identity (35% similarity); however, in the highly conserved catalytic domain II, this sequence identity increases to 24% (43% similarity). This conserved catalytic domain II adopts a thermolysin-like fold and in <i>Pf</i>A-M1, contains an H<sup>496</sup>EYFHX<sub>17</sub>KE<sup>519</sup> zinc-binding motif as well as a G<sup>490</sup>AMEN substrate-guiding motif.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> The catalytic zinc in the active site is coordinated by a catalytic triad His<sup>496</sup>, His<sup>500</sup>, and Glu<sup>519</sup> in the unbound state.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Therefore, we were interested to see how the potent <i>Pf</i>A-M1 inhibitor <b>1</b> would interact with human APN.</div><div class="NLM_p">Our standard fluorescence-based aminopeptidase inhibition assay was used to measure the inhibitory activities of bestatin, Tosedostat, and <b>1</b> against human APN. This assay used recombinant human APN and a commercially available fluorophore 7-amino-4-methylcoumarin as the competitive substrate to determine an inhibitory constant (<i>K</i><sub>i</sub><sup>(app)</sup>). We compared the inhibitory activities of bestatin, Tosedostat, and <b>1</b> against human APN as well as <i>Pf</i>A-M1 (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Bestatin showed a moderate loss in potency toward APN compared to <i>Pf</i>A-M1, whereas Tosedostat exhibited a 6-fold improved potency toward APN compared to <i>Pf</i>A-M1. Interestingly, compound <b>1</b> was significantly more potent than Tosedostat and bestatin, displaying a 10- to 20-fold increase in APN inhibition activity.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. <i>K</i><sub>i</sub><sup>(app)</sup> Comparison of Bestatin, Tosedostat, and Compound <b>1</b> against Human APN and <i>Pf</i>A-M1</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>i</sub><sup>(app)</sup>(APN) ± SEM (nM)</th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>i</sub><sup>(app)</sup>(<i>Pf</i>A-M1) ± SEM (nM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bestatin</td><td class="colsep0 rowsep0" align="char" char="±">2370 ± 350</td><td class="colsep0 rowsep0" align="char" char="±">1530 ± 58</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Tosedostat</td><td class="colsep0 rowsep0" align="char" char="±">1180 ± 8</td><td class="colsep0 rowsep0" align="char" char="±">6150 ± 275</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char="±">118 ± 3</td><td class="colsep0 rowsep0" align="char" char="±">331 ± 12</td></tr></tbody></table></div></div><div class="NLM_p">To understand the mechanism by which <b>1</b> was able to inhibit both APN and <i>Pf</i>A-M1, we investigated the interactions the compound made with the active site of the enzymes. To do this, we used the X-ray crystal structure of <i>Pf</i>A-M1 bound to compound <b>1</b> [Protein Data Bank (PDB) ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZX4">4ZX4</a>]<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> as a scaffold to dock <b>1</b> into the active site of APN (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYQ">4FYQ</a>).<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> The catalytic domains of the two proteins share 24% sequence identity and have a root-mean-square deviation (RMSD) of only 1.244 Å (over 257 Cα atoms in domain II). The co-crystal structure of <b>1</b> bound to <i>Pf</i>A-M1 revealed extensive water-mediated interactions of the 3,4,5-trifluorophenyl ring with backbone residues located at the S1 substrate binding pocket, as well as key hydrophobic interactions with the biaryl system of <b>1</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A).<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Our docking analysis of <b>1</b> bound to APN showed a similar pose to that observed when bound to <i>Pf</i>A-M1 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). Each of the poses obtained from docking <b>1</b> were similar and showed the 3,4,5-trifluorophenyl ring in the same position, located deep within the S1 pocket of APN, and minor rotations of the position of the <i>N</i>-pivaloyl group were observed. In general, there were significantly less interactions observed between <b>1</b> and the active site residues of APN than that of <i>Pf</i>A-M1 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/medium/jm-2019-007578_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) X-ray crystal structure of <b>1</b> bound to <i>Pf</i>A-M1 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZX4">4ZX4</a>). (B) Predicted binding pose of <b>1</b> bound to APN obtained from docking (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYQ">4FYQ</a>). (C) Molecular dynamics (MD) simulation of <b>1</b> docked to APN showed that the trifluorophenyl group formed stable interactions in the S1 pocket. The ligand is shown in magenta, and residues of <i>Pf</i>A-M1 and APN are colored in blue and dark green, respectively. Interactions between the ligand and proteins are depicted as black dashed lines. The zinc is represented by the gray sphere.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00757&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To evaluate any potential dynamics of <b>1</b> bound within the active site of APN as well as any effect from water-mediated interactions, we performed molecular dynamics (MD) simulations of <b>1</b> docked into APN. Molecular modeling for metallo-proteins presents significant challenges, and traditional force fields are often not appropriate for simulation.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> In the case of APN, the presence of zinc in the active site means that this problem cannot be computationally ignored. Recently, our team has produced the necessary parameters to use the zinc Amber force field (ZAFF) to simulate the active site of <i>Pf</i>A-M1.<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a> We used this system to simulate our docking of APN bound to <b>1</b>. MD simulations (<i>n</i> = 3) were performed for the duration of 50 ns, which should be sufficient to observe the movements of small molecules. The results were surprising and showed that in two of the three MD runs, <b>1</b> experienced significant movement within the active site (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_001.pdf" class="ext-link">Figure S2</a>). Investigation into the motion of <b>1</b> indicated that the 3,4,5-trifluorophenyl ring engaged in interactions with a single water molecule and largely maintained face–face stacking interactions with relatively rigid Phe<sup>472</sup> at the S1 pocket of APN (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). In addition, the pivaloyl group was facing the aromatic side chain of Tyr<sup>477</sup> residue. We hypothesized therefore that compounds that are capable of improved interactions in the S1′ subsite as well as extending further to engage residues beyond the S1′ pocket of APN may contribute to improved inhibitory activity toward APN. To this end, we turned our attention to residues Tyr<sup>477</sup>, Arg<sup>381</sup>, and Arg<sup>442</sup> that are located within the S1′ and beyond the S1′ pocket of APN which may allow the formation of hydrophobic interactions as well as polar interactions with inhibitor compound(s).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Substitution of the <i>N</i>-Pivaloyl Group To Optimize Binding at the S1′ Subsite of APN</h3><div class="NLM_p">The molecular modeling revealed the potential to achieve enhanced binding interactions at, and beyond, the S1′ pocket of APN by replacing the <i>N</i>-pivaloyl group of <b>1</b> with aromatic groups that target the Tyr<sup>477</sup> residue. We designed and produced a series of hydroxamic acid analogues that contained elongated alkyl and aryl linkers to reach deeper into the pocket (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). Various hydrogen-bonding groups were also incorporated to capture additional interactions with Arg<sup>381</sup> or Arg<sup>442</sup> residues and improve inhibition activities of the designed compounds. In addition, we were interested to investigate the effect of various heteroaromatic groups, such as indole and indoline. Analogues with benzyl linkers that contain an additional methylene group were also designed to increase flexibility and allow the phenyl group to penetrate more deeply beyond the S1′ pocket.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/medium/jm-2019-007578_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structures of targeted hydroxamic acid analogues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00757&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Chemistry</h3><div class="NLM_p">The synthesis of the key intermediate, the phenyl glycine derivative <b>4</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>), was adapted from Mistry et al.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Installation of the 3,4,5-trifluorophenyl ring was successfully achieved using a Suzuki coupling reaction between 4′-bromoacetophenone and (3,4,5-trifluorophenyl)boronic acid under reported conditions producing acetophenone <b>2</b> in excellent yield (99%).<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Oxidation of <b>2</b> with selenium dioxide in anhydrous pyridine<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> afforded the corresponding α-keto acid <b>3</b> in quantitative yield. The product produced from the reductive amination of <b>3</b> underwent an acid-catalyzed esterification and subsequent debenzylation to successfully afford key precursor <b>4</b> in 47% yield over three steps.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/medium/jm-2019-007578_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0007.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Hydroxamic Acid Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00757&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 3,4,5-trifluorophenylboronic acid, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, 1 M Na<sub>2</sub>CO<sub>3</sub>, tetrahydrofuran (THF), reflux, 2 h; (b) SeO<sub>2</sub>, anhyd. pyridine, reflux, 24 h; (c) (i) benzylamine, Na(OAc)<sub>3</sub>BH, dichloroethane (DCE), room temperature (rt), 24 h; (ii) concd H<sub>2</sub>SO<sub>4</sub>, MeOH, reflux, 24 h; (iii) H<sub>2</sub>, 10% Pd/C, cat. HCl MeOH, rt, 24 h; (d) (i) carbamoylbenzoic acid, (COCl)<sub>2</sub>, dimethylformamide (DMF), dichloromethane (DCM), rt, 1 h; (ii) <b>4</b>, <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA), DCM, rt, 30 min; (e) carboxylic acid, <i>O</i>-(1<i>H</i>-6-chlorobenzotriazol-l-yl)-1,1,3,3-tetramethyluronium hexafluorophosphate (HCTU), DIPEA, DMF, DCM, rt, 24 h; (f) carboxylic acid, 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide (EDCI), 4-dimethylamonipyridine (DMAP), DCM, rt, 24 h; (g) NH<sub>2</sub>OH·HCl, 5 M KOH in anhyd. MeOH, rt; (h) 20% trifluoroacetyl (TFA) in DCM, rt; (i) 1 M BBr<sub>3</sub> in DCM, −78 °C to rt, 2–24 h.</p></p></figure><div class="NLM_p">The key intermediate (<b>4</b>) was then used to incorporate a range of functionalities in place of the <i>N</i>-pivaloyl group present in <b>1</b>, which was predominately achieved with traditional peptide coupling reagents such as HCTU or EDCI. The first synthetic attempt to obtain benzamide analogues <b>5r</b> and <b>5s</b> involved the synthesis of their respective acid chlorides in situ and subsequent acylation with intermediate <b>4</b>. However, benzonitriles <b>5n</b> and <b>5o</b> were produced instead by dehydration of the carboxamide group. Ring-opening reactions of cyclic acid anhydrides were also performed to synthesize alkyl carboxylate analogues. Cyclic anhydrides such as succinic anhydride and Meldrum’s acid have been commonly used in the literature to form amide bonds through ring-opening reactions.<a onclick="showRef(event, 'ref51 ref52 ref53 ref54 ref55 ref56 ref57 ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref51 ref52 ref53 ref54 ref55 ref56 ref57 ref58 ref59">(51−59)</a> The butyric acid analogue (<b>5e</b>) was synthesized from intermediate <b>4</b> via nucleophilic attack on the carbonyl π system present in succinic anhydride. Subsequent (benzotriazol-1-yloxy)tripyrrolidinophosphonium hexafluorophosphate (PyBOP) amide coupling of <b>5e</b> converted the acid moiety to the corresponding carboxamide <b>5f</b>. However, reaction with Meldrum’s acid under identical reaction conditions resulted in decarboxylation, generating the acetamide analogue (<b>5a</b>) instead of the expected propionic analogue.</div><div class="NLM_p">The methyl ester in <b>5</b> was converted to the corresponding hydroxamic acids (<b>6</b>) using hydroxylamine hydrochloride and methanolic potassium hydroxide. In some cases, minor conversion to the carboxylic acid through base-mediated hydrolysis was detected. Occasionally, complete conversion to the desired product was not successful due to unexpected side reactions. For instance, liquid chromatography–mass spectrometry (LC–MS) analysis of the <i>tert</i>-butyl propanoate analogue (<b>5b</b>) indicated only partial conversion. Interestingly, when more of the reagents were added to the reaction mixture, condensation of both methyl and <i>tert</i>-butyl esters occurred, generating the desired mono-hydroxamic acid <b>6b</b> and dihydroxamic acid analogue <b>6d</b>. For the benzonitrile compounds <b>5n</b> and <b>5o</b>, nucleophilic attack of hydroxylamine on the electrophilic nitrile carbon resulted in the formation of amidoxime compounds <b>6p</b> and <b>6q</b>, respectively.<a onclick="showRef(event, 'ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref60 ref61">(60,61)</a> The APN inhibitory activity of these unintended analogues was still evaluated due to their potential ability to form hydrogen bonds or ionic interactions at the S1′ pocket. Compounds with acidic functionality (<b>5c</b>, and phenolic analogues) showed poor solubility in methanol when deprotonated by potassium hydroxide, which consequently resulted in a longer reaction times and poor reaction yields.</div><div class="NLM_p">Further deprotection reactions were required to produce carboxylic acid <b>6c</b>, anilines <b>6ab</b> and <b>6ac</b>, and phenols <b>6ag</b>, <b>6as</b>, <b>6av</b>, <b>6ax</b>, and <b>6az</b>. The <i>tert</i>-butyl propanoate analogue (<b>5b</b>) and Boc-protected aniline analogues (<b>5z</b>, <b>5aa</b>) were hydrolyzed under mild acidic conditions to give propionic acid <b>6c</b> and anilines <b>6ab</b> and <b>6ac</b>, respectively. O-Demethylation was carried out in the presence of boron tribromide to form the corresponding phenols/catechols <b>6ag</b>, <b>6as</b>, <b>6av</b>, <b>6ax</b>, and <b>6az</b>.</div><div class="NLM_p">In addition to the analogues with an amide linker, a benzylamine analogue (<b>9</b>) was also synthesized (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Further physicochemical studies on the parent compound (<b>1</b>) indicate that it has an aqueous solubility ranging between 12 and 25 μM and a Log <i>D</i> of 3.0.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> This study demonstrated that there was room to improve its physicochemical properties for optimal pharmacokinetic profiles. Incorporating bulkier groups inevitably increases the hydrophobicity of molecules, consequently decreasing aqueous solubility. However, the introduction of a secondary amine would allow the molecule to be formulated as a salt to overcome solubility issues. As described previously, reductive amination of intermediate <b>4</b> with 4-fluoro-3-methoxybenzylaldehyde yielded <b>7</b>, which was converted to the corresponding hydroxamic acid (<b>8</b>). The methoxy group in <b>8</b> was readily deprotected using boron tribromide, then reacted with hydrochloric acid to obtain the 4-fluoro-3-hydroxybenzylamine analogue (<b>9</b>) as a hydrochloride salt.</div><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/medium/jm-2019-007578_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0008.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Benzylamine Analogue <b>9</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00757&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) benzaldehyde, Na(OAc)<sub>3</sub>BH, anhyd. DCE, rt, 24 h; (b) NH<sub>2</sub>OH·HCl, 5 M KOH in anhyd. MeOH, rt; (c) (i) 1 M BBr<sub>3</sub> in DCM, −78 °C to rt, 2–24 h; (ii) 1 M HCl, MeOH, rt, 24 h.</p></p></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Substituted Aromatic Group Is Important for Potency toward Human APN</h3><div class="NLM_p">The inhibitory activities (<i>K</i><sub>i</sub><sup>(app)</sup>) of the synthesized hydroxamic acids were measured against recombinant human APN (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Initial experimental triplicates were performed for all compounds to determine compounds of higher priority with <i>K</i><sub>i</sub><sup>(app)</sup> ≤ 100 nM, which were assessed further in biological triplicate. In general, the aliphatic carboxamides (<b>6a</b>–<b>f</b>) were less potent than the lead compound (<b>1</b>) but more potent than bestatin and Tosedostat (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). This suggests that the appended carboxylic acid and carboxamide moieties (in the case of <b>6a</b>, <b>6e</b>, and <b>6f</b>) were unable to make the intended polar interactions with the S1′ subsite of APN, potentially due to the short linker length. The unsubstituted benzamide <b>6g</b> showed over 4-fold loss in inhibitory activity compared to <b>1</b>. However, introduction of hydrogen-bond-donating groups to the phenyl ring led to a recovery in potency. For example, hydroxyl, amidoxime, and carboxamide monosubstituted benzamides <b>6m</b>–<b>s</b> were all more potent than compound <b>1</b>. On the other hand, hydrogen-bond acceptors such as fluorine (<b>6h</b> and <b>6i</b>) and methoxy (<b>6j</b> and <b>6k</b>) were not well tolerated. We also examined eight functionalized carboxamides (<b>6t</b>–<b>x</b>), sulfonamides (<b>6y</b> and <b>6ad</b>), and sulfamide (<i>para</i>-(sulfamoylamino)benzoic acid) (<b>6ae</b>) analogues, which have the capacity to increase ligand binding interactions in the S1′ pocket via hydrogen-bonding interactions. The benzamide derivatives <b>6t</b>–<b>x</b> exhibited a decrease in activity ranging from 2- to 8-fold compared to carboxamide <b>6s</b>. A trend of decreasing activity was observed from the benzamide derivatives as the size of the hydrophobic group increases from methyl, dimethyl, ethyl, and isopropyl, indicating that a loss in polar contacts may result in reduced binding to APN. Additionally, the inhibitory activities of the methyl (<b>6t</b>) and dimethyl (<b>6u</b>) analogues were essentially identical, suggesting that the hydrogen-bond-donating capability of benzamide does not play a significant role in potency. 4-Methylsulfonamide <b>6ad</b> (<i>K</i><sub>i</sub><sup>(app)</sup> = 4.5 nM) and sulfamide <b>6ae</b> (<i>K</i><sub>i</sub><sup>(app)</sup> = 8.2 nM) were the most potent inhibitors of the series and showed a greater than 10-fold improvement in potency compared to <b>1</b>, potentially due to its multiple hydrogen-bond-forming capacity for strong binding interactions with APN.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Summary of Inhibitory Activity of Hydroxamic Acid Analogues against APN</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/medium/jm-2019-007578_0009.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center"><i>K</i><sub>i</sub><sup>(app)</sup> ± SEM (nM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bestatin</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">2370 ± 350</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Tosedostat</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1180 ± 77</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="left">–C(CH<sub>3</sub>)<sub>3</sub></td><td class="colsep0 rowsep0" align="left">118 ± 3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6a</b></td><td class="colsep0 rowsep0" align="left">–CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">560 ± 50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">–CH<sub>2</sub>COOH</td><td class="colsep0 rowsep0" align="left">188 ± 9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6d</b></td><td class="colsep0 rowsep0" align="left">–CH<sub>2</sub>C(O)NHOH</td><td class="colsep0 rowsep0" align="left">348 ± 33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6e</b></td><td class="colsep0 rowsep0" align="left">–CH<sub>2</sub>CH<sub>2</sub>COOH</td><td class="colsep0 rowsep0" align="left">172 ± 9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6f</b></td><td class="colsep0 rowsep0" align="left">–CH<sub>2</sub>CH<sub>2</sub>C(O)NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">497 ± 35</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6g</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>5</sub></td><td class="colsep0 rowsep0" align="left">522 ± 37</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6h</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>4</sub>F (<i>m</i>)</td><td class="colsep0 rowsep0" align="left">919 ± 160</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6i</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>4</sub>F (<i>p</i>)</td><td class="colsep0 rowsep0" align="left">704 ± 68</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6j</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>4</sub>OMe (<i>m</i>)</td><td class="colsep0 rowsep0" align="left">462 ± 6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6k</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>4</sub>OMe (<i>p</i>)</td><td class="colsep0 rowsep0" align="left">745 ± 53</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6l</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>4</sub>OH (<i>m</i>)</td><td class="colsep0 rowsep0" align="left">366 ± 31</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6m</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>4</sub>OH (<i>p</i>)</td><td class="colsep0 rowsep0" align="left">102 ± 5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6p</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>4</sub>C(NHOH)NH<sub>2</sub> (<i>m</i>)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">37.3 ± 2.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6q</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>4</sub>C(NHOH)NH<sub>2</sub> (<i>p</i>)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">49.1 ± 3.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6r</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>4</sub>C(O)NH<sub>2</sub> (<i>m</i>)</td><td class="colsep0 rowsep0" align="left">71.2 ± 6.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6s</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>4</sub>C(O)NH<sub>2</sub> (<i>p</i>)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">82.1 ± 9.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6t</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>4</sub>C(O)NHMe (<i>p</i>)</td><td class="colsep0 rowsep0" align="left">185 ± 22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6u</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>4</sub>C(O)NMe<sub>2</sub> (<i>p</i>)</td><td class="colsep0 rowsep0" align="left">182 ± 5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6v</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>4</sub>C(O)NEt (<i>p</i>)</td><td class="colsep0 rowsep0" align="left">277 ± 22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6w</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>4</sub>C(O)NH<sup><i>i</i></sup>Pr (<i>p</i>)</td><td class="colsep0 rowsep0" align="left">631 ± 70</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6x</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>4</sub>C(O)NHCH<sub>2</sub>CH<sub>2</sub>OH (<i>p</i>)</td><td class="colsep0 rowsep0" align="left">163 ± 26</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6y</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>4</sub>SO<sub>2</sub>NH<sub>2</sub> (<i>p</i>)</td><td class="colsep0 rowsep0" align="left">240 ± 7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6ab</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>4</sub>NH<sub>2</sub> (<i>m</i>)</td><td class="colsep0 rowsep0" align="left">131 ± 10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6ac</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>4</sub>NH<sub>2</sub> (<i>p</i>)</td><td class="colsep0 rowsep0" align="left">205 ± 17</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6ad</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>4</sub>NHSO<sub>2</sub>Me (<i>p</i>)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">4.50 ± 0.80</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6ae</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>4</sub>CNHSO<sub>2</sub>NH<sub>2</sub> (<i>p</i>)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">8.20 ± 0.90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6af</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>3</sub> (3,4-OMe)</td><td class="colsep0 rowsep0" align="left">175 ± 16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6ag</b></td><td class="colsep0 rowsep0" align="left">C<sub>6</sub>H<sub>3</sub> (3,4-OH)</td><td class="colsep0 rowsep0" align="left">430 ± 42</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6ah</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>3</sub> (3-F,4-OH)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">29.1 ± 3.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6ai</b></td><td class="colsep0 rowsep0" align="left">–C<sub>6</sub>H<sub>3</sub> (4-F,3-OH)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">40.0 ± 2.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6aj</b></td><td class="colsep0 rowsep0" align="left">-indol-5-yl</td><td class="colsep0 rowsep0" align="left">111 ± 3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6ak</b></td><td class="colsep0 rowsep0" align="left">-indazol-5yl<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">19.2 ± 2.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6al</b></td><td class="colsep0 rowsep0" align="left">-benzotriazol-5-yl<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">23.4 ± 2.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6am</b></td><td class="colsep0 rowsep0" align="left">-benzimidazol-5-yl<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">170 ± 17</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6an</b></td><td class="colsep0 rowsep0" align="left">-(2-oxoindolin-5-yl)</td><td class="colsep0 rowsep0" align="left">156 ± 9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6ao</b></td><td class="colsep0 rowsep0" align="left">–CH<sub>2</sub>C<sub>6</sub>H<sub>5</sub></td><td class="colsep0 rowsep0" align="left">4420 ± 720</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6ap</b></td><td class="colsep0 rowsep0" align="left">–CH<sub>2</sub>C<sub>6</sub>H<sub>4</sub>F (<i>m</i>)</td><td class="colsep0 rowsep0" align="left">442 ± 36</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6aq</b></td><td class="colsep0 rowsep0" align="left">–CH<sub>2</sub>C<sub>6</sub>H<sub>4</sub>F (<i>p</i>)</td><td class="colsep0 rowsep0" align="left">158 ± 5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6ar</b></td><td class="colsep0 rowsep0" align="left">–CH<sub>2</sub>C<sub>6</sub>H<sub>4</sub>OMe (<i>p</i>)</td><td class="colsep0 rowsep0" align="left">978 ± 120</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6as</b></td><td class="colsep0 rowsep0" align="left">–CH<sub>2</sub>C<sub>6</sub>H<sub>4</sub>OH (<i>p</i>)</td><td class="colsep0 rowsep0" align="left">235 ± 24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6at</b></td><td class="colsep0 rowsep0" align="left">–CH<sub>2</sub>C<sub>6</sub>H<sub>4</sub>C(O)NH<sub>2</sub> (<i>p</i>)</td><td class="colsep0 rowsep0" align="left">604 ± 66</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6au</b></td><td class="colsep0 rowsep0" align="left">–CH<sub>2</sub>C<sub>6</sub>H<sub>3</sub> (3,4-OMe)</td><td class="colsep0 rowsep0" align="left">119 ± 12</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6av</b></td><td class="colsep0 rowsep0" align="left">–CH<sub>2</sub>C<sub>6</sub>H<sub>3</sub> (3,4-OH)</td><td class="colsep0 rowsep0" align="left">137 ± 6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6ax</b></td><td class="colsep0 rowsep0" align="left">–CH<sub>2</sub>C<sub>6</sub>H<sub>3</sub> (3-F,4-OH)</td><td class="colsep0 rowsep0" align="left">43.1 ± 5.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6az</b></td><td class="colsep0 rowsep0" align="left">–CH<sub>2</sub>C<sub>6</sub>H<sub>3</sub> (4-F,3-OH)</td><td class="colsep0 rowsep0" align="left">138 ± 5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">–CH<sub>2</sub>C<sub>6</sub>H<sub>3</sub> (4-F,3-OH)<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">>1 mM</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Biological triplicates were performed for inhibitors with <i>K</i><sub>i</sub><sup>(app)</sup> < 100 nM from initial triage screenings.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">The compound is secondary amine derivative.</p></div></div></div><div class="NLM_p">Among the disubstituted benzamides (<b>6af</b>–<b>ai</b>), the fluorohydroxyl analogues (<b>6ah</b> and <b>6ai</b>) showed improved activity relative to <b>1</b>. Compared to the corresponding monosubstituted benzamide derivatives, disubstituted analogues showed stronger inhibition activity. For example, 3-fluorobenzamide <b>6h</b> and 4-hydroxybenzamide <b>6m</b> displayed activities of 919 and 366 nM, respectively, whereas the potency of the 3-fluoro-4-hydroxyl analogue (<b>6ah</b>) significantly increased to 29 nM, suggesting that both fluoro and hydroxyl are making important interactions with APN. A similar result was observed for the 4-fluoro-3-hydroxy analogue (<b>6ai</b>), which exhibited significantly greater potency (<i>K</i><sub>i</sub><sup>(app)</sup> = 40 nM) than the corresponding monosubstituted analogues 4-fluorobenzamide <b>6i</b> (<i>K</i><sub>i</sub><sup>(app)</sup> = 704 nM) and 3-hydroxybenzamide <b>6l</b> (<i>K</i><sub>i</sub><sup>(app)</sup> = 102 nM). Similarly, the 3,4-dimethoxy analogue (<b>6af)</b> exhibited an increased potency compared to 3- and 4-methoxy analogues, <b>6j</b> and <b>6k</b>, respectively.</div><div class="NLM_p">The activities of compounds where benzamide was replaced with various heterocyclic amides (<b>6aj</b>–<b>an</b>) were also investigated. The indole analogue (<b>6aj</b>) was equipotent with the parent compound (<b>1</b>). A significant increase in the potency was observed from indazole <b>6ak</b> (<i>K</i><sub>i</sub><sup>(app)</sup> = 19.2 nM) and benzotriazole <b>6al</b> (<i>K</i><sub>i</sub><sup>(app)</sup> = 23.4 nM).</div><div class="NLM_p">In general, replacement of benzamide with acetamide to introduce sp<sup>2</sup> characteristics for increased flexibility resulted in a significant loss in potency (<b>6ao</b>–<b>ay</b>). The 3-fluoro-4-hydroxyl analogue <b>6ax</b> was the only compound in this series that proved to be more potent than <b>1</b> with an inhibition activity of 43.1 nM. Inconsistent relationships were observed between the benzamide and matching acetamide analogues. For instance, the potency observed for the benzyl analogue (<b>6ao</b>) reduced drastically compared to the phenyl analogue (<b>6g</b>). This trend was also observed for the 4-methoxy (<b>6k</b> and <b>6ar</b>), 4-hydroxy (<b>6m</b> and <b>6as</b>), 4-fluoro-3-hydroxy (<b>6ai</b> and <b>6az</b>), and 4-benzamide (<b>6s</b> and <b>6at</b>) pairs. In contrast, an increase in potency was observed for the fluoro analogues (<b>6h</b>, <b>6i</b>, <b>6ap</b>, <b>6aq</b>) and the 3,4-dimethoxy analogues (<b>6af</b> and <b>6au</b>), while 3-fluoro-4-hydroxyl compounds <b>6ah</b> and <b>6ax</b> were the most potent compounds in each series. The 4-fluoro-3-hydroxybenzylamine analogue (<b>9</b>) was the weakest inhibitor of this series; full inhibition was not achieved at a concentration of 1 mM, suggesting that carbonyl oxygen of the amide linker is crucial for potent activity, potentially by providing appropriate rigidity and also hydrogen-bonding interaction with nearby residues.</div><div class="NLM_p last">Structure–activity relationship (SAR) investigations around the S1′ pocket of APN through modifications of the <i>N</i>-pivaloyl group of compound <b>1</b> indicated that substituted aryls were essential for inhibitory activity, where hydrogen-bond donors were more favored than hydrogen-bond acceptors. A substantial decrease in activity observed from acetamide and secondary amine analogues revealed that rigidity and the carbonyl oxygen of the benzamide were vital. In addition, studies on functionalized carboxamides and sulfonamides showed that there was a decreasing trend in activity as the size of the hydrophobic group increases, while incorporating polar groups led to a significant rise in activity. This suggested that hydrogen-bonding interactions played a major role to enhance activity against APN.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Core Biaryl System Engages in Hydrophobic Interactions with Flexible Loop at the S1 Pocket of APN</h3><div class="NLM_p">To understand why <b>6ad</b> and <b>6ae</b> displayed such an improvement in potency, we undertook molecular docking followed by MD simulations to generate a model of APN bound to both inhibitors. A rigid-docking into the APN crystal structure (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYQ">4FYQ</a>) of compounds <b>6ad</b> and <b>6ae</b> was performed based on the docked pose of compound <b>1</b> into APN using Surflex Docking software available from SYBYL2.1. The common fragments of the biphenyl core structure and the hydroxamic group were set as constraints. Similar to the molecular docking of compound <b>1</b>, the most preferred structures of <b>6ad</b> and <b>6ae</b> bound to APN were selected based on the total docking score (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_001.pdf" class="ext-link">Table S6</a>). MD simulations showed that the ligands occupy different conformations in a time-dependent manner (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_004.mp4" class="ext-link">Movies S1</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_005.mp4" class="ext-link">S2</a>). Compound <b>6ad</b> participated in water-mediated hydrogen-bonding interactions with the fluorine atoms and the nearby residues at the S1 subsite of APN, both of which were also observed in the co-crystal structure of compound <b>1</b> bound to <i>Pf</i>A-M1.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> One compelling result from the MD simulation was a flexible loop located at domain IV acting as an effective “cap” to close the active site of APN (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). The flexible loop and the Phe<sup>896</sup> residue are believed to play important roles in conformational changes of APN.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> The flexible loop consists of eight amino acid residues (<sup>891</sup>YGGGSFSF<sup>898</sup>) and has been shown to undergo a dramatic change in conformation upon complex formation with a peptide substrate.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> The binding mode of bestatin, which binds APN differently to other M1 aminopeptidases (noncanonical binding pose), is also thought to be related to the flexible loop, in particular Phe<sup>896</sup>. As opposed to other M1–bestatin complexes in which the loop is shorter and different in sequence and therefore does not impede bestatin from binding with canonical geometry, the Phe<sup>896</sup> of APN was positioned away from the binding pocket to accommodate the bulky phenyl group of bestatin at the S1 pocket.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> In our simulations, we saw that the biphenyl ring system of <b>6ad</b> maintained face-to-face stacking interactions with Phe<sup>896</sup> to keep the inhibitor locked in the active site of APN; however, it was clear that this interaction varied within the biphenyl system between different poses. Generally, the side chain of Phe<sup>896</sup> interacted with the top trifluorophenyl (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B,C), but interactions with the central aromatic ring of compound <b>6ad</b> were also observed (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A).</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/medium/jm-2019-007578_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Three conformations of compound <b>6ad</b> (A–C) observed during MD simulation. The carbon atoms of the flexible loop are colored in light pink, and other residues in the binding site are wheat-colored. Interactions between compound <b>6ad</b> (carbon atoms in white) and APN are shown in black dashed lines. (D) Overlay of two predicted binding poses of compound <b>6ae</b>. Residues around compound <b>6ae</b> (carbon atoms in magenta) are wheat-colored (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYQ">4FYQ</a>). Overlaid Arg<sup>442</sup> residues are shown in green. Interactions between the compound and APN are depicted in black dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00757&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Interestingly, stable hydrophobic stacking interactions between the flexible loop and compound <b>6ae</b> were not observed (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Without the flexible loop stabilizing the biaryl core moiety and restricting the movement of the molecule, the flat biphenyl system of <b>6ae</b> gained freedom to move within the S1 pocket. Although prominent stacking interactions that were observed with <b>6ad</b> were missing from <b>6ae</b>, the phenylalanine-rich region provided a favorable environment for the hydrophobic biaryl group to maintain its overall position within the pocket.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Sulfonamide Moiety Provides Multiple Hydrogen-Bonding Interactions at the S1′ Pocket of APN</h3><div class="NLM_p">Based on our MD simulations, the large arylsulfonamide derivatives of both <b>6ad</b> and <b>6ae</b> were able to reach the S1′ pocket of APN. The bottom aromatic group of <b>6ad</b> was flexible enough to interact with the side chain of Phe<sup>898</sup> or Tyr<sup>477</sup> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A–C). Due to the high flexibility of the sulfonamide moiety, these interactions were occasionally lost and replaced with a cation−π interaction between the Arg<sup>381</sup> side chain and the electron-rich aromatic ring of arylsulfonamide. The results also revealed that the sulfonyl oxygen atoms can engage in dual hydrogen bonds with Asn<sup>900</sup> and Arg<sup>442</sup> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_001.pdf" class="ext-link">Table S3</a>), where the arginine residue stayed in a relatively rigid manner throughout simulations.</div><div class="NLM_p">Compound <b>6ae</b> behaved similarly to <b>6ad</b>, exhibiting multiple binding positions. The aromatic ring located in the S1′ pocket of APN displayed extensive interactions with the side chain of Arg<sup>442</sup> and Asn<sup>900</sup>. As described above, the flexible loop was not accessible to the molecule, but the bottom aromatic was located closer to Tyr<sup>477</sup>, forming hydrophobic edge-face stacking interactions (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>D). The backbone amide of Ala<sup>474</sup> could also participate in a hydrogen-bond interaction with a sulfonyl oxygen atom of <b>6ae</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_001.pdf" class="ext-link">Table S4</a>). In the case, where the sulfamide was pointing deeper into the S1′ subsite of APN, the amino group is located in close proximity to Asp<sup>439</sup>.</div><div class="NLM_p">To rationalize our SAR analysis, we also investigated the interactions between one of the weaker inhibitors, <b>6ag</b> (<i>K</i><sub>i</sub><sup>(app)</sup> = 430 nM), which contains 3,4-hydroxyl group (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). In contrast to compounds <b>6ad</b> and <b>6ae</b>, which showed extensive hydrophobic interactions with the flexible loop, the biphenyl system of compound <b>6ag</b> made no interactions with the key phenylalanine residues of the flexible loop. Throughout most of the simulation, the trifluorophenyl moiety participated in a water-mediated hydrogen-bonding network with Ser<sup>469</sup> and hydrophobic interactions between Phe<sup>472</sup>. However, the flexible loop was located too far away from the biphenyl system to capture essential nonpolar interactions, which may explain the significant decrease in the potency. Hydrogen-bonding interactions between the catechol group and the side chain of Glu<sup>418</sup> residue were also observed. However, these interactions appear not able to compensate the missing stacking interactions between the S1′ aromatic group and Tyr<sup>477</sup> or the Phe<sup>898</sup> residues, which may contribute to the reduced inhibition activity of <b>6ag</b> against APN.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/medium/jm-2019-007578_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Comparison of the binding interactions occurred with tight-binding inhibitor <b>6ad</b> (left) and the weaker inhibitor <b>6ag</b> (right) (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYQ">4FYQ</a>). The carbon atoms of the flexible loop are colored in light pink, and other residues in the binding site are wheat-colored. Compounds <b>6ad</b> and <b>6ag</b> are colored in white and green, respectively. Interactions between the ligands and APN are shown in black dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00757&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Role of Glu<sup>389</sup> Residue as the Zinc–Hydroxamic Acid Complex Stabilizer</h3><div class="NLM_p">Another intriguing phenomenon was observed from the formation of the zinc–hydroxamic acid complex. In the M1 aminopeptidases, inhibitors often bind with a pentahedral coordination to the zinc, which mimic the transition state of the activated enzyme.<a onclick="showRef(event, 'ref4 ref62'); return false;" href="javascript:void(0);" class="ref ref4 ref62">(4,62)</a> In the simulations we performed, the hydroxyl group of hydroxamic acid readily lost contact with the zinc, with bond distance ranging from 2.2 to 3.4 Å while the carbonyl oxygen possessed a very stable coordination with a bond distance range of 1.9–2.0 Å, resulting in tetrahedral coordination. Given the challenges associated with metallo-protein simulations, this observation is possibly biased by the ZAFF parameters used. To accurately simulate the change in ligand interaction with the zinc ion, a quantum mechanics/molecular mechanics simulation would be needed. However, we did observe that the loss of pentahedral coordination of <b>6ad</b> and <b>6ae</b> to the zinc ion resulted in the formation of a hydrogen bond with Glu<sup>389</sup> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). This interaction was extremely stable and maintained a bond distance of 2.4–2.9 Å throughout our simulations (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_001.pdf" class="ext-link">Tables S3 and S4</a>). Further computational and experimental investigation is required to determine whether this interaction plays a role in the catalytic mechanism by stabilizing the zinc–ligand complex.</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/medium/jm-2019-007578_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0006.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Hydroxamic acid–zinc complex in the active site of APN. APN residues are wheat-colored, and the ligand is shown in magenta. Interactions between the ligand and the catalytic triad and Glu<sup>389</sup> are illustrated in black dashed lines (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYQ">4FYQ</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00757&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Compounds <b>6ad</b> and <b>6ae</b> Show Selectivity for APN over Matrix Metalloproteinases (MMPs)</h3><div class="NLM_p last">As the hydroxamic acid moiety is a strong zinc chelator, there is a possibility that our potent APN inhibitors may interact with other zinc-dependent enzymes. To assess the selectivity and potential off-target effects of <b>6ad</b> and <b>6ae</b>, we performed quenched fluorescence assays with MMP2, 7, 8, 9, and 13 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_001.pdf" class="ext-link">Figure S1</a>). The activity of the parent compound (<b>1</b>), Tosedostat, and the broad spectrum MMP inhibitor, Marimastat, were also evaluated as comparison and controls (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_001.pdf" class="ext-link">Table 1</a>). MMP2 and 9 were effectively inhibited by Marimastat (IC<sub>50</sub> 0.43 and 3.1 nM, respectively) and weakly by Tosedostat (IC<sub>50</sub>: 0.19 and 1.5 μM, respectively), whereas <b>6ad</b> and <b>6ae</b> weakly inhibited MMP2 (1.3 and 2.1 μM, respectively) and only inhibited MMP9 at relatively high concentrations (IC<sub>50</sub> > 100 μM). Similar observations were made for collagenases MMP8 and MMP13 as well as matrilysin MMP7, which were all inhibited with low nM IC<sub>50</sub> by Marimastat, high nM IC<sub>50</sub> by Tosedostat, and μM to mM IC<sub>50</sub> by the novel inhibitors, with <b>1</b> demonstrating the lowest extent of inhibition (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_001.pdf" class="ext-link">Figure S1</a>). Collectively, these findings show that the novel APN inhibitors demonstrate low off-target inhibitory effects on MMPs.</div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Cellular Toxicity, and Physicochemical and Pharmacokinetic Properties of <b>6ad</b> and <b>6ae</b></h3><div class="NLM_p">Evaluation of cellular toxicity, and physicochemical and stability properties provide important early-stage data to determine whether or not a compound has the necessary features to be pursued further as a drug candidate. In our cellular toxicity study, we used Ad293 cell line, which was used to measure cell toxicity in other literature reports, is derived from the human embryonic kidney 293 (HEK293) cell line, but transfected with a special gene for improved cell adherence to make cell handling easier during cell cultures and assays.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> The compounds show limited cytotoxicity against Ad293 cells with CC<sub>50</sub> values of 41 ± 2 μM for <b>6ad</b> and 149 ± 13 μM for <b>6ae</b>. In vitro physicochemical properties and metabolic/plasma stabilities of two potent APN inhibitors <b>6ad</b> and <b>6ae</b> were measured (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). The kinetic solubility of each compound was determined by nephelometry. Both compounds showed moderate solubility under pH conditions representative of the stomach (pH 2) and upper fasted state small intestine (pH 6.5) suggesting that solubility could be a factor that limits oral absorption. The partition coefficients at pH 7.4 (Log <i>D</i><sub>pH 7.4</sub>) were estimated using a chromatographic method and found to be 2.8 and 2.6, respectively. The metabolic stabilities were assessed by incubating compounds in mouse and human hepatic microsomes at 37 °C and a protein concentration of 0.4 mg/mL. The compounds showed minimal degradation and very long half-lives of longer than 4 h in both mouse and human microsomes, resulting in low in vitro intrinsic clearance values [CL<sub>int</sub> < 7 μL/(min mg protein)]. In vitro plasma stability studies were performed by incubating compounds in human and mouse plasma at 37 °C for up to 4 h. Both compounds displayed minimal loss, indicating that they are not readily susceptible to the action of hydrolytic enzymes present in the plasma.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Physicochemical and Metabolic Properties of <b>6ad</b> and <b>6ae</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/medium/jm-2019-007578_0010.gif" alt="" id="fx2" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="–" /></col><col align="char" char="–" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char="–">kinetic solubility</th><th class="rowsep1 colsep0" colspan="2" align="center">liver microsome stability</th><th class="rowsep1 colsep0" colspan="2" align="center">plasma stability</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center" char="–">Sol<sub>pH 2.0</sub></th><th class="colsep0 rowsep0" align="center" char="–">Sol<sub>pH 6.5</sub></th><th class="colsep0 rowsep0" align="center">mouse <i>t</i><sub>1/2</sub> (min)</th><th class="colsep0 rowsep0" align="center">human <i>t</i><sub>1/2</sub> (min)</th><th class="colsep0 rowsep0" align="center">mouse <i>t</i><sub>1/2</sub> (min)</th><th class="colsep0 rowsep0" align="center">human <i>t</i><sub>1/2</sub> (min)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6ad</b></td><td class="colsep0 rowsep0" align="char" char="–">12.5–25</td><td class="colsep0 rowsep0" align="char" char="–">12.5–25</td><td class="colsep0 rowsep0" align="left">>250</td><td class="colsep0 rowsep0" align="left">>250</td><td class="colsep0 rowsep0" align="left">>1020</td><td class="colsep0 rowsep0" align="left">>1020</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6ae</b></td><td class="colsep0 rowsep0" align="char" char="–">6.3–12.5</td><td class="colsep0 rowsep0" align="char" char="–">12.5–25</td><td class="colsep0 rowsep0" align="left">>250</td><td class="colsep0 rowsep0" align="left">>250</td><td class="colsep0 rowsep0" align="left">>1020</td><td class="colsep0 rowsep0" align="left">>1020</td></tr></tbody></table></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40420" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40420" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Rapid metastasis of cancer cells through complex mechanisms of ECM degradation, angiogenesis, cell invasion, and cell adhesion is a major burden in effective cancer therapy. Therefore, continuous development of novel anticancer agents targeting metastasis is urgently required. Being involved in mechanisms of angiogenesis and metastasis, APN has been broadly studied as a therapeutic target for cancer. Here, we have reported the design, synthesis, and biological evaluations of novel APN inhibitors that were derived from a potent inhibitor of parasitic homologue <i>Pf</i>A-M1. Through comprehensive structure-based design, we were able to generate a small library of novel hydroxamic acid analogues targeting APN and discover a potent compound <b>6ad</b> that showed 527-fold improved inhibition activity than a known APN inhibitor bestatin. Molecular docking and MD simulations highlighted the significance of the flexible loop in domain VI in providing hydrophobic interactions at the S1 pocket. The results also revealed the combination of stacking and dual hydrogen-bonding interactions was crucial to optimize the binding at the S1′ subunit of APN. In addition, cross-activity studies showed that <b>6ad</b> and <b>6ae</b> possessed low off-target activity on MMPs. The evaluation of the cellular activities of <b>6ad</b> and <b>6ae</b> against Ad293 cell line also indicated that they displayed limited cytotoxicity. Moreover, <b>6ad</b> and <b>6ae</b> had favorable metabolic and plasma stability in both human and mouse models. However, the solubility of these compounds is suboptimal and should be addressed by further medicinal chemistry approaches.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27067" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27067" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Chemistry</h3><div class="NLM_p">Chemicals and solvents were purchased from standard suppliers and used without further purification unless otherwise indicated. <sup>1</sup>H NMR, <sup>19</sup>F NMR, and <sup>13</sup>C NMR spectra were recorded on a Bruker Avance Nanobay III 400 MHz Ultrashield Plus spectrometer at 400.13, 376.46, and 100.61 MHz, respectively. NMR experiments were obtained at the temperature of 298 K. Data acquisition and processing was managed using Topspin software package version 3. Chemical shifts (δ) are recorded in parts per million with reference to the chemical shift of the deuterated solvent. Coupling constants (<i>J</i>) and carbon–fluorine coupling constants (<i>J</i><sub>CF</sub>) are recorded in hertz, and multiplicities are described as singlet (s), doublet (d), triplet (t), multiplet (m), doublet of doublets (dd), doublet of triplets (dt), doublet of doublets of doublets (ddd), and broad (br). Overlapped nonequivalent <sup>13</sup>C peaks were identified by heteronuclear single quantum coherence and heteronuclear multiple-bond correlation NMR.</div><div class="NLM_p">Analytical thin-layer chromatography (TLC) was performed on Merck silica gel 60F<sub>254</sub> aluminum-backed plates, which were visualized by fluorescence quenching under a UV lamp at 254 nm or by Fe(III)Cl<sub>3</sub> staining for hydroxamic acid compounds. Flash chromatography was performed with silica gel 60 (particle size, 0.040–0.063 μm).</div><div class="NLM_p">Analytical high-performance liquid chromatography (HPLC) was performed using an Agilent 1260 Infinity Analytical HPLC with a Zorbax Eclipse Plus C18 Rapid Resolution 4.6 × 100 mm<sup>2</sup>, 3.5 μm column. Buffer A: 0.1% TFA in H<sub>2</sub>O and buffer B: 0.1% TFA in MeCN were used. Samples were run at a gradient of 5% buffer B/buffer A (0–9 min) to 100% buffer B (9–10 min) at a flow rate of 1 mL/min. Unless otherwise indicated, all compounds were >95% pure by HPLC (254 and 214 nm) prior to biological evaluation.</div><div class="NLM_p">Preparative HPLC was performed using an Agilent 1260 Infinity instrument coupled with a binary preparative pump and an Agilent 1260 FC-PS fraction collector using Agilent OpenLAB CDS software (revision C.01.04) and an Altima C8 22 × 250 mm<sup>2</sup>, 5 μM column, and a 1260 Infinity diode array detector VL. The following buffers were used: buffer A: 0.1% TFA in H<sub>2</sub>O and buffer B: 0.1% TFA in MeCN. The sample was run at a gradient of 30–100% buffer B over 10 min at a flow rate of 20 mL/min.</div><div class="NLM_p">LC–MS was performed using system A or B. System A: Agilent 6100 series single quadrupole instrument coupled to an Agilent 1200 series HPLC instrument fitted with a Luna 120 C8(2) 5 μm 50 × 4.6 mm<sup>2</sup> column. Samples were run at a flow rate of 0.5 mL/min for 12 min: 5% buffer B/buffer A (0–4 min), 100% buffer B (4–7 min), and 5% buffer B/buffer A (7–12 min). Mass spectra were obtained in positive- and negative-ion modes with a scan range of 100–1000 <i>m</i>/<i>z</i>. UV detection was carried out at 254 nm. System B: Agilent 6120 series single quadrupole instrument coupled to an Agilent 1260 series HPLC instrument fitted with a Poroshell 120 EC-C18 50 × 3.0 mm<sup>2</sup>, 2.7 μm column. The following buffers were used: buffer A: 0.1% formic acid in H<sub>2</sub>O and buffer B: 0.1% formic acid in MeCN. Samples were run at a flow rate of 0.5 mL/min for 5 min: 5% buffer B/buffer A (0–1 min), 100% buffer B (1–2.5 min), and held at this composition until 3.8 min, 5% buffer B/buffer A (3.8–4 min), and held until 5 min at this composition. Mass spectra were obtained in positive- and negative-ion modes with a scan range of 100–1000 <i>m</i>/<i>z</i>. UV detection was carried out at 214 and 254 nm.</div><div class="NLM_p">High-resolution mass spectrometry (HRMS) was carried out using an Agilent 6224 TOF LC–MS mass spectrometer coupled to an Agilent 1290 Infinity device. All data were acquired and referenced via dual-spray electrospray ionization source. Acquisition was performed using Agilent Mass Hunter Data Acquisition software version B.05.00 Build 5.0.5042.2, and analysis was conducted using Mass Hunter Qualitative Analysis version B.05.00 Build 5.0.519.13.</div><div class="NLM_p">Instant JChem was used for data management: Instant JChem 16.9.12.0, ChemAxon (<a href="http://www.chemaxon.com" class="extLink">http://www.chemaxon.com</a>).</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> General Procedure A: Amide Coupling Using HCTU and DIPEA</h4><div class="NLM_p last">Carboxylic acid (1.1 equiv) and HCTU (1.2 equiv) were dissolved in anhydrous DMF (2 mL/mmol) and stirred for 30 min in a N<sub>2</sub>-flushed microwave vial. DIPEA (2.1 equiv) was added dropwise followed by compound <b>4</b> (1.0 equiv) in anhydrous DCM (2 mL/mmol). The reaction mixture was stirred at rt for 1 day. If the reaction did not reach completion after 1 day, then a further 1.2 equiv HCTU and 2.1 equiv DIPEA were added. After completion, the reaction mixture was diluted with sat. NaHCO<sub>3</sub> (10 mL) and extracted with DCM (3 × 15 mL). The combined organic layers were washed with water (2 × 10 mL) and brine (15 mL). The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude was purified by column chromatography using either DCM/MeOH or petroleum ether (PE)/EtOAc as the eluent.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> General Procedure B: Amide Coupling Using EDCI and DMAP</h4><div class="NLM_p last">Compound <b>4</b> (1.0 equiv), carboxylic acid (1.2 equiv), EDCI (1.2 equiv), and DMAP (1.3 equiv) were dissolved in DCM (8 mL/mmol) or DMF (8 mL/mmol) and stirred at rt overnight. If the reaction did not reach completion, then a further 1.2 equiv EDCI and 1.3 equiv DMAP were added. The reaction mixture was diluted with sat. NaHCO<sub>3</sub> (10 mL) and extracted with DCM (3 × 10 mL). The combined organic layers were washed with a 1 M HCl solution (15 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude residue was purified by column chromatography using PE/EtOAc as the eluent.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> General Procedure C: Direct Aminolysis of Methyl Ester to the Hydroxamic Acid</h4><div class="NLM_p last">To a solution of methyl ester (1.0 equiv) in anhydrous MeOH (3 mL/mmol) was added NH<sub>2</sub>OH·HCl (4–10 equiv) followed by KOH (5 M in MeOH, 5–10 equiv). The reaction mixture was stirred at rt and monitored by LC–MS and TLC using an Fe(III)Cl<sub>3</sub> stain. Once the reaction was complete, the suspension was dry-loaded onto silica and purified by column chromatography.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> General Procedure D: O-Demethylation Using BBr<sub>3</sub></h4><div class="NLM_p">To a solution of the methyl ether substrate (1.0 equiv) in DCM (2 mL/mmol) was added BBr<sub>3</sub> (1 M in DCM, 5.0 equiv for mono O-demethylation, 10.0 equiv for double O-demethylation) at −78 °C. The reaction mixture was stirred at rt for 2 h to 1 day. The reaction was quenched by the addition of a 1 M HCl and stirred vigorously for 10 min. The resulting precipitate was filtered and purified by preparative HPLC.</div><div id="sec4_1_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i25" class="anchor-spacer"></div><h5 class="article-section__title" id="_i25"> 1-(3′,4′,5′-Trifluoro-[1,1′-biphenyl]-4-yl)ethan-1-one (<b>2</b>)</h5><div class="NLM_p last">To a nitrogen-flushed 500 mL round-bottom flask were added 4′-bromoacetophenone (5.00 g, 25.1 mmol), 3,4,5-trifluorophenylboronic acid (5.74 g, 32.7 mmol), anhydrous THF (180 mL), and a 1 M Na<sub>2</sub>CO<sub>3</sub> solution (60 mL). PdCl<sub>2</sub>(PPh<sub>3</sub>)<sub>2</sub> (529 mg, 0.754 mmol) was added, and the mixture was heated at reflux for 2 h. The reaction mixture was concentrated under reduced pressure and extracted with Et<sub>2</sub>O (3 × 50 mL). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude product was purified by column chromatography (PE/EtOAc 100:0–50:50) to afford compound <b>2</b> as a yellow-brown solid (6.29 g, 100%); <sup>1</sup>H NMR [<i>d</i><sub>6</sub>-dimethyl sulfoxide (DMSO)] δ 8.03 (d, <i>J</i> = 8.6 Hz, 2H), 7.89 (d, <i>J</i> = 8.5 Hz, 2H), 7.80 (dd, <i>J</i> = 9.6, 6.8 Hz, 2H), 2.62 (s, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.6 (d, <i>J</i> = 21.7 Hz), −162.1 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 197.5, 150.6 (ddd, <i>J</i><sub>CF</sub> = 247.0/9.8/4.2 Hz), 141.1–141.0 (m), 138.79 (dt, <i>J</i><sub>CF</sub> = 250.5/15.6 Hz), 136.5, 135.6 (dt, <i>J</i><sub>CF</sub> = 12.8/6.4 Hz), 128.8, 127.1, 112.2–111.3 (m), 26.8; <i>m</i>/<i>z</i> MS C<sub>14</sub>H<sub>10</sub>F<sub>3</sub>O [MH]<sup>+</sup> calcd 251.1, found 251.0.</div></div><div id="sec4_1_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i26" class="anchor-spacer"></div><h5 class="article-section__title" id="_i26"> 2-Oxo-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetic Acid (<b>3</b>)</h5><div class="NLM_p last">Compound <b>2</b> (6.29 g, 25.1 mmol) and SeO<sub>2</sub> (4.18 g, 37.7 mmol) were dissolved in anhydrous pyridine (200 mL). The reaction mixture was sonicated and then heated at 110 °C overnight under nitrogen. Once the reaction was complete, the mixture was filtered through Celite and the filtrate was concentrated in vacuo. A 1 M HCl solution (20 mL) was added, and the compound was extracted with EtOAc (3 × 50 mL). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered and concentrated under reduced pressure to afford compound <b>3</b> as a brown solid (7.04 g, 100%); <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 8.05–7.93 (m, 4H), 7.84 (dd, <i>J</i> = 9.5/6.7 Hz, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.4 (d, <i>J</i> = 21.7 Hz), −161.3 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 188.0, 165.8, 150.7 (ddd, <i>J</i><sub>CF</sub> = 247.2/9.8/4.1 Hz), 142.9–142.8 (m), 139.1 (dt, <i>J</i><sub>CF</sub> = 248.4/14.2 Hz), 135.8–134.5 (m), 131.6, 130.2, 127.7, 116.5–105.3 (m); <i>m</i>/<i>z</i> MS C<sub>14</sub>H<sub>6</sub>F<sub>3</sub>O<sub>3</sub> [M – H]<sup>−</sup> calcd 279.0, found 279.0.</div></div><div id="sec4_1_4_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i27" class="anchor-spacer"></div><h5 class="article-section__title" id="_i27"> Methyl 2-Amino-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>4</b>)</h5><div class="NLM_p last">Compound <b>3</b> (7.04 g, 25.1 mmol) and benzylamine (4.12 mL, 37.7 mmol) were dissolved in anhydrous DCE (200 mL) and stirred for 30 min. Na(OAc)<sub>3</sub>BH (7.99 g, 56.6 mmol) was added, and the mixture was stirred at rt overnight. Once the reaction was complete, water (30 mL) was added and the mixture was stirred vigorously for 5 min. DCE was removed in vacuo and the solid was filtered and washed with ethanol to give 9.33 g of 2-(benzylamino)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetic acid as a yellow solid. The crude solid was dissolved in MeOH (250 mL) and conc. H<sub>2</sub>SO<sub>4</sub> (5.36 mL, 101 mmol) was added dropwise. The reaction mixture was refluxed for 16 h and then concentrated under reduced pressure. Sat. NaHCO<sub>3</sub> was added and the mixture was extracted with DCM (3 × 150 mL). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure to afford 7.86 g of methyl 2-(benzylamino)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate. The crude ester was subsequently dissolved in MeOH (200 mL), and 32% HCl (5 mL) was added. The flask was evacuated and flushed with nitrogen three times before the addition of 10% Pd/C (1.60 g). The reaction mixture was stirred vigorously under a hydrogen atmosphere at rt overnight. Upon completion, the reaction mixture was filtered through Celite and washed with MeOH (50 mL). The filtrate was concentrated in vacuo followed by dilution with sat. NaHCO<sub>3</sub> and extraction with EtOAc (3 × 100 mL). The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by column chromatography (PE/EtOAc 50:50–0:100) to afford compound <b>4</b> as a sticky yellow solid (3.51 g, 47% over three steps). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 7.47 (m, 4H), 7.20–7.10 (m, 2H), 4.67 (s, 1H), 3.72 (s, 3H), 2.30 (s, 2H); <sup>19</sup>F NMR (CDCl<sub>3</sub>) δ −134.0 (d, <i>J</i> = 20.5 Hz), −162.4 (dd, <i>J</i> = 20.6/20.6 Hz); <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ 174.3, 151.5 (ddd, <i>J</i><sub>CF</sub> = 249.6/10.0/4.3 Hz), 140.5, 140.9–137.9 (m), 138.1–137.9 (m), 136.8 (td, <i>J</i><sub>CF</sub> = 7.8/4.7 Hz), 127.7, 127.3, 111.3–110.9 (m), 58.4, 52.6; <i>m</i>/<i>z</i> MS C<sub>15</sub>H<sub>13</sub>F<sub>3</sub>NO<sub>2</sub> [MH]<sup>+</sup> calcd 296.1, found 296.1.</div></div><div id="sec4_1_4_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28"> Methyl 2-Acetamido-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5a</b>)</h5><div class="NLM_p last">To a mixture of compound <b>4</b> (300 mg, 1.01 mmol) in anhydrous toluene (4 mL) was added Meldrum’s acid (161 mg, 1.12 mmol). The reaction mixture was refluxed for 3 h. After cooling to rt, the resulting precipitate was filtered and washed with Et<sub>2</sub>O to afford compound <b>5a</b> as a white solid (172 mg, 50%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 8.80 (d, <i>J</i> = 7.3 Hz, 1H), 7.89–7.64 (m, 4H), 7.48 (d, <i>J</i> = 8.3 Hz, 2H), 5.48 (d, <i>J</i> = 7.3 Hz, 1H), 3.63 (s, 3H), 1.91 (s, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.8 (d, <i>J</i> = 21.8 Hz), −163.3 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 171.0, 169.3, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.6/9.7/4.2 Hz), 138.4 (dt, <i>J</i><sub>CF</sub> = 249.5/15.7 Hz), 136.9–136.8 (m), 136.7, 136.3 (td, <i>J</i><sub>CF</sub> = 8.1/4.5 Hz), 128.4, 127.2, 117.8–108.1 (m), 55.8, 52.3, 22.2; <i>m</i>/<i>z</i> MS C<sub>17</sub>H<sub>15</sub>F<sub>3</sub>NO<sub>3</sub> [MH]<sup>+</sup> calcd 338.1, found 338.1.</div></div><div id="sec4_1_4_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29"> <i>tert</i>-Butyl 3-((2-Methoxy-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)amino)-3-oxopropanoate (<b>5b</b>)</h5><div class="NLM_p last">3-(<i>tert</i>-Butoxy)-3-oxopropanoic acid (198 mg, 1.24 mmol) was coupled to compound <b>4</b> (330 mg, 1.12 mmol) according to General Procedure A. The crude product was purified by column chromatography (PE/EtOAc 0:100–50:50) to afford compound <b>5b</b> (185 mg, 38%) as an orange oil. <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 9.00 (d, <i>J</i> = 7.3 Hz, 1H), 7.80–7.67 (m, 4H), 7.50 (d, <i>J</i> = 8.3 Hz, 2H), 5.52 (d, <i>J</i> = 7.3 Hz, 1H), 3.65 (s, 3H), 3.31–3.21 (m, 2H), 1.39 (s, 9H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −163.3 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 170.6, 166.9, 165.5, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.6/9.8/4.2 Hz), 138.4 (dt, <i>J</i><sub>CF</sub> = 249.7/15.6 Hz), 136.9–136.8 (m), 136.6, 136.3 (m), 128.3, 127.2, 111.7–111.0 (m), 80.6, 55.8, 52.4, 43.2, 27.7; <i>m</i>/<i>z</i> MS C<sub>22</sub>H<sub>21</sub>F<sub>3</sub>NO<sub>5</sub> [M – H]<sup>−</sup> calcd 436.1, found 436.1.</div></div><div id="sec4_1_4_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30"> 4-((2-Methoxy-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)amino)-4-oxobutanoic Acid (<b>5e</b>)</h5><div class="NLM_p last">To a mixture of compound <b>4</b> (428 mg, 1.45 mmol) in anhydrous toluene (10 mL) was added succinic anhydride (160 mg, 1.60 mmol). The reaction mixture was refluxed for 3 h. After cooling to rt, the resulting precipitate was filtered and washed with Et<sub>2</sub>O to afford compound <b>5e</b> as a white solid (267 mg, 49%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 12.11 (s, 1H), 8.83 (d, <i>J</i> = 7.3 Hz, 1H), 7.80–7.65 (m, 4H), 7.49 (d, <i>J</i> = 8.3 Hz, 2H), 5.49 (d, <i>J</i> = 7.3 Hz, 1H), 3.63 (s, 3H), 2.46–2.39 (m, 4H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.82 (d, <i>J</i> = 21.8 Hz), −163.24 (dd, <i>J</i> = 21.8/21.8 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 173.7, 171.2, 171.0, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.4/9.6/4.0 Hz), 138.4 (m), 137.0 (m), 136.8, 136.3 (m), 128.4, 127.1, 111.5–111.2 (m), 55.8, 52.3, 29.6, 28.9; <i>m</i>/<i>z</i> MS C<sub>19</sub>H<sub>17</sub>F<sub>3</sub>NO<sub>5</sub> [MH]<sup>+</sup> calcd 396.1, found 396.1.</div></div><div id="sec4_1_4_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31"> Methyl 2-(4-Amino-4-oxobutanamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5f</b>)</h5><div class="NLM_p last">Compound <b>5e</b> (272 mg, 0.688 mmol) and PyBOP (531 mg, 1.02 mmol) in DMF (10 mL) were stirred for 10 min. DIPEA (0.2 mL, 1.02 mmol) and ammonium carbonate (332 mg, 3.45 mmol) were added to the reaction mixture which was stirred at rt overnight. After completion, the mixture was diluted with water (10 mL) and extracted with DCM (3 × 15 mL). The combined organic layers were washed with sat. NaHCO<sub>3</sub> (2 × 30 mL) and brine (30 mL). The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude product was purified by column chromatography (PE/EtOAc 50:50–0:100) to afford compound <b>5f</b> as a white solid (84 mg, 31%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 8.81 (d, <i>J</i> = 7.3 Hz, 1H), 7.87–7.60 (m, 4H), 7.49 (d, <i>J</i> = 8.3 Hz, 2H), 7.29 (br s, 1H), 6.75 (br s, 1H), 5.49 (d, <i>J</i> = 7.3 Hz, 1H), 3.63 (s, 3H), 2.47–2.25 (m, 4H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.8 (d, <i>J</i> = 21.8 Hz), −163.3 (dd, <i>J</i> = 21.8/21.8 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 173.4, 171.7, 171.0, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.5/9.7/4.2 Hz), 138.4 (dt, <i>J</i><sub>CF</sub> = 249.6/15.7 Hz), 136.8 (2C), 136.34 (td, <i>J</i><sub>CF</sub> = 8.2/4.2 Hz), 128.4, 127.2, 111.6–110.9 (m), 55.8, 52.3, 30.2, 30.1; <i>m</i>/<i>z</i> MS C<sub>19</sub>H<sub>18</sub>F<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 395.1, found 395.1.</div></div><div id="sec4_1_4_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> Methyl 2-Benzamido-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5g</b>)</h5><div class="NLM_p last">Benzoic acid (99.3 mg, 0.813 mmol) was coupled to compound <b>4</b> (200 mg, 0.678 mmol) according to General Procedure B. The crude product was purified by column chromatography (PE/EtOAc 100:0–50:50) to afford compound <b>5g</b> as a bright yellow foam (244 mg, 90%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 9.26 (d, <i>J</i> = 7.3 Hz, 1H), 8.00–7.91 (m, 2H), 7.79–7.66 (m, 4H), 7.61 (d, <i>J</i> = 8.3 Hz, 2H), 7.58–7.53 (m, 1H), 7.51–7.45 (m, 2H), 5.79 (d, <i>J</i> = 7.3 Hz, 1H), 3.69 (s, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.8 (d, <i>J</i> = 21.7 Hz), −163.4 (dd, <i>J</i> = 21.6/21.6 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 170.9, 166.5, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.7/9.7/4.2 Hz), 138.4 (dt, <i>J</i><sub>CF</sub> = 249.5/15.6 Hz), 136.8, 136.8–136.7 (m), 136.5–136.3 (td, <i>J</i><sub>CF</sub> = 8.1/4.4 Hz), 133.5, 131.6, 128.9, 128.2, 127.7, 127.0, 111.6–111.0 (m), 56.4, 52.3; <i>m</i>/<i>z</i> MS C<sub>22</sub>H<sub>17</sub>F<sub>3</sub>NO<sub>3</sub> [MH]<sup>+</sup> calcd 400.1, found 400.1.</div></div><div id="sec4_1_4_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> Methyl 2-(3-Fluorobenzamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5h</b>)</h5><div class="NLM_p last">3-Fluorobenzoic acid (114 mg, 0.812 mmol) was coupled to compound <b>4</b> (200 mg, 0.677 mmol) according to General Procedure B. The crude product was purified by column chromatography (PE/EtOAc 100:0–50:50) to afford compound <b>5h</b> as a bright yellow foam (243 mg, 86%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 9.35 (d, <i>J</i> = 7.1 Hz, 1H), 7.82–7.66 (m, 6H), 7.60 (d, <i>J</i> = 8.3 Hz, 2H), 7.54 (ddd, <i>J</i> = 8.0/7.9/5.9 Hz, 1H), 7.44–7.38 (m, 1H), 5.77 (d, <i>J</i> = 7.1 Hz, 1H), 3.69 (s, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −112.9, −134.8 (d, <i>J</i> = 21.7 Hz), −163.3 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 170.7, 165.2 (d, <i>J</i><sub>CF</sub> = 2.5 Hz), 161.9 (d, <i>J</i><sub>CF</sub> = 244.3 Hz), 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.7/9.7/4.1 Hz), 138.4 (dt, <i>J</i><sub>CF</sub> = 249.5/15.7 Hz), 136.9–136.6 (m), 136.5, 136.4–136.2 (m), 135.7 (d, <i>J</i><sub>CF</sub> = 6.9 Hz), 130.4 (d, <i>J</i><sub>CF</sub> = 8.0 Hz), 129.0, 127.0, 123.9 (d, <i>J</i><sub>CF</sub> = 2.8 Hz), 118.5 (d, <i>J</i><sub>CF</sub> = 21.2 Hz), 114.5 (d, <i>J</i><sub>CF</sub> = 22.9 Hz), 112.1–110.6 (m), 56.5, 52.4; <i>m</i>/<i>z</i> MS C<sub>22</sub>H<sub>16</sub>F<sub>4</sub>NO<sub>3</sub> [MH]<sup>+</sup> calcd 418.1, found 418.1.</div></div><div id="sec4_1_4_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> Methyl 2-(4-Fluorobenzamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5i</b>)</h5><div class="NLM_p last">4-Fluorobenzoic acid (114 mg, 0.812 mmol) was coupled to compound <b>4</b> (200 mg, 0.677 mmol) according to General Procedure B. The crude product was purified by column chromatography (PE/EtOAc 100:0–50:50) to afford compound <b>5i</b> as a bright yellow foam (150 mg, 53%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 9.28 (d, <i>J</i> = 7.2 Hz, 1H), 8.08–7.97 (m, 2H), 7.75 (d, <i>J</i> = 8.3 Hz, 2H), 7.72–7.64 (m, 2H), 7.60 (d, <i>J</i> = 8.3 Hz, 2H), 7.34–7.25 (m, 2H), 5.77 (d, <i>J</i> = 7.2 Hz, 1H), 3.69 (s, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −108.8, −134.8 (d, <i>J</i> = 21.6 Hz), −163.4 (dd, <i>J</i> = 21.6/21.6 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 170.8, 165.4, 164.1 (d, <i>J</i><sub>CF</sub> = 249.1 Hz), 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.8/9.7/4.1 Hz), 138.4 (dt, <i>J</i><sub>CF</sub> = 249.6/15.6 Hz), 136.8–136.7 (m), 136.6, 136.4 (td, <i>J</i><sub>CF</sub> = 8.1/4.4 Hz), 130.4 (d, <i>J</i><sub>CF</sub> = 9.1 Hz), 123.0 (d, <i>J</i><sub>CF</sub> = 2.9 Hz), 128.9, 127.0, 115.1 (d, <i>J</i><sub>CF</sub> = 21.8 Hz), 111.2 (m), 56.5, 52.3; <i>m</i>/<i>z</i> MS C<sub>22</sub>H<sub>16</sub>F<sub>4</sub>NO<sub>3</sub> [MH]<sup>+</sup> calcd 418.1, found 417.8.</div></div><div id="sec4_1_4_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> Methyl 2-(3-Methoxybenzamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5j</b>)</h5><div class="NLM_p last">3-Methoxybenzoic acid (247 mg, 1.63 mmol) was coupled to compound <b>4</b> (400 mg, 1.36 mmol) according to General Procedure B. The crude product was purified by column chromatography (PE/EtOAc 100:0–50:50) to afford compound <b>5j</b> as a bright yellow foam (578 mg, 99%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 9.25 (d, <i>J</i> = 7.2 Hz, 1H), 7.75 (d, <i>J</i> = 8.4 Hz, 2H), 7.69 (m, 2H), 7.61 (d, <i>J</i> = 8.3 Hz, 2H), 7.56–7.51 (m, 1H), 7.51–7.48 (m, 1H), 7.39 (t, <i>J</i> = 7.9 Hz, 1H), 7.15–7.09 (m, 1H), 5.78 (d, <i>J</i> = 7.2 Hz, 1H), 3.81 (s, 3H), 3.69 (s, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.8 (d, <i>J</i> = 21.7 Hz), −163.3 (dd, <i>J</i> = 21.6/21.6 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 170.9, 166.2, 159.1, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.8/9.8/4.2 Hz), 138.4 (dt, <i>J</i><sub>CF</sub> = 249.5/15.6 Hz), 136.9–136.7 (m, 2C), 136.4 (td, <i>J</i><sub>CF</sub> = 8.1/4.4 Hz), 134.9, 129.4, 128.9, 127.0, 120.0, 117.5, 112.8, 112.0–110.4 (m), 56.5, 55.3, 52.3; <i>m</i>/<i>z</i> MS C<sub>23</sub>H<sub>19</sub>F<sub>3</sub>NO<sub>4</sub> [MH]<sup>+</sup> calcd 430.1, found 429.9.</div></div><div id="sec4_1_4_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> Methyl 2-(4-Methoxybenzamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5k</b>)</h5><div class="NLM_p last">4-Methoxybenzoic acid (199 mg, 1.31 mmol) was coupled to compound <b>4</b> (322 mg, 1.09 mmol) according to General Procedure B. The crude product was purified by column chromatography (PE/EtOAc 100:0–50:50) to afford compound <b>5k</b> as a white solid (154 mg, 33%). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 7.80 (d, <i>J</i> = 8.9 Hz, 2H), 7.54–7.49 (m, 2H), 7.48–7.42 (m, 2H), 7.29 (d, <i>J</i> = 6.7 Hz, 1H), 7.13 (m, 2H), 6.90 (d, <i>J</i> = 8.9 Hz, 2H), 5.81 (d, <i>J</i> = 6.8 Hz, 1H), 3.82 (s, 3H), 3.78 (s, 3H); <sup>19</sup>F NMR (CDCl<sub>3</sub>) δ −134.0 (d, <i>J</i> = 20.5 Hz), −162.3 (dd, <i>J</i> = 20.5 Hz); <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ 171.5, 166.3, 162.7, 151.5 (ddd, <i>J</i><sub>CF</sub> = 249.7/9.9/4.2 Hz), 139.5 (dt, <i>J</i><sub>CF</sub> = 252.2/15.5 Hz), 138.6–138.4 (m), 137.2, 136.7 (td, <i>J</i><sub>CF</sub> = 7.7/4.6 Hz), 129.2, 128.2, 127.5, 125.7, 113.9, 111.1 (m), 56.6, 55.5, 53.1; <i>m</i>/<i>z</i> MS C<sub>23</sub>H<sub>19</sub>F<sub>3</sub>NO<sub>4</sub> [MH]<sup>+</sup> calcd 430.1, found 429.9.</div></div><div id="sec4_1_4_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> Methyl 2-(3-Cyanobenzamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5n</b>)</h5><div class="NLM_p last">Oxalyl chloride (130 μL, 1.51 mmol) was added dropwise to a mixture of 3-carbamoylbenzoic acid (167 mg, 1.01 mmol) in DCM (10 mL) containing a catalytic amount of DMF (20 μL). After stirring the mixture at rt for 1 h, the DCM was concentrated in vacuo. A mixture of DIPEA (126 μL, 1.31 mmol) and compound <b>4</b> (300 mg, 1.01 mmol) in DCM (10 mL) was added to the acid chloride. The reaction mixture was stirred at rt for 30 min and then diluted with water (15 mL) and extracted with DCM (3 × 10 mL). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. LC–MS of the crude product indicated dehydration of the carboxamide group occurred to form a cyano analogue. The crude product was purified by column chromatography (PE/EtOAc 100:0–0:100) to afford compound <b>5n</b> as a clear oil (383 mg, 89%). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 8.12 (s, 1H), 8.06 (d, <i>J</i> = 7.9 Hz, 1H), 7.80 (d, <i>J</i> = 7.8 Hz, 1H), 7.58 (t, <i>J</i> = 7.7 Hz, 1H), 7.54–7.48 (m, 4H), 7.44 (d, <i>J</i> = 6.7 Hz, 1H), 7.19–7.10 (m, 2H), 5.79 (d, <i>J</i> = 6.7 Hz, 1H), 3.81 (s, 3H); <sup>19</sup>F NMR (CDCl<sub>3</sub>) δ −133.7 (d, <i>J</i> = 20.6 Hz), −161.9 (dd, <i>J</i> = 20.6/20.6 Hz); <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ 171.1, 164.7, 151.6 (ddd, <i>J</i><sub>CF</sub> = 250.0/10.1/4.2 Hz), 139.6 (dt, <i>J</i><sub>CF</sub> = 252.4/15.4 Hz), 138.9–138.7 (m), 136.6–136.5 (m), 136.4, 135.3, 134.7, 131.5, 131.2, 129.8, 128.2, 127.7, 117.9, 113.2, 112.0–110.2 (m), 56.8, 53.4; <i>m</i>/<i>z</i> MS C<sub>23</sub>H<sub>14</sub>F<sub>3</sub>N<sub>2</sub>O<sub>3</sub> [M – H]<sup>−</sup> calcd 423.1, found 423.1.</div></div><div id="sec4_1_4_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> Methyl 2-(4-Cyanobenzamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5o</b>)</h5><div class="NLM_p last">Oxalyl chloride (130 μL, 1.51 mmol) was added dropwise to a mixture of 4-carbamoylbenzoic acid (167 mg, 1.01 mmol) in DCM (10 mL) containing a catalytic amount of DMF (20 μL). After stirring the mixture at rt for 1 h, the DCM was removed in vacuo. A mixture of DIPEA (126 μL, 1.31 mmol) and compound <b>4</b> (300 mg, 1.01 mmol) in DCM (10 mL) was added to the acid chloride. The reaction mixture was stirred at rt for 30 min, then diluted with water (15 mL), and extracted with DCM (3 × 10 mL). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. LC–MS of the crude product indicated dehydration of the carboxamide group occurred to form a cyano analogue. The crude product was purified by column chromatography (PE/EtOAc 100:0–0:100) to afford compound <b>5o</b> as a yellow oil (185 mg, 43%). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 7.98–7.73 (m, 4H), 7.54–7.45 (m, 4H), 7.34 (app. t, <i>J</i> = 6.6 Hz, 1H), 7.19–7.11 (m, 2H), 5.79 (d, <i>J</i> = 6.6 Hz, 1H), 3.81 (s, 3H); <sup>19</sup>F NMR (CDCl<sub>3</sub>) δ −133.7 (d, <i>J</i> = 20.5 Hz), −161.9 (dd, <i>J</i> = 20.5/20.5 Hz); <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ 171.1, 164.9, 151.6 (ddd, <i>J</i><sub>CF</sub> = 250.0/10.1/4.3 Hz), 139.6 (dt, <i>J</i><sub>CF</sub> = 252.5/15.3 Hz), 139.0–138.8 (m), 137.4, 136.8–136.3 (m, 2C), 132.7, 128.2, 128.0, 127.7, 118.0, 115.8, 111.9–110.6 (m), 56.7, 53.5; <i>m</i>/<i>z</i> MS C<sub>23</sub>H<sub>16</sub>F<sub>3</sub>N<sub>2</sub>O<sub>3</sub> [MH]<sup>+</sup> calcd 425.1, found 425.1.</div></div><div id="sec4_1_4_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> Methyl 2-(3-Carbamoylbenzamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5r</b>)</h5><div class="NLM_p last">3-Carbamoylbenzoic acid (205 mg, 1.24 mmol) was coupled to compound <b>4</b> (332 mg, 1.13 mmol) according to General Procedure A. The crude product was purified by column chromatography (PE/EtOAc 50:50–0:100) to afford compound <b>5r</b> as a light yellow solid (328 mg, 66%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 9.37 (d, <i>J</i> = 7.1 Hz, 1H), 8.42 (t, <i>J</i> = 1.6 Hz, 1H), 8.07 (br s, 1H), 8.06–7.99 (m, 2H), 7.78–7.69 (m, 4H), 7.64–7.54 (m, 3H), 7.50 (br s, 1H), 5.78 (d, <i>J</i> = 7.1 Hz, 1H), 3.69 (s, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.8 (d, <i>J</i> = 21.7 Hz), −163.3 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 170.9, 167.5, 166.2, 150.7 (ddd, <i>J</i><sub>CF</sub> = 246.6/9.7/4.1 Hz), 138.4 (dt, <i>J</i><sub>CF</sub> = 249.5/15.6 Hz), 136.9–136.8 (m), 136.7, 136.4 (td, <i>J</i><sub>CF</sub> = 8.1/4.4 Hz), 134.5, 133.6, 130.5, 130.4, 129.0, 128.4, 127.1, 126.9, 112.9–109.9 (m), 56.6, 52.5; <i>m</i>/<i>z</i> MS C<sub>23</sub>H<sub>18</sub>F<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 443.1, found 443.1.</div></div><div id="sec4_1_4_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> Methyl 2-(4-Carbamoylbenzamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5s</b>)</h5><div class="NLM_p last">4-Carbamoylbenzoic acid (246 mg, 1.49 mmol) was coupled to compound <b>4</b> (400 mg, 1.35 mmol) according to General Procedure A. The crude product was purified by column chromatography (PE/EtOAc 50:50–0:100) to afford compound <b>5s</b> as a light yellow solid (383 mg, 64%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 9.38 (d, <i>J</i> = 7.2 Hz, 1H), 8.10 (br s, 1H), 8.02–7.91 (m, 4H), 7.84–7.68 (m, 4H), 7.60 (d, <i>J</i> = 8.3 Hz, 2H), 7.52 (br s, 1H), 5.77 (d, <i>J</i> = 7.1 Hz, 1H), 3.68 (s, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.8 (d, <i>J</i> = 21.8 Hz), −163.2 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 170.8, 167.2, 165.9, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.4/9.5/4.1 Hz), 138.4 (dt, <i>J</i><sub>CF</sub> = 249.8/15.7 Hz), 136.9, 136.4 (td, <i>J</i><sub>CF</sub> = 8.2/5.5 Hz), 135.8–135.7 (m), 133.4, 132.2, 129.0, 127.7, 127.4, 127.1, 111.6–111.1 (m), 56.5, 52.5; <i>m</i>/<i>z</i> MS C<sub>23</sub>H<sub>18</sub>F<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 443.1, found 443.1.</div></div><div id="sec4_1_4_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> Methyl 2-(4-(Methylcarbamoyl)benzamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5t</b>)</h5><div class="NLM_p last">4-(Methylcarbamoyl)benzoic acid (97.3 mg, 0.504 mmol) was coupled to compound <b>4</b> (107 mg, 0.364 mmol) according to General Procedure B. Upon completion, the mixture was added to a 0.1 M HCl solution (50 mL) and extracted with EtOAc (3 × 50 mL). The combined organic layers were washed with brine (50 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was then taken up in minimal DMF and added to an ice–water mixture. The precipitate was filtered and washed with minimal MeOH to afford compound <b>5t</b> as a yellow solid (98.4 mg, 59%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 9.38 (d, <i>J</i> = 7.1 Hz, 1H), 8.57 (d, <i>J</i> = 4.5 Hz, 1H), 8.04–7.96 (m, 2H), 7.92 (d, <i>J</i> = 8.4 Hz, 2H), 7.79–7.67 (m, 4H), 7.60 (d, <i>J</i> = 8.3 Hz, 2H), 5.78 (d, <i>J</i> = 7.1 Hz, 1H), 3.69 (s, 3H), 2.80 (d, <i>J</i> = 4.5 Hz, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.8 (d, <i>J</i> = 21.7 Hz), −163.3 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 170.8, 166.0 (2C), 150.7 (ddd, <i>J</i><sub>CF</sub> = 246.4/9.7/4.2 Hz), 139.9–137.1 (m), 137.2, 136.9 (br s), 136.6, 136.5–136.4 (m), 129.0, 127.8, 127.1, 127.0, 112.7–108.5 (m), 56.6, 52.5, 26.3; <i>m</i>/<i>z</i> MS C<sub>24</sub>H<sub>20</sub>F<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 457.1, found 456.8.</div></div><div id="sec4_1_4_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> Methyl 2-(4-(Dimethylcarbamoyl)benzamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5u</b>)</h5><div class="NLM_p last">4-(Dimethylcarbamoyl)benzoic acid (102 mg, 0.491 mmol) was coupled to compound <b>4</b> (114 mg, 0.385 mmol) according to General Procedure B. Upon completion, the mixture was added to a 0.1 M HCl solution (50 mL) and extracted with EtOAc (3 × 50 mL). The combined organic layers were washed with brine (50 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was then taken up in minimal DMF and added to an ice–water mixture. The precipitate was filtered and washed with minimal MeOH to afford compound <b>5u</b> as a white solid (92.0 mg, 51%) <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 9.36 (d, <i>J</i> = 7.2 Hz, 1H), 8.07–7.86 (m, 2H), 7.79–7.68 (m, 4H), 7.60 (d, <i>J</i> = 8.3 Hz, 2H), 7.53–7.46 (m, 2H), 5.77 (d, <i>J</i> = 7.2 Hz, 1H), 3.68 (s, 3H), 2.99 (s, 3H), 2.88 (s, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.8 (d, <i>J</i> = 21.7 Hz), −163.3 (dd, <i>J</i> = 21.8/21.8 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 171.0, 169.7, 166.3, 150.8 (ddd, <i>J</i><sub>CF</sub> = 14.1/9.4/3.6 Hz), 139.9–137.2 (m), 139.7, 137.0 (d, <i>J</i><sub>CF</sub> = 1.5 Hz), 136.8, 136.69–136.36 (m), 134.2, 129.2, 128.0, 127.2, 127.0, 111.4 (dd, <i>J</i><sub>CF</sub> = 16.2/5.3 Hz), 56.7, 52.6, 34.9; <i>m</i>/<i>z</i> MS C<sub>25</sub>H<sub>22</sub>F<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 471.2, found 470.9.</div></div><div id="sec4_1_4_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> Methyl 2-(4-(Ethylcarbamoyl)benzamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5v</b>)</h5><div class="NLM_p last">Carboxylic acid <b>12</b> (102 mg, 0.491 mmol) was coupled to compound <b>4</b> (114 mg, 0.385 mmol) according to General Procedure B. Upon completion, the mixture was added to a 0.1 M HCl solution (50 mL) and extracted with EtOAc (3 × 50 mL). The combined organic layers were washed with brine (50 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was then taken up in minimal DMF and added to an ice–water mixture. The precipitate was filtered and washed with minimal MeOH to afford compound <b>5v</b> as a yellow solid (92.0 mg, 51%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 9.39 (d, <i>J</i> = 7.1 Hz, 1H), 8.61 (t, <i>J</i> = 5.5 Hz, 1H), 8.03–7.98 (m, 2H), 7.93 (d, <i>J</i> = 8.4 Hz, 2H), 7.76 (d, <i>J</i> = 8.3 Hz, 2H), 7.71 (dd, <i>J</i> = 9.4, 6.8 Hz, 2H), 7.61 (d, <i>J</i> = 8.3 Hz, 2H), 5.78 (d, <i>J</i> = 7.1 Hz, 1H), 3.69 (s, 3H), 3.34–3.26 (m, 2H), 1.14 (t, <i>J</i> = 7.2 Hz, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.8 (d, <i>J</i> = 21.7 Hz), −163.3 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 170.8, 165.9, 165.2, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.8/9.6/4.4 Hz), 138.4 (dt, <i>J</i><sub>CF</sub> = 33.6/15.7 Hz), 137.3, 136.8, 136.6, 136.4 (td, <i>J</i><sub>CF</sub> = 8.1/4.6 Hz), 129.0, 127.7, 127.07, 127.05, 111.3 (dd, <i>J</i><sub>CF</sub> = 15.9/5.5 Hz), 56.5, 52.4, 34.2, 14.7; <i>m</i>/<i>z</i> MS C<sub>25</sub>H<sub>22</sub>F<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 471.2, found 470.9.</div></div><div id="sec4_1_4_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> Methyl 2-(4-(Isopropylcarbamoyl)benzamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5w</b>)</h5><div class="NLM_p last">4-(Isopropylcarbamoyl)benzoic acid (119 mg, 0.573 mmol) was coupled to compound <b>4</b> (118 mg, 0.401 mmol) according to General Procedure B. Upon completion, the mixture was added to a 0.1 M HCl solution (50 mL) and extracted with EtOAc (3 × 50 mL). The combined organic layers were washed with brine (50 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was then taken up in minimal DMF and added to an ice–water mixture. The precipitate was filtered and washed with minimal MeOH to afford compound <b>5w</b> as a white solid (177 mg, 91%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 9.40 (d, <i>J</i> = 7.1 Hz, 1H), 8.37 (d, <i>J</i> = 7.7 Hz, 1H), 8.02 (d, <i>J</i> = 8.5 Hz, 2H), 7.95 (d, <i>J</i> = 8.4 Hz, 2H), 7.76 (d, <i>J</i> = 8.3 Hz, 2H), 7.69 (dd, <i>J</i> = 9.3, 6.8 Hz, 2H), 7.62 (d, <i>J</i> = 8.3 Hz, 2H), 5.80 (d, <i>J</i> = 7.1 Hz, 1H), 4.22–4.00 (m, 1H), 3.70 (s, 3H), 1.18 (d, <i>J</i> = 6.6 Hz, 6H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −130.0 (d, <i>J</i> = 21.6 Hz), −158.5 (dd, <i>J</i> = 21.6/21.6 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 170.90, 166.0, 164.7, 153.0–148.6 (m), 138.4 (dt, <i>J</i><sub>CF</sub> = 30.9/13.8 Hz), 137.5, 136.9, 136.6, 136.5–136.3 (m), 135.5, 129.0, 127.7, 127.2, 127.1, 111.3 (dd, <i>J</i><sub>CF</sub> = 18.9/2.5 Hz), 56.6, 52.4, 41.2, 22.3; <i>m</i>/<i>z</i> MS C<sub>26</sub>H<sub>24</sub>F<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 485.2, found 484.9.</div></div><div id="sec4_1_4_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> Methyl 2-(4-((2-Hydroxyethyl)carbamoyl)benzamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5x</b>)</h5><div class="NLM_p last">A sealed vessel containing (1-cyano-2-ethoxy-2-oxoethylidenaminooxy)dimethylamino-morpholino-carbenium hexafluorophosphate (2.23 g, 0.584 mmol), triethylamine (1.10 mL, 7.89 mmol), carboxylic acid <b>14</b> (746 mg, 3.56 mmol), and compound <b>4</b> (761 mg, 2.58 mmol) was purged twice with nitrogen and charged with DMF (5 mL). The reaction mixture was allowed to stir at rt overnight. The mixture was then added to a half saturated NaHCO<sub>3</sub> solution (80 mL) and extracted with EtOAc (3 × 80 mL). The combined organic layers were washed with brine (50 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was then taken up in minimal DMF and added to an ice–water mixture. The precipitate was filtered and washed with minimal MeOH and further purified by column chromatography (DCM/MeOH 98:2–94:6) to afford compound <b>5x</b> as a cream solid after lyophilization (182 mg, 14%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 9.41 (d, <i>J</i> = 7.2 Hz, 1H), 8.61 (t, <i>J</i> = 5.7 Hz, 1H), 8.06–7.99 (m, 2H), 8.00–7.93 (m, 2H), 7.78–7.72 (m, 2H), 7.67 (dd, <i>J</i> = 9.4, 6.6 Hz, 2H), 7.65–7.58 (m, 2H), 5.80 (d, <i>J</i> = 7.1 Hz, 1H), 4.81 (br s, 1H), 3.70 (s, 3H), 3.56 (t, <i>J</i> = 6.2 Hz, 2H), 3.38 (q, <i>J</i> = 6.0, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.8 (d, <i>J</i> = 21.8 Hz), −163.2 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 170.9, 166.1, 165.8, 150.7 (ddd, <i>J</i><sub>CF</sub> = 246.8/9.7/4.2 Hz), 138.5 (dt, <i>J</i><sub>CF</sub> = 30.6/12.4 Hz), 137.3, 137.0, 136.7, 136.5 (td, <i>J</i><sub>CF</sub> = 8.1/4.4 Hz), 135.7, 129.1, 127.8, 127.3, 127.1, 111.3 (dd, <i>J</i><sub>CF</sub> = 16.0/5.5 Hz), 59.8, 56.7, 52.5, 42.4; <i>m</i>/<i>z</i> MS C<sub>25</sub>H<sub>22</sub>F<sub>3</sub>N<sub>2</sub>O<sub>5</sub> [MH]<sup>+</sup> calcd 487.1, found 486.8.</div></div><div id="sec4_1_4_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> Methyl 2-(4-Sulfamoylbenzamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5y</b>)</h5><div class="NLM_p last">4-Carboxybenzenesulfonamide (274 mg, 1.36 mmol) was coupled to compound <b>4</b> (290 mg, 0.981 mmol) according to General Procedure B. Upon completion, the mixture was added to a 0.1 M HCl solution (50 mL) and extracted with EtOAc (3 × 50 mL). The combined organic layers were washed with brine (50 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was then taken up in minimal DMF and added to an ice–water mixture. The fine precipitate was centrifuged, the supernatant removed, and the solid resuspended in EtOAc twice to afford compound <b>5y</b> as a light yellow solid after lyophilization (101 mg, 22%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 9.46 (d, <i>J</i> = 7.1 Hz, 1H), 8.07 (d, <i>J</i> = 8.4 Hz, 2H), 7.91 (d, <i>J</i> = 8.3 Hz, 2H), 7.79–7.69 (m, 4H), 7.60 (d, <i>J</i> = 8.2 Hz, 2H), 7.51 (s, 2H), 5.76 (d, <i>J</i> = 7.0 Hz, 1H), 3.68 (s, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.8 (d, <i>J</i> = 21.4 Hz), −163.2 (dd, <i>J</i> = 21.5/21.5 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 171.2, 166.1, 151.1 (ddd, <i>J</i><sub>CF</sub> = 13.9/9.3/3.7 Hz), 147.1, 140.8–137.5 (m), 137.4, 136.9, 136.8, 129.5, 128.9, 127.5, 126.1, 111.8 (dd, <i>J</i><sub>CF</sub> = 16.0/5.4 Hz), 57.1, 52.9; <i>m</i>/<i>z</i> MS C<sub>22</sub>H<sub>18</sub>F<sub>3</sub>N<sub>2</sub>O<sub>5</sub>S [MH]<sup>+</sup> calcd 479.1, found 478.8.</div></div><div id="sec4_1_4_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> Methyl 2-(3-((<i>tert</i>-Butoxycarbonyl)amino)benzamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5z</b>)</h5><div class="NLM_p last">3-((<i>tert</i>-Butoxycarbonyl)amino)benzoic acid (289 mg, 1.22 mmol) was coupled to compound <b>4</b> (300 mg, 1.02 mmol) according to General Procedure B. The crude product was purified by column chromatography (PE/EtOAc 100:0–50:50) to afford compound <b>5z</b> as a yellow oil (353 mg, 67%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 9.48 (s, 1H), 9.17 (d, <i>J</i> = 7.3 Hz, 1H), 7.99 (m, 1H), 7.78–7.66 (m, 4H), 7.63–7.56 (m, 3H), 7.54–7.48 (m, 1H), 7.34 (t, <i>J</i> = 7.9 Hz, 1H), 5.74 (d, <i>J</i> = 7.1 Hz, 1H), 3.68 (s, 3H), 1.48 (s, 9H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.8 (d, <i>J</i> = 21.7 Hz), −163.4 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 170.8, 166.7, 152.8, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.7/9.7/4.3 Hz), 139.9–136.9 (m) (2C), 136.8, 136.7–136.6 (m), 136.4 (td, <i>J</i><sub>CF</sub> = 8.1/4.4 Hz), 134.3, 128.9, 128.4, 127.0, 121.2, 121.2, 117.8, 113.1–110.3 (m), 79.2, 56.4, 52.3, 28.1; <i>m</i>/<i>z</i> MS C<sub>27</sub>H<sub>26</sub>F<sub>3</sub>N<sub>2</sub>O<sub>5</sub> [MH]<sup>+</sup> calcd 515.2, found 515.1.</div></div><div id="sec4_1_4_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> Methyl 2-(4-((<i>tert</i>-Butoxycarbonyl)amino)benzamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5aa</b>)</h5><div class="NLM_p last">4-((<i>tert</i>-Butoxycarbonyl)amino)benzoic acid (289 mg, 1.22 mmol) was coupled to compound <b>4</b> (300 mg, 1.02 mmol) according to General Procedure B. The crude product was purified by column chromatography (PE/EtOAc 100:0–50:50) to afford compound <b>5aa</b> as a bright yellow foam (370 mg, 70%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 9.62 (s, 1H), 9.04 (d, <i>J</i> = 7.2 Hz, 1H), 7.86 (d, <i>J</i> = 8.8 Hz, 2H), 7.78–7.67 (m, 4H), 7.58 (d, <i>J</i> = 8.3 Hz, 2H), 7.53 (d, <i>J</i> = 8.8 Hz, 2H), 5.74 (d, <i>J</i> = 8.5 Hz, 1H), 3.67 (s, 3H), 1.48 (s, 9H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.8 (d, <i>J</i> = 21.7 Hz), −163.3 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 171.0, 166.0, 152.6, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.5/9.7/4.1 Hz), 142.7, 138.4 (dt, <i>J</i><sub>CF</sub> = 249.9/15.7 Hz), 136.9, 136.8–136.6 (m), 136.4 (td, <i>J</i><sub>CF</sub> = 8.1/4.3 Hz), 128.9, 128.6, 127.0, 126.7, 117.0, 111.5–111.0 (m), 79.5, 56.4, 52.3, 28.0; <i>m</i>/<i>z</i> MS C<sub>27</sub>H<sub>24</sub>F<sub>3</sub>N<sub>2</sub>O<sub>5</sub> [M – H]<sup>−</sup> calcd 513.2, found 512.9.</div></div><div id="sec4_1_4_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> Methyl 2-(4-(Methylsulfonamido)benzamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5ad</b>)</h5><div class="NLM_p last">4-((Methyl)sulfonylamino)benzoic acid (106 mg, 0.494 mmol) was coupled to compound <b>4</b> (118 mg, 0.401 mmol) according to General Procedure B. Upon completion, the mixture was added to a 0.1 M HCl solution (50 mL) and extracted with EtOAc (3 × 50 mL). The combined organic layers were washed with brine (50 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was then taken up in minimal DMF and added to an ice–water mixture. The precipitate was filtered and washed with minimal MeOH to afford compound <b>5ad</b> as a light yellow solid (142 mg, 72%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 10.14 (s, 1H), 9.14 (d, <i>J</i> = 7.2 Hz, 1H), 7.95–7.88 (m, 2H), 7.78–7.68 (m, 4H), 7.58 (d, <i>J</i> = 8.3 Hz, 2H), 7.30–7.18 (m, 2H), 5.74 (d, <i>J</i> = 7.2 Hz, 1H), 3.67 (s, 3H), 3.07 (s, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.8 (d, <i>J</i> = 21.8 Hz), −163.3 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 171.0, 165.9, 150.7 (ddd, <i>J</i><sub>CF</sub> = 246.7/9.7/4.2 Hz), 141.6, 136.8–136.7(m), 136.8, 136.4 (td, <i>J</i><sub>CF</sub> = 7.8/4.2 Hz), 129.2, 129.0, 128.2, 127.0, 117.9, 111.3 (dd, <i>J</i><sub>CF</sub> = 16.0/5.5 Hz), 56.5, 52.4, 40.6; <i>m</i>/<i>z</i> MS C<sub>23</sub>H<sub>20</sub>F<sub>3</sub>N<sub>2</sub>O<sub>5</sub>S [MH]<sup>+</sup> calcd 493.1, found 492.8.</div></div><div id="sec4_1_4_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> Methyl 2-(4-(Sulfamoylamino)benzamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5ae</b>)</h5><div class="NLM_p last">Carboxylic acid <b>15</b> (160 mg, 0.742 mmol) was coupled to compound <b>4</b> (191 mg, 0.645 mmol) according to General Procedure B. Upon completion, the mixture was added to a 0.1 M HCl solution (50 mL) and extracted with EtOAc (3 × 50 mL). The combined organic layers were washed with brine (50 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The residue was then taken up in minimal DMF and added to an ice–water mixture. The precipitate was sonicated in Et<sub>2</sub>O for 10 min and then centrifuged. The supernatant was removed and the solid resuspended in Et<sub>2</sub>O twice to afford compound <b>5ae</b> as a cream solid (170 mg, 53%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 9.94 (s, 1H), 9.05 (d, <i>J</i> = 7.2 Hz, 1H), 7.86 (d, <i>J</i> = 8.8 Hz, 2H), 7.80–7.63 (m, 4H), 7.58 (d, <i>J</i> = 8.3 Hz, 2H), 7.29 (s, 2H), 7.18 (d, <i>J</i> = 8.8 Hz, 2H), 5.73 (d, <i>J</i> = 7.2 Hz, 1H), 3.67 (s, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.8 (d, <i>J</i> = 21.7 Hz), −163.3 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 171.1, 166.1, 150.7 (ddd, <i>J</i><sub>CF</sub> = 246.3/9.6/4.0 Hz), 142.8, 138.5 (dt, <i>J</i><sub>CF</sub> = 30.9/15.2 Hz), 137.0, 136.8, 136.5 (td, <i>J</i><sub>CF</sub> = 8.1/4.4 Hz), 129.0, 128.9, 127.0, 126.3, 116.3, 111.3 (dd, <i>J</i><sub>CF</sub> = 16.0/5.4 Hz), 56.5, 52.4; <i>m</i>/<i>z</i> MS C<sub>22</sub>H<sub>19</sub>F<sub>3</sub>N<sub>3</sub>O<sub>5</sub>S [MH]<sup>+</sup> calcd 494.1, found 493.8.</div></div><div id="sec4_1_4_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> Methyl 2-(3,4-Dimethoxybenzamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5af</b>)</h5><div class="NLM_p last">3,4-Dimethoxybenzoic acid (148 mg, 0.813 mmol) was coupled to compound <b>4</b> (200 mg, 0.677 mmol) according to General Procedure B. The crude product was purified by column chromatography (PE/EtOAc 100:0–50:50) to afford compound <b>5af</b> as a bright yellow foam (233 mg, 75%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 9.10 (d, <i>J</i> = 7.1 Hz, 1H), 7.80–7.65 (m, 4H), 7.64–7.56 (m, 3H), 7.53 (d, <i>J</i> = 2.0 Hz, 1H), 7.03 (d, 7.1 Hz, 1H), 5.76 (d, <i>J</i> = 7.2 Hz, 1H), 3.82 (s, 3H), 3.81 (s, 3H), 3.69 (s, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.8 (d, <i>J</i> = 21.7 Hz), −163.3 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 171.1, 165.9, 151.7, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.7/9.7/4.2 Hz), 148.2, 138.4 (dt, <i>J</i><sub>CF</sub> = 249.7/15.7 Hz), 136.9, 136.8–136.7 (m), 136.4 (td, <i>J</i><sub>CF</sub> = 8.1/4.5 Hz), 129.0, 127.0, 125.6, 121.3, 111.4–111.1 (m), 111.0, 110.8, 56.5, 55.60, 55.58, 52.3; <i>m</i>/<i>z</i> MS C<sub>24</sub>H<sub>21</sub>F<sub>3</sub>NO<sub>5</sub> [MH]<sup>+</sup> calcd 460.1, found 460.1.</div></div><div id="sec4_1_4_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> Methyl 2-(1<i>H</i>-Indole-5-carboxamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5aj</b>)</h5><div class="NLM_p last">1<i>H</i>-Indole-5-carboxylic acid (98 mg, 0.610 mmol) was coupled to compound <b>4</b> (150 mg, 0.508 mmol) according to General Procedure B. Upon completion, the DCM was concentrated in vacuo. The solution was quenched with a 1 M HCl solution and extracted with EtOAc (1 × 30 mL). The organic phase was then washed with a saturated NaHCO<sub>3</sub> solution (20 mL) and finally brine (20 mL). The organic phase was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude product was purified by flash column chromatography (PE/EtOAc 100:0–60:40) to afford compound <b>5aj</b> as a maroon solid (220 mg, 99%). <sup>1</sup>H NMR (MeOD) δ 8.19 (dd, <i>J</i> = 1.7/0.5 Hz, 1H), 7.69–7.58 (m, 5H), 7.46–7.39 (m, 3H), 7.32 (d, <i>J</i> = 3.2 Hz, 1H), 6.56 (dd, <i>J</i> = 3.2/0.8 Hz, 1H), 5.81 (s, 1H), 3.77 (s, 3H); <sup>19</sup>F NMR (MeOD) δ −136.9 (d, <i>J</i> = 19.8 Hz), −166.0 (dd, <i>J</i> = 19.8/19.8 Hz); <sup>13</sup>C NMR (MeOD) δ 172.8, 171.4, 152.6 (ddd, <i>J</i><sub>CF</sub> = 247.8/9.8/4.1 Hz), 141.9–139.2 (m), 139.7, 139.2, 138.5–138.3 (m), 138.3, 129.8 (2C), 129.0, 128.3 (2C), 127.3, 125.6, 121.9, 121.9, 112.3–112.0 (m, 3C), 103.7, 58.4, 53.2; <i>m</i>/<i>z</i> MS (system A) C<sub>24</sub>H<sub>16</sub>F<sub>3</sub>N<sub>2</sub>O<sub>3</sub> [M – H]<sup>−</sup> calcd 437.1, found 437.1.</div></div><div id="sec4_1_4_29" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> Methyl 2-(1<i>H</i>-Indazole-5-carboxamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5ak</b>)</h5><div class="NLM_p last">1<i>H</i>-Indazole-5-carboxylic acid (59 mg, 0.366 mmol) was coupled to compound <b>4</b> (90 mg, 0.305 mmol) according to General Procedure B. Upon completion, the DCM was concentrated in vacuo. The solution was quenched with a 1 M HCl solution and extracted with EtOAc (1 × 30 mL). The organic phase was then washed with a saturated NaHCO<sub>3</sub> solution (20 mL) and finally brine (20 mL). The organic phase was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude product was purified by flash chromatography (PE/EtOAc 100:0–40:60) to afford compound <b>5ak</b> as a white solid (76 mg, 57%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 9.24 (d, <i>J</i> = 7.2 Hz, 1H), 8.46 (s, 1H), 8.22 (s, 1H), 7.91 (dd, <i>J</i> = 8.8/1.5 Hz, 1H), 7.80–7.68 (m, 5H), 7.65–7.56 (m, 3H), 5.79 (d, <i>J</i> = 7.2 Hz, 1H), 3.69 (s, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.8 (d, <i>J</i> = 21.7 Hz), −163.3 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 171.0, 166.8, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.7/9.7/4.1 Hz), 141.0, 139.8–136.9 (m), 136.9, 136.7, 136.6–136.3 (m), 134.9, 128.9 (2C), 127.0 (2C), 126.0, 125.7, 122.3, 121.4, 111.3 (dd, <i>J</i><sub>CF</sub> = 16.1/5.4 Hz), 109.7, 56.5, 52.3; <i>m</i>/<i>z</i> MS C<sub>23</sub>H<sub>17</sub>F<sub>3</sub>N<sub>3</sub>O<sub>3</sub> [MH]<sup>+</sup> calcd 440.1, found 439.8.</div></div><div id="sec4_1_4_30" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> Methyl 2-(1<i>H</i>-Benzo[<i>d</i>][1,2,3]triazole-5-carboxamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5al</b>)</h5><div class="NLM_p last">1<i>H</i>-Benzo[<i>d</i>][1,2,3]triazole-5-carboxylic acid (60 mg, 0.366 mmol) was coupled to compound <b>4</b> (90 mg, 0.305 mmol) according to General Procedure B. Upon completion, the DCM was concentrated in vacuo. The solution was quenched with a 1 M HCl solution and extracted with EtOAc (1 × 30 mL). The organic phase was then washed with a saturated NaHCO<sub>3</sub> solution (20 mL) and finally brine (20 mL). The organic phase was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude product was purified by flash chromatography (PE/EtOAc 100:0–40:60) to afford compound <b>5al</b> as a white solid (100 mg, 75%). <sup>1</sup>H NMR (MeOD) δ 8.47 (s, 1H), 7.98 (dd, <i>J</i> = 8.7/1.5 Hz, 1H), 7.88 (d, <i>J</i> = 8.7 Hz, 1H), 7.66–7.56 (m, 4H), 7.43–7.35 (m, 2H), 5.82 (s, 1H), 3.78 (s, 3H); <sup>19</sup>F NMR (MeOD) δ −136.8 (d, <i>J</i> = 19.9 Hz), −165.8 (dd, <i>J</i> = 19.8/19.8 Hz); <sup>13</sup>C NMR (MeOD) δ 172.5, 169.5, 152.7 (ddd, <i>J</i><sub>CF</sub> = 248.0/9.9/4.2 Hz), 141.9–139.0 (m), 139.5, 139.2, 138.5–138.2 (m), 137.8,<a class="ref internalNav" href="#fn9999a" aria-label="a">a</a> 132.5, 129.9 (2C), 129.8, 128.4, 128.4 (2C), 126.8, 112.4–112.1 (m, 2C), 58.6, 53.2; <i>m</i>/<i>z</i> MS C<sub>22</sub>H<sub>16</sub>F<sub>3</sub>N<sub>4</sub>O<sub>3</sub> [MH]<sup>+</sup> calcd 441.1, found 440.8.</div></div><div id="sec4_1_4_31" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> Methyl 2-(1<i>H</i>-Benzo[<i>d</i>]imidazole-5-carboxamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5am</b>)</h5><div class="NLM_p last">1<i>H</i>-Benzo[<i>d</i>]imidazole-5-carboxylic acid (99 mg, 0.610 mmol) was coupled to compound <b>4</b> (150 mg, 0.508 mmol) according to General Procedure B. Upon completion, the DCM was concentrated in vacuo. The solution was quenched with a 1 M HCl solution and extracted with EtOAc (1 × 30 mL). The organic phase was then washed with a saturated NaHCO<sub>3</sub> solution (20 mL) and finally brine. The organic phase was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude product was purified by flash column chromatography (DCM/MeOH 100:0–90:10) to afford compound <b>5am</b> as a dull yellow/brown solid (174 mg, 78%). <sup>1</sup>H NMR (MeOD) δ 8.30 (s, 1H), 8.21 (d, <i>J</i> = 1.1 Hz, 1H), 7.83 (dd, <i>J</i> = 8.5/1.6 Hz, 1H), 7.69–7.58 (m, 5H), 7.45–7.37 (m, 2H), 5.81 (s, 1H), 3.78 (s, 3H); <sup>19</sup>F NMR (MeOD) δ −136.8 (d, <i>J</i> = 19.8 Hz), −165.9 (dd, <i>J</i> = 19.8/19.8 Hz); <sup>13</sup>C NMR (MeOD) δ 172.6, 170.3, 152.6 (ddd, <i>J</i><sub>CF</sub> = 248.1/10.0/4.1 Hz), 144.8, 141.0–140.8 (m), 141.8–138.8 (m), 139.3, 139.1, 138.6–138.1 (m), 137.9, 129.8 (2C), 129.5, 128.3 (2C), 123.5, 116.9, 115.8, 112.1 (m, 2C), 58.5, 53.3; <i>m</i>/<i>z</i> MS (system A) C<sub>23</sub>H<sub>17</sub>F<sub>3</sub>N<sub>3</sub>O<sub>3</sub> [MH]<sup>+</sup> calcd 440.1, found 439.8.</div></div><div id="sec4_1_4_32" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> Methyl 2-(2-Oxoindoline-5-carboxamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5an</b>)</h5><div class="NLM_p last">2-Oxoindoline-5-carboxylic acid (72 mg, 0.406 mmol) was coupled to compound <b>4</b> (100 mg, 0.339 mmol) in DMF according to General Procedure B. Upon completion, the reaction mixture was poured onto ice-cold water. The precipitate was filtered and washed thoroughly with water and Et<sub>2</sub>O to afford compound <b>5an</b> as a brown solid (152 mg, 99%). <sup>1</sup>H NMR (MeOD) δ 7.83–7.76 (m, 2H), 7.67–7.62 (m, 2H), 7.59–7.55 (m, 2H), 7.42 (dd, <i>J</i> = 9.3/6.6 Hz, 2H), 6.94 (dd, <i>J</i> = 8.1/0.5 Hz, 1H), 5.76 (s, 1H), 3.76 (s, 3H); <sup>19</sup>F NMR (MeOD) δ −136.8 (d, <i>J</i> = 19.8/19.8 Hz), −165.9 (dd, <i>J</i> = 19.8/19.8 Hz); <sup>13</sup>C NMR (MeOD) δ 179.9, 172.6, 169.7, 152.7 (ddd, <i>J</i><sub>CF</sub> = 248.0/10.0/4.1 Hz), 148.3, 141.9–138.9 (m), 139.4–139.1 (m), 138.4–138.2 (m), 138.0, 129.8 (2C), 129.5, 128.6, 128.3 (2C), 127.2, 125.2, 112.2 (m, 2C), 110.3, 58.4, 53.2, 39.6; <i>m</i>/<i>z</i> MS (system A) C<sub>24</sub>H<sub>18</sub>F<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 455.1, found 455.7.</div></div><div id="sec4_1_4_33" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> Methyl 2-(2-Phenylacetamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5ao</b>)</h5><div class="NLM_p last">Phenylacetic acid (99.6 mg, 0.732 mmol) was coupled to compound <b>4</b> (180 mg, 0.610 mmol) according to General Procedure B. The crude product was purified by column chromatography (PE/EtOAc 100:0–50:50) to afford compound <b>5ao</b> as a yellow solid (100 mg, 40%). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 7.43–7.37 (m, 2H), 7.37–7.19 (m, 7H), 7.16–7.04 (m, 2H), 6.60 (d, <i>J</i> = 6.9 Hz, 1H), 5.57 (d, <i>J</i> = 7.0 Hz, 1H), 3.68 (s, 3H), 3.59 (s, 2H); <sup>19</sup>F NMR (CDCl<sub>3</sub>) δ −133.9 (d, <i>J</i> = 20.5 Hz), −162.2 (dd, <i>J</i> = 20.5/20.5 Hz); <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ 171.1, 170.5, 151.5 (ddd, <i>J</i><sub>CF</sub> = 249.8/10.0/4.3 Hz), 139.5 (dt, <i>J</i><sub>CF</sub> = 252.4/15.4 Hz), 138.5–138.4 (m), 136.9, 136.6 (td, <i>J</i><sub>CF</sub> = 7.7/4.5 Hz), 129.5, 129.2, 127.9, 127.6, 127.5, 111.5–110.9 (m), 56.2, 53.1, 43.6; <i>m</i>/<i>z</i> MS C<sub>23</sub>H<sub>19</sub>F<sub>3</sub>NO<sub>3</sub> [MH]<sup>+</sup> calcd 414.1, found 413.9.</div></div><div id="sec4_1_4_34" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> Methyl 2-(2-(3-Fluorophenyl)acetamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5ap</b>)</h5><div class="NLM_p last">2-(3-Fluorophenyl)acetic acid (125 mg, 0.813 mmol) was coupled to compound <b>4</b> (200 mg, 0.667 mmol) according to General Procedure B. The crude product was purified by column chromatography (PE/EtOAc 100:0–50:50) to afford compound <b>5ap</b> as a white solid (248 mg, 85%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 9.08 (d, <i>J</i> = 7.1 Hz, 1H), 7.75 (d, <i>J</i> = 8.4 Hz, 2H), 7.72–7.63 (m, 2H), 7.51 (d, <i>J</i> = 8.3 Hz, 2H), 7.38–7.28 (m, 1H), 7.12 (m, 2H), 7.08–6.98 (m, 1H), 5.51 (d, <i>J</i> = 7.1 Hz, 1H), 3.64 (s, 3H), 3.61 (app. d, <i>J</i> = 4.3 Hz, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −113.9, −134.8 (d, <i>J</i> = 21.6 Hz), −163.3 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 170.8, 169.7, 162.0 (d, <i>J</i><sub>CF</sub> = 242.9 Hz), 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.7/9.7/4.1 Hz), 138.8 (d, <i>J</i><sub>CF</sub> = 7.9 Hz), 138.4 (dt, <i>J</i><sub>CF</sub> = 249.7/15.7 Hz), 136.9 (d, <i>J</i> = 1.2 Hz), 136.5, 136.3 (td, <i>J</i><sub>CF</sub> = 8.1/4.4 Hz), 123.0 (d, <i>J</i><sub>CF</sub> = 8.4 Hz), 128.4, 127.2, 125.2 (d, <i>J</i><sub>CF</sub> = 2.7 Hz), 115.8 (d, <i>J</i><sub>CF</sub> = 21.4 Hz), 113.2 (d, <i>J</i><sub>CF</sub> = 20.8 Hz), 111.8–110.3 (m), 56.0, 52.3, 41.1; <i>m</i>/<i>z</i> MS C<sub>23</sub>H<sub>18</sub>F<sub>4</sub>NO<sub>3</sub> [MH]<sup>+</sup> calcd 432.1, found 431.8.</div></div><div id="sec4_1_4_35" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> Methyl 2-(2-(4-Fluorophenyl)acetamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5aq</b>)</h5><div class="NLM_p last">2-(4-Fluorophenyl)acetic acid (94.0 mg, 0.610 mmol) was coupled to compound <b>4</b> (150 mg, 0.508 mmol) according to General Procedure B. The crude product was purified by column chromatography (PE/EtOAc 100:0–50:50) to afford compound <b>5aq</b> as a white solid (155 mg, 71%). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 7.35 (d, <i>J</i> = 8.4 Hz, 2H), 7.27 (d, <i>J</i> = 8.3 Hz, 2H), 7.19–7.12 (m, 2H), 7.08–7.01 (m, 2H), 6.98–6.91 (m, 2H), 6.61 (d, <i>J</i> = 6.9 Hz, 1H), 5.51 (d, <i>J</i> = 7.0 Hz, 1H), 3.63 (s, 3H), 3.50 (s, 2H); <sup>19</sup>F NMR (CDCl<sub>3</sub>) δ −115.1, −133.9 (d, <i>J</i> = 20.5 Hz), −162.1 (dd, <i>J</i> = 20.5/20.5 Hz); <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ 171.1, 170.2, 162.3 (d, <i>J</i><sub>CF</sub> = 246.1 Hz), 151.5 (ddd, <i>J</i><sub>CF</sub> = 249.8/10.0/4.3 Hz), 140.9–137.8 (dt, <i>J</i><sub>CF</sub> = 252.2/15.2 Hz), 138.7–138.5 (m), 136.8, 136.6 (td, <i>J</i><sub>CF</sub> = 7.8/4.7 Hz), 131.0 (d, <i>J</i><sub>CF</sub> = 8.1 Hz), 130.3 (d, <i>J</i><sub>CF</sub> = 3.3 Hz), 128.0, 127.5, 115.9 (d, <i>J</i><sub>CF</sub> = 21.5 Hz), 111.1 (m), 56.2, 53.1, 42.5; <i>m</i>/<i>z</i> MS C<sub>23</sub>H<sub>18</sub>F<sub>4</sub>NO<sub>3</sub> [MH]<sup>+</sup> calcd 432.1, found 431.8.</div></div><div id="sec4_1_4_36" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> Methyl 2-(2-(4-Methoxyphenyl)acetamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5ar</b>)</h5><div class="NLM_p last">2-(4-Methoxyphenyl)acetic acid (270 mg, 1.63 mmol) was coupled to compound <b>4</b> (400 mg, 1.35 mmol) according to General Procedure B. The crude product was purified by column chromatography (PE/EtOAc 100:0–50:50) to afford compound <b>5ar</b> as a white solid (195 mg, 32%). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 7.42 (d, <i>J</i> = 8.3 Hz, 2H), 7.35 (d, <i>J</i> = 8.3 Hz, 2H), 7.19 (d, <i>J</i> = 8.6 Hz, 2H), 7.12 (m, 2H), 6.89 (d, <i>J</i> = 8.6 Hz, 2H), 6.70 (d, <i>J</i> = 7.0 Hz, 1H), 5.61 (d, <i>J</i> = 7.1 Hz, 1H), 3.80 (s, 3H), 3.71 (s, 3H), 3.55 (s, 2H); <sup>19</sup>F NMR (CDCl<sub>3</sub>) δ −133.9 (d, <i>J</i> = 20.5 Hz), −162.2 (dd, <i>J</i> = 20.5/20.5 Hz); <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ 171.1, 170.9, 159.0, 151.5 (ddd, <i>J</i><sub>CF</sub> = 249.8/10.0/4.2 Hz), 139.5 (dt, <i>J</i><sub>CF</sub> = 32.4/15.4 Hz), 138.4–138.4 (m), 136.9, 136.6 (td, <i>J</i><sub>CF</sub> = 7.8/4.6 Hz), 130.5, 127.9, 127.4, 126.4, 114.5, 111.1 (m), 56.1, 55.3, 53.0, 42.6; <i>m</i>/<i>z</i> MS C<sub>24</sub>H<sub>21</sub>F<sub>3</sub>NO<sub>4</sub> [MH]<sup>+</sup> calcd 444.1, found 443.9.</div></div><div id="sec4_1_4_37" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> Methyl 2-(2-(4-Carbamoylphenyl)acetamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5at</b>)</h5><div class="NLM_p last">2-(4-Carbamoylphenyl)acetic acid (146 mg, 0.813 mmol) was coupled to compound <b>4</b> (200 mg, 0.677 mmol) according to General Procedure B. Upon completion, the reaction mixture was diluted with sat. NaHCO<sub>3</sub> (15 mL) and extracted with DCM (3 × 15 mL). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford compound <b>5at</b> as a white solid (261 mg, 85%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 9.09 (d, <i>J</i> = 7.2 Hz, 1H), 7.91 (br s, 1H), 7.87–7.65 (m, 6H), 7.51 (d, <i>J</i> = 8.3 Hz, 2H), 7.34 (d, <i>J</i> = 8.3 Hz, 2H), 7.30 (br s, 1H), 5.50 (d, <i>J</i> = 7.1 Hz, 1H), 3.64 (s, 3H), 3.62 (d, <i>J</i> = 2.3 Hz, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.8 (d, <i>J</i> = 21.7 Hz), −163.2 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 170.9, 169.8, 167.8, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.7/9.7/4.2 Hz), 139.4, 138.4 (dt, <i>J</i><sub>CF</sub> = 249.5/15.5 Hz), 137.0–136.8 (m), 136.6, 136.2 (td, <i>J</i><sub>CF</sub> = 8.1/4.3 Hz), 132.5, 128.9, 128.5, 127.4, 127.2, 111.6–111.0 (m), 56.0, 52.4, 41.4; <i>m</i>/<i>z</i> MS C<sub>24</sub>H<sub>20</sub>F<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 457.1, found 456.9.</div></div><div id="sec4_1_4_38" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> Methyl 2-(2-(3,4-Dimethoxyphenyl)acetamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5au</b>)</h5><div class="NLM_p last">2-(3,4-Dimethoxyphenyl)acetic acid (319 mg, 1.63 mmol) was coupled to compound <b>4</b> (400 mg, 1.35 mmol) according to General Procedure B. The crude product was purified by column chromatography (PE/EtOAc 100:0–50:50) to afford compound <b>5au</b> as a white solid (446 mg, 70%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 9.03 (d, <i>J</i> = 7.3 Hz, 1H), 7.87–7.62 (m, 4H), 7.51 (d, <i>J</i> = 8.3 Hz, 2H), 6.94–6.75 (m, 3H), 5.51 (d, <i>J</i> = 7.2 Hz, 1H), 3.72 (s, 3H), 3.71 (s, 3H), 3.64 (s, 3H), 3.48 (s, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.8 (d, <i>J</i> = 21.7 Hz), −163.3 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 170.9, 170.4, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.8/9.8/4.3 Hz), 148.5, 147.5, 138.4 (dt, <i>J</i><sub>CF</sub> = 30.6/15.6 Hz), 136.9–136.8 (m), 136.7, 136.3 (td, <i>J</i><sub>CF</sub> = 8.1/4.4 Hz), 128.5, 128.4, 127.1, 121.0, 112.9, 111.7, 111.3 (dd, <i>J</i><sub>CF</sub> = 16.0/5.6 Hz), 55.9, 55.5, 55.3, 52.3, 41.2; <i>m</i>/<i>z</i> MS C<sub>25</sub>H<sub>23</sub>F<sub>3</sub>NO<sub>5</sub> [MH]<sup>+</sup> calcd 474.1, found 473.9.</div></div><div id="sec4_1_4_39" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> Methyl 2-(2-(3-Fluoro-4-methoxyphenyl)acetamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5aw</b>)</h5><div class="NLM_p last">2-(3-Fluoro-4-methoxyphenyl)acetic acid (187 mg, 1.02 mmol) was coupled to compound <b>4</b> (250 mg, 0.847 mmol) according to General Procedure B. The crude product was purified by column chromatography (PE/EtOAc 100:0–50:50) to afford compound <b>5aw</b> as a yellow solid (134 mg, 34%). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 7.45 (d, <i>J</i> = 8.4 Hz, 2H), 7.36 (d, <i>J</i> = 8.2 Hz, 2H), 7.14 (m, 2H), 7.05–6.90 (m, 3H), 6.60 (d, <i>J</i> = 6.9 Hz, 1H), 5.59 (d, <i>J</i> = 7.0 Hz, 1H), 3.89 (s, 3H), 3.73 (s, 3H), 3.54 (s, 2H); <sup>19</sup>F NMR (CDCl<sub>3</sub>) δ −133.9 (d, <i>J</i> = 20.5 Hz), −134.2, −162.1 (dd, <i>J</i> = 20.5/20.5 Hz); <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ 171.1, 170.2, 152.5 (d, <i>J</i><sub>CF</sub> = 246.9 Hz), 151.6 (ddd, <i>J</i><sub>CF</sub> = 249.8/10.0/4.2 Hz), 147.2 (d, <i>J</i><sub>CF</sub> = 10.6 Hz), 139.6 (dt, <i>J</i><sub>CF</sub> = 252.3/15.3 Hz), 138.7–138.5 (m), 136.82, 136.6 (td, <i>J</i><sub>CF</sub> = 7.8/4.7 Hz), 128.0, 127.6, 127.2 (d, <i>J</i><sub>CF</sub> = 6.4 Hz), 125.3 (d, <i>J</i><sub>CF</sub> = 3.6 Hz), 117.2 (d, <i>J</i><sub>CF</sub> = 18.6 Hz), 113.9 (d, <i>J</i><sub>CF</sub> = 2.2 Hz), 111.2 (m), 56.4, 56.3, 53.2, 42.5 (d, <i>J</i><sub>CF</sub> = 1.0 Hz); <i>m</i>/<i>z</i> MS C<sub>23</sub>H<sub>20</sub>F<sub>4</sub>NO<sub>4</sub> [MH]<sup>+</sup> calcd 462.1, found 461.9.</div></div><div id="sec4_1_4_40" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64"> Methyl 2-(2-(4-Fluoro-3-methoxyphenyl)acetamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>5ay</b>)</h5><div class="NLM_p last">2-(4-Fluoro-3-methoxyphenyl)acetic acid (225 mg, 1.22 mmol) was coupled to compound <b>4</b> (300 mg, 1.02 mmol) according to General Procedure B. The crude product was purified by column chromatography (PE/EtOAc 100:0–50:50) to afford compound <b>5ay</b> as a white solid (410 mg, 87%). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 7.47–7.32 (m, 4H), 7.18–7.10 (m, 2H), 7.08–7.00 (m, 1H), 6.89 (dd, <i>J</i> = 8.1/2.1 Hz, 1H), 6.78 (ddd, <i>J</i> = 8.2/4.2/2.1 Hz, 1H), 6.66 (d, <i>J</i> = 7.0 Hz, 1H), 5.60 (d, <i>J</i> = 7.0 Hz, 1H), 3.85 (s, 3H), 3.72 (s, 3H), 3.57 (s, 2H); <sup>19</sup>F NMR (CDCl<sub>3</sub>) δ −133.8 (d, <i>J</i> = 20.5 Hz), −137.0, −162.1 (dd, <i>J</i> = 20.5/20.5 Hz); <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ 171.1, 170.1, 151.9 (d, <i>J</i><sub>CF</sub> = 245.8 Hz), 151.53 (ddd, <i>J</i><sub>CF</sub> = 249.9/10.0/4.2 Hz), 147.97 (d, <i>J</i><sub>CF</sub> = 10.8 Hz), 139.5 (dt, <i>J</i><sub>CF</sub> = 252.4/15.4 Hz), 138.6 (d, <i>J</i><sub>CF</sub> = 1.6 Hz), 136.8, 136.5 (td, <i>J</i><sub>CF</sub> = 7.8/4.7 Hz), 130.76 (d, <i>J</i> = 4.0 Hz), 128.0, 127.5, 121.6 (d, <i>J</i><sub>CF</sub> = 6.9 Hz), 116.4 (d, <i>J</i><sub>CF</sub> = 18.4 Hz), 114.5 (d, <i>J</i><sub>CF</sub> = 2.0 Hz), 111.1 (dd, <i>J</i><sub>CF</sub> = 15.9/6.0 Hz), 56.3, 56.2, 53.1, 43.0; <i>m</i>/<i>z</i> MS C<sub>24</sub>H<sub>20</sub>F<sub>4</sub>NO<sub>4</sub> [MH]<sup>+</sup> calcd 462.1, found 461.8.</div></div><div id="sec4_1_4_41" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65"> 2-Acetamido-<i>N</i>-hydroxy-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetamide (<b>6a</b>)</h5><div class="NLM_p last">Compound <b>5a</b> (101 mg, 0.300 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. The crude product was purified by column chromatography (DCM/MeOH/AcOH 99:0:1–90:9:1) to afford compound <b>6a</b> as a pink solid (64 mg, 65%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.02 (s, 1H), 9.01 (s, 1H), 8.71 (d, <i>J</i> = 8.4 Hz, 1H), 7.86–7.64 (m, 4H), 7.49 (d, <i>J</i> = 8.3 Hz, 2H), 5.41 (d, <i>J</i> = 8.4 Hz, 1H), 1.91 (s, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.8 Hz), −163.5 (dd, <i>J</i> = 21.8/21.8 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 169.1, 166.5, 150.7 (ddd, <i>J</i><sub>CF</sub> = 13.6/9.8/4.3 Hz), 139.4, 134.0–136.8 (m), 136.7–136.4 (m), 136.3–135.9 (m), 127.7, 126.8, 111.3 (dd, <i>J</i><sub>CF</sub> = 16.0/5.4 Hz), 53.5, 22.4; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>16</sub>H<sub>14</sub>F<sub>3</sub>N<sub>2</sub>O<sub>3</sub> [MH]<sup>+</sup> calcd 339.0951, found 339.0953.</div></div><div id="sec4_1_4_42" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66"> 3-((2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)amino)-3-oxopropanoic Acid (<b>6c</b>)</h5><div class="NLM_p last">Compound <b>5b</b> (185 mg, 0.485 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. After 1 day, only partial conversion had occurred and therefore NH<sub>2</sub>OH·HCl (135 mg, 1.94 mmol) and KOH (5 M in MeOH, 0.486 mL) were added. The reaction mixture was stirred for a further 24 h. LC–MS indicated that <b>5b</b> was converted to the desired product and the dihydroxamic acid <i>N</i><sup>1</sup>-hydroxy-<i>N</i><sup>3</sup>-(2-(hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)malonamide (<b>6d</b>). These two compounds were isolated by column chromatography (DCM/MeOH/AcOH 99:0:1–90:9:1). The desired hydroxamic acid was treated with 20% TFA/DCM (5 mL) and stirred at rt overnight. The reaction mixture was concentrated in vacuo and extracted with EtOAc (3 × 5 mL). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude product was purified by column chromatography (DCM/MeOH/AcOH 99:0:1–90:9:1) to afford compound <b>6c</b> as a light brown oil (10 mg, 6% over two steps). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.28 (s, 1H), 9.29 (d, <i>J</i> = 8.2 Hz, 1H), 7.86–7.58 (m, 4H), 7.49 (d, <i>J</i> = 8.3 Hz, 2H), 5.41 (d, <i>J</i> = 8.2 Hz, 1H), 3.26–3.06 (m, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.8/21.8 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 170.2, 166.9, 166.3, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.5/9.7/4.1 Hz), 139.3, 138.3 (dt, <i>J</i><sub>CF</sub> = 251.1/16.7 Hz), 136.6 (td, <i>J</i><sub>CF</sub> = 8.1/4.5 Hz), 136.2–136.1 (m), 127.6, 126.8, 113.1–108.6 (m), 53.7, 43.3; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>17</sub>H<sub>14</sub>F<sub>3</sub>N<sub>2</sub>O<sub>5</sub> [MH]<sup>+</sup> calcd 383.0849, found 383.0857.</div></div><div id="sec4_1_4_43" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> <i>N</i><sup>1</sup>-Hydroxy-<i>N</i><sup>3</sup>-(2-(hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)malonamide (<b>6d</b>)</h5><div class="NLM_p last">The title compound was synthesized by aminolysis of both methyl and <i>tert</i>-butyl esters of compound <b>5b</b> as explained above. <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 10.64 (s, 2H), 9.08 (s, 1H), 8.94 (s, 1H), 8.84 (d, <i>J</i> = 8.2 Hz, 1H), 7.81–7.58 (m, 4H), 7.49 (d, <i>J</i> = 8.3 Hz, 2H), 5.39 (d, <i>J</i> = 8.2 Hz, 1H), 3.07 (s, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 166.1, 165.9, 163.8, 150.6 (ddd, <i>J</i> = 246.5/9.7/4.4 Hz), 139.4, 138.3 (dt, <i>J</i> = 249.8/19.6 Hz), 136.6 (td, <i>J</i> = 8.2/4.5 Hz), 136.2–136.0 (m), 127.5, 126.8, 111.8–109.8 (m), 53.8, 40.3; <i>m</i>/<i>z</i> MS (TOF ES<sup>–</sup>) C<sub>17</sub>H<sub>13</sub>F<sub>3</sub>N<sub>3</sub>O<sub>5</sub> [M – H]<sup>−</sup> calcd 396.1, found 396.0; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>17</sub>H<sub>15</sub>F<sub>3</sub>N<sub>3</sub>O<sub>5</sub> [MH]<sup>+</sup> calcd 398.0958, found 398.0954.</div></div><div id="sec4_1_4_44" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> 4-((2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)amino)-4-oxobutanoic Acid (<b>6e</b>)</h5><div class="NLM_p last">Compound <b>5c</b> (267 mg, 0.675 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. After 1 day, only partial conversion had occurred; therefore, NH<sub>2</sub>OH·HCl (188 mg, 2.71 mmol) was added and the reaction mixture was heated at 40 °C for 2 h. The reaction progressed slowly and therefore the temperature was increased to 50 °C overnight. LC–MS showed degradation of <b>5e</b> to <b>4</b>. The reaction was stopped, and compounds <b>4</b>, <b>5e</b>, and <b>6e</b> were isolated by column chromatography (DCM/MeOH/AcOH 99:0:1–95:4:1). Compound <b>6e</b> was obtained as a white solid (15 mg, 6%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.95 (s, 1H), 11.01 (s, 1H), 9.04 (s, 1H), 8.72 (d, <i>J</i> = 8.3 Hz, 1H), 8.08–7.61 (m, 4H), 7.49 (d, <i>J</i> = 8.3 Hz, 2H), 5.41 (d, <i>J</i> = 8.3 Hz, 1H), 2.49–2.02 (m, 4H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.8 Hz), −163.5 (dd, <i>J</i> = 21.8/21.8 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 174.1, 171.1, 166.5, 150.7 (ddd, <i>J</i><sub>CF</sub> = 246.4/9.7/4.1 Hz), 139.4, 138.4 (dt, <i>J</i><sub>CF</sub> = 248.9/15.1 Hz), 136.6 (td, <i>J</i><sub>CF</sub> = 8.0/4.3 Hz), 136.3–136.0 (m), 127.7, 126.8, 111.5–111.0 (m), 53.6, 29.9, 29.2; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>18</sub>H<sub>16</sub>F<sub>3</sub>N<sub>2</sub>O<sub>5</sub> [MH]<sup>+</sup> calcd 397.1006, found 397.1018.</div></div><div id="sec4_1_4_45" class="NLM_sec NLM_sec_level_4"><div id="ac_i69" class="anchor-spacer"></div><h5 class="article-section__title" id="_i69"> <i>N</i><sup>1</sup>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)succinamide (<b>6f</b>)</h5><div class="NLM_p last">Compound <b>5f</b> (84 mg, 0.213 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. The crude product was purified by column chromatography (DCM/MeOH/AcOH 95:4:1–90:9:1) to afford compound <b>6f</b> as a white solid (43 mg, 52%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.00 (s, 1H), 9.01 (s, 1H), 8.66 (d, <i>J</i> = 8.3 Hz, 1H), 7.79–7.62 (m, 4H), 7.49 (d, <i>J</i> = 8.4 Hz, 2H), 7.27 (br s, 1H), 6.74 (br s, 1H), 5.41 (d, <i>J</i> = 8.3 Hz, 1H), 2.43 (t, <i>J</i> = 7.9 Hz, 2H), 2.28 (t, <i>J</i> = 7.2 Hz, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.8/21.8 Hz);<sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 173.5, 171.4, 166.4, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.4/9.8/4.3 Hz), 139.4, 138.3 (dt, <i>J</i><sub>CF</sub> = 258.4/15.8 Hz), 136.6 (td, <i>J</i><sub>CF</sub> = 8.1/4.2 Hz), 136.2–135.9 (m), 127.7, 126.8, 111.2 (dd, <i>J</i><sub>CF</sub> = 16.0/5.4 Hz), 53.6, 30.4, 30.3; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>18</sub>H<sub>17</sub>F<sub>3</sub>N<sub>3</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 396.1166, found 396.1173.</div></div><div id="sec4_1_4_46" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> <i>N</i>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)benzamide (<b>6g</b>)</h5><div class="NLM_p last">Compound <b>5g</b> (50 mg) was converted to the corresponding hydroxamic acid according to General Procedure C. The crude product was purified by column chromatography (DCM/MeOH 100:0–90:10) followed by preparative HPLC to afford compound <b>6g</b> as a white solid (10.7 mg, 21%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.08 (s, 1H), 9.06 (s, 1H), 8.94 (d, <i>J</i> = 8.1 Hz, 1H), 7.93 (d, <i>J</i> = 7.3 Hz, 2H), 7.78–7.60 (m, 6H), 7.58–7.39 (m, 3H), 5.68 (d, <i>J</i> = 8.1 Hz, 1H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.6 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 166.5, 166.3, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.6/9.7/4.2 Hz), 138.9, 138.3 (dt, <i>J</i><sub>CF</sub> = 249.1/15.4 Hz), 136.6 (td, <i>J</i><sub>CF</sub> = 8.0/4.3 Hz), 136.3–136.1 (m), 133.8, 131.5, 128.2, 128.1, 127.8, 126.8, 113.5–109.6 (m), 54.4; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>21</sub>H<sub>16</sub>F<sub>3</sub>N<sub>2</sub>O<sub>3</sub> [MH]<sup>+</sup> calcd 401.1108, found 401.1104.</div></div><div id="sec4_1_4_47" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> 3-Fluoro-<i>N</i>-(2-(hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)benzamide (<b>6h</b>)</h5><div class="NLM_p last">Compound <b>5h</b> (50 mg, 0.120 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. The crude product was purified by column chromatography (DCM/MeOH 100:0–90:10) followed by preparative HPLC to afford compound <b>6h</b> as a white solid (28.6 mg, 57%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.08 (s, 1H), 9.11 (d, <i>J</i> = 8.0 Hz, 1H), 9.06 (s, 1H), 7.84–7.59 (m, 8H), 7.52 (dd, <i>J</i> = 13.9/7.9 Hz, 1H), 7.43–7.36 (m, 1H), 5.66 (d, <i>J</i> = 8.0 Hz, 1H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −113.1, −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.6/21.6 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 166.3, 165.0 (d, <i>J</i><sub>CF</sub> = 2.4 Hz), 161.8 (d, <i>J</i><sub>CF</sub> = 244.0 Hz), 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.5/9.7/4.2 Hz), 138.6, 138.3 (dt, <i>J</i><sub>CF</sub> = 248.8/15.4 Hz), 136.5 (td, <i>J</i><sub>CF</sub> = 8.2/4.7 Hz), 136.4–136.2 (m), 136.1 (d, <i>J</i><sub>CF</sub> = 6.9 Hz), 130.3 (d, <i>J</i><sub>CF</sub> = 7.9 Hz), 128.2, 126.8, 124.0 (d, <i>J</i><sub>CF</sub> = 2.7 Hz), 118.3 (d, <i>J</i><sub>CF</sub> = 21.1 Hz), 114.6 (d, <i>J</i><sub>CF</sub> = 22.9 Hz), 111.9–110.5 (m), 54.5; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>21</sub>H<sub>15</sub>F<sub>4</sub>N<sub>2</sub>O<sub>3</sub> [MH]<sup>+</sup> calcd 419.1013, found 419.1020.</div></div><div id="sec4_1_4_48" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> 4-Fluoro-<i>N</i>-(2-(hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)benzamide (<b>6i</b>)</h5><div class="NLM_p last">Compound <b>5i</b> (100 mg, 0.240 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. The crude product was purified by column chromatography (DCM/MeOH 100:0–90:10) to afford compound <b>6i</b> as a white solid (48 mg, 48%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.06 (s, 1H), 9.13–8.97 (m, 2H), 8.08–7.96 (m, 2H), 7.78–7.60 (m, 6H), 7.36–7.22 (m, 2H), 5.67 (d, <i>J</i> = 8.0 Hz, 1H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −109.1, −135.0 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.6/21.6 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 166.5, 165.3, 164.0 (d, <i>J</i><sub>CF</sub> = 241.0 Hz), 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.6/9.7/4.2 Hz), 138.8, 138.3 (dt, <i>J</i><sub>CF</sub> = 249.4/15.6 Hz), 136.6 (td, <i>J</i><sub>CF</sub> = 8.0/4.3 Hz), 136.3–136.2 (m), 130.5 (d, <i>J</i><sub>CF</sub> = 9.0 Hz), 130.3 (d, <i>J</i><sub>CF</sub> = 2.9 Hz), 128.1, 126.8, 115.1 (d, <i>J</i><sub>CF</sub> = 21.7 Hz), 111.2 (m), 54.5; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>21</sub>H<sub>15</sub>F<sub>4</sub>N<sub>2</sub>O<sub>3</sub> [MH]<sup>+</sup> calcd 419.1013, found 419.1019.</div></div><div id="sec4_1_4_49" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73"> <i>N</i>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)-3-methoxybenzamide (<b>6j</b>)</h5><div class="NLM_p last">Compound <b>5j</b> (528 mg, 1.23 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. The crude product was purified by column chromatography (DCM/MeOH 100:0–90:10) to afford compound <b>6j</b> as a white solid (350 mg, 66%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 8.99 (d, <i>J</i> = 8.1 Hz, 1H), 7.76–7.61 (m, 6H), 7.53 (m, 2H), 7.37 (dd, <i>J</i> = 7.9/7.9 Hz, 1H), 7.10 (dd, <i>J</i> = 8.1/2.3 Hz, 1H), 5.72 (d, <i>J</i> = 8.0 Hz, 1H), 3.81 (s, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.6 Hz), −163.5 (dd, <i>J</i> = 21.6/21.6 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 166.6, 166.2, 159.2, 150.7 (ddd, <i>J</i><sub>CF</sub> = 246.7/9.7/4.1 Hz), 139.0, 138.4 (dt, <i>J</i><sub>CF</sub> = 249.5/15.6 Hz), 136.7 (td, <i>J</i><sub>CF</sub> = 8.0/4.4 Hz), 136.5–136.0 (m), 135.3, 129.4, 128.2, 126.8, 120.1, 117.6, 112.9, 111.2 (m), 55.3, 54.5; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>22</sub>H<sub>18</sub>F<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 431.1213, found 431.1207.</div></div><div id="sec4_1_4_50" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74"> <i>N</i>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)-4-methoxybenzamide (<b>6k</b>)</h5><div class="NLM_p last">Compound <b>5k</b> (154 mg, 0.359 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. The crude product was purified by column chromatography (DCM/MeOH 100:0–90:10) to afford compound <b>6k</b> as a white solid (95.3 mg, 62%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.06 (s, 1H), 9.05 (s, 1H), 8.76 (d, <i>J</i> = 8.2 Hz, 1H), 7.93 (d, <i>J</i> = 8.9 Hz, 2H), 7.81–7.65 (m, 4H), 7.61 (d, <i>J</i> = 8.4 Hz, 2H), 6.99 (d, <i>J</i> = 8.9 Hz, 2H), 5.66 (d, <i>J</i> = 8.1 Hz, 1H), 3.81 (s, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 166.6, 165.7, 161.8, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.6/9.7/4.2 Hz), 139.1, 138.3 (dt, <i>J</i><sub>CF</sub> = 249.3/15.5 Hz), 136.6 (td, <i>J</i><sub>CF</sub> = 8.1/4.5 Hz), 136.2–136.0 (m), 129.7, 128.1, 126.7, 126.0, 113.4, 111.2 (m), 55.4, 54.3; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>22</sub>H<sub>18</sub>F<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 431.1213, found 431.1216.</div></div><div id="sec4_1_4_51" class="NLM_sec NLM_sec_level_4"><div id="ac_i75" class="anchor-spacer"></div><h5 class="article-section__title" id="_i75"> 3-Hydroxy-<i>N</i>-(2-(hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)benzamide (<b>6l</b>)</h5><div class="NLM_p last">Compound <b>6j</b> (230 mg, 0.534 mmol) was treated with BBr<sub>3</sub> (1 M in DCM, 2.67 mL, 2.67 mmol) according to General Procedure D. The crude product was purified by preparative HPLC to afford compound <b>6l</b> as a white solid (62 mg, 28%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.07 (s, 1H), 9.61 (br s, 1H), 8.79 (d, <i>J</i> = 8.1 Hz, 1H), 7.68 (m, 6H), 7.37 (d, <i>J</i> = 7.8 Hz, 1H), 7.33–7.20 (m, 2H), 6.94 (dd, <i>J</i> = 8.0/1.7 Hz, 1H), 5.65 (d, <i>J</i> = 8.1 Hz, 1H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 166.6, 166.4, 157.3, 150.7 (ddd, <i>J</i><sub>CF</sub> = 246.5/9.7/4.1 Hz), 139.0, 138.4 (dt, <i>J</i><sub>CF</sub> = 249.4/15.6 Hz), 136.6 (td, <i>J</i><sub>CF</sub> = 8.0/4.3 Hz), 136.4–136.1 (m), 135.3, 129.3, 128.1, 126.8, 118.5, 118.3, 114.7, 111.2 (m), 54.4; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>21</sub>H<sub>16</sub>F<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 417.1057, found 417.1064.</div></div><div id="sec4_1_4_52" class="NLM_sec NLM_sec_level_4"><div id="ac_i76" class="anchor-spacer"></div><h5 class="article-section__title" id="_i76"> 4-Hydroxy-<i>N</i>-(2-(hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)benzamide (<b>6m</b>)</h5><div class="NLM_p last">Compound <b>6k</b> (125 mg, 0.290 mmol) was treated with BBr<sub>3</sub> (1 M in DCM, 2.90 mL, 2.90 mmol) according to General Procedure D. The crude product was purified by column chromatography (DCM/MeOH 100:0–90:10) followed by preparative HPLC to afford compound <b>6m</b> as a white solid (3.00 mg, 2.5%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.03 (s, 1H), 10.0 (s, 1H), 9.03 (s, 1H), 8.60 (d, <i>J</i> = 8.2 Hz, 1H), 7.83–7.78 (m, 2H), 7.76–7.65 (m, 4H), 7.60 (d, <i>J</i> = 8.4 Hz, 2H), 6.83–6.77 (m, 2H), 5.63 (d, <i>J</i> = 8.1 Hz, 1H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 166.6, 165.8, 160.4, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.5/9.8/4.3 Hz), 139.2, 138.2 (dt, <i>J</i><sub>CF</sub> = 247.8/15.7 Hz), 136.8–136.3 (m), 136.2–136.1 (m), 129.7, 128.0, 126.7, 124.4, 114.7, 111.7–110.8 (m), 54.2; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>21</sub>H<sub>16</sub>F<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 417.1057, found 417.1072.</div></div><div id="sec4_1_4_53" class="NLM_sec NLM_sec_level_4"><div id="ac_i77" class="anchor-spacer"></div><h5 class="article-section__title" id="_i77"> <i>N</i>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)-3-(<i>N</i>′-hydroxycarbamimidoyl)benzamide (<b>6p</b>)</h5><div class="NLM_p last">Compound <b>5n</b> (372 mg, 0.883 mmol) was converted to the amidoxime <b>6j</b> according to General Procedure C. After stirring at rt for 1 day, the nitrile of <b>5n</b> was completely converted to the corresponding amidoxime, but none of the ester had converted to the hydroxamic acid. The reaction was left for a further 3 days, but only partial conversion to the desired hydroxamic acid <b>6p</b> was observed. NH<sub>2</sub>OH·HCl (244 mg, 3.51 mmol) and KOH (5 M in MeOH, 0.876 mL) were added. After stirring the reaction mixture at rt for 1 day, the reaction was complete and the mixture was purified by column chromatography (DCM/MeOH 100:0–90:10) to afford compound <b>6p</b> as white flakes (132 mg, 33%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.11 (s, 1H), 9.72 (s, 1H), 9.08 (s, 1H), 8.98 (d, <i>J</i> = 8.2 Hz, 1H), 8.20 (s, 1H), 7.86 (dd, <i>J</i> = 20.8/7.9 Hz, 2H), 7.76–7.67 (m, 4H), 7.62 (d, <i>J</i> = 8.4 Hz, 2H), 7.46 (t, <i>J</i> = 7.8 Hz, 1H), 5.93 (s, 2H), 5.68 (d, <i>J</i> = 8.1 Hz, 1H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.8/21.8 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 166.4, 166.1, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.5/9.7/4.2 Hz), 150.3, 139.0, 138.3 (dt, <i>J</i><sub>CF</sub> = 247.1/14.4 Hz), 136.7–136.4 (m), 136.4–136.2 (m), 133.7, 133.3, 128.3 (2C), 128.2, 128.0, 126.8, 124.7, 112.2–109.2 (m), 54.3; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>22</sub>H<sub>18</sub>F<sub>3</sub>N<sub>4</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 459.1275, found 459.1276.</div></div><div id="sec4_1_4_54" class="NLM_sec NLM_sec_level_4"><div id="ac_i78" class="anchor-spacer"></div><h5 class="article-section__title" id="_i78"> <i>N</i>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)-4-(<i>N</i>′-hydroxycarbamimidoyl)benzamide (<b>6q</b>)</h5><div class="NLM_p last">Compound <b>5o</b> (212 mg, 0.500 mmol) was converted to the corresponding amidoxime <b>6q</b> according to General Procedure C. The crude product was purified by column chromatography (DCM/MeOH 100:0–95:5) to afford <b>6q</b> as a white solid (34 mg, 15%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.05 (s, 1H), 9.81 (s, 1H), 9.07 (s, 1H), 9.00 (d, <i>J</i> = 8.1 Hz, 1H), 7.92 (d, <i>J</i> = 8.5 Hz, 2H), 7.79–7.66 (m, 6H), 7.62 (d, <i>J</i> = 8.4 Hz, 2H), 5.91 (s, 2H), 5.66 (d, <i>J</i> = 8.1 Hz, 1H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.8 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 166.5, 165.9, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.2/9.5/4.1 Hz), 150.2, 138.9, 138.3 (m), 136.7–136.4 (m), 136.3–136.2 (m), 136.1, 133.9, 128.1, 127.7, 126.8, 125.0, 111.2 (m), 54.4; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>22</sub>H<sub>18</sub>F<sub>3</sub>N<sub>4</sub>O<sub>4</sub> [M + H]<sup>+</sup> calcd 459.1275, found 459.1281; purity 85%.</div></div><div id="sec4_1_4_55" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79"> <i>N</i>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)isophthalamide (<b>6r</b>)</h5><div class="NLM_p last">Compound <b>5r</b> (328 mg, 0.741 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. The crude product was purified by column chromatography (DCM/MeOH 95:5–90:10) to afford compound <b>6r</b> as a white solid (153 mg, 47%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.11 (s, 1H), 9.09 (s, 1H), 9.07 (d, <i>J</i> = 8.2 Hz, 1H), 8.41 (br s, 1H), 8.07 (br s, 1H), 8.05–7.95 (m, 2H), 7.77–7.67 (m, 4H), 7.62 (d, <i>J</i> = 8.4 Hz, 2H), 7.55 (dd, <i>J</i> = 7.8/7.8 Hz, 1H), 7.50 (s, 1H), 5.68 (d, <i>J</i> = 8.1 Hz, 1H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.8 Hz), −163.4 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 167.5, 166.4, 165.9, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.5/9.7/4.1 Hz), 140.2–136.8 (m), 138.9, 136.6 (td, <i>J</i><sub>CF</sub> = 8.0/4.3 Hz), 136.4–136.3 (m), 134.3, 133.9, 130.6, 130.4, 128.3, 128.1, 126.8, 126.7, 112.6–109.7 (m), 54.4; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>22</sub>H<sub>17</sub>F<sub>3</sub>N<sub>3</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 444.1166, found 444.1172.</div></div><div id="sec4_1_4_56" class="NLM_sec NLM_sec_level_4"><div id="ac_i80" class="anchor-spacer"></div><h5 class="article-section__title" id="_i80"> <i>N</i>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)terephthalamide (<b>6s</b>)</h5><div class="NLM_p last">Compound <b>5s</b> (383 mg, 0.865 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. The crude product was purified by column chromatography (DCM/MeOH 100:0–90:10) to afford compound <b>6s</b> as a white solid (38 mg, 10%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.08 (s, 1H), 9.11 (d, <i>J</i> = 8.1 Hz, 1H), 9.07 (br s, 1H), 8.09 (br s, 1H), 8.01–7.89 (m, 4H), 7.78–7.58 (m, 6H), 7.50 (br s, 1H), 5.67 (d, <i>J</i> = 8.1 Hz, 1H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.8 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 167.2, 166.4, 165.8, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.5/9.9/4.3 Hz), 138.7, 138.2 (dt, <i>J</i><sub>CF</sub> = 262.9/14.7 Hz), 136.7, 136.5 (dt, <i>J</i><sub>CF</sub> = 6.4/5.2 Hz), 136.4–136.2 (m), 136.1, 128.1, 127.8, 127.3, 126.8, 112.3 (dd, <i>J</i><sub>CF</sub> = 16.0/5.4 Hz), 54.4; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>22</sub>H<sub>17</sub>F<sub>3</sub>N<sub>3</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 444.1166, found 444.1172.</div></div><div id="sec4_1_4_57" class="NLM_sec NLM_sec_level_4"><div id="ac_i81" class="anchor-spacer"></div><h5 class="article-section__title" id="_i81"> <i>N</i><sup>1</sup>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)-<i>N</i><sup>4</sup>-methylterephthalamide (<b>6t</b>)</h5><div class="NLM_p last">Compound <b>5t</b> (52.7 mg, 0.115 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. Upon completion, the mixture was purified by preparative HPLC to afford compound <b>6t</b> as a white fluffy solid after lyophilization (24.5 mg, 47%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.09 (s, 1H), 9.11 (d, <i>J</i> = 8.1 Hz, 1H), 9.07 (br s, 1H), 8.57 (d, <i>J</i> = 4.6 Hz, 1H), 8.00 (d, <i>J</i> = 8.4 Hz, 2H), 7.90 (d, <i>J</i> = 8.4 Hz, 2H), 7.78–7.66 (m, 4H), 7.62 (d, <i>J</i> = 8.3 Hz, 2H), 5.67 (d, <i>J</i> = 8.0 Hz, 1H), 2.80 (d, <i>J</i> = 4.5 Hz, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) −134.9 (d, <i>J</i> = 21.8 Hz), −163.5 (dd, <i>J</i> = 21.8/21.8 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 166.4, 165.9, 165.8, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.2/9.3/3.9 Hz), 138.8, 139.8–137.0 (m), 137.0, 136.6 (td, <i>J</i><sub>CF</sub> = 7.9/4.8 Hz), 136.3, 135.9, 128.1, 127.8, 126.9, 126.8, 111.2 (dd, <i>J</i><sub>CF</sub> = 16.2/5.2 Hz), 54.4, 26.3; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>23</sub>H<sub>19</sub>N<sub>3</sub>O<sub>4</sub>F<sub>3</sub> [MH]<sup>+</sup> calcd 458.1322, found 458.1323.</div></div><div id="sec4_1_4_58" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> <i>N</i><sup>1</sup>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)-<i>N</i><sup>4</sup>,<i>N</i><sup>4</sup>-dimethylterephthalamide (<b>6u</b>)</h5><div class="NLM_p last">Compound <b>5u</b> (51.3 mg, 0.109 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. Upon completion, the mixture was purified by column chromatography (DCM/PE/AcOH 1:1:0.1 to DCM/MeOH/AcOH 95:5:0.1) followed by preparative HPLC to afford compound <b>6u</b> as a white fluffy solid after lyophilization (8.0 mg, 16%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.07 (s, 1H), 9.08 (d, <i>J</i> = 8.1 Hz, 2H), 7.96 (d, <i>J</i> = 8.4 Hz, 2H), 7.78–7.64 (m, 4H), 7.62 (d, <i>J</i> = 8.4 Hz, 2H), 7.47 (d, <i>J</i> = 8.4 Hz, 2H), 5.66 (d, <i>J</i> = 8.0 Hz, 1H), 2.99 (s, 3H), 2.89 (s, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.6/21.6 Hz); <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>24</sub>H<sub>21</sub>N<sub>3</sub>O<sub>4</sub>F<sub>3</sub> [MH]<sup>+</sup> calcd 472.1479, found 472.1480.</div></div><div id="sec4_1_4_59" class="NLM_sec NLM_sec_level_4"><div id="ac_i83" class="anchor-spacer"></div><h5 class="article-section__title" id="_i83"> <i>N</i><sup>1</sup>-Ethyl-<i>N</i><sup>4</sup>-(2-(hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)terephthalamide (<b>6v</b>)</h5><div class="NLM_p last">Compound <b>5v</b> (50.7 mg, 0.108 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. Upon completion, the resulting mixture was purified by column chromatography (DCM/PE/AcOH 1:1:0.1 to DCM/MeOH/AcOH 95:5:0.1) followed by preparative HPLC to afford compound <b>6v</b> as a white fluffy solid after lyophilization (10.0 mg, 13%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.08 (s, 1H), 9.10 (d, <i>J</i> = 8.1 Hz, 1H), 9.06 (br s, 1H), 8.59 (t, <i>J</i> = 5.2 Hz, 1H), 7.99 (d, <i>J</i> = 8.4 Hz, 2H), 7.90 (d, <i>J</i> = 8.3 Hz, 2H), 7.75–7.68 (m, 4H), 7.62 (d, <i>J</i> = 8.3 Hz, 2H), 5.66 (d, <i>J</i> = 8.0 Hz, 1H), 3.30 (q, <i>J</i> = 7.1 Hz, 2H), 1.13 (t, <i>J</i> = 7.2 Hz, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.8 Hz), −163.5 (dd, <i>J</i> = 21.6/21.6 Hz); <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>24</sub>H<sub>21</sub>N<sub>3</sub>O<sub>4</sub>F<sub>3</sub> [MH]<sup>+</sup> calcd 472.1479, found 472.1480.</div></div><div id="sec4_1_4_60" class="NLM_sec NLM_sec_level_4"><div id="ac_i84" class="anchor-spacer"></div><h5 class="article-section__title" id="_i84"> <i>N</i><sup>1</sup>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)-<i>N</i><sup>4</sup>-isopropylterephthalamide (<b>6w</b>)</h5><div class="NLM_p last">Compound <b>5w</b> (56.8 mg, 0.117 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. Upon completion, the mixture was purified by column chromatography (DCM/PE/AcOH 1:1:0.1 to DCM/MeOH/AcOH 95:5:0.1) followed by preparative HPLC to afford compound <b>6w</b> as a white fluffy solid after lyophilization (11.6 mg, 20%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.09 (s, 1H), 9.10 (d, <i>J</i> = 8.1 Hz, 1H), 9.07 (d, <i>J</i> = 0.8 Hz, 1H), 8.35 (d, <i>J</i> = 7.7 Hz, 1H), 7.99 (d, <i>J</i> = 8.5 Hz, 2H), 7.91 (d, <i>J</i> = 8.5 Hz, 2H), 7.77–7.66 (m, 4H), 7.62 (d, <i>J</i> = 8.4 Hz, 2H), 5.67 (d, <i>J</i> = 8.1 Hz, 1H), 4.19–3.97 (m, 1H), 1.18 (d, <i>J</i> = 6.6 Hz, 6H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.8 Hz), −163.5 (dd, <i>J</i> = 21.8/21.8 Hz); <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>25</sub>H<sub>23</sub>N<sub>3</sub>O<sub>4</sub>F<sub>3</sub> [MH]<sup>+</sup> calcd 486.1635, found 486.1638.</div></div><div id="sec4_1_4_61" class="NLM_sec NLM_sec_level_4"><div id="ac_i85" class="anchor-spacer"></div><h5 class="article-section__title" id="_i85"> <i>N</i><sup>1</sup>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)-<i>N</i><sup>4</sup>-(2-hydroxyethyl)terephthalamide (<b>6x</b>)</h5><div class="NLM_p last">Compound <b>5x</b> (54.1 mg, 0.111 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. Upon completion, the mixture was purified by preparative HPLC to afford compound <b>6x</b> as a white fluffy solid after lyophilization (18.4 mg, 34%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.08 (s, 1H), 9.11 (d, <i>J</i> = 8.1 Hz, 1H), 9.07 (s, 1H), 8.57 (t, <i>J</i> = 5.6 Hz, 1H), 8.00 (d, <i>J</i> = 8.4 Hz, 2H), 7.92 (d, <i>J</i> = 8.4 Hz, 2H), 7.78–7.66 (m, 4H), 7.62 (d, <i>J</i> = 8.3 Hz, 2H), 5.67 (d, <i>J</i> = 8.0 Hz, 1H), 4.74 (s, 1H), 3.52 (q, <i>J</i> = 9.7 Hz, 2H), 3.40–3.30 (m, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −163.4 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 166.4, 165.8, 165.7, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.8/9.5/4.3 Hz), 138.7, 138.3 (dt, <i>J</i><sub>CF</sub> = 31.2/16.1 Hz), 137.0, 136.6 (td, <i>J</i><sub>CF</sub> = 8.2/4.7 Hz), 136.3, 135.9, 128.2, 127.8, 127.1, 126.8, 112.4–109.4 (m), 59.7, 54.4, 42.3; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>24</sub>H<sub>21</sub>N<sub>3</sub>O<sub>5</sub>F<sub>3</sub> [MH]<sup>+</sup> calcd 488.1428, found 488.1425.</div></div><div id="sec4_1_4_62" class="NLM_sec NLM_sec_level_4"><div id="ac_i86" class="anchor-spacer"></div><h5 class="article-section__title" id="_i86"> <i>N</i>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)-4-sulfamoylbenzamide (<b>6y</b>)</h5><div class="NLM_p last">Compound <b>5y</b> (55.2 mg, 0.115 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. Upon completion, the mixture was then purified by column chromatography (DCM/PE/AcOH 1:1:0.1 to DCM/MeOH/AcOH 95:5:0.1) followed by preparative HPLC to afford compound <b>6y</b> as a white fluffy solid after lyophilization (10.0 mg, 18%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.09 (d, <i>J</i> = 0.9 Hz, 1H), 9.22 (d, <i>J</i> = 8.0 Hz, 1H), 9.07 (d, <i>J</i> = 1.1 Hz, 1H), 8.15–7.99 (m, 2H), 7.97–7.84 (m, 2H), 7.71 (dt, <i>J</i> = 10.6/6.2 Hz, 4H), 7.62 (d, <i>J</i> = 8.4 Hz, 2H), 7.50 (s, 2H), 5.66 (d, <i>J</i> = 8.0 Hz, 1H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −163.4 (dd, <i>J</i> = 21.6/21.6 Hz); <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>21</sub>H<sub>17</sub>F<sub>3</sub>N<sub>3</sub>O<sub>5</sub>S [MH]<sup>+</sup> calcd 480.0836, found 480.0835.</div></div><div id="sec4_1_4_63" class="NLM_sec NLM_sec_level_4"><div id="ac_i87" class="anchor-spacer"></div><h5 class="article-section__title" id="_i87"> 3-Amino-<i>N</i>-(2-(hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)benzamide TFA Salt (<b>6ab</b>)</h5><div class="NLM_p last">Compound <b>5z</b> (228 mg, 0.443 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. Upon completion, the mixture was purified by column chromatography (DCM/MeOH 100:0–95:5) to afford <i>tert</i>-butyl (3-((2-(hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)carbamoyl)phenyl)carbamate (108 mg). This was dissolved in 20% TFA/DCM and stirred at rt for 4 h to remove the boc group. The reaction mixture was concentrated in vacuo and purified by preparative HPLC to afford compound <b>6ab</b> as a pale brown solid (28.5 mg, 12% over two steps). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.09 (s, 1H), 8.82 (d, <i>J</i> = 8.0 Hz, 1H), 7.77–7.65 (m, 4H), 7.61 (d, <i>J</i> = 8.3 Hz, 2H), 7.51–7.42 (m, 2H), 7.33 (t, <i>J</i> = 7.8 Hz, 1H), 7.09 (dd, <i>J</i> = 7.9/1.3 Hz, 1H), 6.72 (br s, 3H), 5.64 (d, <i>J</i> = 8.0 Hz, 1H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −74.5, −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO, TFA signals are not included) δ 166.4, 166.1, 150.7 (ddd, <i>J</i><sub>CF</sub> = 246.6/9.7/4.0 Hz), 141.2–141.0 (m), 138.9, 138.3 (dt, <i>J</i><sub>CF</sub> = 31.0/13.9 Hz), 136.6 (td, <i>J</i><sub>CF</sub> = 8.3/4.6 Hz), 136.3, 135.0, 129.2, 128.0, 126.8, 121.1, 120.6, 117.5, 111.7–110.6 (m), 54.3; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>21</sub>H<sub>17</sub>F<sub>3</sub>N<sub>3</sub>O<sub>3</sub> [MH]<sup>+</sup> calcd. 416.1217, found 416.1229.</div></div><div id="sec4_1_4_64" class="NLM_sec NLM_sec_level_4"><div id="ac_i88" class="anchor-spacer"></div><h5 class="article-section__title" id="_i88"> 4-Amino-<i>N</i>-(2-(hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)benzamide TFA Salt (<b>6ac</b>)</h5><div class="NLM_p last">Compound <b>5aa</b> (235 mg, 0.457 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. Upon completion, the mixture was purified by column chromatography (DCM/MeOH 100:0–95:5) to afford <i>tert</i>-butyl (4-((2-(hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)carbamoyl)phenyl)carbamate (128 mg). This was dissolved in 20% TFA/DCM and stirred at rt for 4 h to remove the boc group. The reaction mixture was concentrated in vacuo and purified by preparative HPLC to afford compound <b>6ac</b> as a pale brown solid (69 mg, 28% over two steps). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.08 (br s, 1H), 8.53 (d, <i>J</i> = 8.1 Hz, 1H), 7.78 (d, <i>J</i> = 8.6 Hz, 2H), 7.74–7.56 (m, 6H), 7.17 (br s, 3H), 6.80 (d, <i>J</i> = 8.5 Hz, 2H), 5.66 (d, <i>J</i> = 8.0 Hz, 1H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −74.5, −134.9 (d, <i>J</i> = 21.6 Hz), −163.6 (dd, <i>J</i> = 21.6/21.6 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO, TFA signals are not included) δ 166.8, 166.0, 150.7 (ddd, <i>J</i><sub>CF</sub> = 246.6/9.7/4.2 Hz), 147.5, 139.4, 138.4 (dt, <i>J</i><sub>CF</sub> = 249.4/15.6 Hz), 136.7 (td, <i>J</i><sub>CF</sub> = 8.1/4.6 Hz), 136.3 (m), 129.4, 128.1, 127.0, 123.9, 115.4, 111.7–110.6 (m), 54.3; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>21</sub>H<sub>17</sub>F<sub>3</sub>N<sub>3</sub>O<sub>3</sub> [MH]<sup>+</sup> calcd 416.1217, found 416.1234.</div></div><div id="sec4_1_4_65" class="NLM_sec NLM_sec_level_4"><div id="ac_i89" class="anchor-spacer"></div><h5 class="article-section__title" id="_i89"> <i>N</i>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)-4-(methylsulfonamido)benzamide (<b>6ad</b>)</h5><div class="NLM_p last">Compound <b>5ad</b> (65.2 mg, 0.132 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. Upon completion, the mixture was purified by column chromatography (DCM/PE/AcOH 1:1:0.1 to DCM/MeOH/AcOH 95:5:0.1) followed by preparative HPLC to afford compound <b>6ad</b> as a white fluffy solid after lyophilization (30.0 mg, 46%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.07 (s, 1H), 10.13 (s, 1H), 9.06 (s, 1H), 8.83 (d, <i>J</i> = 8.1 Hz, 1H), 7.92 (d, <i>J</i> = 8.8 Hz, 2H), 7.79–7.65 (m, 4H), 7.60 (d, <i>J</i> = 8.4 Hz, 2H), 7.25 (d, <i>J</i> = 8.8 Hz, 2H), 5.65 (d, <i>J</i> = 8.1 Hz, 1H), 3.07 (s, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.8 Hz), −163.5 (dd, <i>J</i> = 21.8/21.8 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 166.5, 165.6, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.5/9.6/4.3 Hz), 141.4, 139.0, 138.3 (dt, <i>J</i><sub>CF</sub> = 31.4/16.1 Hz), 136.6 (td, <i>J</i><sub>CF</sub> = 8.2/4.2 Hz), 136.3–136.1 (m), 129.2, 128.5, 128.0, 126.8, 117.8, 111.2 (dd, <i>J</i><sub>CF</sub> = 15.8/5.5 Hz), 54.3, 40.6; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>22</sub>H<sub>19</sub>F<sub>3</sub>N<sub>3</sub>O<sub>5</sub>S [MH]<sup>+</sup> calcd 494.0992, found 494.0993.</div></div><div id="sec4_1_4_66" class="NLM_sec NLM_sec_level_4"><div id="ac_i90" class="anchor-spacer"></div><h5 class="article-section__title" id="_i90"> <i>N</i>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)-4-(sulfamoylamino)benzamide (<b>6ae</b>)</h5><div class="NLM_p last">Compound <b>5ae</b> (54.2 mg, 0.110 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. Upon completion, the mixture was purified by preparative HPLC to afford compound <b>6ae</b> as a beige fluffy solid after lyophilization (39.3 mg, 72%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.07 (s, 1H), 9.93 (s, 1H), 8.75 (d, <i>J</i> = 8.2 Hz, 1H), 7.88 (d, <i>J</i> = 8.7 Hz, 2H), 7.81–7.65 (m, 4H), 7.61 (d, <i>J</i> = 8.4 Hz, 2H), 7.29 (s, 2H), 7.19 (d, <i>J</i> = 8.8 Hz, 2H), 5.66 (d, <i>J</i> = 8.1 Hz, 1H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 166.6, 165.8, 150.7 (ddd, <i>J</i><sub>CF</sub> = 246.7/9.8/4.3 Hz), 142.5, 139.1, 138.3 (dt, <i>J</i><sub>CF</sub> = 31.1/15.9 Hz), 136.6 (td, <i>J</i><sub>CF</sub> = 8.1/4.6 Hz), 136.2 (d, <i>J</i> = 1.5 Hz), 128.8, 128.1, 126.8, 126.6, 116.2, 111.2 (dd, <i>J</i><sub>CF</sub> = 16.0/5.4 Hz), 54.2; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>21</sub>H<sub>18</sub>F<sub>3</sub>N<sub>4</sub>O<sub>5</sub>S [MH]<sup>+</sup> calcd 495.0945, found 495.0947.</div></div><div id="sec4_1_4_67" class="NLM_sec NLM_sec_level_4"><div id="ac_i91" class="anchor-spacer"></div><h5 class="article-section__title" id="_i91"> <i>N</i>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)-3,4-dimethoxybenzamide (<b>6af</b>)</h5><div class="NLM_p last">Compound <b>5af</b> (183 mg, 0.398 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. The crude product was purified by column chromatography (DCM/MeOH 100:0–90:10) to afford compound <b>6af</b> as a white solid (93.4 mg, 51%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.08 (s, 1H), 9.07 (s, 1H), 8.86 (d, <i>J</i> = 8.2 Hz, 1H), 7.79–7.47 (m, 8H), 7.01 (d, <i>J</i> = 8.6 Hz, 1H), 5.68 (d, <i>J</i> = 8.1 Hz, 1H), 3.82 (s, 3H), 3.81 (s, 3H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 166.6, 165.8, 151.5, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.6/9.7/4.2 Hz), 148.1, 139.1, 138.3 (dt, <i>J</i><sub>CF</sub> = 248.0/15.7 Hz), 136.6 (td, <i>J</i><sub>CF</sub> = 8.2/4.5 Hz), 136.3–135.7 (m), 128.1, 126.8, 126.0, 121.3, 111.4–111.1 (m), 111.1, 110.8, 55.6 (2C), 54.3; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>23</sub>H<sub>20</sub>F<sub>3</sub>N<sub>2</sub>O<sub>5</sub> [MH]<sup>+</sup> calcd 461.1319, found 461.1322.</div></div><div id="sec4_1_4_68" class="NLM_sec NLM_sec_level_4"><div id="ac_i92" class="anchor-spacer"></div><h5 class="article-section__title" id="_i92"> 3,4-Dihydroxy-<i>N</i>-(2-(hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)benzamide (<b>6ag</b>)</h5><div class="NLM_p last">Compound <b>6af</b> (200 mg, 0.434 mmol) was treated with BBr<sub>3</sub> (1 M in DCM, 4.34 mL, 4.34 mmol) according to General Procedure D to form the corresponding catechol. The crude product was purified by preparative HPLC to afford compound <b>6ag</b> as a white solid (69.3 mg, 37%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.05 (s, 1H), 9.49 (s, 1H), 9.15 (s, 1H), 9.06 (s, 1H), 8.49 (d, <i>J</i> = 8.1 Hz, 1H), 7.80–7.64 (m, 4H), 7.61 (d, <i>J</i> = 8.3 Hz, 2H), 7.39–7.21 (m, 2H), 6.78 (d, <i>J</i> = 8.2 Hz, 1H), 5.63 (d, <i>J</i> = 8.1 Hz, 1H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.6 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 166.8, 166.0, 150.7 (ddd, <i>J</i><sub>CF</sub> = 246.6/9.7/4.2 Hz), 148.8, 144.9, 139.3, 138.4 (dt, <i>J</i><sub>CF</sub> = 249.5/15.8 Hz), 136.7 (td, <i>J</i><sub>CF</sub> = 8.1/4.4 Hz), 136.3–136.2 (m), 128.1, 126.8, 125.0, 119.7, 115.5, 114.9, 112.7–109.9 (m), 54.3; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>21</sub>H<sub>16</sub>F<sub>3</sub>N<sub>2</sub>O<sub>5</sub> [MH]<sup>+</sup> calcd 433.1006, found 433.1003.</div></div><div id="sec4_1_4_69" class="NLM_sec NLM_sec_level_4"><div id="ac_i93" class="anchor-spacer"></div><h5 class="article-section__title" id="_i93"> 3-Fluoro-4-hydroxy-<i>N</i>-(2-(hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)benzamide (<b>6ah</b>)</h5><div class="NLM_p last">3-Fluoro-4-hydroxybenzoic acid (194 mg, 1.24 mmol) was coupled to compound <b>4</b> (330 mg, 1.12 mmol) according to General Procedure A. Upon completion, the reaction mixture was diluted with a 1 M HCl solution (10 mL) and extracted with DCM (3 × 15 mL). The combined organic layers were washed with water (2 × 20 mL) and brine (20 mL). The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by column chromatography (DCM/MeOH 100:0–95:5) to afford 430 mg of the desired product (64% of purity). The crude product was converted to the corresponding hydroxamic acid according to General Procedure C. The mixture was purified by column chromatography (DCM/MeOH 100:0–90:10) to afford compound <b>6ah</b> as a white solid (30 mg, 7%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.05 (s, 1H), 10.53 (s, 1H), 9.05 (s, 1H), 8.82 (d, <i>J</i> = 8.1 Hz, 1H), 7.85–7.47 (m, 8H), 6.99 (t, <i>J</i> = 8.7 Hz, 1H), 5.63 (d, <i>J</i> = 8.1 Hz, 1H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.8 Hz), −136.6, −163.5 (dd, <i>J</i> = 21.8/21.8 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 166.5, 164.9 (d, <i>J</i><sub>CF</sub> = 2.0 Hz), 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.5/9.8/4.2 Hz), 150.3 (d, <i>J</i><sub>CF</sub> = 241.0 Hz), 148.1 (d, <i>J</i><sub>CF</sub> = 12.1 Hz), 138.9, 138.3 (dt, <i>J</i><sub>CF</sub> = 249.1/15.9 Hz), 136.8–136.4 (m), 136.3–136.1 (m), 128.1, 126.7, 125.0 (d, <i>J</i><sub>CF</sub> = 2.8 Hz), 124.9 (d, <i>J</i><sub>CF</sub> = 5.3 Hz), 117.0 (d, <i>J</i><sub>CF</sub> = 2.9 Hz), 115.9 (d, <i>J</i><sub>CF</sub> = 19.6 Hz), 111.6–110.6 (m), 54.4; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>21</sub>H<sub>15</sub>F<sub>4</sub>N<sub>2</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 435.0962, found 435.0968.</div></div><div id="sec4_1_4_70" class="NLM_sec NLM_sec_level_4"><div id="ac_i94" class="anchor-spacer"></div><h5 class="article-section__title" id="_i94"> 4-Fluoro-3-hydroxy-<i>N</i>-(2-(hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)benzamide (<b>6ai</b>)</h5><div class="NLM_p last">4-Fluoro-3-hydroxybenzoic acid (233 mg, 1.49 mmol) was coupled to compound <b>4</b> (400 mg, 1.35 mmol) according to General Procedure A. Upon completion, the reaction mixture was diluted with a 1 M HCl solution (10 mL) and extracted with DCM (3 × 15 mL). The combined organic layers were washed with water (2 × 20 mL) and brine (20 mL). The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude was purified by column chromatography (DCM/MeOH 98:2–90:10) to afford 577 mg of a white solid (53% purity). The product was converted to the corresponding hydroxamic acid according to General Procedure C. After stirring at rt for 1 day, 40% of the starting material had converted to compound <b>6ai</b>. The reaction was left for a further 6 days reaching 70% conversion according to LC–MS. The reaction was stopped, and the starting material and compound <b>6ai</b> were isolated by column chromatography (DCM/MeOH 100:0–90:10), in which 68 mg (13%) of compound <b>6ai</b> was obtained as a pale yellow solid and 10 mg of the starting material was recovered. <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.04 (s, 1H), 10.11 (s, 1H), 9.04 (s, 1H), 8.87 (d, <i>J</i> = 8.1 Hz, 1H), 7.74–7.66 (m, 4H), 7.60 (d, <i>J</i> = 8.4 Hz, 2H), 7.49 (dd, <i>J</i> = 8.6/2.2 Hz, 1H), 7.41 (ddd, <i>J</i> = 8.4/4.3/2.2 Hz, 1H), 7.21 (dd, <i>J</i> = 11.0/8.5 Hz, 1H), 5.60 (d, <i>J</i> = 8.1 Hz, 1H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −132.2, −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 166.5, 165.5, 153.1 (d, <i>J</i><sub>CF</sub> = 246.2 Hz), 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.4/10.2/4.0 Hz), 144.6 (d, <i>J</i><sub>CF</sub> = 12.4 Hz), 138.9, 138.3 (dt, <i>J</i><sub>CF</sub> = 243.0/12.5 Hz), 136.6 (td, <i>J</i><sub>CF</sub> = 7.9/4.6 Hz), 136.3–136.1 (m), 130.6 (d, <i>J</i><sub>CF</sub> = 3.1 Hz), 128.1, 126.8, 119.3 (d, <i>J</i><sub>CF</sub> = 7.4 Hz), 117.7 (d, <i>J</i><sub>CF</sub> = 3.9 Hz), 115.8 (d, <i>J</i><sub>CF</sub> = 18.9 Hz), 111.9–110.6 (m), 54.4; <i>m</i>/<i>z</i> MS (TOF ES<sup>–</sup>) C<sub>21</sub>H<sub>13</sub>F<sub>4</sub>N<sub>2</sub>O<sub>4</sub> [M – H]<sup>−</sup> calcd 433.1, found 433.1; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>21</sub>H<sub>15</sub>F<sub>4</sub>N<sub>2</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 435.0962, found 435.0973.</div></div><div id="sec4_1_4_71" class="NLM_sec NLM_sec_level_4"><div id="ac_i95" class="anchor-spacer"></div><h5 class="article-section__title" id="_i95"> <i>N</i>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)-1<i>H</i>-indole-5-carboxamide (<b>6aj</b>)</h5><div class="NLM_p last">Compound <b>5aj</b> (200 mg, 0.456 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. Upon completion, the solvent was concentrated in vacuo. An aqueous 10% citric acid solution was added dropwise to the crude residue. The precipitate was filtered and purified by flash column chromatography (PE/EtOAc 100:0–15:85) to afford compound <b>6aj</b> as a white solid (129 mg, 64%). <sup>1</sup>H NMR (MeOD) δ 8.19 (d, <i>J</i> = 1.3 Hz, 1H), 7.69–7.59 (m, 5H), 7.47–7.36 (m, 3H), 7.32 (d, <i>J</i> = 3.2 Hz, 1H), 6.56 (dd, <i>J</i> = 3.2, 0.8 Hz, 1H), 5.73 (s, 1H); <sup>19</sup>F NMR (MeOD) δ −136.9 (d, <i>J</i> = 19.8 Hz), −166.0 (dd, <i>J</i> = 19.9/19.9 Hz); <sup>13</sup>C NMR (MeOD) δ 171.0, 169.5, 152.6 (ddd, <i>J</i><sub>CF</sub> = 247.9/9.9/4.2 Hz), 141.8–139.2 (m), 139.7, 139.5, 139.0, 138.5 (td, <i>J</i><sub>CF</sub> = 7.8/4.5 Hz), 129.2 (2C), 129.0, 128.2 (2C), 127.4, 125.6, 121.8, 121.7, 112.3–111.9 (m, 3C), 103.7, 56.2; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>23</sub>H<sub>17</sub>F<sub>3</sub>N<sub>3</sub>O<sub>3</sub> [MH]<sup>+</sup> calcd 440.1217, found 440.1210.</div></div><div id="sec4_1_4_72" class="NLM_sec NLM_sec_level_4"><div id="ac_i96" class="anchor-spacer"></div><h5 class="article-section__title" id="_i96"> <i>N</i>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)-1<i>H</i>-indazole-5-carboxamide (<b>6ak</b>)</h5><div class="NLM_p last">Compound <b>5ak</b> (45 mg, 0.102 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. Upon completion, the solvent was concentrated in vacuo. An aqueous 10% citric acid solution was added dropwise to the crude residue. The precipitate was filtered and washed thoroughly with water, then triturated with Et<sub>2</sub>O to afford compound <b>6ak</b> as an off-white solid (40 mg, 89%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 13.32 (s, 1H), 11.11 (s, 1H), 9.09 (s, 1H), 8.93 (d, <i>J</i> = 8.1 Hz, 1H), 8.48 (s, 1H), 8.21 (s, 1H), 7.91 (dd, <i>J</i> = 8.8/1.2 Hz, 1H), 7.80–7.52 (m, 7H), 5.70 (d, <i>J</i> = 8.0 Hz, 1H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 167.0, 166.9, 150.9 (ddd, <i>J</i><sub>CF</sub> = 246.9/9.7/4.2 Hz), 141.4, 139.2, 138.6 (dt, <i>J</i><sub>CF</sub> = 31.4/15.9 Hz), 136.8 (td, <i>J</i><sub>CF</sub> = 7.9/4.4 Hz), 136.6, 135.3, 128.4 (2C), 127.1 (2C), 126.4, 126.0, 122.6, 121.7, 111.4 (m, 2C), 110.1, 54.8; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>22</sub>H<sub>16</sub>F<sub>3</sub>N<sub>4</sub>O<sub>3</sub> [MH]<sup>+</sup> calcd 441.1169, found 441.1176.</div></div><div id="sec4_1_4_73" class="NLM_sec NLM_sec_level_4"><div id="ac_i97" class="anchor-spacer"></div><h5 class="article-section__title" id="_i97"> <i>N</i>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)-1<i>H</i>-benzo[<i>d</i>][1,2,3]triazole-5-carboxamide (<b>6al</b>)</h5><div class="NLM_p last">Compound <b>5al</b> (90 mg, 0.204 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. Upon completion, the solvent was concentrated in vacuo. An aqueous 10% citric acid solution was added dropwise to the crude residue. The precipitate was filtered and washed thoroughly with water, then triturated in Et<sub>2</sub>O to afford compound <b>6al</b> as a yellow solid (75 mg, 83%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.09 (s, 1H), 9.22 (d, <i>J</i> = 8.0 Hz, 1H), 9.07 (s, 1H), 8.60 (s, 1H), 8.02–7.91 (m, 2H), 7.79–7.61 (m, 7H), 5.75–5.63 (m, 1H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO, rotamers) δ 171.9, 171.5, 166.6, 166.3, 150.8 (ddd, <i>J</i><sub>CF</sub> = 246.8/9.8/4.3 Hz), 138.9 (2C), 138.5 (dt, <i>J</i><sub>CF</sub> = 248.9/15.7 Hz), 136.8–136.6 (m),<a class="ref internalNav" href="#fn9999a" aria-label="a">a</a> 136.5, 129.0, 128.4 (3C), 127.0, 126.9 (3C), 111.6–111.2 (m), 54.8; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>21</sub>H<sub>15</sub>F<sub>3</sub>N<sub>5</sub>O<sub>3</sub> [MH]<sup>+</sup> calcd 442.1122, found 442.1127.</div></div><div id="sec4_1_4_74" class="NLM_sec NLM_sec_level_4"><div id="ac_i98" class="anchor-spacer"></div><h5 class="article-section__title" id="_i98"> <i>N</i>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)-1<i>H</i>-benzo[<i>d</i>]imidazole-5-carboxamide (<b>6am</b>)</h5><div class="NLM_p last">Compound <b>5am</b> (80 mg, 0.182 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. Upon completion, the solvent was concentrated in vacuo. An aqueous 10% citric acid solution was added dropwise to the crude residue. The resultant solid was filtered and washed thoroughly with water, then triturated in Et<sub>2</sub>O to afford compound <b>6am</b> as an off-white solid (75 mg, 94%). <sup>1</sup>H NMR (MeOD) δ 8.33 (s, 1H), 8.22 (d, <i>J</i> = 1.0 Hz, 1H), 7.85 (dd, <i>J</i> = 8.5/1.6 Hz, 1H), 7.71–7.61 (m, 5H), 7.46–7.37 (m, 2H), 5.73 (s, 1H); <sup>19</sup>F NMR (MeOD) δ −136.8 (d, <i>J</i> = 19.8 Hz), −165.9 (dd, <i>J</i> = 19.8/19.8 Hz); <sup>13</sup>C NMR (MeOD) δ 170.0, 169.4, 152.6 (ddd, <i>J</i><sub>CF</sub> = 247.9/9.9/4.1 Hz), 144.8, 141.9–138.3 (m), 140.7, 139.2, 139.1, 138.6, 138.5–138.3 (m), 129.8, 129.3 (2C), 128.2 (2C), 123.6, 116.8, 115.8, 112.1 (m, 2C), 56.5; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>22</sub>H<sub>16</sub>F<sub>3</sub>N<sub>4</sub>O<sub>3</sub> [MH]<sup>+</sup> calcd 441.1169, found 441.1176.</div></div><div id="sec4_1_4_75" class="NLM_sec NLM_sec_level_4"><div id="ac_i99" class="anchor-spacer"></div><h5 class="article-section__title" id="_i99"> <i>N</i>-(2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)-2-oxoindoline-5-carboxamide (<b>6an</b>)</h5><div class="NLM_p last">Compound <b>5an</b> (70 mg, 0.154 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. Upon completion, the solvent was concentrated in vacuo. An aqueous 10% citric acid solution was added dropwise to the crude residue. The resultant solid was filtered and washed thoroughly with water. The solid was purified by flash column chromatography (DCM/MeOH 100:0–90:10) to afford compound <b>6an</b> as a dull yellow solid (30 mg, 43%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.05 (s, 1H), 10.63 (s, 1H), 9.05 (s, 1H), 8.72 (d, <i>J</i> = 8.1 Hz, 1H), 7.82 (d, <i>J</i> = 7.7 Hz, 2H), 7.76–7.65 (m, 4H), 7.60 (d, <i>J</i> = 8.4 Hz, 2H), 6.86 (d, <i>J</i> = 8.5 Hz, 1H), 5.64 (d, <i>J</i> = 8.1 Hz, 1H), 3.53 (s, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.8 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 177.9, 167.3, 166.9, 151.2 (ddd, <i>J</i><sub>CF</sub> = 247.0/9.8/4.2 Hz), 147.18, 139.12, 138.9 (dt, <i>J</i><sub>CF</sub> = 32.1/15.4 Hz), 137.1–136.8 (m, 2C), 128.6 (3C), 127.4 (2C), 127.2, 126.3, 124.5, 111.7 (m, 2C), 109.3, 54.9, 36.1; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>23</sub>H<sub>17</sub>F<sub>3</sub>N<sub>3</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 456.1166, found 456.1172.</div></div><div id="sec4_1_4_76" class="NLM_sec NLM_sec_level_4"><div id="ac_i100" class="anchor-spacer"></div><h5 class="article-section__title" id="_i100"> <i>N</i>-Hydroxy-2-(2-phenylacetamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetamide (<b>6ao</b>)</h5><div class="NLM_p last">Compound <b>5ao</b> (54.0 mg, 0.131 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. The crude product was purified by column chromatography (DCM/MeOH 100:0–90:0) to afford compound <b>6ao</b> as a white solid (26.7 mg, 49%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.0 (s, 1H), 9.04 (s, 1H), 8.92 (d, <i>J</i> = 8.3 Hz, 1H), 7.76–7.64 (m, 4H), 7.50 (d, <i>J</i> = 8.3 Hz, 2H), 7.32–7.17 (m, 5H), 5.42 (d, <i>J</i> = 8.3 Hz, 1H), 3.58 (s, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 169.9, 166.3, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.4/9.8/4.3 Hz), 139.3, 138.3 (dt, <i>J</i><sub>CF</sub> = 31.3/16.9 Hz), 136.5 (td, <i>J</i><sub>CF</sub> = 7.8/4.1 Hz), 136.4, 136.2–136.1 (m), 129.1, 128.2, 127.5, 126.8, 126.3, 111.8–110.3 (m), 53.5, 41.7; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>22</sub>H<sub>18</sub>F<sub>3</sub>N<sub>2</sub>O<sub>3</sub> [MH]<sup>+</sup> calcd 415.1264, found 415.1264.</div></div><div id="sec4_1_4_77" class="NLM_sec NLM_sec_level_4"><div id="ac_i101" class="anchor-spacer"></div><h5 class="article-section__title" id="_i101"> 2-(2-(3-Fluorophenyl)acetamido)-<i>N</i>-hydroxy-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetamide (<b>6ap</b>)</h5><div class="NLM_p last">Compound <b>5ap</b> (198 mg, 0.459 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. The crude product was purified by column chromatography (DCM/MeOH 100:0–90:10) followed by preparative HPLC to afford compound <b>6ap</b> as a white solid (101 mg, 51%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.07 (s, 1H), 9.07 (s, 1H), 8.99 (d, <i>J</i> = 8.2 Hz, 1H), 7.84–7.61 (m, 4H), 7.51 (d, <i>J</i> = 8.2 Hz, 2H), 7.32 (dd, <i>J</i> = 14.5/7.5 Hz, 1H), 7.18–6.90 (m, 3H), 5.43 (d, <i>J</i> = 8.2 Hz, 1H), 3.63 (s, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −113.8, −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.6/21.6 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 169.4, 166.4, 162.0 (d, <i>J</i><sub>CF</sub> = 243.0 Hz), 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.7/9.7/4.1 Hz), 139.2 (d, <i>J</i> = 4.3 Hz), 139.1, 138.3 (dt, <i>J</i><sub>CF</sub> = 249.5/15.7 Hz), 136.5 (td, <i>J</i><sub>CF</sub> = 8.1/4.4 Hz), 136.3 (m), 130.0 (d, <i>J</i><sub>CF</sub> = 8.4 Hz), 127.6, 126.8, 125.2 (d, <i>J</i><sub>CF</sub> = 2.5 Hz), 115.8 (d, <i>J</i><sub>CF</sub> = 21.3 Hz), 113.2 (d, <i>J</i><sub>CF</sub> = 20.8 Hz), 111.2 (m), 53.6, 41.3; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>22</sub>H<sub>17</sub>F<sub>4</sub>N<sub>2</sub>O<sub>3</sub> [MH]<sup>+</sup> calcd 433.1170, found 433.1188.</div></div><div id="sec4_1_4_78" class="NLM_sec NLM_sec_level_4"><div id="ac_i102" class="anchor-spacer"></div><h5 class="article-section__title" id="_i102"> 2-(2-(4-Fluorophenyl)acetamido)-<i>N</i>-hydroxy-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetamide (<b>6aq</b>)</h5><div class="NLM_p last">Compound <b>5aq</b> (47 mg, 0.109 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. The crude product was purified by column chromatography (DCM/MeOH 100:0–90:10) followed by preparative HPLC to afford compound <b>6aq</b> as a white solid (40.6 mg, 38%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.05 (s, 1H), 9.06 (s, 1H), 8.94 (d, <i>J</i> = 8.3 Hz, 1H), 7.82–7.61 (m, 4H), 7.50 (d, <i>J</i> = 8.2 Hz, 2H), 7.31 (m, 2H), 7.10 (t, <i>J</i> = 8.8 Hz, 2H), 5.42 (d, <i>J</i> = 8.3 Hz, 1H), 3.58 (s, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −116.78 (s), −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 169.8, 166.4, 161.0 (d, <i>J</i><sub>CF</sub> = 241.9 Hz), 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.7/9.8/4.2 Hz), 139.2, 138.3 (dt, <i>J</i><sub>CF</sub> = 249.5/15.6 Hz), 136.5 (td, <i>J</i><sub>CF</sub> = 8.0/4.4 Hz), 136.3–136.1 (m), 132.5 (d, <i>J</i><sub>CF</sub> = 3.0 Hz), 130.9 (d, <i>J</i><sub>CF</sub> = 8.0 Hz), 127.5, 126.8, 114.9 (d, <i>J</i> = 21.1 Hz), 111.5–110.7 (m), 53.5, 40.8; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>22</sub>H<sub>17</sub>F<sub>4</sub>N<sub>2</sub>O<sub>3</sub> [MH]<sup>+</sup> calcd 433.1170, found 433.1173.</div></div><div id="sec4_1_4_79" class="NLM_sec NLM_sec_level_4"><div id="ac_i103" class="anchor-spacer"></div><h5 class="article-section__title" id="_i103"> <i>N</i>-Hydroxy-2-(2-(4-methoxyphenyl)acetamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetamide (<b>6ar</b>)</h5><div class="NLM_p last">Compound <b>5ar</b> (195 mg, 0.440 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. The crude product was purified by column chromatography (DCM/MeOH 100:0–90:10) to afford compound <b>6ar</b> as a white solid (124 mg, 64%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.03 (s, 1H), 9.04 (s, 1H), 8.83 (d, <i>J</i> = 8.4 Hz, 1H), 7.95–7.61 (m, 4H), 7.49 (d, <i>J</i> = 8.3 Hz, 2H), 7.19 (d, <i>J</i> = 8.7 Hz, 2H), 6.84 (d, <i>J</i> = 8.7 Hz, 2H), 5.41 (d, <i>J</i> = 8.3 Hz, 1H), 3.71 (s, 3H), 3.50 (s, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 170.2, 166.4, 157.9, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.6/9.7/4.1 Hz), 139.3, 138.3 (dt, <i>J</i><sub>CF</sub> = 249.2/15.6 Hz), 136.5 (td, <i>J</i><sub>CF</sub> = 8.1/4.4 Hz), 136.2–136.0 (m), 130.0, 128.3, 127.5, 126.8, 113.6, 111.2 (m), 55.0, 53.4, 40.9; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>23</sub>H<sub>20</sub>F<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 445.1370, found 445.1362.</div></div><div id="sec4_1_4_80" class="NLM_sec NLM_sec_level_4"><div id="ac_i104" class="anchor-spacer"></div><h5 class="article-section__title" id="_i104"> <i>N</i>-Hydroxy-2-(2-(4-hydroxyphenyl)acetamido)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetamide (<b>6as</b>)</h5><div class="NLM_p last">Compound <b>6ar</b> (184 mg, 0.414 mmol) was treated with BBr<sub>3</sub> (1 M in DCM, 2.07 mL, 2.07 mmol) according to General Procedure D. The crude product was purified by preparative HPLC to afford compound <b>6as</b> as a white solid (58 mg, 32%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.05 (s, 1H), 9.21 (s, 1H), 9.05 (s, 1H), 8.78 (d, <i>J</i> = 8.4 Hz, 1H), 7.75–7.63 (m, 4H), 7.50 (d, <i>J</i> = 8.3 Hz, 2H), 7.07 (d, <i>J</i> = 8.5 Hz, 2H), 6.68 (d, <i>J</i> = 8.5 Hz, 2H), 5.42 (d, <i>J</i> = 8.3 Hz, 1H), 3.45 (s, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 170.5, 166.4, 155.9, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.6/9.7/4.2 Hz), 139.4, 138.3 (dt, <i>J</i><sub>CF</sub> = 249.3/15.7 Hz), 136.6 (td, <i>J</i><sub>CF</sub> = 8.1/4.4 Hz), 136.3–136.0 (m), 130.0, 127.5, 126.8, 126.5, 115.0, 111.6–110.9 (m), 53.5, 41.0; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>22</sub>H<sub>18</sub>F<sub>3</sub>N<sub>2</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 431.1213, found 431.1214.</div></div><div id="sec4_1_4_81" class="NLM_sec NLM_sec_level_4"><div id="ac_i105" class="anchor-spacer"></div><h5 class="article-section__title" id="_i105"> 4-(2-((2-(Hydroxyamino)-2-oxo-1-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)ethyl)amino)-2-oxoethyl)benzamide (<b>6at</b>)</h5><div class="NLM_p last">Compound <b>5at</b> (198 mg, 0.434 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. The crude product was purified by column chromatography followed by preparative HPLC to afford compound <b>6at</b> as a white solid (33.5 mg, 17%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.05 (s, 1H), 9.05 (s, 1H), 8.99 (d, <i>J</i> = 8.3 Hz, 1H), 7.90 (br s, 1H), 7.78 (d, <i>J</i> = 8.2 Hz, 2H), 7.74–7.64 (m, 4H), 7.50 (d, <i>J</i> = 8.3 Hz, 2H), 7.33 (d, <i>J</i> = 8.2 Hz, 2H), 7.29 (br s, 1H), 5.41 (d, <i>J</i> = 8.3 Hz, 1H), 3.64 (s, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 169.5, 167.8, 166.3, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.3/9.4/4.6 Hz), 139.7, 139.9–139.0 (m), 139.2, 136.5 (td, <i>J</i><sub>CF</sub> = 5.3/2.2 Hz), 136.3–136.1 (m), 132.5, 128.9, 127.5, 127.4, 126.8, 111.4–111.0 (m), 53.5, 41.5; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>23</sub>H<sub>19</sub>F<sub>3</sub>N<sub>3</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 458.1322, found 458.1320.</div></div><div id="sec4_1_4_82" class="NLM_sec NLM_sec_level_4"><div id="ac_i106" class="anchor-spacer"></div><h5 class="article-section__title" id="_i106"> 2-(2-(3,4-Dimethoxyphenyl)acetamido)-<i>N</i>-hydroxy-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetamide (<b>6au</b>)</h5><div class="NLM_p last">Compound <b>5au</b> (413 mg, 0.872 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. The crude product was purified by column chromatography (DCM/MeOH 100:0–90:0) to afford compound <b>6au</b> as a white solid (100 mg, 24%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.04 (s, 1H), 9.04 (s, 1H), 8.84 (d, <i>J</i> = 8.4 Hz, 1H), 7.75–7.63 (m, 4H), 7.50 (d, <i>J</i> = 8.3 Hz, 2H), 6.92 (d, <i>J</i> = 1.9 Hz, 1H), 6.85 (d, <i>J</i> = 8.2 Hz, 1H), 6.78 (dd, <i>J</i> = 8.2/1.9 Hz, 1H), 5.42 (d, <i>J</i> = 8.4 Hz, 1H), 3.72 (s, 3H), 3.71 (s, 3H), 3.49 (s, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 170.1, 166.4, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.5,/9.7/4.2 Hz), 148.5, 147.5, 139.3, 138.3 (dt, <i>J</i><sub>CF</sub> = 31.0/15.7 Hz), 136.5 (td, <i>J</i><sub>CF</sub> = 8.0/4.3 Hz), 136.2–136.1 (m), 128.8, 127.5, 126.8, 121.0, 112.9, 111.8, 111.5–110.5 (m), 55.5, 55.4, 53.4, 41.4; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>24</sub>H<sub>22</sub>F<sub>3</sub>N<sub>2</sub>O<sub>5</sub> [MH]<sup>+</sup> calcd 475.1475, found 475.1477.</div></div><div id="sec4_1_4_83" class="NLM_sec NLM_sec_level_4"><div id="ac_i107" class="anchor-spacer"></div><h5 class="article-section__title" id="_i107"> 2-(2-(3,4-Dihydroxyphenyl)acetamido)-<i>N</i>-hydroxy-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetamide (<b>6av</b>)</h5><div class="NLM_p last">Compound <b>6au</b> (213 mg, 0.449 mmol) was treated with BBr<sub>3</sub> (1 M in DCM, 4.49 mL, 4.49 mmol) according to General Procedure D to form the corresponding catechol. The crude product was purified by preparative HPLC to afford compound <b>6av</b> as a white solid (97.9 mg, 49%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.05 (s, 1H), 9.05 (br s, 1H), 8.85–8.53 (m, 3H), 7.76–7.62 (m, 4H), 7.49 (d, <i>J</i> = 8.3 Hz, 2H), 6.69 (d, <i>J</i> = 2.0 Hz, 1H), 6.64 (d, <i>J</i> = 8.0 Hz, 1H), 6.53 (dd, <i>J</i> = 8.0/2.0 Hz, 1H), 5.41 (d, <i>J</i> = 8.3 Hz, 1H), 3.39 (s, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 170.5, 166.4, 150.7 (ddd, <i>J</i><sub>CF</sub> = 246.5/9.6/4.1 Hz), 145.0, 143.9, 139.4, 138.3 (dt, <i>J</i><sub>CF</sub> = 236.0/15.9 Hz), 136.6 (td, <i>J</i><sub>CF</sub> = 8.2/4.4 Hz), 136.3–136.2 (m), 127.5, 127.1, 126.8, 119.9, 116.6, 115.4, 112.6–109.0 (m), 53.4, 41.3; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>22</sub>H<sub>18</sub>F<sub>3</sub>N<sub>2</sub>O<sub>5</sub> [MH]<sup>+</sup> calcd 447.1162, found 444.1162.</div></div><div id="sec4_1_4_84" class="NLM_sec NLM_sec_level_4"><div id="ac_i108" class="anchor-spacer"></div><h5 class="article-section__title" id="_i108"> 2-(2-(3-Fluoro-4-hydroxyphenyl)acetamido)-<i>N</i>-hydroxy-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetamide (<b>6ax</b>)</h5><div class="NLM_p last">Compound <b>5aw</b> (104 mg, 0.225 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C, affording 67.2 mg of 2-(2-(3-fluoro-4-methoxyphenyl)acetamido)-<i>N</i>-hydroxy-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetamide (<b>6aw</b>). This was treated with BBr<sub>3</sub> (1 M in DCM, 0.724 μL, 0.724 μmol) according to General Procedure D. The resulting precipitate was filtered to afford compound <b>6ax</b> as an off-white solid (46.5 mg, 72%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) 11.07 (s, 1H), 9.71 (br s, 1H), 8.84 (d, <i>J</i> = 8.2 Hz, 1H), 7.79–7.60 (m, 4H), 7.49 (d, <i>J</i> = 8.3 Hz, 2H), 7.11–6.98 (m, 1H), 6.93–6.74 (m, 2H), 5.40 (d, <i>J</i> = 8.2 Hz, 1H), 3.47 (s, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.8 (d, <i>J</i> = 21.7 Hz), −136.8, −163.5 (dd, <i>J</i> = 21.6/21.6 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 170.2, 166.5, 150.8 (d, <i>J</i> = 240.0 Hz), 150.7 (ddd, <i>J</i> = 12.7/9.6/4.2 Hz), 143.4 (d, <i>J</i> = 12.1 Hz), 139.3, 138.4 (dt, <i>J</i> = 249.1/15.4 Hz), 136.6 (td, <i>J</i> = 8.2/4.5 Hz), 136.4–136.3 (m), 127.7, 127.6, 126.9, 125.2 (d, <i>J</i> = 3.0 Hz), 117.5 (d, <i>J</i> = 3.0 Hz), 116.7 (d, <i>J</i> = 18.4 Hz), 111.3 (m), 53.6, 40.7; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>22</sub>H<sub>17</sub>F<sub>4</sub>N<sub>2</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 449.1117, found 449.1125.</div></div><div id="sec4_1_4_85" class="NLM_sec NLM_sec_level_4"><div id="ac_i109" class="anchor-spacer"></div><h5 class="article-section__title" id="_i109"> 2-(2-(4-Fluoro-3-methoxyphenyl)acetamido)-<i>N</i>-hydroxy-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetamide (<b>6ay</b>)</h5><div class="NLM_p last">Compound <b>5ay</b> (367 mg, 0.795 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. The crude product was purified by column chromatography (DCM/MeOH 100:0–90:0) to afford the desired hydroxamic acid analogue as a white solid (<b>6ay</b>) (226 mg, 61%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.05 (s, 1H), 9.05 (s, 1H), 8.95 (d, <i>J</i> = 8.3 Hz, 1H), 7.81–7.63 (m, 4H), 7.50 (d, <i>J</i> = 8.3 Hz, 2H), 7.19–7.01 (m, 2H), 6.81 (ddd, <i>J</i> = 8.2/4.3/2.0 Hz, 1H), 5.41 (d, <i>J</i> = 8.3 Hz, 1H), 3.80 (s, 3H), 3.55 (s, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −138.9 (s), −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 169.7, 166.3, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.7/9.8/4.3 Hz), 150.4 (d, <i>J</i><sub>CF</sub> = 242.0 Hz), 146.7 (d, <i>J</i><sub>CF</sub> = 10.6 Hz), 139.3, 138.3 (dt, <i>J</i><sub>CF</sub> = 31.4/15.9 Hz), 136.5 (td, <i>J</i><sub>CF</sub> = 8.1/4.4 Hz), 136.3–136.1 (m), 133.1 (d, <i>J</i><sub>CF</sub> = 3.7 Hz), 127.5, 126.8, 121.2 (d, <i>J</i><sub>CF</sub> = 6.7 Hz), 115.4 (d, <i>J</i><sub>CF</sub> = 17.9 Hz), 114.6 (d, <i>J</i><sub>CF</sub> = 1.5 Hz), 111.7–110.4 (m), 55.8, 53.5, 41.3; <i>m</i>/<i>z</i> MS C<sub>23</sub>H<sub>19</sub>F<sub>4</sub>N<sub>2</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 463.1, found 462.8.</div></div><div id="sec4_1_4_86" class="NLM_sec NLM_sec_level_4"><div id="ac_i110" class="anchor-spacer"></div><h5 class="article-section__title" id="_i110"> 2-(2-(4-Fluoro-3-hydroxyphenyl)acetamido)-<i>N</i>-hydroxy-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetamide (<b>6az</b>)</h5><div class="NLM_p last">Compound <b>6ay</b> (205 mg, 0.443 mmol) was treated with BBr<sub>3</sub> (1 M in DCM, 2.22 mL, 2.22 mmol) according to General Procedure D to form the corresponding phenol. The crude product was purified by preparative HPLC to afford compound <b>6az</b> as a white solid (156 mg, 78%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.05 (s, 1H), 9.73 (s, 1H), 9.06 (s, 1H), 8.89 (d, <i>J</i> = 8.3 Hz, 1H), 7.76–7.63 (m, 4H), 7.49 (d, <i>J</i> = 8.3 Hz, 2H), 7.07–6.96 (m, 1H), 6.88 (dd, <i>J</i> = 8.7/2.1 Hz, 1H), 6.67 (ddd, <i>J</i> = 8.2/4.3/2.1 Hz, 1H), 5.40 (d, <i>J</i> = 8.3 Hz, 1H), 3.48 (s, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.9 (d, <i>J</i> = 21.7 Hz), −139.8, −163.5 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 169.9, 166.4, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.5/9.7/4.1 Hz), 150.0 (d, <i>J</i><sub>CF</sub> = 239.0 Hz), 144.4 (d, <i>J</i><sub>CF</sub> = 12.2 Hz), 139.3, 138.3 (dt, <i>J</i><sub>CF</sub> = 249.1/15.5 Hz), 136.6 (td, <i>J</i><sub>CF</sub> = 8.0/4.1 Hz), 136.3–136.0 (m), 132.7 (d, <i>J</i><sub>CF</sub> = 3.4 Hz), 127.5, 126.8, 119.9 (d, <i>J</i><sub>CF</sub> = 6.4 Hz), 118.5 (d, <i>J</i><sub>CF</sub> = 2.7 Hz), 115.6 (d, <i>J</i><sub>CF</sub> = 18.0 Hz), 111.8–110.8 (m), 53.5, 41.0; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>22</sub>H<sub>17</sub>F<sub>4</sub>N<sub>2</sub>O<sub>4</sub> [MH]<sup>+</sup> calcd 449.1117, found 449.1113.</div></div><div id="sec4_1_4_87" class="NLM_sec NLM_sec_level_4"><div id="ac_i111" class="anchor-spacer"></div><h5 class="article-section__title" id="_i111"> Methyl 2-((4-Fluoro-3-methoxybenzyl)amino)-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetate (<b>7</b>)</h5><div class="NLM_p last">4-Fluoro-3-methoxybenzaldehyde (172 mg, 1.12 mmol) was added to a solution of compound <b>4</b> (300 mg, 1.02 mmol) in DCE (10 mL) and stirred for 30 min before adding Na(OAc)<sub>3</sub>BH (540 mg, 2.55 mmol). The reaction mixture was stirred at rt overnight. Upon completion, the reaction mixture was diluted with sat. NaHCO<sub>3</sub> (20 mL) and extracted with DCM (3 × 20 mL). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo. The crude product was purified by column chromatography (PE/EtOAc 100:0–50:50) to afford compound <b>7</b> as a clear oil (405 mg, 92%). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ 7.53–7.41 (m, 4H), 7.22–7.13 (m, 2H), 7.01–6.93 (m, 2H), 6.83–6.76 (m, 1H), 4.41 (s, 1H), 3.87 (s, 3H), 3.72 (s, 3H), 3.71 (s, 2H), 2.42 (br s, 1H); <sup>19</sup>F NMR (CDCl<sub>3</sub>) δ −134.0 (d, <i>J</i> = 20.6 Hz), −137.4, −162.3 (dd, <i>J</i> = 20.5/20.5 Hz); <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ 173.2, 151.7 (d, <i>J</i><sub>CF</sub> = 244.9 Hz), 151.5 (ddd, <i>J</i><sub>CF</sub> = 249.7/10.1/4.3 Hz), 147.6 (d, <i>J</i><sub>CF</sub> = 10.8 Hz), 140.9–137.9 (m), 138.3, 138.1–138.0 (m), 136.7 (td, <i>J</i><sub>CF</sub> = 7.8/4.7 Hz), 135.6 (d, <i>J</i><sub>CF</sub> = 3.7 Hz), 128.3, 127.3, 120.5 (d, <i>J</i><sub>CF</sub> = 6.8 Hz), 115.8 (d, <i>J</i><sub>CF</sub> = 18.3 Hz), 113.4 (d, <i>J</i><sub>CF</sub> = 1.9 Hz), 111.4–110.6 (m), 63.9, 56.2, 52.5, 51.1; <i>m</i>/<i>z</i> MS C<sub>23</sub>H<sub>20</sub>F<sub>4</sub>NO<sub>3</sub> [MH]<sup>+</sup> calcd 434.1, found 433.9.</div></div><div id="sec4_1_4_88" class="NLM_sec NLM_sec_level_4"><div id="ac_i112" class="anchor-spacer"></div><h5 class="article-section__title" id="_i112"> 2-((4-Fluoro-3-methoxybenzyl)amino)-<i>N</i>-hydroxy-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetamide (<b>8</b>)</h5><div class="NLM_p last">Compound <b>7</b> (365 mg, 0.842 mmol) was converted to the corresponding hydroxamic acid according to General Procedure C. The crude product was purified by column chromatography (DCM/MeOH 100:0–90:0) to afford compound <b>8</b> as a white solid (242 mg, 66%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 10.80 (s, 1H), 8.95 (s, 1H), 7.77–7.61 (m, 4H), 7.51 (d, <i>J</i> = 8.3 Hz, 2H), 7.18–7.06 (m, 2H), 6.85 (ddd, <i>J</i> = 8.1/4.4/1.8 Hz, 1H), 4.09 (s, 1H), 3.82 (s, 3H), 3.61 (s, 2H), 3.10 (br s, 1H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −135.0 (d, <i>J</i> = 21.7 Hz), −138.4, −163.7 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 168.3, 150.6 (ddd, <i>J</i><sub>CF</sub> = 246.5/9.7/4.2 Hz), 150.5 (d, <i>J</i><sub>CF</sub> = 242.0 Hz), 146.9 (d, <i>J</i><sub>CF</sub> = 10.6 Hz), 140.3, 139.8–136.9 (m), 137.0 (d, <i>J</i><sub>CF</sub> = 3.5 Hz), 136.7 (td, <i>J</i><sub>CF</sub> = 8.1/4.3 Hz), 136.1–135.9 (m), 128.0, 126.6, 119.9 (d, <i>J</i><sub>CF</sub> = 6.7 Hz), 115.3 (d, <i>J</i><sub>CF</sub> = 17.8 Hz), 113.3 (d, <i>J</i><sub>CF</sub> = 1.4 Hz), 111.9–110.6 (m), 61.8, 55.8, 49.9; <i>m</i>/<i>z</i> MS C<sub>22</sub>H<sub>19</sub>F<sub>4</sub>N<sub>2</sub>O<sub>3</sub> [MH]<sup>+</sup> calcd 435.1, found 434.9</div></div><div id="sec4_1_4_89" class="NLM_sec NLM_sec_level_4"><div id="ac_i113" class="anchor-spacer"></div><h5 class="article-section__title" id="_i113"> 2-((4-Fluoro-3-hydroxybenzyl)amino)-<i>N</i>-hydroxy-2-(3′,4′,5′-trifluoro-[1,1′-biphenyl]-4-yl)acetamide HCl Salt (<b>9</b>)</h5><div class="NLM_p last">Compound <b>8</b> (200 mg, 0.460 mmol) was treated with BBr<sub>3</sub> (1 M in DCM, 2.30 mL, 2.30 mmol) according to General Procedure D. Upon completion, the reaction was quenched with a 1 M HCl solution (3 mL) and stirred vigorously for 10 min. The mixture was neutralized to pH 7 with sat. NaHCO<sub>3</sub> and extracted with EtOAc (3 × 10 mL). The combined organic layers were dried over anhydrous Na<sub>2</sub>SO<sub>3</sub>, filtered, and concentrated in vacuo. To convert the neutralized product to the corresponding HCl salt, the crude product (126 mg, 0.300 mmol) was dissolved in MeOH (8 mL) before addition of 1 M HCl (600 μL, 0.600 mmol) and stirred overnight at rt. The reaction mixture was concentrated in vacuo and lyophilized to remove excess HCl to afford compound <b>9</b> as a white solid (125 mg, 27%). <sup>1</sup>H NMR (<i>d</i><sub>6</sub>-DMSO) δ 11.55 (s, 1H), 10.21 (s, 1H), 10.13 (br s, 2H), 9.40 (br s, 1H), 7.84 (d, <i>J</i> = 8.4 Hz, 2H), 7.81–7.73 (m, 2H), 7.70 (d, <i>J</i> = 8.4 Hz, 2H), 7.23–7.09 (m, 2H), 6.90 (ddd, <i>J</i> = 8.1/4.0/2.1 Hz, 1H), 4.84 (s, 1H), 3.93 (q, <i>J</i> = 12.6 Hz, 2H); <sup>19</sup>F NMR (<i>d</i><sub>6</sub>-DMSO) δ −134.7 (d, <i>J</i> = 21.7 Hz), −135.7, −162.8 (dd, <i>J</i> = 21.7/21.7 Hz); <sup>13</sup>C NMR (<i>d</i><sub>6</sub>-DMSO) δ 162.8, 151.5 (d, <i>J</i><sub>CF</sub> = 242.8 Hz), 150.7 (ddd, <i>J</i><sub>CF</sub> = 246.7/9.7/4.2 Hz), 144.9 (d, <i>J</i> = 12.3 Hz), 138.6 (dt, <i>J</i><sub>CF</sub> = 249.0/15.5 Hz), 137.89–137.7 (m), 135.9 (dt, <i>J</i><sub>CF</sub> = 8.4/4.0 Hz), 132.4, 129.1, 127.2, 121.7 (d, <i>J</i><sub>CF</sub> = 6.6 Hz), 120.2 (d, <i>J</i><sub>CF</sub> = 2.9 Hz), 116.1 (d, <i>J</i><sub>CF</sub> = 18.5 Hz), 111.8–110.9 (m), 59.3, 48.5; <i>m</i>/<i>z</i> HRMS (TOF ES<sup>+</sup>) C<sub>21</sub>H<sub>17</sub>F<sub>4</sub>N<sub>2</sub>O<sub>3</sub> [MH]<sup>+</sup> calcd 421.1170, found 421.1175.</div></div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i114" class="anchor-spacer"></div><h3 class="article-section__title" id="_i114"> Biology</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i115" class="anchor-spacer"></div><h4 class="article-section__title" id="_i115"> Protein Expression and Purification</h4><div class="NLM_p last">A soluble form of human APN ectodomain was expressed and purified as reported previously.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> In brief, human APN was expressed in a stably transfected HEK293S GnT1̅ cell line, which was a kind gift from Prof. James Rini from the University of Toronto, Canada. The cell growth and passaging of the cells, as well as collection of the culture supernatant, were conducted by Monash Protein Production Unit. Cells were grown in Dulbecco’s modified Eagle’s medium (DMEM)/F-12 supplemented with 3% fetal bovine serum (FBS) Invitrogen, 1× penicillin–streptomycin (Invitrogen), 1 mg/L doxycycline (Sigma), and 1 mg/L aprotinin (Bioshop Canada). The APN–protein A fusion protein was purified by IgG-Sepharose (GE Healthcare) affinity chromatography. The protein A tag was removed by on-column tobacco etch virus protease digestion and the liberated APN was further purified by size exclusion chromatography on a Superdex S200 10/300 column in 50 mM <i>N</i>-(2-hydroxyethyl)piperazine-<i>N</i>′-ethanesulfonic acid pH 8.0, 300 mM NaCl, and 5% glycerol buffer. Biochemical parameters <i>K</i><sub>m</sub> (31 ± 5 μM) and <i>k</i><sub>cat</sub> (456 ± 14 FU/s) were determined in the presence of <span class="smallcaps smallerCapital">l</span>-leucine-7-amido-4-methylcoumarin hydrochloride (H-Leu-NHMec) (Sigma L2145).</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i116" class="anchor-spacer"></div><h4 class="article-section__title" id="_i116"> APN Enzymatic Analysis</h4><div class="NLM_p last">Aminopeptidase assays were based on the modified report of  Drinkwater et al.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> The activity of APN was determined by measuring the release of the fluorogenic leaving group, NH<sub>2</sub>Mec, from the fluorogenic peptide H-Leu-NHMec (Sigma L2145). The reactions were carried out in 384-well microtiter plates, 50 μL total volume at 37 °C using a spectrofluorimeter (BMG FLUOstar) with excitation at 355 nm and emission at 460 nm. APN was preincubated in 100 mM Tris pH 8.0 at 37 °C with the inhibitors for 10 min prior to the addition of substrate (25 μM). Inhibitor concentrations were assayed with highest working concentrations of 2–320 μM, diluted 1:4 to assess an overall 1000-fold concentration series. Initially, assays were performed in experimental triplicate, and if a <i>K</i><sub>i</sub><sup>(app)</sup> value of ≤100 nM was calculated, the inhibitor was considered of high priority, and assessed further in biological triplicate. The fluorescence signal was monitored for 1 h at 37 °C. Only a linear range of velocity was considered in data analysis. <i>K</i><sub>i</sub><sup>(app)</sup> values were then calculated by using the Morrison equation for competitive and tight-binding inhibitors.<a onclick="showRef(event, 'ref65 ref66'); return false;" href="javascript:void(0);" class="ref ref65 ref66">(64,65)</a> Analysis and graphical output were performed in GraphPad Prism 7.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i117" class="anchor-spacer"></div><h4 class="article-section__title" id="_i117"> Ad293 Cellular Viability Assay</h4><div class="NLM_p last">Compounds were prepared as a 0.1 M stock solution in DMSO. A total of 5000 cells/well were plated out in a 96-well sterile clear TC plates in 100 μL of DMEM + 10% FBS + 1% Penstrep (Invitrogen). The plates were incubated at 37 °C and 5% CO<sub>2</sub> overnight. Media was aspirated off followed by washing with 100 μL of PBS pH 7.4 (Invitrogen). In a separate plate, 240 μL of each compound was prepared at the working concentration by diluting the stock solutions with the media. This was serially diluted in 1:2 ratios across well plate. From the dilution plate, 100 μL of each compound dose was dispensed into the cell-seeded plate. Final concentration ranges of 0–800 μM (bestatin) and 0–400 μM (compound <b>6ad</b> and <b>6ae</b>) were achieved. After addition of the compound, the plate was incubated for an additional 72 h. At the 72 h mark, 10 μL of CellTiter-Blue (Promega) was added to the cells followed by an additional 4 h of incubation. Fluorescence readings were obtained on EnVision (PerkinElmer) at 565/595 nm. The results were expressed as % proliferation relative to a negative DMSO control. CC<sub>50</sub> values were calculated from dose–response curves (log[compound concentration] vs % proliferation) using GraphPad Prism 7.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i118" class="anchor-spacer"></div><h4 class="article-section__title" id="_i118"> Estimation of Kinetic Solubility Using Nephelometry</h4><div class="NLM_p last">Compounds in DMSO was spiked into either pH 6.5 phosphate buffer or 0.01 M HCl (pH ∼ 2.0) with the final DMSO concentration being 1%. After 30 min had elapsed, the samples were then analyzed via Nephelometry to determine a solubility range.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(66)</a></div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i119" class="anchor-spacer"></div><h4 class="article-section__title" id="_i119"> Estimation of Distribution Coefficient Using Chromatography</h4><div class="NLM_p last">Partition coefficient values (Log <i>D</i>) of the test compounds were estimated at pH 7.4 by correlation of their chromatographic retention properties against the characteristics of a series of standard compounds with known partition coefficient values. The method employed is a gradient HPLC-based derivation of the method developed by Lombardo et al.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(67)</a></div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i120" class="anchor-spacer"></div><h4 class="article-section__title" id="_i120"> In Vitro Metabolic Stability</h4><div class="NLM_p last">The metabolic stability assay was performed by incubating each compound in human or mouse liver microsomes at 37 °C and a protein concentration of 0.4 mg/mL. The metabolic reaction was initiated by the addition of a reduced nicotinamide adenine dinucleotide phosphate (NADPH)-regenerating system and quenched at various time points over a 60 min incubation period by the addition of acetonitrile containing diazepam as internal standard. Control samples (containing no NADPH) were included (and quenched at 2, 30, and 60 min) to monitor for potential degradation in the absence of cofactor. The human liver microsomes used in this experiment were supplied by XenoTech, lot # 1410230. The mouse liver microsomes used in this experiment were supplied by XenoTech, lot # 1510256. Microsomal incubations were performed at a substrate concentration of 1 μM.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i121" class="anchor-spacer"></div><h4 class="article-section__title" id="_i121"> In Vitro Plasma Stability</h4><div class="NLM_p last">Human plasma (pooled; <i>n</i> = 3 donors) was separated from whole blood procured from the Victorian Blood Donor Registry. Mouse plasma (pooled; multiple mice) from male Swiss outbred mice was collected in-house. Aliquots of plasma were spiked with DMSO/MeCN/H<sub>2</sub>O solutions of test compound to a nominal compound concentration of 1000 ng/mL. The maximum final DMSO and acetonitrile concentrations were 0.2% (v/v) and 0.4% (v/v), respectively. Immediately after compound spiking, plasma was vortex-mixed and aliquots of spiked plasma (50 μL) were transferred into fresh microcentrifuge tubes and maintained at 37 °C under 10% CO<sub>2</sub> conditions. At various time points over the 240 min incubation period, triplicate plasma samples were taken and immediately snap-frozen in dry ice. All samples were stored frozen at −80 °C until analysis by LC–MS. Plasma samples were quantified relative to calibration standards prepared using blank plasma of the same species. Calibration standards were spiked with test compound over the range of 0.5–10 000 ng/mL. Internal standard (diazepam) was added to calibration standards and incubation samples and then immediately quenched using two volumes of MeCN to precipitate plasma proteins. Samples were vortex-mixed and centrifuged (10 000 rpm for 3 min) in a microcentrifuge and the supernatant analyzed by LC–MS (Water Micromass Quattro Premier coupled to a Waters Acquity UPLC). The bioanalytical method was validated with respect to calibration range, linearity, accuracy, and precision. The mean and standard deviation of measured plasma concentrations were calculated for each time point and expressed as a percentage remaining relative to the initial time point (5 min).</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i122" class="anchor-spacer"></div><h3 class="article-section__title" id="_i122"> Computational Chemistry</h3><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i123" class="anchor-spacer"></div><h4 class="article-section__title" id="_i123"> Molecular Modeling</h4><div class="NLM_p last">Molecular docking of compounds <b>1</b>, <b>6ad</b>, <b>6ae</b>, and <b>6ag</b> were carried out using Surflex docking interfaced with SYBYL2.1.1 in SFXC mode. The domain II of reference <i>Pf</i>A-M1 structure (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZX4">4ZX4</a>, residues 392–649) and APN structure (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYQ">4FYQ</a>, residues 287–546) were chosen for alignment due to the fact this domain incorporates the active site of interest and also shares the highest structural and sequence similarity across M1 aminopeptidase superfamily. A superimposed structural alignment was then performed using Pymol 1.8.23<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(68)</a> with domain II of both <i>Pf</i>A-M1 and APN. The coordinates of compound <b>1</b> in the superimposed APN was subsequently extracted for later use of fragment constraints. The bond constants and charge distribution for <b>1</b> were derived using the GAFF<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(69)</a> and AM1-BCC models,<a onclick="showRef(event, 'ref71 ref72'); return false;" href="javascript:void(0);" class="ref ref71 ref72">(70,71)</a> respectively. To prepare the crystal structures, water molecules were removed and any missing regions were modeled using the program Modeller v9.10.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(72)</a> The flexible loop <sup>891</sup>YGGGSFSF<sup>898</sup> was not resolved in the unbound structure of APN (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYQ">4FYQ</a>); however, it was resolved in electron density when APN was bound to either small-molecule inhibitors such as bestatin (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYR">4FYR</a>) and amastatin (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYT">4FYT</a>) or a peptide substrate angiotensin IV (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYS">4FYS</a>).<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> We rebuilt the missing loop in <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYQ">4FYQ</a> using Modeller, and the structure was overlaid with the bound APN structures to examine if the flexible loop was modeled into a sensible position. Side-chain amides were protonated and fixed alongside charge addition using the Gasteiger–Marsili method.<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(73)</a> The charge for zinc metal was 2.0. The two-dimensional ligand structures were prepared with default settings from SYBYL2.1.1. In grid generation, the core biphenyl system and hydroxamic acid were set as fragment constraints with a constraint penalty of 10. A total of 20 poses were produced for each ligand that were similar in their conformations and interactions. The differences in the total docking score between the most preferred pose and the second best pose of <b>1</b>, <b>6ad</b>, <b>6ae</b>, and <b>6ag</b> were 0.02, 0.003, 0.127, and 0.046, respectively. The best pose according to the total docking scores was selected to perform MD simulations.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i124" class="anchor-spacer"></div><h4 class="article-section__title" id="_i124"> MD Simulations</h4><div class="NLM_p last">The APN–ligand complex models were solvated in a rectangular simulation box leaving at least 12 Å of water shell thickness at all sides of the protein with a periodic box of 117 Å × 125 Å × 120 Å. System charges were neutralized with sodium counterions.<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(74)</a> Proteins and ions were modeled using the AMBER force field FF14SB and waters represented using the three-particle TIP3P model.<a onclick="showRef(event, 'ref75 ref76'); return false;" href="javascript:void(0);" class="ref ref75 ref76">(74,75)</a> M1 aminopeptidase zinc and zinc-binding residues (His<sup>388</sup>, His<sup>392</sup>, and Glu<sup>411</sup>) were prepared as described previously.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> MD simulations were performed using NAMD 2.9 on an IBM Blue Gene/Q.<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(76)</a> Equilibration was performed in three stages. First, potential steric clashes in the initial configuration were relieved with 2000 steps of energy minimization. Initial velocities for each system were then assigned randomly according to a Maxwell–Boltzmann distribution at 100 K. Each system was then heated to 300 K over 0.1 ns, under the isothermal–isometric ensemble (NVT) conditions, with the protein atoms (excluding hydrogens) harmonically restrained [with a force constant of 10 kcal/(mol A<sup>2</sup>)]. Following this, each system was simulated for another 0.1 ns under the isothermal–isobaric ensemble (NPT) with applied harmonic restraints. For each system, we repeated the above process three times to initiate the production simulations with different initial velocities in NPT. Simulation time step was set to 2 fs and the SHAKE algorithm was used to constrain all bonds involving hydrogen atoms.<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(77)</a> All simulations were run at constant temperature (300 K) and pressure (1 atm), using a Langevin damping coefficient of 0.5 fs<sup>–1</sup> and a Berendsen thermostat relaxation time of τ<sub>P</sub> = 1.0 ps.<a onclick="showRef(event, 'ref79 ref80'); return false;" href="javascript:void(0);" class="ref ref79 ref80">(78,79)</a> For each simulated system, periodic boundary conditions were used together with the particle-mesh Ewald method for long-range electrostatic interactions, and a real space cutoff of 10 Å was used.<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(80)</a> To increase the efficiency of sampling, production MD simulations for APN were run in triplicate for 50 ns. The conformation was sampled every 5000 steps (one snapshot per 10 ps). We then used one snapshot per 100 ps to analyze the MD trajectories.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i125"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00757" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35765" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35765" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00757" class="ext-link">10.1021/acs.jmedchem.9b00757</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Synthetic procedures (compounds <b>10</b>–<b>16</b>); MMP enzymatic analysis; RMSD of the Cα and ligands of <b>1</b>, <b>6ad</b>, <b>6ae</b>, and <b>6ag</b> along the molecular modeling trajectory; hydrogen-bond occupancy analysis of <b>1</b>, <b>6ad</b>, <b>6ae</b>, and <b>6ag</b> with APN; binding scores for the docked poses of <b>1</b>, <b>6ad</b>, <b>6ae</b>, and <b>6ag</b> into APN; and HPLC traces of biologically evaluated compounds (<b>6a</b>–<b>6az</b> and <b>9</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">PDB file for compound <b>1</b> docking (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">MD simulations of APN <b>6ad</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_004.mp4" class="ext-link">MP4</a>)</p></li><li><p class="inline">MD simulations of APN <b>6ae</b> ( <a href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_005.mp4" class="ext-link">MP4</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_001.pdf">jm9b00757_si_001.pdf (4.92 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_002.csv">jm9b00757_si_002.csv (15.36 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_003.pdb">jm9b00757_si_003.pdb (822.47 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_004.mp4">jm9b00757_si_004.mp4 (44.15 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_005.mp4">jm9b00757_si_005.mp4 (44.14 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The atomic coordinates for the X-ray crystal structures used as templates for molecular docking are available from the Protein Data Bank (PDB). <i>Pf</i>A-M1-<b>1</b> PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZX4">4ZX4</a> and human APN PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYQ">4FYQ</a>.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b00757" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70107" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70107" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sheena McGowan</span> - <span class="hlFld-Affiliation affiliation">Infection
and Immunity Program, Biomedicine Discovery Institute and Department
of Microbiology, Monash University, Clayton Campus, Clayton, VIC 3800, Australia</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-6863-1106" title="Orcid link">http://orcid.org/0000-0001-6863-1106</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#46152e23232827680b250129312728062b292827352e68232233"><span class="__cf_email__" data-cfemail="40132825252e216e0d23072f37212e002d2f2e2133286e252435">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter J. Scammells</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Monash Institute of Pharmaceutical Sciences and , Monash University, Parkville Campus, Parkville, VIC 3052, Australia</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2930-895X" title="Orcid link">http://orcid.org/0000-0003-2930-895X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3262574657401c6151535f5f575e5e41725f5d5c53415a1c575647"><span class="__cf_email__" data-cfemail="83d3e6f7e6f1add0e0e2eeeee6efeff0c3eeecede2f0ebade6e7f6">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jisook Lee</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Monash Institute of Pharmaceutical Sciences and , Monash University, Parkville Campus, Parkville, VIC 3052, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Natalie B. Vinh</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Monash Institute of Pharmaceutical Sciences and , Monash University, Parkville Campus, Parkville, VIC 3052, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nyssa Drinkwater</span> - <span class="hlFld-Affiliation affiliation">Infection
and Immunity Program, Biomedicine Discovery Institute and Department
of Microbiology, Monash University, Clayton Campus, Clayton, VIC 3800, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wei Yang</span> - <span class="hlFld-Affiliation affiliation">Infection
and Immunity Program, Biomedicine Discovery Institute and Department
of Microbiology, Monash University, Clayton Campus, Clayton, VIC 3800, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Komagal Kannan Sivaraman</span> - <span class="hlFld-Affiliation affiliation">Infection
and Immunity Program, Biomedicine Discovery Institute and Department
of Microbiology, Monash University, Clayton Campus, Clayton, VIC 3800, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Luke S. Schembri</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Monash Institute of Pharmaceutical Sciences and , Monash University, Parkville Campus, Parkville, VIC 3052, Australia</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5588-1077" title="Orcid link">http://orcid.org/0000-0001-5588-1077</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michelle Gazdik</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Monash Institute of Pharmaceutical Sciences and , Monash University, Parkville Campus, Parkville, VIC 3052, Australia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter M. Grin</span> - <span class="hlFld-Affiliation affiliation">Department of Biochemistry
& Molecular Biology, 
    Centre for Blood Research, Life Sciences Institute, 
    , , and , University of British Columbia, Vancouver, British Columbia V6T 1Z3, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Georgina S. Butler</span> - <span class="hlFld-Affiliation affiliation">Department of Biochemistry
& Molecular Biology, 
    Department of Oral Biological &
Medical Sciences, 
    Centre for Blood Research, Life Sciences Institute, 
    , , and , University of British Columbia, Vancouver, British Columbia V6T 1Z3, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christopher M. Overall</span> - <span class="hlFld-Affiliation affiliation">Department of Biochemistry
& Molecular Biology, 
    Department of Oral Biological &
Medical Sciences, 
    Centre for Blood Research, Life Sciences Institute, 
    , , and , University of British Columbia, Vancouver, British Columbia V6T 1Z3, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5844-2731" title="Orcid link">http://orcid.org/0000-0001-5844-2731</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Susan A. Charman</span> - <span class="hlFld-Affiliation affiliation">Centre for Drug
Candidate Optimisation, Monash Institute of Pharmaceutical Sciences and , Monash University, Parkville Campus, Parkville, VIC 3052, Australia</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e13241-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i128">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31588" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31588" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the National Health and Medical Research Council (Project Grant 1063786 to S.M. and P.J.S.). The authors thank Prof. James Rini at the University of Toronto, Canada, for the generous donation of the cell line that expresses the recombinant human APN ectodomain. They also thank Miss Cassandra Yong for the kind donation of the Ad293 cell line and for training to perform the cell toxicity assay. They acknowledge the Monash Platforms (Protein Production and Crystallization) for technical assistance. J.L. was supported by Monash University, Sir James McNeil Foundation, and the Cancer Therapeutics CRC, CTx. W.Y. was supported by a postgraduate scholarship from Monash University.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">HCTU</td><td class="NLM_def"><p class="first last"><i>O</i>-(1<i>H</i>-6-chlorobenzotriazol-l-yl)-1,1,3,3-tetramethyluronium hexafluorophosphate</p></td></tr><tr><td class="NLM_term">DMAP</td><td class="NLM_def"><p class="first last">4-dimethylamonipyridine</p></td></tr><tr><td class="NLM_term">EDCI</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3-dimethylaminopropyl)carbodiimide</p></td></tr><tr><td class="NLM_term">PyBOP</td><td class="NLM_def"><p class="first last">(benzotriazol-1-yloxy)tripyrrolidinophosphonium hexafluorophosphate</p></td></tr></table><span class="title" id="d56738093e10734">Additional Note</span><div class="NLM_fn" id="fn9999a"><sup>a</sup> <p class="last">May account for two carbons, as there are many overlapping quaternary carbons in this region.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i131">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50852" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50852" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 80 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maroux, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louvard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baratti, J.</span></span> <span> </span><span class="NLM_article-title">The aminopeptidase from hog intestinal brush border</span>. <i>Biochim. Biophys. Acta, Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>321</i></span>,  <span class="NLM_fpage">282</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1016/0005-2744(73)90083-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2F0005-2744%2873%2990083-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=4750768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaE2cXksFSmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=1973&pages=282-295&author=S.+Marouxauthor=D.+Louvardauthor=J.+Baratti&title=The+aminopeptidase+from+hog+intestinal+brush+border&doi=10.1016%2F0005-2744%2873%2990083-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Aminopeptidase from hog intestinal brush border</span></div><div class="casAuthors">Maroux, S.; Louvard, D.; Baratti, J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Enzymology</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">321</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">282-95</span>CODEN:
                <span class="NLM_cas:coden">BBEZAD</span>;
        ISSN:<span class="NLM_cas:issn">0924-1086</span>.
    </div><div class="casAbstract">The aminopeptidase bound to the brush border membrane of porcine jejunum and ileum was purified.  After treatment of the mucosa with Triton X-100, the enzyme was still attached to very small membrane fragments (the aminopeptidase complex) from which it was liberated by trypsin.  The free enzyme could be further purified to homogeneity by using classical techniques.  The high-molecular-weight complex contained, besides aminopeptidase, at least 1 other protein inactive on the substrates of this enzyme.  The free aminopeptidase had a molecular weight of ∼280,000 and contained 23% sugars and 2 atoms of Zn per mole.  Gel electrophoresis in Na dodecyl sulfate showed 4 bands.  One, however, is probably an artifact arising from trypsin treatment.  The other 3 suggest the existence in aminopeptidase of 3 subunits with molecular weights of 130,000, 97,000, and 49,000 and which were also obtained from the complex.  The purified aminopeptidase accounts for all, or almost all, of the peptidase activity of the brush border membrane.  It also accounts for all of the arylamidase activity, and a small but apparently significant substrate, ∼ half the tripeptidase activity, and a small but apparently significant part of the dipeptidase activity of the whole jejunal and ileal mucosa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqkqw_YjJkBLVg90H21EOLACvtfcHk0lj2mW2tcNrXHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXksFSmsQ%253D%253D&md5=aab1fece3802aaa127064adcfa44392b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2F0005-2744%2873%2990083-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0005-2744%252873%252990083-1%26sid%3Dliteratum%253Aachs%26aulast%3DMaroux%26aufirst%3DS.%26aulast%3DLouvard%26aufirst%3DD.%26aulast%3DBaratti%26aufirst%3DJ.%26atitle%3DThe%2520aminopeptidase%2520from%2520hog%2520intestinal%2520brush%2520border%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Enzymol.%26date%3D1973%26volume%3D321%26spage%3D282%26epage%3D295%26doi%3D10.1016%2F0005-2744%2873%2990083-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaklamanis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turley, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leek, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gatter, K. C.</span></span> <span> </span><span class="NLM_article-title">Expression of aminopeptidase-n (CD 13) in normal tissues and malignant neoplasms of epithelial and lymphoid origin</span>. <i>J. Clin. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">47</span>, <span class="refDoi"> DOI: 10.1136/jcp.47.1.43</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1136%2Fjcp.47.1.43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=7907609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A280%3ADyaK2c7nvFGhtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=1994&pages=43-47&author=J.+Dixonauthor=L.+Kaklamanisauthor=H.+Turleyauthor=I.+D.+Hicksonauthor=R.+D.+Leekauthor=A.+L.+Harrisauthor=K.+C.+Gatter&title=Expression+of+aminopeptidase-n+%28CD+13%29+in+normal+tissues+and+malignant+neoplasms+of+epithelial+and+lymphoid+origin&doi=10.1136%2Fjcp.47.1.43"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of aminopeptidase-n (CD 13) in normal tissues and malignant neoplasms of epithelial and lymphoid origin</span></div><div class="casAuthors">Dixon J; Kaklamanis L; Turley H; Hickson I D; Leek R D; Harris A L; Gatter K C</div><div class="citationInfo"><span class="NLM_cas:title">Journal of clinical pathology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-7</span>
        ISSN:<span class="NLM_cas:issn">0021-9746</span>.
    </div><div class="casAbstract">AIMS:  To provide a detailed knowledge of the distribution of the CD13 molecule, also known as the protease aminopeptidase-N, on both normal tissues and malignant neoplasms of epithelial and lymphoid origin.  METHODS:  CD13 antigen was examined by immunocytochemistry, using a recently produced antibody (VS5E) alongside a commercially available anti-CD13 monoclonal antibody.  The VS5E recognising CD13 was produced by immunising a doxorubicin resistant breast cancer cell line (MCF-7-ADr).  A striking feature of this antibody was that it stained the doxorubicin resistant cells but not the parental cell line.  Both antibodies were tested on a broad range of normal tissues and three common types of epithelial malignancy (colon n = 28, lung n = 30, breast n = 35), and 12 cases of Hodgkin's and 52 of non-Hodgkin's lymphomas.  RESULTS:  CD13 was expressed on many tissue and cell types outside the haematopoietic system.  In particular it was present on breast epithelium and in 20% (seven of 35) of breast carcinomas, but absent in normal and neoplastic colonic and bronchial tissues and lymphomas.  CONCLUSIONS:  This study provides not only detailed information about the expression of the CD13 antigen, but also raises the important possibility that CD13 expression may correlate with drug resistance in breast carcinomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRHCFYsz4T-0ax48Rj5GnGofW6udTcc2eYcMEf0Et7Etbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2c7nvFGhtQ%253D%253D&md5=6cd79912e08bdf02227d88d22b78a781</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1136%2Fjcp.47.1.43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fjcp.47.1.43%26sid%3Dliteratum%253Aachs%26aulast%3DDixon%26aufirst%3DJ.%26aulast%3DKaklamanis%26aufirst%3DL.%26aulast%3DTurley%26aufirst%3DH.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26aulast%3DLeek%26aufirst%3DR.%2BD.%26aulast%3DHarris%26aufirst%3DA.%2BL.%26aulast%3DGatter%26aufirst%3DK.%2BC.%26atitle%3DExpression%2520of%2520aminopeptidase-n%2520%2528CD%252013%2529%2520in%2520normal%2520tissues%2520and%2520malignant%2520neoplasms%2520of%2520epithelial%2520and%2520lymphoid%2520origin%26jtitle%3DJ.%2520Clin.%2520Pathol.%26date%3D1994%26volume%3D47%26spage%3D43%26epage%3D47%26doi%3D10.1136%2Fjcp.47.1.43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mina-Osorio, P.</span></span> <span> </span><span class="NLM_article-title">The moonlighting enzyme CD13: old and new functions to target</span>. <i>Trends Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">371</span>, <span class="refDoi"> DOI: 10.1016/j.molmed.2008.06.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.molmed.2008.06.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=18603472" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD1cXpsVCltrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=361-371&author=P.+Mina-Osorio&title=The+moonlighting+enzyme+CD13%3A+old+and+new+functions+to+target&doi=10.1016%2Fj.molmed.2008.06.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The moonlighting enzyme CD13: old and new functions to target</span></div><div class="casAuthors">Mina-Osorio, Paola</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Molecular Medicine</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">361-371</span>CODEN:
                <span class="NLM_cas:coden">TMMRCY</span>;
        ISSN:<span class="NLM_cas:issn">1471-4914</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Aminopeptidase N (CD13) is a widely expressed ectoenzyme with functions that do not always depend on its enzymic activity: an aspect that has been overlooked.  Numerous CD13-targeting tools have been developed in the last few years.  Several of them are already undergoing clin. trials, and there are promising reports on the effectiveness of others in animal models of disease.  However, their efficacy might be obscured by their effects on unrecognized functions of CD13, resulting in unexpected complications.  The purpose of this review is (i) to discuss the various functions ascribed to CD13 and the possible mechanisms behind them and (ii) to consider some of the questions that need to be answered to achieve a better understanding of the biol. relevance of these functions, a more precise interpretation of the results obtained after their manipulation and a more rational design of CD13-targeting agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVuM5Zcq3wFrVg90H21EOLACvtfcHk0lgzxAoLlUNNvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXpsVCltrc%253D&md5=10447fdd2c56d40b650bd904363f5fc5</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2008.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2008.06.003%26sid%3Dliteratum%253Aachs%26aulast%3DMina-Osorio%26aufirst%3DP.%26atitle%3DThe%2520moonlighting%2520enzyme%2520CD13%253A%2520old%2520and%2520new%2520functions%2520to%2520target%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2008%26volume%3D14%26spage%3D361%26epage%3D371%26doi%3D10.1016%2Fj.molmed.2008.06.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wong, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rini, J. M.</span></span> <span> </span><span class="NLM_article-title">The X-ray crystal structure of human aminopeptidase N reveals a novel dimer and the basis for peptide processing</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">36804</span>– <span class="NLM_lpage">36813</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.398842</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1074%2Fjbc.M112.398842" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=22932899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFOqsrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=36804-36813&author=A.+H.+Wongauthor=D.+Zhouauthor=J.+M.+Rini&title=The+X-ray+crystal+structure+of+human+aminopeptidase+N+reveals+a+novel+dimer+and+the+basis+for+peptide+processing&doi=10.1074%2Fjbc.M112.398842"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The X-ray Crystal Structure of Human Aminopeptidase N Reveals a Novel Dimer and the Basis for Peptide Processing</span></div><div class="casAuthors">Wong, Alan H. M.; Zhou, Dongxia; Rini, James M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">36804-36813</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Human aminopeptidase N (hAPN/hCD13) is a dimeric membrane protein and a member of the M1 family of zinc metallopeptidases.  Within the rennin-angiotensin system, its enzymic activity is responsible for processing peptide hormones angiotensin III and IV.  In addn., hAPN is also involved in cell adhesion, endocytosis, and signal transduction and it is an important target for cancer therapy.  Reported here are the high resoln. x-ray crystal structures of the dimeric ectodomain of hAPN and its complexes with angiotensin IV and the peptidomimetic inhibitors, amastatin and bestatin.  Each monomer of the dimer is found in what has been termed the closed form in other M1 enzymes and each monomer is characterized by an internal cavity surrounding the catalytic site as well as a unique substrate/inhibitor-dependent loop ordering, which in the case of the bestatin complex suggests a new route to inhibitor design.  The hAPN structure provides the first example of a dimeric M1 family member and the obsd. structural features, in conjunction with a model for the open form, provide novel insights into the mechanism of peptide processing and signal transduction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1AkeQE7nfNLVg90H21EOLACvtfcHk0lgzxAoLlUNNvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFOqsrnF&md5=7b4950107a47e98f51b22d02695b8363</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.398842&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.398842%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DA.%2BH.%26aulast%3DZhou%26aufirst%3DD.%26aulast%3DRini%26aufirst%3DJ.%2BM.%26atitle%3DThe%2520X-ray%2520crystal%2520structure%2520of%2520human%2520aminopeptidase%2520N%2520reveals%2520a%2520novel%2520dimer%2520and%2520the%2520basis%2520for%2520peptide%2520processing%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D36804%26epage%3D36813%26doi%3D10.1074%2Fjbc.M112.398842" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fournie-Zaluski, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chauvel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roques, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corvol, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llorens-Cortes, C.</span></span> <span> </span><span class="NLM_article-title">Identification of metabolic pathways of brain angiotensin II and III using specific aminopeptidase inhibitors: predominant role of angiotensin III in the control of vasopressin release</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">11968</span>– <span class="NLM_lpage">11973</span>, <span class="refDoi"> DOI: 10.1073/pnas.93.21.11968</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1073%2Fpnas.93.21.11968" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=8876246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaK28XmtlarsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1996&pages=11968-11973&author=S.+Ziniauthor=M.+C.+Fournie-Zaluskiauthor=E.+Chauvelauthor=B.+P.+Roquesauthor=P.+Corvolauthor=C.+Llorens-Cortes&title=Identification+of+metabolic+pathways+of+brain+angiotensin+II+and+III+using+specific+aminopeptidase+inhibitors%3A+predominant+role+of+angiotensin+III+in+the+control+of+vasopressin+release&doi=10.1073%2Fpnas.93.21.11968"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of metabolic pathways of brain angiotensin II and III using specific aminopeptidase inhibitors: Predominant role of angiotensin III in the control of vasopressin release</span></div><div class="casAuthors">Zini, Sylvie; Fournie-Zaluski, Marie-Claude; Chauvel, Eric; Roques, Bernard P.; Corvol, Pierre; Llorens-Cortex, Catherine</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11968-11973</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Angiotensin (Ang) II and Ang III are two peptide effectors of the brain renin-angiotensin system that participate in the control of blood pressure and increase water consumption and vasopressin release.  To delineate the resp. roles of these peptides in the regulation of vasopressin secretion, their metabolic pathways and their effects on vasopressin release were identified in vivo.  For this purpose, we used recently developed selective inhibitors of aminopeptidase A (APA) and aminopeptidase N (APN), two enzymes that are believed to be responsible for the N-terminal cleavage of Ang II and Ang III, resp.  Mice received [3H]Ang II intracerebroventricularly (i.c.v.) in the presence or absence of the APA inhibitor, EC 33 (3-amino-4-thio-Bu sulfonate) or the APN inhibitor, EC 27 (2-amino-pentan-1,5-dithiol).  [3H]Ang II and [3H]Ang III levels were evaluated from hypothalamus homogenates by HPLC.  EC 33 increased the half-life of [3H]Ang II 2.6-fold and completely blocked the formation of [3H]Ang III, whereas EC 27 increased the half-life of [3H]Ang III 2.3-fold.  In addn., the effects of EC 33 and EC 27 on Ang-induced vasopressin release were studied in mice.  Ang II was injected i.c.v. in the presence or absence of EC 33, and plasma vasopressin levels were estd. by RIA.  While vasopressin levels were increased 2-fold by Ang II (5 ng), EC 33 inhibited Ang II-induced vasopressin release in a dose-dependent manner.  In contrast, EC 27 injected alone increased in a dose-dependent manner vasopressin levels.  The EC 27-induced vasopressin release was completely blocked by the coadministration of the Ang receptor antagonist [Sar1-Ala8]Ang II.  Thus, APA and APN are involved in vivo in the metab. of brain Ang II and Ang III, resp., and the action of Ang II on vasopressin release depends upon the prior conversion of Ang II to Ang III.  This shows that Ang III behaves as one of the main effector peptides of the brain renin-angiotensin system in the control of vasopressin release.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpOvvF5qOP0bVg90H21EOLACvtfcHk0ljT0RhBDThGew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmtlarsrY%253D&md5=0526bbe74b15a9984199e2163c3a6702</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1073%2Fpnas.93.21.11968&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.93.21.11968%26sid%3Dliteratum%253Aachs%26aulast%3DZini%26aufirst%3DS.%26aulast%3DFournie-Zaluski%26aufirst%3DM.%2BC.%26aulast%3DChauvel%26aufirst%3DE.%26aulast%3DRoques%26aufirst%3DB.%2BP.%26aulast%3DCorvol%26aufirst%3DP.%26aulast%3DLlorens-Cortes%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520metabolic%2520pathways%2520of%2520brain%2520angiotensin%2520II%2520and%2520III%2520using%2520specific%2520aminopeptidase%2520inhibitors%253A%2520predominant%2520role%2520of%2520angiotensin%2520III%2520in%2520the%2520control%2520of%2520vasopressin%2520release%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1996%26volume%3D93%26spage%3D11968%26epage%3D11973%26doi%3D10.1073%2Fpnas.93.21.11968" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giros, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gros, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solhonne, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, J. C.</span></span> <span> </span><span class="NLM_article-title">Characterization of aminopeptidases responsible for inactivating endogenous (Met5)enkephalin in brain slices using peptidase inhibitors and anti-aminopeptidase M antibodies</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">281</span>– <span class="NLM_lpage">287</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=2869404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaL28XhvVaitbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1986&pages=281-287&author=B.+Girosauthor=C.+Grosauthor=B.+Solhonneauthor=J.+C.+Schwartz&title=Characterization+of+aminopeptidases+responsible+for+inactivating+endogenous+%28Met5%29enkephalin+in+brain+slices+using+peptidase+inhibitors+and+anti-aminopeptidase+M+antibodies"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of aminopeptidases responsible for inactivating endogenous (Met5)enkephalin in brain slices using peptidase inhibitors and anti-aminopeptidase M antibodies</span></div><div class="casAuthors">Giros, Bruno; Gros, Claude; Solhonne, Brigitte; Schwartz, Jean Charles</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">281-7</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    </div><div class="casAbstract">In addn. to enkephalinase (EC 3.4.24.11), 2 enkephalin-hydrolyzing aminopeptidases recently identified in cerebral membranes [aminopeptidase M (EC 3.4.11.2) and a puromycin-sensitive aminopeptidase (also designated MII or aminoenkephalinase)] are potentially involved in endogenous enkephalin inactivation.  Their participation in the hydrolysis of the endogenous (Met5)enkephalin released by depolarization of slices from rat globus pallidus was assessed by using 3 inhibitory agents: bestatin, puromycin, and anti-aminopeptidase M antibodies.  The selectivity and potency of these agents were first detn. by evaluating their IC50 (concn. for 50% max. inhibition) values for inhibition of [3H](Met5)enkephalin hydrolysis by increasingly complex prepns. comprising semipurified aminopeptidases, pallidal membranes, and pallidal slices.  Bestatin was a fairly potent inhibitor but lacked selectivity, as there was only a 3-fold difference between its IC50 values for the 2 aminopeptidases, and it displayed restricted diffusion and degrdn. in the slice prepn.  Puromycin discriminated well between the 2 aminopeptidases (30-fold difference in IC50 values) and did not show any apparent restricted diffusion in the slice prepn.  Anti-aminopeptidase M antibodies were highly discriminant (>300-fold difference in IC50 values for the 2 aminopeptidases) but displayed restricted diffusion.  Anal. of the concn.-protection curves of the 3 agents for recovery of the (Met5)enkephalin released from pallidal slices in the presence of the enkephalinase inhibitor thiorphan indicated that both aminopeptidases participated in enkephalin degrdn. but that the role of aminopeptidase M was largely predominant, in contrast with its low relative activity in the prepn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouT8HLGNgJjrVg90H21EOLACvtfcHk0ljT0RhBDThGew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28XhvVaitbg%253D&md5=f65e26f08374b1474c519c768613683e</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGiros%26aufirst%3DB.%26aulast%3DGros%26aufirst%3DC.%26aulast%3DSolhonne%26aufirst%3DB.%26aulast%3DSchwartz%26aufirst%3DJ.%2BC.%26atitle%3DCharacterization%2520of%2520aminopeptidases%2520responsible%2520for%2520inactivating%2520endogenous%2520%2528Met5%2529enkephalin%2520in%2520brain%2520slices%2520using%2520peptidase%2520inhibitors%2520and%2520anti-aminopeptidase%2520M%2520antibodies%26jtitle%3DMol.%2520Pharmacol.%26date%3D1986%26volume%3D29%26spage%3D281%26epage%3D287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheinberg, D. A.</span></span> <span> </span><span class="NLM_article-title">Substance P and bradykinin are natural inhibitors of CD13/aminopeptidase N</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>208</i></span>,  <span class="NLM_fpage">664</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1006/bbrc.1995.1390</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1006%2Fbbrc.1995.1390" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=7535053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaK2MXksVKhsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=208&publication_year=1995&pages=664-674&author=Y.+Xuauthor=D.+Wellnerauthor=D.+A.+Scheinberg&title=Substance+P+and+bradykinin+are+natural+inhibitors+of+CD13%2Faminopeptidase+N&doi=10.1006%2Fbbrc.1995.1390"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Substance P and bradykinin are natural inhibitors of CD13/aminopeptidase N</span></div><div class="casAuthors">Xu, Yang; Wellner, Daniel; Scheinberg, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">208</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">664-74</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Aminopeptidase N (EC 3.4.11.2) is an important enzyme that is involved in the degrdn. of regulatory peptides including enkephalins.  The authors report here that purified and native membrane-bound aminopeptidase N will sequentially and completely hydrolyze both Leu-enkephalin and Met-enkephalin from the amino terminus.  Both purified pig aminopeptidase N and the enzyme of live HL60 cells displayed similar Km values for enkephalin.  The naturally occurring neuropeptides substance P and bradykinin, and the morphine agonist, morphiceptin, were not hydrolyzed by aminopeptidase N and each inhibited the enzymic activity.  Each of these peptides contains a proline at the second residue.  The Ki values for substance P (0.44 μM), bradykinin (9.4 μM), and morphiceptin (169 μM) were obtained with the enzyme on live HL60 cells.  The values for the purified enzyme from pig were similar.  The potent inhibition of aminopeptidase N by substance P and bradykinin suggests that these peptides may be natural inhibitors of the enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKLpukdjBcGLVg90H21EOLACvtfcHk0ljT0RhBDThGew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXksVKhsbw%253D&md5=67009d9bb902fd3a97c57f9128373e14</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.1995.1390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.1995.1390%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DY.%26aulast%3DWellner%26aufirst%3DD.%26aulast%3DScheinberg%26aufirst%3DD.%2BA.%26atitle%3DSubstance%2520P%2520and%2520bradykinin%2520are%2520natural%2520inhibitors%2520of%2520CD13%252Faminopeptidase%2520N%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1995%26volume%3D208%26spage%3D664%26epage%3D674%26doi%3D10.1006%2Fbbrc.1995.1390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bosch, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smits, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haagmans, B. L.</span></span> <span> </span><span class="NLM_article-title">Membrane ectopeptidases targeted by human coronaviruses</span>. <i>Curr. Opin. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1016/j.coviro.2014.03.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.coviro.2014.03.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=24762977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVarsLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=55-60&author=B.+J.+Boschauthor=S.+L.+Smitsauthor=B.+L.+Haagmans&title=Membrane+ectopeptidases+targeted+by+human+coronaviruses&doi=10.1016%2Fj.coviro.2014.03.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Membrane ectopeptidases targeted by human coronaviruses</span></div><div class="casAuthors">Bosch, Berend Jan; Smits, Saskia L.; Haagmans, Bart L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Virology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">55-60</span>CODEN:
                <span class="NLM_cas:coden">COVUAF</span>;
        ISSN:<span class="NLM_cas:issn">1879-6257</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">A review.  Six coronaviruses, including the recently identified Middle East respiratory syndrome coronavirus, are known to target the human respiratory tract causing mild to severe disease.  Their interaction with receptors expressed on cells located in the respiratory tract is an essential first step in the infection.  Thus far three membrane ectopeptidases, dipeptidyl peptidase 4 (DPP4), angiotensin-converting enzyme 2 (ACE2) and aminopeptidase N (APN), have been identified as entry receptors for four human-infecting coronaviruses.  Although the catalytic activity of the ACE2, APN, and DPP4 peptidases is not required for virus entry, co-expression of other host proteases allows efficient viral entry.  In addn., evolutionary conservation of these receptors may permit interspecies transmissions.  Because of the physiol. function of these peptidase systems, pathogenic host responses may be potentially amplified and cause acute respiratory distress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohFxBacvTPabVg90H21EOLACvtfcHk0ljT0RhBDThGew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVarsLjL&md5=b86845672b1f34fddab209e22e199bce</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.coviro.2014.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coviro.2014.03.011%26sid%3Dliteratum%253Aachs%26aulast%3DBosch%26aufirst%3DB.%2BJ.%26aulast%3DSmits%26aufirst%3DS.%2BL.%26aulast%3DHaagmans%26aufirst%3DB.%2BL.%26atitle%3DMembrane%2520ectopeptidases%2520targeted%2520by%2520human%2520coronaviruses%26jtitle%3DCurr.%2520Opin.%2520Virol.%26date%3D2014%26volume%3D6%26spage%3D55%26epage%3D60%26doi%3D10.1016%2Fj.coviro.2014.03.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegyi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maile, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heister, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagemann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddell, S. G.</span></span> <span> </span><span class="NLM_article-title">Molecular analysis of the coronavirus-receptor function of aminopeptidase N</span>. <i>Adv. Exp. Med. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>440</i></span>,  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">67</span>, <span class="refDoi"> DOI: 10.1007/978-1-4615-5331-1_8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1007%2F978-1-4615-5331-1_8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=9782265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaK1cXmvFKgu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=440&publication_year=1998&pages=61-67&author=A.+F.+Kolbauthor=A.+Hegyiauthor=J.+Maileauthor=A.+Heisterauthor=M.+Hagemannauthor=S.+G.+Siddell&title=Molecular+analysis+of+the+coronavirus-receptor+function+of+aminopeptidase+N&doi=10.1007%2F978-1-4615-5331-1_8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular analysis of the coronavirus-receptor function of aminopeptidase N</span></div><div class="casAuthors">Kolb, Andreas F.; Hegyi, Annette; Maile, Julia; Heister, Angelien; Hagemann, Margitta; Siddell, Stuart G.</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Experimental Medicine and Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">440</span>
        (<span class="NLM_cas:issue">Coronaviruses and Arteriviruses</span>),
    <span class="NLM_cas:pages">61-67</span>CODEN:
                <span class="NLM_cas:coden">AEMBAP</span>;
        ISSN:<span class="NLM_cas:issn">0065-2598</span>.
    
            (<span class="NLM_cas:orgname">Plenum Publishing Corp.</span>)
        </div><div class="casAbstract">Aminopeptidase N (APN) is a major cell surface for coronaviruses of the serogroup I.  By using chimeric APN proteins assembled from human, porcine and feline APN the authors have identified determinants which are critically involved in the coronavirus-APN interaction.  The results indicate that human coronavirus 229E (HCV 229E) is distinct from the other serogroup I coronaviruses in that determinants located within the N-terminal parts of the human and feline APN proteins mediate the infection of HCV 229E, whereas determinants located within the C-terminal parts of porcine, feline and canine APN mediate the infection of transmissible gastro-enteritis virus (TGEV), feline infectious peritonitis virus (FIPV) and canine coronavirus (CCV), resp.  A further anal. of the mapped amino acid segments by site directed mutagenesis revealed that a short stretch of 8 amino acids in the hAPN protein plays a decisive role in mediating HCV 229E reception.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKoYqOIhFVh7Vg90H21EOLACvtfcHk0liDHMRY-doLpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmvFKgu78%253D&md5=088b2ac1d1445e968d14097778819963</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2F978-1-4615-5331-1_8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4615-5331-1_8%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DA.%2BF.%26aulast%3DHegyi%26aufirst%3DA.%26aulast%3DMaile%26aufirst%3DJ.%26aulast%3DHeister%26aufirst%3DA.%26aulast%3DHagemann%26aufirst%3DM.%26aulast%3DSiddell%26aufirst%3DS.%2BG.%26atitle%3DMolecular%2520analysis%2520of%2520the%2520coronavirus-receptor%2520function%2520of%2520aminopeptidase%2520N%26jtitle%3DAdv.%2520Exp.%2520Med.%2520Biol.%26date%3D1998%26volume%3D440%26spage%3D61%26epage%3D67%26doi%3D10.1007%2F978-1-4615-5331-1_8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yeager, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashmun, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardellichio, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Look, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, K. V.</span></span> <span> </span><span class="NLM_article-title">Human aminopeptidase N is a receptor for human coronavirus 229E</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>357</i></span>,  <span class="NLM_fpage">420</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1038/357420a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1038%2F357420a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1350662" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaK38Xkt1CjtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=357&publication_year=1992&pages=420-422&author=C.+L.+Yeagerauthor=R.+A.+Ashmunauthor=R.+K.+Williamsauthor=C.+B.+Cardellichioauthor=L.+H.+Shapiroauthor=A.+T.+Lookauthor=K.+V.+Holmes&title=Human+aminopeptidase+N+is+a+receptor+for+human+coronavirus+229E&doi=10.1038%2F357420a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Human aminopeptidase N is a receptor for human coronavirus 229E</span></div><div class="casAuthors">Yeager, Curtis L.; Ashmun, Richard A.; Williams, Richard K.; Cardellichio, Christine B.; Shapiro, Linda H.; Look, A. Thomas; Holmes, Kathryn V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">357</span>
        (<span class="NLM_cas:issue">6377</span>),
    <span class="NLM_cas:pages">420-2</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Transmissible gastroenteritis virus (TGEV) replicates selectively in the differentiated enterocytes covering the villi of the small intestine.  In order to investigate the mol. determinants of the infection, the surface mol. used by the virus for binding and entry into host cells was studied.  Aminopeptidase N, an ectoenzyme abundantly expressed at the apical membrane of the enterocyte serves as a receptor for TGEV.  Monoclonal antibodies were selected for their ability to block infection by TGEV of porcine cell lines.  They recognized a brush border membrane protein of Mr 150K, which was identified as aminopeptidase N by N-terminal sequencing.  Two lines of evidence supported the view that the peptidase itself acts as a receptor.  First, virions bound specifically to aminopeptidase N that was purified to homogeneity.  Second, recombinant expression of aminopeptidase N conferred infectivity by TGEV to an otherwise non-permissive cell line.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphfMWdeAI-0rVg90H21EOLACvtfcHk0liDHMRY-doLpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xkt1CjtL4%253D&md5=e911c3bd8d7b89cc84e41beca7151b7a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2F357420a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F357420a0%26sid%3Dliteratum%253Aachs%26aulast%3DYeager%26aufirst%3DC.%2BL.%26aulast%3DAshmun%26aufirst%3DR.%2BA.%26aulast%3DWilliams%26aufirst%3DR.%2BK.%26aulast%3DCardellichio%26aufirst%3DC.%2BB.%26aulast%3DShapiro%26aufirst%3DL.%2BH.%26aulast%3DLook%26aufirst%3DA.%2BT.%26aulast%3DHolmes%26aufirst%3DK.%2BV.%26atitle%3DHuman%2520aminopeptidase%2520N%2520is%2520a%2520receptor%2520for%2520human%2520coronavirus%2520229E%26jtitle%3DNature%26date%3D1992%26volume%3D357%26spage%3D420%26epage%3D422%26doi%3D10.1038%2F357420a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mina-Osorio, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, E.</span></span> <span> </span><span class="NLM_article-title">Aminopeptidase N (CD13) functionally interacts with FcgammaRs in human monocytes</span>. <i>J. Leukocyte Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">1008</span>– <span class="NLM_lpage">1017</span>, <span class="refDoi"> DOI: 10.1189/jlb.1204714</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1189%2Fjlb.1204714" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=15758076" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltValurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2005&pages=1008-1017&author=P.+Mina-Osorioauthor=E.+Ortega&title=Aminopeptidase+N+%28CD13%29+functionally+interacts+with+FcgammaRs+in+human+monocytes&doi=10.1189%2Fjlb.1204714"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Aminopeptidase N (CD13) functionally interacts with FcγRs in human monocytes</span></div><div class="casAuthors">Mina-Osorio, Paola; Ortega, Enrique</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Leukocyte Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1008-1017</span>CODEN:
                <span class="NLM_cas:coden">JLBIE7</span>;
        ISSN:<span class="NLM_cas:issn">0741-5400</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Aminopeptidase N (E.C. 3.4.11.2) is a membrane-bound metalloproteinase expressed in many tissues.  Although its cytoplasmic portion has only eight amino acids, crosslinking of CD13 by monoclonal antibodies (mAb) has been shown to trigger intracellular signaling.  A functional assocn. between CD13 and receptors for IgG (FcγRs) has been proposed.  In this work, we evaluated possible functional interactions between CD13 and FcγRs in human peripheral blood monocytes and in U-937 promonocytic cells.  Our results show that during FcγR-mediated phagocytosis, CD13 redistributes to the phagocytic cup and is internalized into the phagosomes.  Moreover, modified erythrocytes that interact with the monocytic cell membrane through FcγRI and CD13 are ingested simultaneously, more efficiently than those that interact through the FcγRI only.  Also, co-crosslinking of CD13 with FcγRI by specific mAbs increases the level and duration of Syk phosphorylation induced by FcγRI crosslinking.  Finally, FcγRI and CD13 colocalize in zones of cellular polarization and coredistribute after aggregation of either of them.  These results demonstrate that CD13 and FcγRI can functionally interact on the monocytic cell membrane and suggest that CD13 may act as a signal regulator of FcγR function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8zhizZl_ECbVg90H21EOLACvtfcHk0liDHMRY-doLpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltValurc%253D&md5=8fac3c179e79f9a914cc6058b57d6482</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1189%2Fjlb.1204714&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1189%252Fjlb.1204714%26sid%3Dliteratum%253Aachs%26aulast%3DMina-Osorio%26aufirst%3DP.%26aulast%3DOrtega%26aufirst%3DE.%26atitle%3DAminopeptidase%2520N%2520%2528CD13%2529%2520functionally%2520interacts%2520with%2520FcgammaRs%2520in%2520human%2520monocytes%26jtitle%3DJ.%2520Leukocyte%2520Biol.%26date%3D2005%26volume%3D77%26spage%3D1008%26epage%3D1017%26doi%3D10.1189%2Fjlb.1204714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pasqualini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koivunen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kain, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahdenranta, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stryhn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashmun, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arap, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruoslahti, E.</span></span> <span> </span><span class="NLM_article-title">Aminopeptidase N is a receptor for tumor-homing peptides and a target for inhibiting angiogenesis</span>. <i>Cancer. Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">722</span>– <span class="NLM_lpage">727</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10676659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD3cXht1Gmtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2000&pages=722-727&author=R.+Pasqualiniauthor=E.+Koivunenauthor=R.+Kainauthor=J.+Lahdenrantaauthor=M.+Sakamotoauthor=A.+Stryhnauthor=R.+A.+Ashmunauthor=L.+H.+Shapiroauthor=W.+Arapauthor=E.+Ruoslahti&title=Aminopeptidase+N+is+a+receptor+for+tumor-homing+peptides+and+a+target+for+inhibiting+angiogenesis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Aminopeptidase N is a receptor for tumor-homing peptides and a target for inhibiting angiogenesis</span></div><div class="casAuthors">Pasqualini, Renata; Koivunen, Erkki; Kain, Renate; Lahdenranta, Johanna; Sakamoto, Michiie; Stryhn, Anette; Ashmun, Richard A.; Shapiro, Linda H.; Arap, Wadih; Ruoslahti, Erkki</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">722-727</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">AACR Subscription Office</span>)
        </div><div class="casAbstract">Phage that display a surface peptide with the NGR sequence motif home selectively to tumor vasculature in vivo.  A drug coupled to an NGR peptide has more potent antitumor effects than the free drug.  The authors show here that the receptor for the NGR peptides in tumor vasculature is aminopeptidase N (APN; also called CD13).  NGR phage specifically bound to immunocaptured APN and to cells engineered to express APN on their surface.  Antibodies against APN inhibited in vivo tumor homing by the NGR phage.  Immunohistochem. staining showed that APN expression is up-regulated in endothelial cells within mouse and human tumors.  In another tissue that undergoes angiogenesis, corpus luteum, blood vessels also expressed APN, but APN was not detected in blood vessels of various other normal tissues stained under the same conditions.  APN antagonists specifically inhibited angiogenesis in chorioallantoic membranes and in the retina and suppressed tumor growth.  Thus, APN is involved in angiogenesis and can serve as a target for delivering drugs into tumors and for inhibiting angiogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobSSRsFaZLubVg90H21EOLACvtfcHk0liqCx8SKwqF5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXht1Gmtbc%253D&md5=b5731d5e35221aeef1a2191e040486d1</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPasqualini%26aufirst%3DR.%26aulast%3DKoivunen%26aufirst%3DE.%26aulast%3DKain%26aufirst%3DR.%26aulast%3DLahdenranta%26aufirst%3DJ.%26aulast%3DSakamoto%26aufirst%3DM.%26aulast%3DStryhn%26aufirst%3DA.%26aulast%3DAshmun%26aufirst%3DR.%2BA.%26aulast%3DShapiro%26aufirst%3DL.%2BH.%26aulast%3DArap%26aufirst%3DW.%26aulast%3DRuoslahti%26aufirst%3DE.%26atitle%3DAminopeptidase%2520N%2520is%2520a%2520receptor%2520for%2520tumor-homing%2520peptides%2520and%2520a%2520target%2520for%2520inhibiting%2520angiogenesis%26jtitle%3DCancer.%2520Res.%26date%3D2000%26volume%3D60%26spage%3D722%26epage%3D727" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mina-Osorio, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, E.</span></span> <span> </span><span class="NLM_article-title">CD13 in cell adhesion: aminopeptidase N (CD13) mediates homotypic aggregation of monocytic cells</span>. <i>J. Leukocyte Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">719</span>– <span class="NLM_lpage">730</span>, <span class="refDoi"> DOI: 10.1189/jlb.0705425</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1189%2Fjlb.0705425" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=16415167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsFCmurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2006&pages=719-730&author=P.+Mina-Osorioauthor=L.+H.+Shapiroauthor=E.+Ortega&title=CD13+in+cell+adhesion%3A+aminopeptidase+N+%28CD13%29+mediates+homotypic+aggregation+of+monocytic+cells&doi=10.1189%2Fjlb.0705425"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">CD13 in cell adhesion: aminopeptidase N (CD13) mediates homotypic aggregation of monocytic cells</span></div><div class="casAuthors">Mina-Osorio, Paola; Shapiro, Linda H.; Ortega, Enrique</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Leukocyte Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">719-730</span>CODEN:
                <span class="NLM_cas:coden">JLBIE7</span>;
        ISSN:<span class="NLM_cas:issn">0741-5400</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Homotypic aggregation (HA) of cells plays key roles in physiol. and pathol. processes, such as embryogenesis, immune responses, angiogenesis, tumor cell invasion, and metastasis.  Aminopeptidase N (CD13) has been implicated in most of these phenomena, although its participation has been attributed to its enzymic activity, while its role as an adhesion mol. has been almost unexplored.  Here, we show that certain anti-CD13 monoclonal antibodies induce HA of monocytic U-937 cells, independently of their effect on enzymic activity.  The phenomenon is related to binding to a specific site on the CD13 mol. and is independent of integrins.  It is abrogated by low temp., by the glycolysis inhibitor 2-deoxyglucose, and by inhibitors of tyrosine and mitogen-activated protein kinases.  The inhibitor of microtubule polymn. colchicine has a synergistic effect on CD13-mediated aggregation, suggesting an inhibitory role of microtubules in this process.  Finally, during HA, CD13 actively redistributes to the zones of cell-cell contact, as detd. by live cell imaging studies, demonstrating a direct role of CD13 in the adhesion phenomenon.  Together, these data show for the first time the participation of CD13 in monocytic cell adhesion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrvQUEMsZK9rVg90H21EOLACvtfcHk0liqCx8SKwqF5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsFCmurY%253D&md5=adf38b17294b6cf1eca0b7de7e81e97e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1189%2Fjlb.0705425&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1189%252Fjlb.0705425%26sid%3Dliteratum%253Aachs%26aulast%3DMina-Osorio%26aufirst%3DP.%26aulast%3DShapiro%26aufirst%3DL.%2BH.%26aulast%3DOrtega%26aufirst%3DE.%26atitle%3DCD13%2520in%2520cell%2520adhesion%253A%2520aminopeptidase%2520N%2520%2528CD13%2529%2520mediates%2520homotypic%2520aggregation%2520of%2520monocytic%2520cells%26jtitle%3DJ.%2520Leukocyte%2520Biol.%26date%3D2006%26volume%3D79%26spage%3D719%26epage%3D730%26doi%3D10.1189%2Fjlb.0705425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span> <span> </span><span class="NLM_article-title">Structural basis for multifunctional roles of mammalian aminopeptidase N</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">17966</span>– <span class="NLM_lpage">17971</span>, <span class="refDoi"> DOI: 10.1073/pnas.1210123109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1073%2Fpnas.1210123109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=23071329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=17966-17971&author=L.+Chenauthor=Y.+L.+Linauthor=G.+Pengauthor=F.+Li&title=Structural+basis+for+multifunctional+roles+of+mammalian+aminopeptidase+N&doi=10.1073%2Fpnas.1210123109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for multifunctional roles of mammalian aminopeptidase N</span></div><div class="casAuthors">Chen, Lang; Lin, Yi-Lun; Peng, Guiqing; Li, Fang</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">17966-17971, S17966/1-S17966/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Mammalian aminopeptidase N (APN) plays multifunctional roles in many physiol. processes, including peptide metab., cell motility and adhesion, and coronavirus entry.  Here we detd. crystal structures of porcine APN at 1.85 Å resoln. and its complexes with a peptide substrate and a variety of inhibitors.  APN is a cell surface-anchored and seahorse-shaped zinc-aminopeptidase that forms head-to-head dimers.  Captured in a catalytically active state, these structures of APN illustrate a detailed catalytic mechanism for its aminopeptidase activity.  The active site and peptide-binding channel of APN reside in cavities with wide openings, allowing easy access to peptides.  The cavities can potentially open up further to bind the exposed N terminus of proteins.  The active site anchors the N-terminal neutral residue of peptides/proteins, and the peptide-binding channel binds the remainder of the peptides/proteins in a sequence-independent fashion.  APN also provides an exposed outer surface for coronavirus binding, without its physiol. functions being affected.  These structural features enable APN to function ubiquitously in peptide metab., interact with other proteins to mediate cell motility and adhesion, and serve as a coronavirus receptor.  This study elucidates multifunctional roles of APN and can guide therapeutic efforts to treat APN-related diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7DJL9z8NePrVg90H21EOLACvtfcHk0liqCx8SKwqF5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvE&md5=630e3553442e5c38718e68488f37aee8</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1210123109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1210123109%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DY.%2BL.%26aulast%3DPeng%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DF.%26atitle%3DStructural%2520basis%2520for%2520multifunctional%2520roles%2520of%2520mammalian%2520aminopeptidase%2520N%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D17966%26epage%3D17971%26doi%3D10.1073%2Fpnas.1210123109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span> <span> </span><span class="NLM_article-title">Aminopeptidase fingerprints, an integrated approach for identification of good substrates and optimal inhibitors</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">3310</span>– <span class="NLM_lpage">3318</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.060418</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1074%2Fjbc.M109.060418" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=19948737" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVektw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=3310-3318&author=M.+Dragauthor=M.+Bogyoauthor=J.+A.+Ellmanauthor=G.+S.+Salvesen&title=Aminopeptidase+fingerprints%2C+an+integrated+approach+for+identification+of+good+substrates+and+optimal+inhibitors&doi=10.1074%2Fjbc.M109.060418"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Aminopeptidase Fingerprints, an Integrated Approach for Identification of Good Substrates and Optimal Inhibitors</span></div><div class="casAuthors">Drag, Marcin; Bogyo, Matthew; Ellman, Jonathan A.; Salvesen, Guy S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">3310-3318</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Aminopeptidases process the N-terminal amino acids of target substrates by sequential cleavage of one residue at a time.  They are found in all cell compartments of prokaryotes and eukaryotes, being implicated in the major proteolytic events of cell survival, defense, growth, and development.  We present a new approach for the fast and reliable evaluation of the substrate specificity of individual aminopeptidases.  Using solid phase chem. with the 7-amino-4-carbamoylmethylcoumarin fluorophore, we have synthesized a library of 61 individual natural and unnatural amino acid substrates, chosen to cover a broad spectrum of the possible interactions in the S1 pocket of this type of protease.  As proof of concept, we detd. the substrate specificity of human, pig, and rat orthologs of aminopeptidase N (CD13), a highly conserved cell surface protease that inactivates enkephalins and other bioactive peptides.  Our data reveal a large and hydrophobic character for the S1 pocket of aminopeptidase N that is conserved with aminopeptidase Ns.  Our approach, which can be applied in principle to all aminopeptidases, yields useful information for the design of specific inhibitors, and more importantly, reveals a relation between the kinetics of substrate hydrolysis and the kinetics of enzyme inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT41yqFNTF3bVg90H21EOLACvtfcHk0liqCx8SKwqF5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVektw%253D%253D&md5=eb4f0c442d396c5f290dd6aabdb64898</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.060418&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.060418%26sid%3Dliteratum%253Aachs%26aulast%3DDrag%26aufirst%3DM.%26aulast%3DBogyo%26aufirst%3DM.%26aulast%3DEllman%26aufirst%3DJ.%2BA.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26atitle%3DAminopeptidase%2520fingerprints%252C%2520an%2520integrated%2520approach%2520for%2520identification%2520of%2520good%2520substrates%2520and%2520optimal%2520inhibitors%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D3310%26epage%3D3318%26doi%3D10.1074%2Fjbc.M109.060418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGowan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowther, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stack, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golding, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner-Adams, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trenholme, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teuscher, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donnelly, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mucha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kafarski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degori, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckle, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardiner, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whisstock, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalton, J. P.</span></span> <span> </span><span class="NLM_article-title">Structural basis for the inhibition of the essential <i>Plasmodium falciparum</i> M1 neutral aminopeptidase</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">2537</span>– <span class="NLM_lpage">2542</span>, <span class="refDoi"> DOI: 10.1073/pnas.0807398106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1073%2Fpnas.0807398106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=19196988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD1MXislahsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=2537-2542&author=S.+McGowanauthor=C.+J.+Porterauthor=J.+Lowtherauthor=C.+M.+Stackauthor=S.+J.+Goldingauthor=T.+S.+Skinner-Adamsauthor=K.+R.+Trenholmeauthor=F.+Teuscherauthor=S.+M.+Donnellyauthor=J.+Grembeckaauthor=A.+Muchaauthor=P.+Kafarskiauthor=R.+Degoriauthor=A.+M.+Buckleauthor=D.+L.+Gardinerauthor=J.+C.+Whisstockauthor=J.+P.+Dalton&title=Structural+basis+for+the+inhibition+of+the+essential+Plasmodium+falciparum+M1+neutral+aminopeptidase&doi=10.1073%2Fpnas.0807398106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for the inhibition of the essential Plasmodium falciparum M1 neutral aminopeptidase</span></div><div class="casAuthors">McGowan, Sheena; Porter, Corrine J.; Lowther, Jonathan; Stack, Colin M.; Golding, Sarah J.; Skinner-Adams, Tina S.; Trenholme, Katharine R.; Teuscher, Franka; Donnelly, Sheila M.; Grembecka, Jolanta; Mucha, Artur; Kafarski, Pawel; DeGori, Ross; Buckle, Ashley M.; Gardiner, Donald L.; Whisstock, James C.; Dalton, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2537-2542</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Plasmodium falciparum parasites are responsible for the major global disease malaria, which results in >2 million deaths each year.  With the rise of drug-resistant malarial parasites, novel drug targets and lead compds. are urgently required for the development of new therapeutic strategies.  Here, we address this important problem by targeting the malarial neutral aminopeptidases that are involved in the terminal stages of Hb digestion and essential for the provision of amino acids used for parasite growth and development within the erythrocyte.  We characterize the structure and substrate specificity of one such aminopeptidase, PfA-M1, a validated drug target.  The X-ray crystal structure of PfA-M1 alone and in complex with the generic inhibitor, bestatin, and a phosphinate dipeptide analog with potent in vitro and in vivo antimalarial activity, hPheP[CH2]Phe, reveals features within the protease active site that are crit. to its function as an aminopeptidase and can be exploited for drug development.  These results set the groundwork for the development of antimalarial therapeutics that target the neutral aminopeptidases of the parasite.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-6_ZNvWMUP7Vg90H21EOLACvtfcHk0lhNFLlXkHUoaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXislahsrY%253D&md5=23bc5efa86832c3fbc10e900f42f3216</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0807398106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0807398106%26sid%3Dliteratum%253Aachs%26aulast%3DMcGowan%26aufirst%3DS.%26aulast%3DPorter%26aufirst%3DC.%2BJ.%26aulast%3DLowther%26aufirst%3DJ.%26aulast%3DStack%26aufirst%3DC.%2BM.%26aulast%3DGolding%26aufirst%3DS.%2BJ.%26aulast%3DSkinner-Adams%26aufirst%3DT.%2BS.%26aulast%3DTrenholme%26aufirst%3DK.%2BR.%26aulast%3DTeuscher%26aufirst%3DF.%26aulast%3DDonnelly%26aufirst%3DS.%2BM.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DMucha%26aufirst%3DA.%26aulast%3DKafarski%26aufirst%3DP.%26aulast%3DDegori%26aufirst%3DR.%26aulast%3DBuckle%26aufirst%3DA.%2BM.%26aulast%3DGardiner%26aufirst%3DD.%2BL.%26aulast%3DWhisstock%26aufirst%3DJ.%2BC.%26aulast%3DDalton%26aufirst%3DJ.%2BP.%26atitle%3DStructural%2520basis%2520for%2520the%2520inhibition%2520of%2520the%2520essential%2520Plasmodium%2520falciparum%2520M1%2520neutral%2520aminopeptidase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2009%26volume%3D106%26spage%3D2537%26epage%3D2542%26doi%3D10.1073%2Fpnas.0807398106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hitzerd, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verbrugge, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ossenkoppele, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, G. J.</span></span> <span> </span><span class="NLM_article-title">Positioning of aminopeptidase inhibitors in next generation cancer therapy</span>. <i>Amino Acids</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">793</span>– <span class="NLM_lpage">808</span>, <span class="refDoi"> DOI: 10.1007/s00726-013-1648-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1007%2Fs00726-013-1648-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=24385243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVOltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2014&pages=793-808&author=S.+M.+Hitzerdauthor=S.+E.+Verbruggeauthor=G.+Ossenkoppeleauthor=G.+Jansenauthor=G.+J.+Peters&title=Positioning+of+aminopeptidase+inhibitors+in+next+generation+cancer+therapy&doi=10.1007%2Fs00726-013-1648-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Positioning of aminopeptidase inhibitors in next generation cancer therapy</span></div><div class="casAuthors">Hitzerd, Sarina M.; Verbrugge, Sue Ellen; Ossenkoppele, Gert; Jansen, Gerrit; Peters, Godefridus J.</div><div class="citationInfo"><span class="NLM_cas:title">Amino Acids</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">793-808</span>CODEN:
                <span class="NLM_cas:coden">AACIE6</span>;
        ISSN:<span class="NLM_cas:issn">0939-4451</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag GmbH</span>)
        </div><div class="casAbstract">A review.  Aminopeptidases represent a class of (zinc) metalloenzymes that catalyze the cleavage of amino acids nearby the N-terminus of polypeptides, resulting in hydrolysis of peptide bonds.  Aminopeptidases operate downstream of the ubiquitin-proteasome pathway and are implicated in the final step of intracellular protein degrdn. either by trimming proteasome-generated peptides for antigen presentation or full hydrolysis into free amino acids for recycling in renewed protein synthesis.  This review focuses on the function and subcellular location of five key aminopeptidases (aminopeptidase N, leucine aminopeptidase, puromycin-sensitive aminopeptidase, leukotriene A4 hydrolase and endoplasmic reticulum aminopeptidase 1/2) and their assocn. with different diseases, in particular cancer and their current position as target for therapeutic intervention by aminopeptidase inhibitors.  Historically, bestatin was the first prototypical aminopeptidase inhibitor that entered the clinic 35 years ago and is still used for the treatment of lung cancer.  More recently, new generation aminopeptidase inhibitors became available, including the aminopeptidase inhibitor prodrug tosedostat, which is currently tested in phase II clin. trials for acute myeloid leukemia.  Beyond bestatin and tosedostat, medicinal chem. has emerged with addnl. series of potential aminopeptidases inhibitors which are still in an early phase of (pre)clin. investigations.  The expanded knowledge of the unique mechanism of action of aminopeptidases has revived interest in aminopeptidase inhibitors for drug combination regimens in anti-cancer treatment.  In this context, this review will discuss relevant features and mechanisms of action of aminopeptidases and will also elaborate on factors contributing to aminopeptidase inhibitor efficacy and/or loss of efficacy due to drug resistance-related phenomena.  Together, a growing body of data point to aminopeptidase inhibitors as attractive tools for combination chemotherapy, hence their implementation may be a step forward in a new era of personalized treatment of cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrABQrQu0W5GrVg90H21EOLACvtfcHk0lhNFLlXkHUoaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVOltQ%253D%253D&md5=c17cc3a379f6a21b0126d672f3680dc6</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1007%2Fs00726-013-1648-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00726-013-1648-0%26sid%3Dliteratum%253Aachs%26aulast%3DHitzerd%26aufirst%3DS.%2BM.%26aulast%3DVerbrugge%26aufirst%3DS.%2BE.%26aulast%3DOssenkoppele%26aufirst%3DG.%26aulast%3DJansen%26aufirst%3DG.%26aulast%3DPeters%26aufirst%3DG.%2BJ.%26atitle%3DPositioning%2520of%2520aminopeptidase%2520inhibitors%2520in%2520next%2520generation%2520cancer%2520therapy%26jtitle%3DAmino%2520Acids%26date%3D2014%26volume%3D46%26spage%3D793%26epage%3D808%26doi%3D10.1007%2Fs00726-013-1648-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bauvois, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dauzonne, D.</span></span> <span> </span><span class="NLM_article-title">Aminopeptidase-N/CD13 (EC 3.4.11.2) inhibitors: chemistry, biological evaluations, and therapeutic prospects</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">88</span>– <span class="NLM_lpage">130</span>, <span class="refDoi"> DOI: 10.1002/med.20044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1002%2Fmed.20044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=16216010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD28Xmt1CitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=88-130&author=B.+Bauvoisauthor=D.+Dauzonne&title=Aminopeptidase-N%2FCD13+%28EC+3.4.11.2%29+inhibitors%3A+chemistry%2C+biological+evaluations%2C+and+therapeutic+prospects&doi=10.1002%2Fmed.20044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Aminopeptidase-N/CD13 (EC 3.4.11.2) inhibitors: chemistry, biological evaluations, and therapeutic prospects</span></div><div class="casAuthors">Bauvois, Brigitte; Dauzonne, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-130</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Aminopeptidase N (APN)/CD13 (EC 3.4.11.2) is a transmembrane protease present in a wide variety of human tissues and cell types (endothelial, epithelial, fibroblast, leukocyte).  APN/CD13 expression is dysregulated in inflammatory diseases and in cancers (solid and hematol. tumors).  APN/CD13 serves as a receptor for coronaviruses.  Natural and synthetic inhibitors of APN activity have been characterized.  These inhibitors have revealed that APN is able to modulate bioactive peptide responses (pain management, vasopressin release) and to influence immune functions and major biol. events (cell proliferation, secretion, invasion, angiogenesis).  Therefore, inhibition of APN/CD13 may lead to the development of anti-cancer and anti-inflammatory drugs.  This review provides an update on the biol. and pharmacol. profiles of known natural and synthetic APN inhibitors.  Current status on their potential use as therapeutic agents is discussed with regard to toxicity and specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7XAstbaJj6bVg90H21EOLACvtfcHk0lhNFLlXkHUoaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xmt1CitQ%253D%253D&md5=b1377159539c0e5cdaeb5e5d736554e0</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fmed.20044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.20044%26sid%3Dliteratum%253Aachs%26aulast%3DBauvois%26aufirst%3DB.%26aulast%3DDauzonne%26aufirst%3DD.%26atitle%3DAminopeptidase-N%252FCD13%2520%2528EC%25203.4.11.2%2529%2520inhibitors%253A%2520chemistry%252C%2520biological%2520evaluations%252C%2520and%2520therapeutic%2520prospects%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2006%26volume%3D26%26spage%3D88%26epage%3D130%26doi%3D10.1002%2Fmed.20044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wickström, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nygren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gullbo, J.</span></span> <span> </span><span class="NLM_article-title">Aminopeptidase N (CD13) as a target for cancer chemotherapy</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">501</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1111/j.1349-7006.2010.01826.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1111%2Fj.1349-7006.2010.01826.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=21205077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A280%3ADC%252BC3M3gtFKkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2011&pages=501-508&author=M.+Wickstr%C3%B6mauthor=R.+Larssonauthor=P.+Nygrenauthor=J.+Gullbo&title=Aminopeptidase+N+%28CD13%29+as+a+target+for+cancer+chemotherapy&doi=10.1111%2Fj.1349-7006.2010.01826.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Aminopeptidase N (CD13) as a target for cancer chemotherapy</span></div><div class="casAuthors">Wickstrom Malin; Larsson Rolf; Nygren Peter; Gullbo Joachim</div><div class="citationInfo"><span class="NLM_cas:title">Cancer science</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">501-8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The enzyme aminopeptidase N (APN, also known as CD13) is a Zn(2+) dependent membrane-bound ectopeptidase that degrades preferentially proteins and peptides with a N-terminal neutral amino acid.  Aminopeptidase N has been associated with the growth of different human cancers and suggested as a suitable target for anti-cancerous therapy.  Different approaches have been used to develop new drugs directed to this target, including enzyme inhibitors as well as APN-targeted carrier constructs.  This review discusses the prevalence and possible function of APN in malignant diseases, mainly solid tumors, as well as its "drugability" evaluated in preclinical in vivo models, and also provides a brief overview of current clinical trials focused on APN.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR-AFmpp47W5PEeosoOnpOLfW6udTcc2ebd_x6dYmcmTLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M3gtFKkuw%253D%253D&md5=907f8f3efe78ae795d81408d68986b44</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1111%2Fj.1349-7006.2010.01826.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1349-7006.2010.01826.x%26sid%3Dliteratum%253Aachs%26aulast%3DWickstr%25C3%25B6m%26aufirst%3DM.%26aulast%3DLarsson%26aufirst%3DR.%26aulast%3DNygren%26aufirst%3DP.%26aulast%3DGullbo%26aufirst%3DJ.%26atitle%3DAminopeptidase%2520N%2520%2528CD13%2529%2520as%2520a%2520target%2520for%2520cancer%2520chemotherapy%26jtitle%3DCancer%2520Sci.%26date%3D2011%26volume%3D102%26spage%3D501%26epage%3D508%26doi%3D10.1111%2Fj.1349-7006.2010.01826.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amin, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, T.</span></span> <span> </span><span class="NLM_article-title">Design of aminopeptidase N inhibitors as anti-cancer agents</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6468</span>– <span class="NLM_lpage">6490</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00782</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00782" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A280%3ADC%252BC1Mjgt1Wjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6468-6490&author=S.+A.+Aminauthor=N.+Adhikariauthor=T.+Jha&title=Design+of+aminopeptidase+N+inhibitors+as+anti-cancer+agents&doi=10.1021%2Facs.jmedchem.7b00782"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Design of Aminopeptidase N Inhibitors as Anti-cancer Agents</span></div><div class="casAuthors">Amin Sk Abdul; Adhikari Nilanjan; Jha Tarun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6468-6490</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Aminopeptidase N (APN) is an important metalloenzyme.  It regulates multivariate cellular functions by different mechanisms such as enzymatic cleavage of peptides.  This may play a role in endocytosis and regulate signal transduction.  APN, a member of the M1 zinc metallopeptidase family, plays crucial roles in a variety of functions such as migration and invasion, and angiogenesis and metastasis of tumor cells.  Therefore, APN inhibitors may be useful for the treatment of cancer.  In this Perspective, structure-activity relationships of APN inhibitors are discussed to get an idea of possible lead candidates.  APN inhibitors should possess an aryl hydrophobic function along with a zinc binding group attached to the hydrophobic group(s) to achieve high potency.  This and other design aspects of APN inhibitors are discussed in this Perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRteeQg7iu8x6qcneEtCi92fW6udTcc2ebd_x6dYmcmTLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mjgt1Wjug%253D%253D&md5=9b277fdc4171daeee7560bf0154046c9</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00782%26sid%3Dliteratum%253Aachs%26aulast%3DAmin%26aufirst%3DS.%2BA.%26aulast%3DAdhikari%26aufirst%3DN.%26aulast%3DJha%26aufirst%3DT.%26atitle%3DDesign%2520of%2520aminopeptidase%2520N%2520inhibitors%2520as%2520anti-cancer%2520agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6468%26epage%3D6490%26doi%3D10.1021%2Facs.jmedchem.7b00782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ashmun, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Look, A. T.</span></span> <span> </span><span class="NLM_article-title">Metalloprotease activity of CD13/aminopeptidase N on the surface of human myeloid cells</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">469</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1967220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaK3cXpsF2luw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=1990&pages=462-469&author=R.+A.+Ashmunauthor=A.+T.+Look&title=Metalloprotease+activity+of+CD13%2Faminopeptidase+N+on+the+surface+of+human+myeloid+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Metalloprotease activity of CD13/aminopeptidase N on the surface of human myeloid cells</span></div><div class="casAuthors">Ashmun, Richard A.; Look, A. Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-9</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    </div><div class="casAbstract">It was previously found that the myeloid cell surface glycoprotein CD13 (gp150) is identical to aminopeptidase N (EC 3.4.11.2), a widely distributed membrane-bound, Zn-dependent metallogprotease with an extracellular enzymic domain that cleaves N-terminal amino acid residues from oligopeptides.  As a 1st step toward defining the function of this mol. on myeloid cells, we the cell surface-assocd. N-terminal peptidase was assessed activity by sensitive spectrophotometric measurements of the cleavage of p-nitroanilide amino acid derivs.  Aminopeptidase activity detected on the surface of normal and malignant hematopoietic cells coincided with the level of cell surface CD13 expression as measured by flow cytometry.  The enzyme was specifically inhibited by the Zn-binding metalloprotease inhibitors, bestatin, 1,10-phenanthroline, or 2.2'-dipyridyl, but was not affected by several inhibitors of other classes of proteases.  Aminopeptidase activity was demonstrated for CD13 mols. specifically immunopptd. from the surface of CD13-pos. cells and was blocked by the metalloprotease inhibitor 1,10-phenanthroline.  Moreover, cell surface aminopeptidase activity was partially inhibited when viable cells were incubated with 2 of a panel of 11 monoclonal antibodies (MoAbs) known to be specific for extracellular epitopes of human CD13.  This inhibition was apparent in the absence of detectable downmodulation of CD13 mols. from the cell surface, suggesting that these MoAbs either phys. interfere with substrate binding or alter the Zn-coordinating properties of aminopeptidase N mols.  Aminopeptidase N could play an important role in modulating signals generated by peptides at the surface of myeloid cells, either by removing key N-terminal residues from active peptides or by converting inactive peptides to active forms.  The inhibitory antibodies used in this study should prove useful in delineating the physiol. roles of CD13/aminopeptidase N on normal and malignant myeloid cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlZp8__KZ8-7Vg90H21EOLACvtfcHk0li6fJoNchpMHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXpsF2luw%253D%253D&md5=932fb4f057da474e269fc2955bd106de</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAshmun%26aufirst%3DR.%2BA.%26aulast%3DLook%26aufirst%3DA.%2BT.%26atitle%3DMetalloprotease%2520activity%2520of%2520CD13%252Faminopeptidase%2520N%2520on%2520the%2520surface%2520of%2520human%2520myeloid%2520cells%26jtitle%3DBlood%26date%3D1990%26volume%3D75%26spage%3D462%26epage%3D469" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hashida, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takabayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adachi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kondo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohno, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaoka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyake, M.</span></span> <span> </span><span class="NLM_article-title">Aminopeptidase N is involved in cell motility and angiogenesis: its clinical significance in human colon cancer</span>. <i>Gastroenterology</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>122</i></span>,  <span class="NLM_fpage">376</span>– <span class="NLM_lpage">386</span>, <span class="refDoi"> DOI: 10.1053/gast.2002.31095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1053%2Fgast.2002.31095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=11832452" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD38XhsVGns7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2002&pages=376-386&author=H.+Hashidaauthor=A.+Takabayashiauthor=M.+Kanaiauthor=M.+Adachiauthor=K.+Kondoauthor=N.+Kohnoauthor=Y.+Yamaokaauthor=M.+Miyake&title=Aminopeptidase+N+is+involved+in+cell+motility+and+angiogenesis%3A+its+clinical+significance+in+human+colon+cancer&doi=10.1053%2Fgast.2002.31095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Aminopeptidase N is involved in cell motility and angiogenesis: its clinical significance in human colon cancer</span></div><div class="casAuthors">Hashida, Hiroki; Takabayashi, Arimichi; Kanai, Michiyuki; Adachi, Masashi; Kondo, Keiichi; Kohno, Nobuoki; Yamaoka, Yoshio; Miyake, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Gastroenterology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">376-386</span>CODEN:
                <span class="NLM_cas:coden">GASTAB</span>;
        ISSN:<span class="NLM_cas:issn">0016-5085</span>.
    
            (<span class="NLM_cas:orgname">W. B. Saunders Co.</span>)
        </div><div class="casAbstract">Background & Aims: The mol. basis of cell motility is highly complex and is controlled by a no. of mol. systems, whereas angiogenesis is an important biol. component of tumor progression.  The aims of this study were to investigate the possible involvement of proteins at the cell surface in controlling cell motility and angiogenesis, and to identify the cell surface mols. involved in gastrointestinal tumors.  Methods: We addressed these issues using functional monoclonal antibodies, which inhibit cell motility, endothelial cell migration, and tube formation.  Furthermore, we investigated the relationship between this antigen and colon cancer, and showed the prognostic significance in human colon cancer.  Results: We established a murine monoclonal antibody MH8-11, which inhibits cell motility and in vitro angiogenesis.  This epitope was a 165-kilodalton protein, and the sequencing anal. revealed that it was almost identical to aminopeptidase N (APN)/cluster of differentiation (CD) 13.  APN/CD13 expression was assocd. with tumor status (P = 0.025).  The disease-free and overall survival rate for patients with pos. APN/CD13 expression tumors was significantly lower than that for patients with neg. APN/CD13 expression tumors (P = 0.014, 0.033, resp.).  Among 47 node-pos. patients, the survival rate of patients with neg. APN/CD13 expression was better than that of those with pos. APN/CD13 expression.  Conclusions: Our data suggest that APN/CD13 is involved in cell motility and angiogenesis, and APN/CD13 expression may be a useful indicator of a poor prognosis for node-pos. patients with colon cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhHjwhNpLFZrVg90H21EOLACvtfcHk0lg_75G5VZc_vA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhsVGns7s%253D&md5=0ee1dfdc5e46097c8193b604d93d11e0</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1053%2Fgast.2002.31095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fgast.2002.31095%26sid%3Dliteratum%253Aachs%26aulast%3DHashida%26aufirst%3DH.%26aulast%3DTakabayashi%26aufirst%3DA.%26aulast%3DKanai%26aufirst%3DM.%26aulast%3DAdachi%26aufirst%3DM.%26aulast%3DKondo%26aufirst%3DK.%26aulast%3DKohno%26aufirst%3DN.%26aulast%3DYamaoka%26aufirst%3DY.%26aulast%3DMiyake%26aufirst%3DM.%26atitle%3DAminopeptidase%2520N%2520is%2520involved%2520in%2520cell%2520motility%2520and%2520angiogenesis%253A%2520its%2520clinical%2520significance%2520in%2520human%2520colon%2520cancer%26jtitle%3DGastroenterology%26date%3D2002%26volume%3D122%26spage%3D376%26epage%3D386%26doi%3D10.1053%2Fgast.2002.31095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitaichi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komoto, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaganoi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyake, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamasaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kondo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imamura, M.</span></span> <span> </span><span class="NLM_article-title">Clinicopathological significance of <tep-common:author-query>AQ7: Please provide a DOI number for ref 23 or indicate if one doesn&amp;#x2019;t exist.</tep-common:author-query>aminopeptidase N/CD13 expression in human gastric carcinoma</span>. <i>Hepatogastroenterology</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.1016/j.cellbi.2008.01.130</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.cellbi.2008.01.130" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=17419227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A280%3ADC%252BD2s3hsFWltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2007&pages=36-40&author=J.+Kawamuraauthor=Y.+Shimadaauthor=H.+Kitaichiauthor=I.+Komotoauthor=Y.+Hashimotoauthor=J.+Kaganoiauthor=M.+Miyakeauthor=S.+Yamasakiauthor=K.+Kondoauthor=M.+Imamura&title=Clinicopathological+significance+of+%3Ctep-common%3Aauthor-query%3EAQ7%3A+Please+provide+a+DOI+number+for+ref+23+or+indicate+if+one+doesn%26amp%3B%23x2019%3Bt+exist.%3C%2Ftep-common%3Aauthor-query%3Eaminopeptidase+N%2FCD13+expression+in+human+gastric+carcinoma&doi=10.1016%2Fj.cellbi.2008.01.130"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Clinicopathological significance of aminopeptidase N/CD13 expression in human gastric carcinoma</span></div><div class="casAuthors">Kawamura Junichiro; Shimada Yutaka; Kitaichi Hidenori; Komoto Izumi; Hashimoto Yosuhe; Kaganoi Junichi; Miyake Masayuki; Yamasaki Seiji; Kondo Kan; Imamura Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Hepato-gastroenterology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">73</span>),
    <span class="NLM_cas:pages">36-40</span>
        ISSN:<span class="NLM_cas:issn">0172-6390</span>.
    </div><div class="casAbstract">BACKGROUND/AIMS:  Aminopeptidase N (APN)/CD13 is a transmembrane ectoenzyme occurring in a wide variety of cells.  Recently, APN/CD13 has been reported to be involved in tumor invasion and metastasis.  However, precise functions in tumor cells, and its role in gastric carcinoma remain unclear.  METHODOLOGY:  To evaluate the role of APN/CD13 in gastric carcinoma, we conducted immunohistochemical staining for APN/CD13 in 121 gastric carcinoma specimens, using anti-APN monoclonal antibody.  The relationship between APN/CD13 expression and various prognostic factors of gastric carcinoma were investigated.  RESULTS:  Of the 121 patients with gastric carcinoma studied, 48 were strongly positive, 36 were weakly positive, and 37 were negative.  Overall survival rate of the patients with negative APN/CD13 expression was significantly lower than that of the patients with positive APN/CD13 expression.  APN/CD13 expression was negatively associated with lymph node metastasis.  Multivariate analysis showed APN/CD13 expression to be a significant prognostic factor.  CONCLUSIONS:  Decreased expression of APN/CD13 was associated with a poor prognosis.  Hence, our results demonstrate that the immunohistochemical detection of APN/CD13 could provide useful information as one of the prognostic factors in gastric cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR_WCHzzJ7alTjk2McQZ6XmfW6udTcc2ea5GxOOD94K_rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s3hsFWltA%253D%253D&md5=0ca2651b5c05608b8d76991b421b5f04</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.cellbi.2008.01.130&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellbi.2008.01.130%26sid%3Dliteratum%253Aachs%26aulast%3DKawamura%26aufirst%3DJ.%26aulast%3DShimada%26aufirst%3DY.%26aulast%3DKitaichi%26aufirst%3DH.%26aulast%3DKomoto%26aufirst%3DI.%26aulast%3DHashimoto%26aufirst%3DY.%26aulast%3DKaganoi%26aufirst%3DJ.%26aulast%3DMiyake%26aufirst%3DM.%26aulast%3DYamasaki%26aufirst%3DS.%26aulast%3DKondo%26aufirst%3DK.%26aulast%3DImamura%26aufirst%3DM.%26atitle%3DClinicopathological%2520significance%2520of%2520%253Ctep-common%253Aauthor-query%253EAQ7%253A%2520Please%2520provide%2520a%2520DOI%2520number%2520for%2520ref%252023%2520or%2520indicate%2520if%2520one%2520doesn%2526amp%253B%2523x2019%253Bt%2520exist.%253C%252Ftep-common%253Aauthor-query%253Eaminopeptidase%2520N%252FCD13%2520expression%2520in%2520human%2520gastric%2520carcinoma%26jtitle%3DHepatogastroenterology%26date%3D2007%26volume%3D54%26spage%3D36%26epage%3D40%26doi%3D10.1016%2Fj.cellbi.2008.01.130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span> <span> </span><span class="NLM_article-title">Possible contribution of aminopeptidase N (APN/CD13) to migration and invasion of human osteosarcoma cell lines</span>. <i>Int. J. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">2475</span>– <span class="NLM_lpage">2485</span>, <span class="refDoi"> DOI: 10.3892/ijo.2014.2664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.3892%2Fijo.2014.2664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=25340499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVamt7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2014&pages=2475-2485&author=W.+Liangauthor=B.+Gaoauthor=G.+Xuauthor=D.+Wengauthor=M.+Xieauthor=Y.+Qian&title=Possible+contribution+of+aminopeptidase+N+%28APN%2FCD13%29+to+migration+and+invasion+of+human+osteosarcoma+cell+lines&doi=10.3892%2Fijo.2014.2664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Possible contribution of aminopeptidase N (APN/CD13) to migration and invasion of human osteosarcoma cell lines</span></div><div class="casAuthors">Liang, Wenqing; Gao, Bo; Xu, Guojian; Weng, Dong; Xie, Minghua; Qian, Yu</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2475-2485</span>CODEN:
                <span class="NLM_cas:coden">IJONES</span>;
        ISSN:<span class="NLM_cas:issn">1019-6439</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Osteosarcoma is the most common primary malignancy of the bone.  Aminopeptidase N (APN/CD13), a Zn+2-dependent ectopeptidase localized on the cell surface, is widely considered to influence the invasion mechanism.  This study explores the potential involvement of APN in migration and invasion of human osteosarcoma cells in vitro using inhibitors and activators of APN.  Cells treated with APN inhibitor bestatin displayed decreased migration and invasion in a Boyden chamber Transwell assay.  Western blotting revealed reduced levels of mitogen-activated protein kinase (MAPK) and phosphatidylinositol 3-kinase (PI3K) pathway proteins, reduced phosphorylation of p38, ERK1/2 and JNK and decreased levels of NF-κB.  Bestatin treatment also lowered APN, matrix metalloproteinase (MMP)-2 and -9 enzymic activity and their mRNA expression.  Reduced MMP-2 and -9 protein levels were also obsd.  By comparison, cells treated with cytokine interleukin-6 (IL-6), a stimulator of APN, displayed increased migration and invasion.  Western blotting revealed increased levels of MAPK and PI3K pathway proteins, phosphorylated p38, ERK1/2 and JNK, and NF-κB.  IL-6 treatment also increased APN and MMP-2 and -9 enzymic activity.  An increase of APN, MMP-2 and -9 mRNA levels, and MMP-2 and -9 protein levels was also obsd.  Together these expts. reveal potential enzymic and signalling roles for APN in osteosarcoma and establish a starting point for an in-depth anal. of the role of APN in regulating invasiveness.  A deeper knowledge about the regulatory mechanisms of APN may contribute to the development of anti-metastatic therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprlLnunSv0_LVg90H21EOLACvtfcHk0lg_75G5VZc_vA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVamt7vJ&md5=50a91f5dc8f5c9c8bfd3bb1fad9f4a23</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.3892%2Fijo.2014.2664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijo.2014.2664%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DW.%26aulast%3DGao%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DWeng%26aufirst%3DD.%26aulast%3DXie%26aufirst%3DM.%26aulast%3DQian%26aufirst%3DY.%26atitle%3DPossible%2520contribution%2520of%2520aminopeptidase%2520N%2520%2528APN%252FCD13%2529%2520to%2520migration%2520and%2520invasion%2520of%2520human%2520osteosarcoma%2520cell%2520lines%26jtitle%3DInt.%2520J.%2520Oncol.%26date%3D2014%26volume%3D45%26spage%3D2475%26epage%3D2485%26doi%3D10.3892%2Fijo.2014.2664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nohara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakuragi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yanagihara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwanuma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kajiyama, Y.</span></span> <span> </span><span class="NLM_article-title">Aminopeptidase N (APN/CD13) as a target molecule for scirrhous gastric cancer</span>. <i>Clin. Res. Hepatol. Gastroenterol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">494</span>– <span class="NLM_lpage">503</span>, <span class="refDoi"> DOI: 10.1016/j.clinre.2015.11.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.clinre.2015.11.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=26774363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVenurfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2016&pages=494-503&author=S.+Noharaauthor=K.+Katoauthor=D.+Fujiwaraauthor=N.+Sakuragiauthor=K.+Yanagiharaauthor=Y.+Iwanumaauthor=Y.+Kajiyama&title=Aminopeptidase+N+%28APN%2FCD13%29+as+a+target+molecule+for+scirrhous+gastric+cancer&doi=10.1016%2Fj.clinre.2015.11.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Aminopeptidase N (APN/CD13) as a target molecule for scirrhous gastric cancer</span></div><div class="casAuthors">Nohara, Shigeo; Kato, Kazunori; Fujiwara, Daisuke; Sakuragi, Naoya; Yanagihara, Kazuyoshi; Iwanuma, Yoshimi; Kajiyama, Yoshiaki</div><div class="citationInfo"><span class="NLM_cas:title">Clinics and Research in Hepatology and Gastroenterology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">494-503</span>CODEN:
                <span class="NLM_cas:coden">CRHGAQ</span>;
        ISSN:<span class="NLM_cas:issn">2210-7401</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Scirrhous gastric cancer is assocd. with peritoneal dissemination and advanced lymph node metastasis from an early stage, and the prognosis is still poor.  In this study, we aimed to analyze candidate mols. for targeted therapy of scirrhous gastric cancer.  We searched for mols./metabolic activity that might be predominantly expressed in a subpopulation of scirrhous gastric cancer cells and might function as cancer stem cell markers.For this purpose, we investigated the expression of various cell surface markers and of aldehyde dehydrogenase (ALDH) activity.  These analyses showed that the scirrhous gastric cancer cell lines HSC-58 and HSC-44PE heterogeneously expressed CD13, while CD44, CDCP1, EpCAM and ABCG2 were expressed uniformly.  Moreover, 10% of the total HSC-58 cell population expressed ALDH enzyme activity.  A subpopulation of cells strongly pos. for ALDH also expressed high levels of CD13, both of which are known as cancer stem cell markers.  HSC-58 cells expressing high levels of CD13 showed lower sensitivity to a cancer drug cisplatin than cells with low levels of CD13.  In contrast, CD13-high subpopulation of HSC-58 was more sensitive to an aminopeptidase N inhibitor bestatin.  In terms of antibody-drug therapy, anti-CD13-immunotoxin was highly cytotoxic towards HSC-58 cells and was more cytotoxic than anti-EpCAM-immunotoxin.These data suggest that CD13 is a suitable cell surface candidate for targeted antibody-drug therapy of scirrhous gastric cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolezJYMlPnvLVg90H21EOLACvtfcHk0lhpWBt-RT8-MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVenurfK&md5=bcb17ef6086f2fad9a3fa7b3996435b0</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.clinre.2015.11.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.clinre.2015.11.003%26sid%3Dliteratum%253Aachs%26aulast%3DNohara%26aufirst%3DS.%26aulast%3DKato%26aufirst%3DK.%26aulast%3DFujiwara%26aufirst%3DD.%26aulast%3DSakuragi%26aufirst%3DN.%26aulast%3DYanagihara%26aufirst%3DK.%26aulast%3DIwanuma%26aufirst%3DY.%26aulast%3DKajiyama%26aufirst%3DY.%26atitle%3DAminopeptidase%2520N%2520%2528APN%252FCD13%2529%2520as%2520a%2520target%2520molecule%2520for%2520scirrhous%2520gastric%2520cancer%26jtitle%3DClin.%2520Res.%2520Hepatol.%2520Gastroenterol.%26date%3D2016%26volume%3D40%26spage%3D494%26epage%3D503%26doi%3D10.1016%2Fj.clinre.2015.11.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brand, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stucke-Ring, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schliemann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrach, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorlich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillejan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller-Tidow, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardelmann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiewrodt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berdel, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwoppe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, W.</span></span> <span> </span><span class="NLM_article-title">Potential therapeutic impact of CD13 expression in non-small cell lung cancer</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0177146</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0177146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1371%2Fjournal.pone.0177146" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=28604784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1agsrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=L.+H.+Schmidtauthor=C.+Brandauthor=J.+Stucke-Ringauthor=C.+Schliemannauthor=T.+Kesslerauthor=S.+Harrachauthor=M.+Mohrauthor=D.+Gorlichauthor=A.+Marraauthor=L.+Hillejanauthor=C.+Muller-Tidowauthor=G.+Lenzauthor=E.+Wardelmannauthor=R.+Wiewrodtauthor=W.+E.+Berdelauthor=C.+Schwoppeauthor=W.+Hartmann&title=Potential+therapeutic+impact+of+CD13+expression+in+non-small+cell+lung+cancer&doi=10.1371%2Fjournal.pone.0177146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Potential therapeutic impact of CD13 expression in non-small cell lung cancer</span></div><div class="casAuthors">Schmidt, Lars Henning; Brand, Caroline; Stucke-Ring, Janine; Schliemann, Christoph; Kessler, Torsten; Harrach, Saliha; Mohr, Michael; Goerlich, Dennis; Marra, Alessandro; Hillejan, Ludger; Mueller-Tidow, Carsten; Lenz, Georg; Wardelmann, Eva; Wiewrodt, Rainer; Berdel, Wolfgang E.; Schwoeppe, Christian; Hartmann, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e0177146/1-e0177146/16</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Background: Aminopeptidase N (CD13) is a zinc-binding protease that has functional effects on both cancerogenesis and tumor angiogenesis.  Since CD13 is an antigen suitable for mol. targeted therapies (e.g. tTF-NGR induced tumor vascular infarction), we evaluated its impact in NSCLC patients, and tested the effects of the CD13-targeted fusion protein tTF-NGR (truncated tissue factor (tTF) contg. the NGR motif: asparagine-glycine-arginine) in vivo in nude mice.  Methods: Expression of both CD13 and CD31 was studied in 270 NSCLC patients by immunohistochem.  Clin. correlations and prognostic effects of the expression profiles were analyzed using univariate and multivariate analyses.  In addn., a microarray-based anal. on the basis of the KM plotter database was performed.  The in vivo effects of the CD13-targeted fusion protein tTF-NGR on tumor growth were tested in CD1 nude mice carrying A549 lung carcinoma xenotransplants.  Results: CD13 expression in tumor endothelial and vessel assocd. stromal cells was found in 15% of the investigated samples, while expression in tumor cells was obsd. in 7%.  Although no significant prognostic impact was obsd. in the full NSCLC study cohort, both univariate and multivariate models identified vascular CD13 protein expression to correlate with poor overall survival in stage III and pN2+ NSCLC patients.  Microarray-based mRNA anal. for either adenocarcinomas or squamous cell carcinomas did not reveal any significant effect.  However, the anal. of CD13 mRNA expression for all lung cancer histologies demonstrated a pos. prognostic effect.  In vivo, systemic application of CD13-targeted tissue factor tTF-NGR significantly reduced CD13+ A549 tumor growth in nude mice.  Conclusions Our results contribute a data basis for prioritizing clin. testing of tTF-NGR and other antitumor mols. targeted by NGR-peptides in NSCLC.  Because: CD13 expression in NSCLC tissues was found only in a specific subset of NSCLC patients, rigorous pre-therapeutic testing will help to select patients for these studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLBFxsuIJ6lLVg90H21EOLACvtfcHk0lhpWBt-RT8-MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1agsrzP&md5=41ed1bce9f3f7ed37b54617f97242acd</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0177146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0177146%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DL.%2BH.%26aulast%3DBrand%26aufirst%3DC.%26aulast%3DStucke-Ring%26aufirst%3DJ.%26aulast%3DSchliemann%26aufirst%3DC.%26aulast%3DKessler%26aufirst%3DT.%26aulast%3DHarrach%26aufirst%3DS.%26aulast%3DMohr%26aufirst%3DM.%26aulast%3DGorlich%26aufirst%3DD.%26aulast%3DMarra%26aufirst%3DA.%26aulast%3DHillejan%26aufirst%3DL.%26aulast%3DMuller-Tidow%26aufirst%3DC.%26aulast%3DLenz%26aufirst%3DG.%26aulast%3DWardelmann%26aufirst%3DE.%26aulast%3DWiewrodt%26aufirst%3DR.%26aulast%3DBerdel%26aufirst%3DW.%2BE.%26aulast%3DSchwoppe%26aufirst%3DC.%26aulast%3DHartmann%26aufirst%3DW.%26atitle%3DPotential%2520therapeutic%2520impact%2520of%2520CD13%2520expression%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0177146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tokuhara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishida, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higashiyama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyake, M.</span></span> <span> </span><span class="NLM_article-title">Clinical significance of aminopeptidase N in non-small cell lung cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">3971</span>– <span class="NLM_lpage">3978</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-06-0338</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1158%2F1078-0432.CCR-06-0338" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=16818694" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsValtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2006&pages=3971-3978&author=T.+Tokuharaauthor=N.+Hattoriauthor=H.+Ishidaauthor=T.+Hiraiauthor=M.+Higashiyamaauthor=K.+Kodamaauthor=M.+Miyake&title=Clinical+significance+of+aminopeptidase+N+in+non-small+cell+lung+cancer&doi=10.1158%2F1078-0432.CCR-06-0338"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical Significance of Aminopeptidase N in Non-Small Cell Lung Cancer</span></div><div class="casAuthors">Tokuhara, Takahiro; Hattori, Noboru; Ishida, Hisao; Hirai, Tatsuya; Higashiyama, Masahiko; Kodama, Ken; Miyake, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3971-3978</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: The aim of our study is to investigate the mechanism of metastasis, to detect novel metastasis-assocd. mols., and to evaluate the mols. from the point of view of clin. application.  A monoclonal antibody MH8-11, which we established, recognizes a glycoprotein that is identical to aminopeptidase N (APN/CD13).  APN/CD13 degrades the extracellular matrix, while it is also involved in cell motility and improves angiogenesis.  Exptl. Design: We investigated the expression of APN/CD13 in 194 cases of non-small cell lung cancer (NSCLC) by immunohistochem. analyses and reverse transcription-PCR assay to det. the significance of this prognostic factor; 95 tumors were stage I, 36 were stage II, 39 were stage IIIA, and 24 were stage IIIB.  Moreover, we investigated that the relationship between the expression of APN/CD13 and angiogenesis and prognosis for patients with NSCLC.  Results: We found a correlation between the expression of APN/CD13 and angiogenesis (r = 0.659; P < 0.0001).  In the 194 patients with NSCLC, we found 68 patients to be APN/CD13+ and 126 patients to be APN/CD13-.  The 5-yr survival rate in patients with APN/CD13+ tumors was significantly lower than in those whose tumors had neg. APN/CD13 (48.3% vs. 67.1%; P = 0.0001).  Conclusion: Our data suggest the expression of APN/CD13 for patients with NSCLC to be assocd. with a poor prognosis and angiogenesis.  This is the first study to show the relationship between the expression of APN/CD13 and the prognosis of patients with NSCLC.  The inhibition of APN/CD13 may be an effective new mol. target therapy for patients with NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8_LPwESR3ALVg90H21EOLACvtfcHk0lhpWBt-RT8-MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsValtbo%253D&md5=a797d87b167bbc2630d3f9269e1b4bf6</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-06-0338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-06-0338%26sid%3Dliteratum%253Aachs%26aulast%3DTokuhara%26aufirst%3DT.%26aulast%3DHattori%26aufirst%3DN.%26aulast%3DIshida%26aufirst%3DH.%26aulast%3DHirai%26aufirst%3DT.%26aulast%3DHigashiyama%26aufirst%3DM.%26aulast%3DKodama%26aufirst%3DK.%26aulast%3DMiyake%26aufirst%3DM.%26atitle%3DClinical%2520significance%2520of%2520aminopeptidase%2520N%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2006%26volume%3D12%26spage%3D3971%26epage%3D3978%26doi%3D10.1158%2F1078-0432.CCR-06-0338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valastyan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Tumor metastasis: molecular insights and evolving paradigms</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">275</span>– <span class="NLM_lpage">292</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.09.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.cell.2011.09.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=22000009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlaksb3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2011&pages=275-292&author=S.+Valastyanauthor=R.+A.+Weinberg&title=Tumor+metastasis%3A+molecular+insights+and+evolving+paradigms&doi=10.1016%2Fj.cell.2011.09.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor Metastasis: Molecular Insights and Evolving Paradigms</span></div><div class="casAuthors">Valastyan, Scott; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">275-292</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Metastases represent the end products of a multistep cell-biol. process termed the invasion-metastasis cascade, which involves dissemination of cancer cells to anatomically distant organ sites and their subsequent adaptation to foreign tissue microenvironments.  Each of these events is driven by the acquisition of genetic and/or epigenetic alterations within tumor cells and the co-option of nonneoplastic stromal cells, which together endow incipient metastatic cells with traits needed to generate macroscopic metastases.  Recent advances provide provocative insights into these cell-biol. and mol. changes, which have implications regarding the steps of the invasion-metastasis cascade that appear amenable to therapeutic targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVqOPVKwPqtbVg90H21EOLACvtfcHk0lhpWBt-RT8-MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlaksb3N&md5=fa62d15c6b61bf90cfcdc1d51bbe98b9</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.09.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.09.024%26sid%3Dliteratum%253Aachs%26aulast%3DValastyan%26aufirst%3DS.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DTumor%2520metastasis%253A%2520molecular%2520insights%2520and%2520evolving%2520paradigms%26jtitle%3DCell%26date%3D2011%26volume%3D147%26spage%3D275%26epage%3D292%26doi%3D10.1016%2Fj.cell.2011.09.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span> <span> </span><span class="NLM_article-title">Development of synthetic aminopeptidase N/CD13 inhibitors to overcome cancer metastasis and angiogenesis</span>. <i>ACS. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">959</span>– <span class="NLM_lpage">964</span>, <span class="refDoi"> DOI: 10.1021/ml3000758</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml3000758" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2ls7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=959-964&author=L.+Suauthor=J.+Caoauthor=Y.+Jiaauthor=X.+Zhangauthor=H.+Fangauthor=W.+Xu&title=Development+of+synthetic+aminopeptidase+N%2FCD13+inhibitors+to+overcome+cancer+metastasis+and+angiogenesis&doi=10.1021%2Fml3000758"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Development of synthetic aminopeptidase N/CD13 inhibitors to overcome cancer metastasis and angiogenesis</span></div><div class="casAuthors">Su, Li; Cao, Jiangying; Jia, Yuping; Zhang, Xiaonan; Fang, Hao; Xu, Wenfang</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">959-964</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cancer metastasis is a major barrier to its treatment and an important cause of patient death.  Antimetastatic agents hold promise for patients with advanced metastatic tumors.  Aminopeptidase N/CD13 (APN) is being pursued by many as an important target against cancer metastasis and angiogenesis, but there are few reports on the in vivo evaluation of synthetic APN inhibitors.  Herein, a series of compds. targeting APN were synthesized and evaluated for their antimetastasis and antiangiogenesis potency both in vitro and in vivo.  Excitingly, compds. (I) (R1 = 1-naphthyl, CH2-3-OMeC6H4, CH2-1-naphthyl), with the most potent APN inhibitory activities, displayed significant antimetastasis and antiangiogenesis effects in vitro and in vivo, suggesting that those synthetic APN inhibitors have the potential to overcome cancer metastasis and angiogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMkj2KpECZfrVg90H21EOLACvtfcHk0ljFokza2Lqg7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2ls7vL&md5=342276e4b1621b1afb642bbcf0ae2464</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fml3000758&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml3000758%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DJia%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DFang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DW.%26atitle%3DDevelopment%2520of%2520synthetic%2520aminopeptidase%2520N%252FCD13%2520inhibitors%2520to%2520overcome%2520cancer%2520metastasis%2520and%2520angiogenesis%26jtitle%3DACS.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D959%26epage%3D964%26doi%3D10.1021%2Fml3000758" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wirtz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konstantopoulos, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Searson, P. C.</span></span> <span> </span><span class="NLM_article-title">The physics of cancer: the role of physical interactions and mechanical forces in metastasis</span>. <i>Nat. Rev. Cancer.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">512</span>– <span class="NLM_lpage">522</span>, <span class="refDoi"> DOI: 10.1038/nrc3080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1038%2Fnrc3080" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=21701513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvFSqtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=512-522&author=D.+Wirtzauthor=K.+Konstantopoulosauthor=P.+C.+Searson&title=The+physics+of+cancer%3A+the+role+of+physical+interactions+and+mechanical+forces+in+metastasis&doi=10.1038%2Fnrc3080"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">The physics of cancer: the role of physical interactions and mechanical forces in metastasis</span></div><div class="casAuthors">Wirtz, Denis; Konstantopoulos, Konstantinos; Searson, Peter C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">512-522</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Metastasis is a complex, multistep process responsible for >90% of cancer-related deaths.  In addn. to genetic and external environmental factors, the phys. interactions of cancer cells with their microenvironment, as well as their modulation by mech. forces, are key determinants of the metastatic process.  We reconstruct the metastatic process and describe the importance of key phys. and mech. processes at each step of the cascade.  The emerging insight into these phys. interactions may help to solve some long-standing questions in disease progression and may lead to new approaches to developing cancer diagnostics and therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRh9IWK-3RjLVg90H21EOLACvtfcHk0ljFokza2Lqg7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvFSqtb0%253D&md5=a87cc9cd6d6383a52bb37932cfcb10a7</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnrc3080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3080%26sid%3Dliteratum%253Aachs%26aulast%3DWirtz%26aufirst%3DD.%26aulast%3DKonstantopoulos%26aufirst%3DK.%26aulast%3DSearson%26aufirst%3DP.%2BC.%26atitle%3DThe%2520physics%2520of%2520cancer%253A%2520the%2520role%2520of%2520physical%2520interactions%2520and%2520mechanical%2520forces%2520in%2520metastasis%26jtitle%3DNat.%2520Rev.%2520Cancer.%26date%3D2011%26volume%3D11%26spage%3D512%26epage%3D522%26doi%3D10.1038%2Fnrc3080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaffer, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">A perspective on cancer cell metastasis</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>331</i></span>,  <span class="NLM_fpage">1559</span>– <span class="NLM_lpage">1564</span>, <span class="refDoi"> DOI: 10.1126/science.1203543</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1126%2Fscience.1203543" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=21436443" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjs1Cgtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=331&publication_year=2011&pages=1559-1564&author=C.+L.+Chafferauthor=R.+A.+Weinberg&title=A+perspective+on+cancer+cell+metastasis&doi=10.1126%2Fscience.1203543"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A Perspective on Cancer Cell Metastasis</span></div><div class="casAuthors">Chaffer, Christine L.; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">331</span>
        (<span class="NLM_cas:issue">6024</span>),
    <span class="NLM_cas:pages">1559-1564</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Metastasis causes most cancer deaths, yet this process remains one of the most enigmatic aspects of the disease.  Building on new mechanistic insights emerging from recent research, we offer our perspective on the metastatic process and reflect on possible paths of future exploration.  We suggest that metastasis can be portrayed as a 2-phase process: The first phase involves the phys. translocation of a cancer cell to a distant organ, whereas the second encompasses the ability of the cancer cell to develop into a metastatic lesion at that distant site.  Although much remains to be learned about the second phase, we feel that an understanding of the first phase is now within sight, due in part to a better understanding of how cancer cell behavior can be modified by a cell-biol. program called the epithelial-to-mesenchymal transition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS1rLIOdukebVg90H21EOLACvtfcHk0ljFokza2Lqg7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjs1Cgtbo%253D&md5=a361a1380ba9a72d974189059ebd9210</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1126%2Fscience.1203543&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1203543%26sid%3Dliteratum%253Aachs%26aulast%3DChaffer%26aufirst%3DC.%2BL.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DA%2520perspective%2520on%2520cancer%2520cell%2520metastasis%26jtitle%3DScience%26date%3D2011%26volume%3D331%26spage%3D1559%26epage%3D1564%26doi%3D10.1126%2Fscience.1203543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saiki, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoneda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuruo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azuma, I.</span></span> <span> </span><span class="NLM_article-title">Role of aminopeptidase N (CD13) in tumor-cell invasion and extracellular matrix degradation</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">137</span>– <span class="NLM_lpage">143</span>, <span class="refDoi"> DOI: 10.1002/ijc.2910540122</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1002%2Fijc.2910540122" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=8097496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaK3sXlsFOhs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1993&pages=137-143&author=I.+Saikiauthor=H.+Fujiiauthor=J.+Yonedaauthor=F.+Abeauthor=M.+Nakajimaauthor=T.+Tsuruoauthor=I.+Azuma&title=Role+of+aminopeptidase+N+%28CD13%29+in+tumor-cell+invasion+and+extracellular+matrix+degradation&doi=10.1002%2Fijc.2910540122"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Role of aminopeptidase N (CD13) in tumor-cell invasion and extracellular matrix degradation</span></div><div class="casAuthors">Saiki, Ikuo; Fujii, Hideji; Yoneda, Junya; Abe, Tuminori; Nakajima, Motowo; Tsuruo, Takashi; Azuma, Ichiro</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">137-43</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    </div><div class="casAbstract">The effect of monoclonal antibodies (MAbs) specific for aminopeptidase N/CD13 on the invasion of human metastatic tumor cells into reconstituted basement membrane (Matrigel) was studied.  The invasion of human metastatic tumor cells (SN12M renal-cell carcinoma, HT 1080 fibrosarcoma and A375M melanoma) into Matrigel-coated filters was inhibited by an anti-CD13 MAb, WM15, in a concn.-dependent manner.  However, this MAb did not have any effect on tumor-cell adhesion and migration to the extracellular matrixes, which may be involved in tumor-cell invasion.  MAb WM15 inhibited the degrdn. of type-IV collagen by tumor cells in a concn.-dependent manner.  Also, WM15 inhibited hydrolyzing activities towards substrates of aminopeptidases in 3 different tumor cells.  Since a previous study indicated that bestatin, an aminopeptidase inhibitor, was able to inhibit tumor-cell invasion, as well as aminopeptidase activities of murine and human metastatic tumor cells, cell-surface aminopeptidase N/CD13 may be partly involved in the activation mechanism for type-IV collagenolysis to achieve tumor-cell invasion, and anti-CD13 MAb WM15 may inhibit tumor-cell invasion through a mechanism involving its inhibitory action on the aminopeptidase N in tumor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogwayYWiR2P7Vg90H21EOLACvtfcHk0lgMCGf0kZ69dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXlsFOhs7w%253D&md5=36333c86bcefb529543bce7af4132376</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fijc.2910540122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.2910540122%26sid%3Dliteratum%253Aachs%26aulast%3DSaiki%26aufirst%3DI.%26aulast%3DFujii%26aufirst%3DH.%26aulast%3DYoneda%26aufirst%3DJ.%26aulast%3DAbe%26aufirst%3DF.%26aulast%3DNakajima%26aufirst%3DM.%26aulast%3DTsuruo%26aufirst%3DT.%26aulast%3DAzuma%26aufirst%3DI.%26atitle%3DRole%2520of%2520aminopeptidase%2520N%2520%2528CD13%2529%2520in%2520tumor-cell%2520invasion%2520and%2520extracellular%2520matrix%2520degradation%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D1993%26volume%3D54%26spage%3D137%26epage%3D143%26doi%3D10.1002%2Fijc.2910540122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoneda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saiki, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kojima, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azuma, I.</span></span> <span> </span><span class="NLM_article-title">Inhibition of tumor invasion and extracellular matrix degradation by ubenimex (bestatin)</span>. <i>Clin. Exp. Metastasis</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">59</span>, <span class="refDoi"> DOI: 10.1007/BF00163576</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1007%2FBF00163576" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1733647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaK38XitVOnsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1992&pages=49-59&author=J.+Yonedaauthor=I.+Saikiauthor=H.+Fujiiauthor=F.+Abeauthor=Y.+Kojimaauthor=I.+Azuma&title=Inhibition+of+tumor+invasion+and+extracellular+matrix+degradation+by+ubenimex+%28bestatin%29&doi=10.1007%2FBF00163576"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of tumor invasion and extracellular matrix degradation by ubenimex (bestatin)</span></div><div class="casAuthors">Yoneda, Junya; Saiki, Ikuo; Fujii, Hideji; Abe, Fuminori; Kojima, Yutaka; Azuma, Ichiro</div><div class="citationInfo"><span class="NLM_cas:title">Clinical & Experimental Metastasis</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">49-59</span>CODEN:
                <span class="NLM_cas:coden">CEXMD2</span>;
        ISSN:<span class="NLM_cas:issn">0262-0898</span>.
    </div><div class="casAbstract">The authors have investigated the effect of the immunomodulator ubenimex (bestatin) on the enzymic degrdn. of the extracellular matrix by human renal cell carcinoma SN12M cells during the invasive process.  The invasion of SN12M cells into reconstituted basement membrane (Matrigel) was inhibited by the presence of bestatin in a concn.-dependent manner.  However, bestatin did not have any effect on tumor cell adhesion and migration to the extracellular matrixes which may be involved in tumor cell invasion.  Bestatin inhibited the degrdn. of type IV collagen by tumor cells, but not by tumor-conditioned medium (TCM), in a concn.-dependent manner.  The authors also found that bestatin inhibited hydrolyzing activities towards substrates of aminopeptidases in SN12M cells.  Since bestatin was found to inhibit aminopeptidase activity, the inhibition of tumor invasion by bestatin is likely to be assocd. with its action as an enzyme inhibitor.  Bestatin only slightly inhibited tumor cell plasmin activity, which can lead to the conversion of the latent collagenase to the active form, but this slight effect was not significant.  The zymog. of TCM from SN12M cells showed that the treatment of tumor cells with bestatin resulted in the disappearance of the 689 kDa type IV collagenase-enzyme level (active form) and slight redn. of the 72 kDa type IV collagenase-enzyme level (latent form).  These results indicated that bestatin may inhibit tumor cell invasion through a mechanism involving its inhibitory action on aminopeptidases in tumor cells, suggesting that the aminopeptidase may partly be assocd. with the conversion of a latent form of type IV procollagenase to an active form or the secretion of the collagenases from tumor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooBzxijM_O1rVg90H21EOLACvtfcHk0lgMCGf0kZ69dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XitVOnsbk%253D&md5=b6239f33e3bcf9169498553a53225a0a</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1007%2FBF00163576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00163576%26sid%3Dliteratum%253Aachs%26aulast%3DYoneda%26aufirst%3DJ.%26aulast%3DSaiki%26aufirst%3DI.%26aulast%3DFujii%26aufirst%3DH.%26aulast%3DAbe%26aufirst%3DF.%26aulast%3DKojima%26aufirst%3DY.%26aulast%3DAzuma%26aufirst%3DI.%26atitle%3DInhibition%2520of%2520tumor%2520invasion%2520and%2520extracellular%2520matrix%2520degradation%2520by%2520ubenimex%2520%2528bestatin%2529%26jtitle%3DClin.%2520Exp.%2520Metastasis%26date%3D1992%26volume%3D10%26spage%3D49%26epage%3D59%26doi%3D10.1007%2FBF00163576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mucha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalton, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kafarski, P.</span></span> <span> </span><span class="NLM_article-title">Metallo-aminopeptidase inhibitors</span>. <i>Biochimie</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">1509</span>– <span class="NLM_lpage">1529</span>, <span class="refDoi"> DOI: 10.1016/j.biochi.2010.04.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.biochi.2010.04.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=20457213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2isLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2010&pages=1509-1529&author=A.+Muchaauthor=M.+Dragauthor=J.+P.+Daltonauthor=P.+Kafarski&title=Metallo-aminopeptidase+inhibitors&doi=10.1016%2Fj.biochi.2010.04.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Metallo-aminopeptidase inhibitors</span></div><div class="casAuthors">Mucha, Artur; Drag, Marcin; Dalton, John P.; Kafarski, Pawel</div><div class="citationInfo"><span class="NLM_cas:title">Biochimie</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1509-1529</span>CODEN:
                <span class="NLM_cas:coden">BICMBE</span>;
        ISSN:<span class="NLM_cas:issn">0300-9084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Aminopeptidases are enzymes that selectively hydrolyze an amino acid residue from the N-terminus of proteins and peptides.  They are important for the proper functioning of prokaryotic and eukaryotic cells, but very often are central players in the devastating human diseases like cancer, malaria and diabetes.  The largest aminopeptidase group include enzymes contg. metal ion(s) in their active centers, which often dets. the type of inhibitors that are the most suitable for them.  Effective ligands mostly bind in a non-covalent mode by forming complexes with the metal ion(s).  Here, we present several approaches for the design of inhibitors for metallo-aminopeptidases.  The optimized structures should be considered as potential leads in the drug discovery process against endogenous and infectious diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGGCPbxscFkrVg90H21EOLACvtfcHk0lgMCGf0kZ69dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2isLrF&md5=5b9bb175448ac8e3058f1e4ad8f5654c</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.biochi.2010.04.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biochi.2010.04.026%26sid%3Dliteratum%253Aachs%26aulast%3DMucha%26aufirst%3DA.%26aulast%3DDrag%26aufirst%3DM.%26aulast%3DDalton%26aufirst%3DJ.%2BP.%26aulast%3DKafarski%26aufirst%3DP.%26atitle%3DMetallo-aminopeptidase%2520inhibitors%26jtitle%3DBiochimie%26date%3D2010%26volume%3D92%26spage%3D1509%26epage%3D1529%26doi%3D10.1016%2Fj.biochi.2010.04.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span> <span> </span><span class="NLM_article-title">Recent advance in aminopeptidase N (APN/CD13) inhibitor research</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">5011</span>– <span class="NLM_lpage">5021</span>, <span class="refDoi"> DOI: 10.2174/092986711797535155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.2174%2F092986711797535155" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=22050749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVOrtLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=5011-5021&author=X.+Zhangauthor=H.+Fangauthor=J.+Zhangauthor=Y.+Yuanauthor=W.+Xu&title=Recent+advance+in+aminopeptidase+N+%28APN%2FCD13%29+inhibitor+research&doi=10.2174%2F092986711797535155"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advance in aminopeptidase N (APN/CD13) inhibitor research</span></div><div class="casAuthors">Zhang, X.; Fang, H.; Zhang, J.; Yuan, Y.; Xu, W.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">5011-5021</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  APN is an important zinc dependent metallo-exopeptidase, which can degrade the extracellular matrix, and plays an important role in tumor invasion, metastasis and angiogenesis, it has been considered as a suitable target for anti-cancer drug design.  Recently, research of structure-based APN inhibitors is becoming an interesting field, and many APN inhibitors have been reported.  Here we review a series of APN inhibitors and the recent progress in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8lQD3shghIrVg90H21EOLACvtfcHk0lgMCGf0kZ69dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVOrtLrP&md5=76c3d93c00979eda9f2ccf67c9b31145</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.2174%2F092986711797535155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986711797535155%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DFang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DW.%26atitle%3DRecent%2520advance%2520in%2520aminopeptidase%2520N%2520%2528APN%252FCD13%2529%2520inhibitor%2520research%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2011%26volume%3D18%26spage%3D5011%26epage%3D5021%26doi%3D10.2174%2F092986711797535155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scornik, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botbol, V.</span></span> <span> </span><span class="NLM_article-title">Bestatin as an experimental tool in mammals</span>. <i>Curr. Drug Metab.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">85</span>, <span class="refDoi"> DOI: 10.2174/1389200013338748</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.2174%2F1389200013338748" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=11465152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD3MXisFWjurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2001&pages=67-85&author=O.+A.+Scornikauthor=V.+Botbol&title=Bestatin+as+an+experimental+tool+in+mammals&doi=10.2174%2F1389200013338748"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Bestatin as an experimental tool in mammals</span></div><div class="casAuthors">Scornik, O. A.; Botbol, V.</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Metabolism</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-85</span>CODEN:
                <span class="NLM_cas:coden">CDMUBU</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review with 163 refs.  Bestatin, an antibiotic of microbial origin, is a potent inhibitor of some, but not all aminopeptidases.  It can be administered, with low toxicity, to cultured cells, intact animals and humans.  It has become a useful tool in elucidating the physiol. role of some mammalian exopeptidases in the regulation of the immune system, in the growth of tumors and their invasion of surrounding tissues, and in the degrdn. of cellular proteins.  Bestatin-sensitive enzymes play important roles in the digestion and absorption of peptides in the brush border of the intestine and the kidney, in the reproductive system, and in the metab. of opioid peptides and leukotrienes.  Aminopeptidase N emerges as the major target for the effects of bestatin on the immune system and some of its effects on tumor growth and the endometrium.  It is also the major bestatin-sensitive enzyme involved in the degrdn. of oligopeptides on the surface of intestine and kidney brush borders, and the inactivation of enkephalins in the brain.  Bestatin-sensitive cytosolic exopeptidases are important in the degrdn. to amino acids of di- and tripeptides generated in most cells by cellular protein degrdn., as well as those absorbed through the brush border of intestine and kidney.  Inhibition of one of these exopeptidases, cytosol alanine aminopeptidase, results in apoptosis.  Bestatin-sensitive cystinyl aminopeptidase is abundant in placenta.  Two bestatin-sensitive enzymes, aminopeptidase B and nardilysin, are particularly abundant in late spermatids.  Finally bestatin-sensitive LTA4 hydrolase generates the potent chemotactic agent, LTB4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6UOKu-QtgUrVg90H21EOLACvtfcHk0lgYT-YDZAeXxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXisFWjurY%253D&md5=02986f26a62a737ffb23b19b5e38109f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.2174%2F1389200013338748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389200013338748%26sid%3Dliteratum%253Aachs%26aulast%3DScornik%26aufirst%3DO.%2BA.%26aulast%3DBotbol%26aufirst%3DV.%26atitle%3DBestatin%2520as%2520an%2520experimental%2520tool%2520in%2520mammals%26jtitle%3DCurr.%2520Drug%2520Metab.%26date%3D2001%26volume%3D2%26spage%3D67%26epage%3D85%26doi%3D10.2174%2F1389200013338748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Umezawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoyagi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeuchi, T.</span></span> <span> </span><span class="NLM_article-title">Bestatin, an inhibitor of aminopeptidase B, produced by actinomycetes</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">97</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.29.97</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.7164%2Fantibiotics.29.97" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=931798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaE28XktlWmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1976&pages=97-99&author=H.+Umezawaauthor=T.+Aoyagiauthor=H.+Sudaauthor=M.+Hamadaauthor=T.+Takeuchi&title=Bestatin%2C+an+inhibitor+of+aminopeptidase+B%2C+produced+by+actinomycetes&doi=10.7164%2Fantibiotics.29.97"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Bestatin, an inhibitor of aminopeptidase B, produced by actinomycetes</span></div><div class="casAuthors">Umezawa, Hamao; Aoyagi, Takaaki; Suda, Hiroyuki; Hamada, Masa; Takeuchi, Tomio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">97-9</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">Bestatin [58970-76-6], a specific inhibitor of aminopeptidase B (EC 3.4.11.6) [9073-92-1], was produced by Streptomyces olivoreticuli strain MD976-C7.  A typical medium contained maltose 1.5, yeast ext. 0.3, N-Z amine 1, and NaCl 0.3% (pH 7.4).  Max. prodn. was attained in 2-3 days of shaking at 27°, and prodn. was maintained for 2-6 days thereafter.  Bestatin was extd. from the culture filtrate by adsorption on activated C or Amberlite-XAD or by extn. with an org. solvent.  Bestatin also inhibited leucine aminopeptidase (EC 3.4.11.1) [9001-61-0].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6vWkQ-jPexbVg90H21EOLACvtfcHk0lgYT-YDZAeXxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XktlWmt7g%253D&md5=eefb781bea66a640c0967769a24ea231</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.29.97&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.29.97%26sid%3Dliteratum%253Aachs%26aulast%3DUmezawa%26aufirst%3DH.%26aulast%3DAoyagi%26aufirst%3DT.%26aulast%3DSuda%26aufirst%3DH.%26aulast%3DHamada%26aufirst%3DM.%26aulast%3DTakeuchi%26aufirst%3DT.%26atitle%3DBestatin%252C%2520an%2520inhibitor%2520of%2520aminopeptidase%2520B%252C%2520produced%2520by%2520actinomycetes%26jtitle%3DJ.%2520Antibiot.%26date%3D1976%26volume%3D29%26spage%3D97%26epage%3D99%26doi%3D10.7164%2Fantibiotics.29.97" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ichinose, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koike, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iioka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakuma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, M.</span></span> <span> </span><span class="NLM_article-title">Randomized double-blind placebo-controlled trial of bestatin in patients with resected stage I squamous-cell lung carcinoma</span>. <i>JNCI, J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">610</span>, <span class="refDoi"> DOI: 10.1093/jnci/95.8.605</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1093%2Fjnci%2F95.8.605" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=12697853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjtFWqsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2003&pages=605-610&author=Y.+Ichinoseauthor=K.+Genkaauthor=T.+Koikeauthor=H.+Katoauthor=Y.+Watanabeauthor=T.+Moriauthor=S.+Iiokaauthor=A.+Sakumaauthor=M.+Ohta&title=Randomized+double-blind+placebo-controlled+trial+of+bestatin+in+patients+with+resected+stage+I+squamous-cell+lung+carcinoma&doi=10.1093%2Fjnci%2F95.8.605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Randomized double-blind placebo-controlled trial of bestatin in patients with resected stage I squamous-cell lung carcinoma</span></div><div class="casAuthors">Ichinose, Yukito; Genka, Keiichiro; Koike, Teruaki; Kato, Harubumi; Watanabe, Yoh; Mori, Takashi; Iioka, Sogo; Sakuma, Akira; Ohta, Mitsuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">605-610</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Background: Bestatin is a potent aminopeptidase inhibitor that has immunostimulant and antitumor activity.  We conducted a prospective randomized, double-blind, placebo-controlled trial to det. whether postoperative adjuvant treatment with bestatin could prolong the survival of patients with completely resected stage I squamous-cell lung carcinoma.  Methods: Patients with confirmed, resected stage I squamous-cell lung carcinoma were randomly assigned to receive either bestatin (30 mg) or placebo daily by mouth for 2 yr.  We assessed whether bestatin treatment was assocd. with overall survival and 5-yr cancer-free survival and assessed its safety.  All statistical tests were two-sided.  Results: From July 8, 1992, through Mar. 30, 1995, 402 patients were entered in the study, 202 in the bestatin group and 198 in the placebo group.  The median follow-up for surviving patients was 76 mo (range = 58-92 mo).  The 5-yr overall survival was 81% in the bestatin group and 74% in the placebo group for a difference of 7% (95% confidence interval [CI] = -1.4% to 15.0%).  The 5-yr cancer-free survival was 71% in the bestatin group and 62% in the placebo group for a difference of 9% (95% CI = -0.7% to 17.8%).  Overall survival (P = .033, log-rank test) and cancer-free survival (P = .017, log-rank test) were statistically significantly different by Kaplan-Meier anal.  Few adverse events were obsd. in either group.  Conclusions: Survival was statistically significantly better for patients with completely resected stage I squamous-cell lung carcinoma who were treated with bestatin as a postoperative adjuvant therapy than for those who received a placebo.  This result requires confirmation in other phase III trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKPQhXgUTESbVg90H21EOLACvtfcHk0lgYT-YDZAeXxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjtFWqsb4%253D&md5=f794262982689c02a29e138a93d4e44f</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2F95.8.605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252F95.8.605%26sid%3Dliteratum%253Aachs%26aulast%3DIchinose%26aufirst%3DY.%26aulast%3DGenka%26aufirst%3DK.%26aulast%3DKoike%26aufirst%3DT.%26aulast%3DKato%26aufirst%3DH.%26aulast%3DWatanabe%26aufirst%3DY.%26aulast%3DMori%26aufirst%3DT.%26aulast%3DIioka%26aufirst%3DS.%26aulast%3DSakuma%26aufirst%3DA.%26aulast%3DOhta%26aufirst%3DM.%26atitle%3DRandomized%2520double-blind%2520placebo-controlled%2520trial%2520of%2520bestatin%2520in%2520patients%2520with%2520resected%2520stage%2520I%2520squamous-cell%2520lung%2520carcinoma%26jtitle%3DJNCI%252C%2520J.%2520Natl.%2520Cancer%2520Inst.%26date%3D2003%26volume%3D95%26spage%3D605%26epage%3D610%26doi%3D10.1093%2Fjnci%2F95.8.605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ota, K.</span></span> <span> </span><span class="NLM_article-title">Review of ubenimex (Bestatin): clinical research</span>. <i>Biomed. Pharmacother.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1016/0753-3322(91)90123-B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2F0753-3322%2891%2990123-B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1912370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A280%3ADyaK38%252Fgtlalsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=1991&pages=55-60&author=K.+Ota&title=Review+of+ubenimex+%28Bestatin%29%3A+clinical+research&doi=10.1016%2F0753-3322%2891%2990123-B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Review of ubenimex (Bestatin): clinical research</span></div><div class="casAuthors">Ota K</div><div class="citationInfo"><span class="NLM_cas:title">Biomedicine & pharmacotherapy = Biomedecine & pharmacotherapie</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">55-60</span>
        ISSN:<span class="NLM_cas:issn">0753-3322</span>.
    </div><div class="casAbstract">An immunomodulating agent, ubenimex (Bestatin) has low toxicity even after long-term oral administration and brings about significant modifications in immunological response.  In a cooperative randomized controlled study of Bestatin immunotherapy for adult acute nonlymphocytic leukemia, prolongation of remission duration and survival was achieved with Bestatin immunotherapy combined with remission maintenance chemotherapy.  The significant prolongation of remission duration and survival was seen in the Bestatin group, especially in the elderly patients.  Randomized controlled studies of Bestatin immunotherapy were performed in solid tumors including malignant melanoma, carcinoma of the lung, stomach, bladder, head and neck and esophagus, and therapeutic benefits regarding disease free-interval or survival were observed in certain types of the above-mentioned cancers; however, Bestatin immunotherapy for these cancers should be further investigated in large-scale controlled studies to confirm its activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQgjoNzGIjBCcScOqQTSkOffW6udTcc2eY2Q6NRcRRPm7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK38%252Fgtlalsw%253D%253D&md5=17a5625acdeaf93d830fd935bd3475ee</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2F0753-3322%2891%2990123-B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0753-3322%252891%252990123-B%26sid%3Dliteratum%253Aachs%26aulast%3DOta%26aufirst%3DK.%26atitle%3DReview%2520of%2520ubenimex%2520%2528Bestatin%2529%253A%2520clinical%2520research%26jtitle%3DBiomed.%2520Pharmacother.%26date%3D1991%26volume%3D45%26spage%3D55%26epage%3D60%26doi%3D10.1016%2F0753-3322%2891%2990123-B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ota, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurita, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masaoka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uzuka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogawa, N.</span></span> <span> </span><span class="NLM_article-title">Immunotherapy with bestatin for acute nonlymphocytic leukemia in adults</span>. <i>Cancer Immunol. Immunother.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1007/BF00205548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1007%2FBF00205548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=3533256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A280%3ADyaL2s%252FivVWjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1986&pages=5-10&author=K.+Otaauthor=S.+Kuritaauthor=K.+Yamadaauthor=T.+Masaokaauthor=Y.+Uzukaauthor=N.+Ogawa&title=Immunotherapy+with+bestatin+for+acute+nonlymphocytic+leukemia+in+adults&doi=10.1007%2FBF00205548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Immunotherapy with bestatin for acute nonlymphocytic leukemia in adults</span></div><div class="casAuthors">Ota K; Kurita S; Yamada K; Masaoka T; Uzuka Y; Ogawa N</div><div class="citationInfo"><span class="NLM_cas:title">Cancer immunology, immunotherapy : CII</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-10</span>
        ISSN:<span class="NLM_cas:issn">0340-7004</span>.
    </div><div class="casAbstract">In a cooperative randomized control study of immunotherapy with bestatin in combination with chemotherapy in adults with acute nonlymphocytic leukemia (ANLL), 101 patients (48 in the bestatin group and 53 in the control group) out of 115 patients registered were evaluated as eligible.  The bestatin group achieved a statistically significant prolongation of survival compared with the control group in overall ANLL and acute myelogenous leukemia.  In the analysis of patient age, the bestatin group achieved a statistically significant prolongation of both the remission duration and survival in patients aged 50 to 65 years, while the differences were not significant in the 15 to 49 age group.  The bestatin group tended to achieve a higher rate of reinduction of remission in patients who had recurrence of leukemia.  Side effects developed in only 5 (9.6%) of 52 patients treated with bestatin.  None of these side effects were particularly serious in nature.  It is concluded that bestatin is useful for prolongation of survival of adult patients with ANLL, making for a longer remission duration especially in elderly patients and with few side effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRj-k_x41rUPOqrn-c4Of3KfW6udTcc2eY2Q6NRcRRPm7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL2s%252FivVWjug%253D%253D&md5=23e6deab11cc762aa7df680ac674cdde</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1007%2FBF00205548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00205548%26sid%3Dliteratum%253Aachs%26aulast%3DOta%26aufirst%3DK.%26aulast%3DKurita%26aufirst%3DS.%26aulast%3DYamada%26aufirst%3DK.%26aulast%3DMasaoka%26aufirst%3DT.%26aulast%3DUzuka%26aufirst%3DY.%26aulast%3DOgawa%26aufirst%3DN.%26atitle%3DImmunotherapy%2520with%2520bestatin%2520for%2520acute%2520nonlymphocytic%2520leukemia%2520in%2520adults%26jtitle%3DCancer%2520Immunol.%2520Immunother.%26date%3D1986%26volume%3D23%26spage%3D5%26epage%3D10%26doi%3D10.1007%2FBF00205548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Urabe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mutoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizoguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takaku, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogawa, N.</span></span> <span> </span><span class="NLM_article-title">Ubenimex in the treatment of acute nonlymphocytic leukemia in adults</span>. <i>Ann. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1007/BF01788128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1007%2FBF01788128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=8347731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A280%3ADyaK3szltVSntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=1993&pages=63-66&author=A.+Urabeauthor=Y.+Mutohauthor=H.+Mizoguchiauthor=F.+Takakuauthor=N.+Ogawa&title=Ubenimex+in+the+treatment+of+acute+nonlymphocytic+leukemia+in+adults&doi=10.1007%2FBF01788128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Ubenimex in the treatment of acute nonlymphocytic leukemia in adults</span></div><div class="casAuthors">Urabe A; Mutoh Y; Mizoguchi H; Takaku F; Ogawa N</div><div class="citationInfo"><span class="NLM_cas:title">Annals of hematology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">63-6</span>
        ISSN:<span class="NLM_cas:issn">0939-5555</span>.
    </div><div class="casAbstract">A multi-institutional randomized study for the evaluation of ubenimex (Bestatin) in the treatment of adult acute nonlymphocytic leukemia was performed.  One hundred and ninety-five patients were registered from February 1988 to December 1990.  Patients who had reached complete remission by one or two courses of remission induction chemotherapy were divided into the ubenimex group or the control group by randomization.  Patients of the ubenimex group started to receive 30 mg ubenimex orally once a day when maintenance therapy began and continued as long as possible.  Remission duration and survival were analyzed based on the data as of August 31, 1991.  Remission duration of the ubenimex group was superior to that of the control group (generalized Wilcoxon test: p = 0.0338).  Fifty percent remission duration was 508 days in the ubenimex group and 386 days in the control group.  There has been no statistical difference in survival between the two groups as yet.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR-t0sTByak7WK_SFaaT9INfW6udTcc2eY2Q6NRcRRPm7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK3szltVSntQ%253D%253D&md5=4a8545655f9819fdbaef9b3bbd012d9c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2FBF01788128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01788128%26sid%3Dliteratum%253Aachs%26aulast%3DUrabe%26aufirst%3DA.%26aulast%3DMutoh%26aufirst%3DY.%26aulast%3DMizoguchi%26aufirst%3DH.%26aulast%3DTakaku%26aufirst%3DF.%26aulast%3DOgawa%26aufirst%3DN.%26atitle%3DUbenimex%2520in%2520the%2520treatment%2520of%2520acute%2520nonlymphocytic%2520leukemia%2520in%2520adults%26jtitle%3DAnn.%2520Hematol.%26date%3D1993%26volume%3D67%26spage%3D63%26epage%3D66%26doi%3D10.1007%2FBF01788128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krige, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Needham, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bawden, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flores, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farmer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miles, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callaghan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirwin-Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legris, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Box, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laber, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odedra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eccles, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bone, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayscough, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drummond, A. H.</span></span> <span> </span><span class="NLM_article-title">CHR-2797: an antiproliferative aminopeptidase inhibitor that leads to amino acid deprivation in human leukemic cells</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">6669</span>– <span class="NLM_lpage">6679</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-07-6627</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1158%2F0008-5472.CAN-07-6627" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=18701491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD1cXpslKms70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=6669-6679&author=D.+Krigeauthor=L.+A.+Needhamauthor=L.+J.+Bawdenauthor=N.+Floresauthor=H.+Farmerauthor=L.+E.+Milesauthor=E.+Stoneauthor=J.+Callaghanauthor=S.+Chandlerauthor=V.+L.+Clarkauthor=P.+Kirwin-Jonesauthor=V.+Legrisauthor=J.+Owenauthor=T.+Patelauthor=S.+Woodauthor=G.+Boxauthor=D.+Laberauthor=R.+Odedraauthor=A.+Wrightauthor=L.+M.+Woodauthor=S.+A.+Ecclesauthor=E.+A.+Boneauthor=A.+Ayscoughauthor=A.+H.+Drummond&title=CHR-2797%3A+an+antiproliferative+aminopeptidase+inhibitor+that+leads+to+amino+acid+deprivation+in+human+leukemic+cells&doi=10.1158%2F0008-5472.CAN-07-6627"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">CHR-2797: An Antiproliferative Aminopeptidase Inhibitor that Leads to Amino Acid Deprivation in Human Leukemic Cells</span></div><div class="casAuthors">Krige, David; Needham, Lindsey A.; Bawden, Lindsay J.; Flores, Nicolas; Farmer, Hannah; Miles, Lauren E. C.; Stone, Erica; Callaghan, Juliana; Chandler, Stephen; Clark, Vanessa L.; Kirwin-Jones, Patricia; Legris, Valerie; Owen, Jo; Patel, Thakor; Wood, Steve; Box, Gary; Laber, David; Odedra, Rajesh; Wright, Annette; Wood, L. Michael; Eccles, Suzanne A.; Bone, Elisabeth A.; Ayscough, Andrew; Drummond, Alan H.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6669-6679</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">CHR-2797 is a novel metalloenzyme inhibitor that is converted into a pharmacol. active acid product (CHR-79888) inside cells.  CHR-79888 is a potent inhibitor of a no. of intracellular aminopeptidases, including leucine aminopeptidase.  CHR-2797 exerts antiproliferative effects against a range of tumor cell lines in vitro and in vivo and shows selectivity for transformed over nontransformed cells.  Its antiproliferative effects are at least 300 times more potent than the prototypical aminopeptidase inhibitor, bestatin.  However, the mechanism by which inhibition of these enzymes leads to proliferative changes is not understood.  Gene expression microarrays were used to profile changes in mRNA expression levels in the human promyelocytic leukemia cell line HL-60 treated with CHR-2797.  This anal. showed that CHR-2797 treatment induced a transcriptional response indicative of amino acid depletion, the amino acid deprivation response, which involves up-regulation of amino acid synthetic genes, transporters, and tRNA synthetases.  These changes were confirmed in other leukemic cell lines sensitive to the antiproliferative effects of CHR-2797.  Furthermore, CHR-2797 treatment inhibited phosphorylation of mTOR substrates and reduced protein synthesis in HL-60 cells, both also indicative of amino acid depletion.  Treatment with CHR-2797 led to an increase in the concn. of intracellular small peptides, the substrates of aminopeptidases.  It is suggested that aminopeptidase inhibitors, such as CHR-2797 and bestatin, deplete sensitive tumor cells of amino acids by blocking protein recycling, and this generates an antiproliferative effect.  CHR-2797 is orally bioavailable and currently undergoing phase II clin. investigation in the treatment of myeloid leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmWE9oaA_yI7Vg90H21EOLACvtfcHk0lhWoG2gkUse3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXpslKms70%253D&md5=e116fdd3ec63512bb64c45001ece3351</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-07-6627&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-07-6627%26sid%3Dliteratum%253Aachs%26aulast%3DKrige%26aufirst%3DD.%26aulast%3DNeedham%26aufirst%3DL.%2BA.%26aulast%3DBawden%26aufirst%3DL.%2BJ.%26aulast%3DFlores%26aufirst%3DN.%26aulast%3DFarmer%26aufirst%3DH.%26aulast%3DMiles%26aufirst%3DL.%2BE.%26aulast%3DStone%26aufirst%3DE.%26aulast%3DCallaghan%26aufirst%3DJ.%26aulast%3DChandler%26aufirst%3DS.%26aulast%3DClark%26aufirst%3DV.%2BL.%26aulast%3DKirwin-Jones%26aufirst%3DP.%26aulast%3DLegris%26aufirst%3DV.%26aulast%3DOwen%26aufirst%3DJ.%26aulast%3DPatel%26aufirst%3DT.%26aulast%3DWood%26aufirst%3DS.%26aulast%3DBox%26aufirst%3DG.%26aulast%3DLaber%26aufirst%3DD.%26aulast%3DOdedra%26aufirst%3DR.%26aulast%3DWright%26aufirst%3DA.%26aulast%3DWood%26aufirst%3DL.%2BM.%26aulast%3DEccles%26aufirst%3DS.%2BA.%26aulast%3DBone%26aufirst%3DE.%2BA.%26aulast%3DAyscough%26aufirst%3DA.%26aulast%3DDrummond%26aufirst%3DA.%2BH.%26atitle%3DCHR-2797%253A%2520an%2520antiproliferative%2520aminopeptidase%2520inhibitor%2520that%2520leads%2520to%2520amino%2520acid%2520deprivation%2520in%2520human%2520leukemic%2520cells%26jtitle%3DCancer%2520Res.%26date%3D2008%26volume%3D68%26spage%3D6669%26epage%3D6679%26doi%3D10.1158%2F0008-5472.CAN-07-6627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lowenberg, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ossenkoppele, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burnett, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zachee, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duhrsen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dierickx, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller-Tidow, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonneveld, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krug, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bone, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flores, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hooftman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zweegman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, F.</span></span> <span> </span><span class="NLM_article-title">Phase I/II clinical study of Tosedostat, an inhibitor of aminopeptidases, in patients with acute myeloid leukemia and myelodysplasia</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">4333</span>– <span class="NLM_lpage">4338</span>, <span class="refDoi"> DOI: 10.1200/JCO.2009.27.6295</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1200%2FJCO.2009.27.6295" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=20733120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A280%3ADC%252BC3cflvFOntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2010&pages=4333-4338&author=B.+Lowenbergauthor=G.+Morganauthor=G.+J.+Ossenkoppeleauthor=A.+K.+Burnettauthor=P.+Zacheeauthor=U.+Duhrsenauthor=D.+Dierickxauthor=C.+Muller-Tidowauthor=P.+Sonneveldauthor=U.+Krugauthor=E.+Boneauthor=N.+Floresauthor=A.+F.+Richardsonauthor=L.+Hooftmanauthor=C.+Jenkinsauthor=S.+Zweegmanauthor=F.+Davies&title=Phase+I%2FII+clinical+study+of+Tosedostat%2C+an+inhibitor+of+aminopeptidases%2C+in+patients+with+acute+myeloid+leukemia+and+myelodysplasia&doi=10.1200%2FJCO.2009.27.6295"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Phase I/II clinical study of Tosedostat, an inhibitor of aminopeptidases, in patients with acute myeloid leukemia and myelodysplasia</span></div><div class="casAuthors">Lowenberg Bob; Morgan Gareth; Ossenkoppele Gert J; Burnett Alan K; Zachee Pierre; Duhrsen Ulrich; Dierickx Daan; Muller-Tidow Carsten; Sonneveld Pieter; Krug Utz; Bone Elisabeth; Flores Nicolas; Richardson Alison F; Hooftman Leon; Jenkins Chris; Zweegman Sonja; Davies Faith</div><div class="citationInfo"><span class="NLM_cas:title">Journal of clinical oncology : official journal of the American Society of Clinical Oncology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">4333-8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">PURPOSE:  To identify the maximum-tolerated dose (MTD) and to evaluate the antileukemic activity of tosedostat (formerly CHR-2797), an orally bioavailable aminopeptidase inhibitor.  PATIENTS AND METHODS:  In phase I, the MTD of once daily oral doses of tosedostat in hematologic malignancies was defined.  In phase II, the therapeutic activity of the maximum-acceptable dose (MAD) of tosedostat was evaluated in elderly and/or relapsing patients with acute myeloid leukemia (AML) or myelodysplastic syndrome.  RESULTS:  In phase I, 16 patients were treated in four cohorts with tosedostat (60 mg to 180 mg) for 28 days.  Three patients reported dose-limiting toxicities: two with reversible thrombocytopenia (> 75% reduction in platelet count) at 180 mg (MTD) and one with a Common Toxicity Criteria (CTC) grade 3 ALT elevation at 130 mg (MAD).  In phase II, 41 patients were treated with 130 mg tosedostat.  In phases I and II, the most common severe (CTC grades 3 to 5) adverse event was a reduction in the platelet count.  Of the 51 AML patients in this study, seven reached complete marrow response (< 5% marrow blasts), with three achieving complete remission, and a further seven patients reaching a partial marrow response (between 5% and 15% marrow blasts).  The overall response rate was therefore 27%.  All responders were age > 60 years, and 79% had either relapsed or refractory AML.  CONCLUSION:  This phase I/II study demonstrates that oral once daily dosing with 130 mg tosedostat is well tolerated and has significant antileukemic activity.  The favorable risk-benefit profile suggests that further clinical trials are warranted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSC4BwIVFbbZpPq5LRczm8ffW6udTcc2ebk--AzBcQDW7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cflvFOntQ%253D%253D&md5=f1209aab57f7448877dba45b96d4c4b1</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1200%2FJCO.2009.27.6295&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2009.27.6295%26sid%3Dliteratum%253Aachs%26aulast%3DLowenberg%26aufirst%3DB.%26aulast%3DMorgan%26aufirst%3DG.%26aulast%3DOssenkoppele%26aufirst%3DG.%2BJ.%26aulast%3DBurnett%26aufirst%3DA.%2BK.%26aulast%3DZachee%26aufirst%3DP.%26aulast%3DDuhrsen%26aufirst%3DU.%26aulast%3DDierickx%26aufirst%3DD.%26aulast%3DMuller-Tidow%26aufirst%3DC.%26aulast%3DSonneveld%26aufirst%3DP.%26aulast%3DKrug%26aufirst%3DU.%26aulast%3DBone%26aufirst%3DE.%26aulast%3DFlores%26aufirst%3DN.%26aulast%3DRichardson%26aufirst%3DA.%2BF.%26aulast%3DHooftman%26aufirst%3DL.%26aulast%3DJenkins%26aufirst%3DC.%26aulast%3DZweegman%26aufirst%3DS.%26aulast%3DDavies%26aufirst%3DF.%26atitle%3DPhase%2520I%252FII%2520clinical%2520study%2520of%2520Tosedostat%252C%2520an%2520inhibitor%2520of%2520aminopeptidases%252C%2520in%2520patients%2520with%2520acute%2520myeloid%2520leukemia%2520and%2520myelodysplasia%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2010%26volume%3D28%26spage%3D4333%26epage%3D4338%26doi%3D10.1200%2FJCO.2009.27.6295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drinkwater, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinh, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamert, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggeri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holleran, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loganathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paiardini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avery, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGowan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scammells, P. J.</span></span> <span> </span><span class="NLM_article-title">Potent dual inhibitors of <i>Plasmodium falciparum</i> M1 and M17 aminopeptidases through optimization of S1 pocket interactions</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">64</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.01.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.ejmech.2016.01.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=26807544" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtl2mur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2016&pages=43-64&author=N.+Drinkwaterauthor=N.+B.+Vinhauthor=S.+N.+Mistryauthor=R.+S.+Bamertauthor=C.+Ruggeriauthor=J.+P.+Holleranauthor=S.+Loganathanauthor=A.+Paiardiniauthor=S.+A.+Charmanauthor=A.+K.+Powellauthor=V.+M.+Averyauthor=S.+McGowanauthor=P.+J.+Scammells&title=Potent+dual+inhibitors+of+Plasmodium+falciparum+M1+and+M17+aminopeptidases+through+optimization+of+S1+pocket+interactions&doi=10.1016%2Fj.ejmech.2016.01.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Potent dual inhibitors of Plasmodium falciparum M1 and M17 aminopeptidases through optimization of S1 pocket interactions</span></div><div class="casAuthors">Drinkwater, Nyssa; Vinh, Natalie B.; Mistry, Shailesh N.; Bamert, Rebecca S.; Ruggeri, Chiara; Holleran, John P.; Loganathan, Sasdekumar; Paiardini, Alessandro; Charman, Susan A.; Powell, Andrew K.; Avery, Vicky M.; McGowan, Sheena; Scammells, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43-64</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Malaria remains a global health problem, and though international efforts for treatment and eradication have made some headway, the emergence of drug-resistant parasites threatens this progress.  Antimalarial therapeutics acting via novel mechanisms are urgently required.  Plasmodium falciparum M1 and M17 are neutral aminopeptidases which are essential for parasite growth and development.  Previous work in our group has identified inhibitors capable of dual inhibition of PfA-M1 and PfA-M17, and revealed further regions within the protease S1 pockets that could be exploited in the development of ligands with improved inhibitory activity.  Herein, we report the structure-based design and synthesis of novel hydroxamic acid analogs that are capable of potent inhibition of both PfA-M1 and PfA-M17.  Furthermore, the developed compds. potently inhibit Pf growth in culture, including the multi-drug resistant strain Dd2.  The ongoing development of dual PfA-M1/PfA-M17 inhibitors continues to be an attractive strategy for the design of novel antimalarial therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWA9kTGB140LVg90H21EOLACvtfcHk0lhWoG2gkUse3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtl2mur8%253D&md5=0ca2a60568b2469364c5a75070d67da2</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.01.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.01.015%26sid%3Dliteratum%253Aachs%26aulast%3DDrinkwater%26aufirst%3DN.%26aulast%3DVinh%26aufirst%3DN.%2BB.%26aulast%3DMistry%26aufirst%3DS.%2BN.%26aulast%3DBamert%26aufirst%3DR.%2BS.%26aulast%3DRuggeri%26aufirst%3DC.%26aulast%3DHolleran%26aufirst%3DJ.%2BP.%26aulast%3DLoganathan%26aufirst%3DS.%26aulast%3DPaiardini%26aufirst%3DA.%26aulast%3DCharman%26aufirst%3DS.%2BA.%26aulast%3DPowell%26aufirst%3DA.%2BK.%26aulast%3DAvery%26aufirst%3DV.%2BM.%26aulast%3DMcGowan%26aufirst%3DS.%26aulast%3DScammells%26aufirst%3DP.%2BJ.%26atitle%3DPotent%2520dual%2520inhibitors%2520of%2520Plasmodium%2520falciparum%2520M1%2520and%2520M17%2520aminopeptidases%2520through%2520optimization%2520of%2520S1%2520pocket%2520interactions%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D110%26spage%3D43%26epage%3D64%26doi%3D10.1016%2Fj.ejmech.2016.01.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kannan
Sivaraman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paiardini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sienczyk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggeri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oellig, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalton, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scammells, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGowan, S.</span></span> <span> </span><span class="NLM_article-title">Synthesis and structure-activity relationships of phosphonic arginine mimetics as inhibitors of the M1 and M17 aminopeptidases from <i>Plasmodium falciparum</i></span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5213</span>– <span class="NLM_lpage">5217</span>, <span class="refDoi"> DOI: 10.1021/jm4005972</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4005972" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A280%3ADC%252BC3snns1yjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=5213-5217&author=K.+Kannan%0ASivaramanauthor=A.+Paiardiniauthor=M.+Sienczykauthor=C.+Ruggeriauthor=C.+A.+Oelligauthor=J.+P.+Daltonauthor=P.+J.+Scammellsauthor=M.+Dragauthor=S.+McGowan&title=Synthesis+and+structure-activity+relationships+of+phosphonic+arginine+mimetics+as+inhibitors+of+the+M1+and+M17+aminopeptidases+from+Plasmodium+falciparum&doi=10.1021%2Fjm4005972"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and structure-activity relationships of phosphonic arginine mimetics as inhibitors of the M1 and M17 aminopeptidases from Plasmodium falciparum</span></div><div class="casAuthors">Kannan Sivaraman Komagal; Paiardini Alessandro; Sienczyk Marcin; Ruggeri Chiara; Oellig Christine A; Dalton John P; Scammells Peter J; Drag Marcin; McGowan Sheena</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5213-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The malaria parasite Plasmodium falciparum employs two metallo-aminopeptidases, PfA-M1 and PfA-M17, which are essential for parasite survival.  Compounds that inhibit the activity of either enzyme represent leads for the development of new antimalarial drugs.  Here we report the synthesis and structure-activity relationships of a small library of phosphonic acid arginine mimetics that probe the S1 pocket of both enzymes and map the necessary interactions that would be important for a dual inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTyoiyTPibjo4IRAdBLKY41fW6udTcc2eaKS9C_Sr645Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3snns1yjsw%253D%253D&md5=7a25fc0da999cc98324351269f8df212</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm4005972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4005972%26sid%3Dliteratum%253Aachs%26aulast%3DKannan%2BSivaraman%26aufirst%3DK.%26aulast%3DPaiardini%26aufirst%3DA.%26aulast%3DSienczyk%26aufirst%3DM.%26aulast%3DRuggeri%26aufirst%3DC.%26aulast%3DOellig%26aufirst%3DC.%2BA.%26aulast%3DDalton%26aufirst%3DJ.%2BP.%26aulast%3DScammells%26aufirst%3DP.%2BJ.%26aulast%3DDrag%26aufirst%3DM.%26aulast%3DMcGowan%26aufirst%3DS.%26atitle%3DSynthesis%2520and%2520structure-activity%2520relationships%2520of%2520phosphonic%2520arginine%2520mimetics%2520as%2520inhibitors%2520of%2520the%2520M1%2520and%2520M17%2520aminopeptidases%2520from%2520Plasmodium%2520falciparum%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D5213%26epage%3D5217%26doi%3D10.1021%2Fjm4005972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drinkwater, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggeri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivaraman, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loganathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paiardini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avery, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scammells, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGowan, S.</span></span> <span> </span><span class="NLM_article-title">Two-pronged attack: dual inhibition of <i>Plasmodium falciparum</i> M1 and M17 metalloaminopeptidases by a novel series of hydroxamic acid-based inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9168</span>– <span class="NLM_lpage">9183</span>, <span class="refDoi"> DOI: 10.1021/jm501323a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501323a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslahs77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9168-9183&author=S.+N.+Mistryauthor=N.+Drinkwaterauthor=C.+Ruggeriauthor=K.+K.+Sivaramanauthor=S.+Loganathanauthor=S.+Fletcherauthor=M.+Dragauthor=A.+Paiardiniauthor=V.+M.+Averyauthor=P.+J.+Scammellsauthor=S.+McGowan&title=Two-pronged+attack%3A+dual+inhibition+of+Plasmodium+falciparum+M1+and+M17+metalloaminopeptidases+by+a+novel+series+of+hydroxamic+acid-based+inhibitors&doi=10.1021%2Fjm501323a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Two-Pronged Attack: Dual Inhibition of Plasmodium falciparum M1 and M17 Metalloaminopeptidases by a Novel Series of Hydroxamic Acid-Based Inhibitors</span></div><div class="casAuthors">Mistry, Shailesh N.; Drinkwater, Nyssa; Ruggeri, Chiara; Sivaraman, Komagal Kannan; Loganathan, Sasdekumar; Fletcher, Sabine; Drag, Marcin; Paiardini, Alessandro; Avery, Vicky M.; Scammells, Peter J.; McGowan, Sheena</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9168-9183</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Plasmodium parasites, the causative agents of malaria, have developed resistance to most of the current antimalarial therapies, including artemisinin combination therapies which are widely described as the last line of defense.  Antimalarial agents with a novel mode of action are urgently required.  Two Plasmodium falciparum aminopeptidases, PfA-M1 and PfA-M17, play crucial roles in the erythrocytic stage of infection and have been validated as potential antimalarial targets.  Using compd.-bound crystal structures of both enzymes, the authors have used a structure-guided approach to develop a novel series of inhibitors capable of potent inhibition of both PfA-M1 and PfA-M17 activity and parasite growth in culture.  Herein the authors describe the design, synthesis, and evaluation of a series of hydroxamic acid-based inhibitors I [R1 = PhCO, 2-FC6H4CO, 3-H2NC6H4CO, etc.; R2 = HO, HONH] and demonstrate the compds. to be exciting new leads for the development of novel antimalarial therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTJUvJd2YOGLVg90H21EOLACvtfcHk0liBQMqkHFxZaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslahs77E&md5=6b6fe308bc3b434d686268e65014d9c1</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm501323a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501323a%26sid%3Dliteratum%253Aachs%26aulast%3DMistry%26aufirst%3DS.%2BN.%26aulast%3DDrinkwater%26aufirst%3DN.%26aulast%3DRuggeri%26aufirst%3DC.%26aulast%3DSivaraman%26aufirst%3DK.%2BK.%26aulast%3DLoganathan%26aufirst%3DS.%26aulast%3DFletcher%26aufirst%3DS.%26aulast%3DDrag%26aufirst%3DM.%26aulast%3DPaiardini%26aufirst%3DA.%26aulast%3DAvery%26aufirst%3DV.%2BM.%26aulast%3DScammells%26aufirst%3DP.%2BJ.%26aulast%3DMcGowan%26aufirst%3DS.%26atitle%3DTwo-pronged%2520attack%253A%2520dual%2520inhibition%2520of%2520Plasmodium%2520falciparum%2520M1%2520and%2520M17%2520metalloaminopeptidases%2520by%2520a%2520novel%2520series%2520of%2520hydroxamic%2520acid-based%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9168%26epage%3D9183%26doi%3D10.1021%2Fjm501323a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, K. M.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Metal ion modeling using classical mechanics</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">1564</span>– <span class="NLM_lpage">1686</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00440</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00440" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvVSg" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2017&pages=1564-1686&author=P.+Liauthor=K.+M.+Merz&title=Metal+ion+modeling+using+classical+mechanics&doi=10.1021%2Facs.chemrev.6b00440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Metal Ion Modeling Using Classical Mechanics</span></div><div class="casAuthors">Li, Pengfei; Merz, Kenneth M., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1564-1686</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Metal ions play significant roles in numerous fields including chem., geochem., biochem. and materials science.  With computational tools increasingly becoming important in chem. research, methods have emerged to effectively face the challenge of modeling metal ions in the gas, aq. and solid phases.  Herein we review both quantum and classical modeling strategies for metal ion contg. systems that have been developed over the past few decades.  This review focuses on classical metal ion modeling based on unpolarized models (including the nonbonded, bonded, cationic dummy atom, and combined models), polarizable models (e.g., the fluctuating charge, Drude oscillator, and the induced dipole models), the angular overlap model, and valence bond based models.  Quantum mech. studies of metal ion contg. systems at the semiempirical, ab initio and d. functional levels of theory are reviewed as well with a particular focus on how these methods inform classical modeling efforts.  Finally, conclusions and future prospects and directions are offered that will further enhance the classical modeling of metal ion contg. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5FykOPt7hi7Vg90H21EOLACvtfcHk0liBQMqkHFxZaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvVSg&md5=1cd2a84bd580b3b4e3493bfdd4bc4da1</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00440%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DP.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26atitle%3DMetal%2520ion%2520modeling%2520using%2520classical%2520mechanics%26jtitle%3DChem.%2520Rev.%26date%3D2017%26volume%3D117%26spage%3D1564%26epage%3D1686%26doi%3D10.1021%2Facs.chemrev.6b00440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porebski, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kass, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckle, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGowan, S.</span></span> <span> </span><span class="NLM_article-title">Generation of AMBER force field parameters for zinc centres of M1 and M17 family aminopeptidases</span>. <i>J. Biomol. Struct. Dyn.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_fpage">2595</span>– <span class="NLM_lpage">2604</span>, <span class="refDoi"> DOI: 10.1080/07391102.2017.1364669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1080%2F07391102.2017.1364669" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=28782419" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVans7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=2595-2604&author=W.+Yangauthor=B.+T.+Rileyauthor=X.+Leiauthor=B.+T.+Porebskiauthor=I.+Kassauthor=A.+M.+Buckleauthor=S.+McGowan&title=Generation+of+AMBER+force+field+parameters+for+zinc+centres+of+M1+and+M17+family+aminopeptidases&doi=10.1080%2F07391102.2017.1364669"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Generation of AMBER force field parameters for zinc centres of M1 and M17 family aminopeptidases</span></div><div class="casAuthors">Yang, Wei; Riley, Blake T.; Lei, Xiangyun; Porebski, Benjamin T.; Kass, Itamar; Buckle, Ashley M.; McGowan, Sheena</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Structure and Dynamics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2595-2604</span>CODEN:
                <span class="NLM_cas:coden">JBSDD6</span>;
        ISSN:<span class="NLM_cas:issn">0739-1102</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">The M1 and M17 aminopeptidases are metallo-exopeptidases that rely on the presence of divalent cations, usually zinc, in their active site for proteolytic activity.  They are from sep. protease superfamilies, however, members often have overlapping substrate specificity.  Inhibitors of one or both enzymes can be used to modulate hypertension, reduce proliferation of certain types of cancers and control malaria parasites.  Current inhibitors act to chelate the zinc ions in the active site, locking the enzymes in an inactive transition state.  We were interested in using a computational approach to understand the structure and dynamics of the M1 and M17 aminopeptidases, however, the presence of the essential metal ions in the proteases presents a challenge to classical mol. dynamics (MD) simulation.  The zinc amber force field does not contain applicable descriptions of the zinc coordination environment present in either of these two protease families.  To provide tools for the study of these two enzymes, we have used the metal center parameter builder to generate new hybrid bonded/nonbonded force field (FF) parameters to correctly describe the active site architecture for each enzyme.  The new parameters were evaluated by fitting the normal mode frequencies of mol. mechanics to the quantum mechanics frequencies and validated by performing short MD simulations.  The new FF parameters now enable more accurate and reliable MD simulations for any member of the M1 or M17 aminopeptidase superfamilies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql0MSrgIHsmLVg90H21EOLACvtfcHk0lhjxvd1EdHclA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVans7fP&md5=b47e57600f925e5bf8e9b7053743c7cc</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1080%2F07391102.2017.1364669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F07391102.2017.1364669%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DW.%26aulast%3DRiley%26aufirst%3DB.%2BT.%26aulast%3DLei%26aufirst%3DX.%26aulast%3DPorebski%26aufirst%3DB.%2BT.%26aulast%3DKass%26aufirst%3DI.%26aulast%3DBuckle%26aufirst%3DA.%2BM.%26aulast%3DMcGowan%26aufirst%3DS.%26atitle%3DGeneration%2520of%2520AMBER%2520force%2520field%2520parameters%2520for%2520zinc%2520centres%2520of%2520M1%2520and%2520M17%2520family%2520aminopeptidases%26jtitle%3DJ.%2520Biomol.%2520Struct.%2520Dyn.%26date%3D2018%26spage%3D2595%26epage%3D2604%26doi%3D10.1080%2F07391102.2017.1364669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porebski, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kass, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckle, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGowan, S.</span></span> <span> </span><span class="NLM_article-title">Mapping the pathway and dynamics of bestatin inhibition of the <i>Plasmodium falciparum</i> M1 aminopeptidase, PfA-M1</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2504</span>– <span class="NLM_lpage">2513</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201800563</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1002%2Fcmdc.201800563" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=30318749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFejt7nO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=2504-2513&author=W.+Yangauthor=B.+Rileyauthor=X.+Leiauthor=B.+T.+Porebskiauthor=I.+Kassauthor=A.+M.+Buckleauthor=S.+McGowan&title=Mapping+the+pathway+and+dynamics+of+bestatin+inhibition+of+the+Plasmodium+falciparum+M1+aminopeptidase%2C+PfA-M1&doi=10.1002%2Fcmdc.201800563"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Mapping the Pathway and Dynamics of Bestatin Inhibition of the Plasmodium falciparum M1 Aminopeptidase PfA-M1</span></div><div class="casAuthors">Yang, Wei; Riley, Blake T.; Lei, Xiangyun; Porebski, Benjamin T.; Kass, Itamar; Buckle, Ashley M.; McGowan, Sheena</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2504-2513</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The M1 metallo-aminopeptidase from Plasmodium falciparum, PfA-M1, is an attractive drug target for the design of new antimalarials.  Bestatin, a broad-spectrum metalloprotease inhibitor, is a moderate inhibitor of PfA-M1, and has been used to provide structure-activity relationships to inform drug design.  The crystal structure of PfA-M1 with bestatin bound within its active site has been detd.; however, dynamics of the inhibitor and the assocn. or dissocn. pathway have yet to be characterized.  Here we present an all-atom mol. dynamics study where we have generated a hidden Markov state model from 2.3 μs of mol. dynamics simulation.  Our hidden Markov state model identifies five macrostates that clearly show the events involved in bestatin dissocn. from the PfA-M1 active site.  The results show for the first time that bestatin can escape the substrate specificity pockets of the enzyme, primarily due to weak interactions within the pockets.  Our approach identifies relevant conformational sampling of the inhibitor inside the enzyme and the protein dynamics that could be exploited to produce potent and selective inhibitors that can differentiate between similar members of the M1 aminopeptidase superfamily.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpM9tv3inDxhrVg90H21EOLACvtfcHk0lhjxvd1EdHclA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFejt7nO&md5=b9c9757cb142807f3dc986123f71bbd9</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201800563&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201800563%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DW.%26aulast%3DRiley%26aufirst%3DB.%26aulast%3DLei%26aufirst%3DX.%26aulast%3DPorebski%26aufirst%3DB.%2BT.%26aulast%3DKass%26aufirst%3DI.%26aulast%3DBuckle%26aufirst%3DA.%2BM.%26aulast%3DMcGowan%26aufirst%3DS.%26atitle%3DMapping%2520the%2520pathway%2520and%2520dynamics%2520of%2520bestatin%2520inhibition%2520of%2520the%2520Plasmodium%2520falciparum%2520M1%2520aminopeptidase%252C%2520PfA-M1%26jtitle%3DChemMedChem%26date%3D2018%26volume%3D13%26spage%3D2504%26epage%3D2513%26doi%3D10.1002%2Fcmdc.201800563" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beebe, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilius, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merta, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soni, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beutel, B. A.</span></span> <span> </span><span class="NLM_article-title">Synthesis and SAR evaluation of oxadiazolopyrazines as selective Haemophilus influenzae antibacterial agents</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">3133</span>– <span class="NLM_lpage">3136</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(03)00727-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2FS0960-894X%2803%2900727-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=12951079" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvV2lsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=3133-3136&author=X.+Beebeauthor=A.+M.+Niliusauthor=P.+J.+Mertaauthor=N.+B.+Soniauthor=M.+H.+Buiauthor=R.+Wagnerauthor=B.+A.+Beutel&title=Synthesis+and+SAR+evaluation+of+oxadiazolopyrazines+as+selective+Haemophilus+influenzae+antibacterial+agents&doi=10.1016%2FS0960-894X%2803%2900727-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and SAR evaluation of oxadiazolopyrazines as selective Haemophilus influenzae antibacterial agents</span></div><div class="casAuthors">Beebe, Xenia; Nilius, Angela M.; Merta, Philip J.; Soni, Niru B.; Bui, Mai H.; Wagner, Rolf; Beutel, Bruce A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3133-3136</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">The parallel synthesis and antibacterial activity of 5-hydroxy-1,2,5-oxadiazolo[3,4-b]pyrazines, e.g., I, is reported.  The compds. were synthesized by condensing diaminofurazan (II) with α-keto acids to give a variety of aryl-substituted analogs.  Halogenated Ph groups at C-6 give rise to the greatest Haemophilus influenzae antibacterial activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXyJKZ9hzxF7Vg90H21EOLACvtfcHk0lhjxvd1EdHclA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvV2lsLk%253D&md5=a64b56982c86d0e7a16897346c203a03</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2803%2900727-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252803%252900727-3%26sid%3Dliteratum%253Aachs%26aulast%3DBeebe%26aufirst%3DX.%26aulast%3DNilius%26aufirst%3DA.%2BM.%26aulast%3DMerta%26aufirst%3DP.%2BJ.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DWagner%26aufirst%3DR.%26aulast%3DBeutel%26aufirst%3DB.%2BA.%26atitle%3DSynthesis%2520and%2520SAR%2520evaluation%2520of%2520oxadiazolopyrazines%2520as%2520selective%2520Haemophilus%2520influenzae%2520antibacterial%2520agents%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2003%26volume%3D13%26spage%3D3133%26epage%3D3136%26doi%3D10.1016%2FS0960-894X%2803%2900727-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adediran, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabaret, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohier, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakselman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, R. F.</span></span> <span> </span><span class="NLM_article-title">Substituted aryl malonamates as new serine beta-lactamase substrates: structure-activity studies</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">282</span>– <span class="NLM_lpage">291</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2009.10.056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.bmc.2009.10.056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=19932622" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFykurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=282-291&author=S.+A.+Adediranauthor=D.+Cabaretauthor=J.+F.+Lohierauthor=M.+Wakselmanauthor=R.+F.+Pratt&title=Substituted+aryl+malonamates+as+new+serine+beta-lactamase+substrates%3A+structure-activity+studies&doi=10.1016%2Fj.bmc.2009.10.056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Substituted aryl malonamates as new serine β-lactamase substrates: Structure-activity studies</span></div><div class="casAuthors">Adediran, S. A.; Cabaret, D.; Lohier, J.-F.; Wakselman, M.; Pratt, R. F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">282-291</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of substituted aryl malonamates have been prepd.  These compds. are analogs of aryl phenaceturates where the amido side chain has been replaced by a retro-amide.  Like the phenaceturates, these compds. are substrates of typical class A and class C β-lactamases, particularly of the latter, and of sol. DD-peptidases.  The effect of substituents α to the ester carbonyl group on turnover by these enzymes is similar to that in the phenaceturates.  On the other hand, N-alkylation of the side chain amide of malonamates, but not of phenaceturates, retains the susceptibility of the compds. to hydrolysis by β-lactamases.  This reactivity is not enhanced, however, by bridging the amide nitrogen and Cα atoms.  A phosphonate analog of the malonamates was found to be an irreversible inhibitor of the β-lactamases.  These results, therefore, provide further evidence for the covalent access of compds. bearing retro-amide side chains to the active sites of β-lactam-recognizing enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMbLiZVPEKNbVg90H21EOLACvtfcHk0lhIdKc6p0b3SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFykurvK&md5=82f87374778b0963de10dec3d4d0bcdc</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2009.10.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2009.10.056%26sid%3Dliteratum%253Aachs%26aulast%3DAdediran%26aufirst%3DS.%2BA.%26aulast%3DCabaret%26aufirst%3DD.%26aulast%3DLohier%26aufirst%3DJ.%2BF.%26aulast%3DWakselman%26aufirst%3DM.%26aulast%3DPratt%26aufirst%3DR.%2BF.%26atitle%3DSubstituted%2520aryl%2520malonamates%2520as%2520new%2520serine%2520beta-lactamase%2520substrates%253A%2520structure-activity%2520studies%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2010%26volume%3D18%26spage%3D282%26epage%3D291%26doi%3D10.1016%2Fj.bmc.2009.10.056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Im, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. C.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationships of heteroaromatic esters as human rhinovirus 3C protease inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">3632</span>– <span class="NLM_lpage">3636</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2009.04.114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.bmcl.2009.04.114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=19464175" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmvFGisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=3632-3636&author=I.+Imauthor=E.+S.+Leeauthor=S.+J.+Choiauthor=J.+Y.+Leeauthor=Y.+C.+Kim&title=Structure-activity+relationships+of+heteroaromatic+esters+as+human+rhinovirus+3C+protease+inhibitors&doi=10.1016%2Fj.bmcl.2009.04.114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity relationships of heteroaromatic esters as human rhinovirus 3C protease inhibitors</span></div><div class="casAuthors">Im, Isak; Lee, Eui Seung; Choi, Soo Jeong; Lee, Ju-Yeon; Kim, Yong-Chul</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3632-3636</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Human rhinovirus 3C protease (HRV 3Cpro) is known to be a promising target for development of therapeutic agents against the common cold because of the importance of the protease in viral replication as well as its expression in a large no. of serotypes.  To explore non-peptidic inhibitors of HRV 3Cpro, a series of novel heteroarom. esters was synthesized and evaluated for inhibitory activity against HRV 3Cpro, to det. the structure-activity relationships.  The most potent inhibitor, 7, with a 5-bromopyridinyl group, had an IC50 value of 80 nM.  In addn., the binding mode of a novel analog, 19 (I), with the 4-hydroxyquinolinone moiety, was explored by mol. docking, suggesting a new interaction in the S1 pocket.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4DsOHx3pVs7Vg90H21EOLACvtfcHk0lhIdKc6p0b3SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmvFGisbk%253D&md5=60df4e967a2ee30e11661a0990e62825</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.04.114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.04.114%26sid%3Dliteratum%253Aachs%26aulast%3DIm%26aufirst%3DI.%26aulast%3DLee%26aufirst%3DE.%2BS.%26aulast%3DChoi%26aufirst%3DS.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DKim%26aufirst%3DY.%2BC.%26atitle%3DStructure-activity%2520relationships%2520of%2520heteroaromatic%2520esters%2520as%2520human%2520rhinovirus%25203C%2520protease%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D3632%26epage%3D3636%26doi%3D10.1016%2Fj.bmcl.2009.04.114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kam, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhee, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Back, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Na, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choo, H. Y.</span></span> <span> </span><span class="NLM_article-title">Synthesis and bradykinin inhibitory activity of novel non-peptide compounds, and evaluation of in vivo analgesic activity</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2327</span>– <span class="NLM_lpage">2336</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2010.01.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.bmc.2010.01.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=20189403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtVCrtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=2327-2336&author=Y.+L.+Kamauthor=H.+K.+Rheeauthor=H.+J.+Kimauthor=S.+K.+Backauthor=H.+S.+Naauthor=H.+Y.+Choo&title=Synthesis+and+bradykinin+inhibitory+activity+of+novel+non-peptide+compounds%2C+and+evaluation+of+in+vivo+analgesic+activity&doi=10.1016%2Fj.bmc.2010.01.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and bradykinin inhibitory activity of novel non-peptide compounds, and evaluation of in vivo analgesic activity</span></div><div class="casAuthors">Kam, Yoo Lim; Rhee, Hee-Kyung; Kim, Hwa-Jung; Back, Seung Keun; Na, Heung Sik; Choo, Hea-Young Park</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2327-2336</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of novel non-peptide diamide compds. was synthesized and evaluated as antibradykinin agents by utilizing guinea-pig ileum smooth muscle.  Among the final compds., (Z)-4-(4-(bis(4-fluorophenyl)methyl)piperazin-1-yl)-4-oxo-N-(4-phenylbutan-2-yl)but-2-enamide I showed most favorable bradykinin inhibitory activity and demonstrated analgesic efficacies in the rat models of inflammatory and neuropathic pain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzrCYHX2555bVg90H21EOLACvtfcHk0lhIdKc6p0b3SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtVCrtrc%253D&md5=1b64a06ed2b8a5108d00fa82388b94ee</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2010.01.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2010.01.050%26sid%3Dliteratum%253Aachs%26aulast%3DKam%26aufirst%3DY.%2BL.%26aulast%3DRhee%26aufirst%3DH.%2BK.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DBack%26aufirst%3DS.%2BK.%26aulast%3DNa%26aufirst%3DH.%2BS.%26aulast%3DChoo%26aufirst%3DH.%2BY.%26atitle%3DSynthesis%2520and%2520bradykinin%2520inhibitory%2520activity%2520of%2520novel%2520non-peptide%2520compounds%252C%2520and%2520evaluation%2520of%2520in%2520vivo%2520analgesic%2520activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2010%26volume%3D18%26spage%3D2327%26epage%3D2336%26doi%3D10.1016%2Fj.bmc.2010.01.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williams, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zammit, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shackleford, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morizzi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krum, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, D. J.</span></span> <span> </span><span class="NLM_article-title">3′,4′-Bis-difluoromethoxycinnamoylanthranilate (FT061): an orally-active antifibrotic agent that reduces albuminuria in a rat model of progressive diabetic nephropathy</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">6868</span>– <span class="NLM_lpage">6873</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.09.100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.bmcl.2013.09.100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=24169234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1yqurrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=6868-6873&author=S.+J.+Williamsauthor=S.+C.+Zammitauthor=A.+J.+Coxauthor=D.+M.+Shacklefordauthor=J.+Morizziauthor=Y.+Zhangauthor=A.+K.+Powellauthor=R.+E.+Gilbertauthor=H.+Krumauthor=D.+J.+Kelly&title=3%E2%80%B2%2C4%E2%80%B2-Bis-difluoromethoxycinnamoylanthranilate+%28FT061%29%3A+an+orally-active+antifibrotic+agent+that+reduces+albuminuria+in+a+rat+model+of+progressive+diabetic+nephropathy&doi=10.1016%2Fj.bmcl.2013.09.100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">3',4'-Bis-difluoromethoxycinnamoylanthranilate (FT061): An orally-active antifibrotic agent that reduces albuminuria in a rat model of progressive diabetic nephropathy</span></div><div class="casAuthors">Williams, Spencer J.; Zammit, Steven C.; Cox, Alison J.; Shackleford, David M.; Morizzi, Julia; Zhang, Yuan; Powell, Andrew K.; Gilbert, Richard E.; Krum, Henry; Kelly, Darren J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6868-6873</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cinnamoylanthranilates including tranilast have been identified as promising antifibrotics that can reduce fibrosis occurring in the kidney during diabetes, thereby delaying and/or preventing kidney dysfunction.  Structure-activity relationships aimed at improving potency and metabolic stability have led to the discovery of FT061.  This compd., which bears a bis-difluoromethoxy catechol, attenuates TGF-β-stimulated prodn. of collagen in cultured renal mesangial cells (approx 50% at 3 μM).  When dosed orally at 20 mg/kg to male Sprague Dawley rats, FT061 exhibited a high bioavailability (73%), Cmax of 200 μM and Tmax of 150 min, and a half-life of 5.4 h.  FT061 reduced albuminuria when orally dosed in rats at 200 mg kg/day in a late intervention study of a rat model of progressive diabetic nephropathy.  The prepn. of cinnamoylanthranilates is also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGog6CM_Mf_lprVg90H21EOLACvtfcHk0lim-kTYIAGyUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1yqurrN&md5=2f74b5facaf9c7c186e18ce7bac5dfbb</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.09.100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.09.100%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DS.%2BJ.%26aulast%3DZammit%26aufirst%3DS.%2BC.%26aulast%3DCox%26aufirst%3DA.%2BJ.%26aulast%3DShackleford%26aufirst%3DD.%2BM.%26aulast%3DMorizzi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DPowell%26aufirst%3DA.%2BK.%26aulast%3DGilbert%26aufirst%3DR.%2BE.%26aulast%3DKrum%26aufirst%3DH.%26aulast%3DKelly%26aufirst%3DD.%2BJ.%26atitle%3D3%25E2%2580%25B2%252C4%25E2%2580%25B2-Bis-difluoromethoxycinnamoylanthranilate%2520%2528FT061%2529%253A%2520an%2520orally-active%2520antifibrotic%2520agent%2520that%2520reduces%2520albuminuria%2520in%2520a%2520rat%2520model%2520of%2520progressive%2520diabetic%2520nephropathy%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D6868%26epage%3D6873%26doi%3D10.1016%2Fj.bmcl.2013.09.100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Butters, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill-Cousins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kariuki, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ooi, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wordingham, S. V.</span></span> <span> </span><span class="NLM_article-title">Synthesis and stereochemical determination of batzelladine C methyl ester</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">5001</span>– <span class="NLM_lpage">5009</span>, <span class="refDoi"> DOI: 10.1039/b914744f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1039%2Fb914744f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=19907792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVSgtbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=5001-5009&author=M.+Buttersauthor=C.+D.+Daviesauthor=M.+C.+Elliottauthor=J.+Hill-Cousinsauthor=B.+M.+Kariukiauthor=L.+L.+Ooiauthor=J.+L.+Woodauthor=S.+V.+Wordingham&title=Synthesis+and+stereochemical+determination+of+batzelladine+C+methyl+ester&doi=10.1039%2Fb914744f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and stereochemical determination of batzelladine C methyl ester</span></div><div class="casAuthors">Butters, Michael; Davies, Christopher D.; Elliott, Mark C.; Hill-Cousins, Joseph; Kariuki, Benson M.; Ooi, Li-ling; Wood, John L.; Wordingham, Stuart V.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5001-5009</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Batzelladine C (I) is a tricyclic guanidine alkaloid of unknown stereochem. at one center as well as unknown abs. stereochem.  The two possible diastereoisomers of the Me ester corresponding to this compd. have been synthesized, permitting the relative and abs. stereochem. of this compd. to be assigned.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA38AGKZAuULVg90H21EOLACvtfcHk0lim-kTYIAGyUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVSgtbbI&md5=292546c0516a3ed7053d859e9322a90d</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1039%2Fb914744f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb914744f%26sid%3Dliteratum%253Aachs%26aulast%3DButters%26aufirst%3DM.%26aulast%3DDavies%26aufirst%3DC.%2BD.%26aulast%3DElliott%26aufirst%3DM.%2BC.%26aulast%3DHill-Cousins%26aufirst%3DJ.%26aulast%3DKariuki%26aufirst%3DB.%2BM.%26aulast%3DOoi%26aufirst%3DL.%2BL.%26aulast%3DWood%26aufirst%3DJ.%2BL.%26aulast%3DWordingham%26aufirst%3DS.%2BV.%26atitle%3DSynthesis%2520and%2520stereochemical%2520determination%2520of%2520batzelladine%2520C%2520methyl%2520ester%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2009%26volume%3D7%26spage%3D5001%26epage%3D5009%26doi%3D10.1039%2Fb914744f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. J.</span></span> <span> </span><span class="NLM_article-title">Rational design of an indolebutanoic acid derivative as a novel aldose reductase inhibitor based on docking and 3D QSAR studies of phenethylamine derivatives</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">5619</span>– <span class="NLM_lpage">5627</span>, <span class="refDoi"> DOI: 10.1021/jm0205346</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0205346" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD3sXptlOhsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=5619-5627&author=W.+S.+Sunauthor=Y.+S.+Parkauthor=J.+Yooauthor=K.+D.+Parkauthor=S.+H.+Kimauthor=J.+H.+Kimauthor=H.+J.+Park&title=Rational+design+of+an+indolebutanoic+acid+derivative+as+a+novel+aldose+reductase+inhibitor+based+on+docking+and+3D+QSAR+studies+of+phenethylamine+derivatives&doi=10.1021%2Fjm0205346"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design of an Indolebutanoic Acid Derivative as a Novel Aldose Reductase Inhibitor Based on Docking and 3D QSAR Studies of Phenethylamine Derivatives</span></div><div class="casAuthors">Sun, Won Suck; Park, Yoon Sun; Yoo, Jakyung; Park, Ki Duk; Kim, Sung Han; Kim, Jung-Han; Park, Hyun-Ju</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">5619-5627</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of 45 phenethylamine derivs. were synthesized and evaluated for their inhibitory activity against pig kidney aldose reductase (ALR2, EC 1.1.1.21).  Their IC50 values ranged from 400 μM to 24 μM.  The binding modes of compds. at the active site of ALR2 were examd. using flexible docking.  The results indicated that phenethylamine derivs. nicely fit into the active pocket of ALR2 by forming various hydrogen bonding and hydrophobic interactions.  3D-QSAR anal. was also conducted using FlexX-docked alignment of the compds.  The best prediction was obtained by CoMSIA combined with hydrophobic and hydrogen bond donor/acceptor field (q2 = 0.557, r2 = 0.934).  A new deriv., 4-oxo-4-(4-hydroxyindole)butanoic acid, was designed, taking into account the CoMSIA field and the binding mode derived by FlexX docking.  This rationally designed compd. exhibits an ALR2 inhibition with an IC50 value of 7.4 μM, which compares favorably to that of a well-known ALR2 inhibitor, tolrestat (IC50 = 16 μM) and represents a potency approx. 240-fold higher than that of an original phenethylamine lead compd., YUA001.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvwAtyhojGd7Vg90H21EOLACvtfcHk0lim-kTYIAGyUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXptlOhsro%253D&md5=1439f168344cc235b2cb3225cacdc9ca</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fjm0205346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0205346%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DW.%2BS.%26aulast%3DPark%26aufirst%3DY.%2BS.%26aulast%3DYoo%26aufirst%3DJ.%26aulast%3DPark%26aufirst%3DK.%2BD.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DKim%26aufirst%3DJ.%2BH.%26aulast%3DPark%26aufirst%3DH.%2BJ.%26atitle%3DRational%2520design%2520of%2520an%2520indolebutanoic%2520acid%2520derivative%2520as%2520a%2520novel%2520aldose%2520reductase%2520inhibitor%2520based%2520on%2520docking%2520and%25203D%2520QSAR%2520studies%2520of%2520phenethylamine%2520derivatives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D5619%26epage%3D5627%26doi%3D10.1021%2Fjm0205346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sadovoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">K, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golubeva, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sviridova, L.</span></span> <span> </span><span class="NLM_article-title">Regioselective synthesis of 1-alkyl-5-(indol-3-yl- and -2-yl)pyrrolidin-2-ones from available reagents</span>. <i>Chem. Heterocycl. Compd.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1215</span>– <span class="NLM_lpage">1223</span>, <span class="refDoi"> DOI: 10.1007/s10593-011-0655-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1007%2Fs10593-011-0655-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisVajsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2011&pages=1215-1223&author=A.+Sadovoyauthor=A.+Kauthor=G.+Golubevaauthor=L.+Sviridova&title=Regioselective+synthesis+of+1-alkyl-5-%28indol-3-yl-+and+-2-yl%29pyrrolidin-2-ones+from+available+reagents&doi=10.1007%2Fs10593-011-0655-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Regioselective synthesis of 1-alkyl-5-(indol-3-yl- and -2-yl)pyrrolidin-2-ones from available reagents</span></div><div class="casAuthors">Sadovoy, A. V.; Kovrov, A. E.; Golubeva, G. A.; Sviridova, L. A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry of Heterocyclic Compounds (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1215-1223</span>CODEN:
                <span class="NLM_cas:coden">CHCCAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-3122</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The reaction under mild conditions of 1-alkyl-5-hydroxypyrrolidin-2-ones with different indoles having a free 3 position leads exclusively to 1-alkyl-5-(indol-3-yl)pyrrolidin-2-ones but if position 3 is occupied to 1-alkyl-5-(indol-2-yl)pyrrolidin-2-ones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd1MPCXnyqxrVg90H21EOLACvtfcHk0lhJyAXp4INijA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisVajsbo%253D&md5=87ab5cb35d35413ca21abbbe57d62aa3</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1007%2Fs10593-011-0655-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10593-011-0655-x%26sid%3Dliteratum%253Aachs%26aulast%3DSadovoy%26aufirst%3DA.%26aulast%3DK%26aufirst%3DA.%26aulast%3DGolubeva%26aufirst%3DG.%26aulast%3DSviridova%26aufirst%3DL.%26atitle%3DRegioselective%2520synthesis%2520of%25201-alkyl-5-%2528indol-3-yl-%2520and%2520-2-yl%2529pyrrolidin-2-ones%2520from%2520available%2520reagents%26jtitle%3DChem.%2520Heterocycl.%2520Compd.%26date%3D2011%26volume%3D46%26spage%3D1215%26epage%3D1223%26doi%3D10.1007%2Fs10593-011-0655-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romeo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salerno, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittala, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modica, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siracusa, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Materia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buccioni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marucci, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minneman, K. P.</span></span> <span> </span><span class="NLM_article-title">High affinity ligands and potent antagonists for the alpha1D-adrenergic receptor. Novel 3,8-disubstituted [1]benzothieno[3,2-d]pyrimidine derivatives</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">419</span>– <span class="NLM_lpage">432</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2014.06.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.ejmech.2014.06.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=24992070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1aks7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2014&pages=419-432&author=G.+Romeoauthor=L.+Salernoauthor=V.+Pittalaauthor=M.+N.+Modicaauthor=M.+A.+Siracusaauthor=L.+Materiaauthor=M.+Buccioniauthor=G.+Marucciauthor=K.+P.+Minneman&title=High+affinity+ligands+and+potent+antagonists+for+the+alpha1D-adrenergic+receptor.+Novel+3%2C8-disubstituted+%5B1%5Dbenzothieno%5B3%2C2-d%5Dpyrimidine+derivatives&doi=10.1016%2Fj.ejmech.2014.06.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">High affinity ligands and potent antagonists for the α1D-adrenergic receptor. Novel 3,8-disubstituted [1]benzothieno[3,2-d]pyrimidine derivatives</span></div><div class="casAuthors">Romeo, Giuseppe; Salerno, Loredana; Pittala, Valeria; Modica, Maria N.; Siracusa, Maria A.; Materia, Luisa; Buccioni, Michela; Marucci, Gabriella; Minneman, Kenneth P.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">419-432</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A new series of high affinity ligands and antagonists for the α1D-adrenergic receptor (AR) has been discovered.  New mols. present a [1]benzothieno[3,2-d]pyrimidin-2,4(1H,3H)-dione or a [1]benzothieno[3,2-d]pyrimidin-4(3H)-one scaffold and bear a 2-[4-(2-methoxyphenyl)piperazin-1-yl]ethyl moiety in the 3-position and various amide substituents in the 8-position.  In binding assays at the three human cloned α1A-, α1B-, and α1D-AR subtypes, they showed high affinity values, particularly for the α1D-AR subtype.  Compd. 22 (RX18), N1-methyl-N5-[3-[2-[4-(2-methoxyphenyl)piperazin-1-yl]ethyl]-2,4-dioxo-1,2,3,4-tetrahydro[1]benzothieno[3,2-d]pyrimidin-8-yl]-N1-(phenylmethyl)pentanediamide, was the most interesting in the series displaying very high affinity (pKi = 10.25) and potent antagonism (pKb = 9.15) when tested in a functional assay at the α1D-AR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB0qdtVlxDw7Vg90H21EOLACvtfcHk0lhJyAXp4INijA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1aks7vI&md5=9bf0c8a151b4f424fa07515b1ea5a158</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2014.06.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2014.06.057%26sid%3Dliteratum%253Aachs%26aulast%3DRomeo%26aufirst%3DG.%26aulast%3DSalerno%26aufirst%3DL.%26aulast%3DPittala%26aufirst%3DV.%26aulast%3DModica%26aufirst%3DM.%2BN.%26aulast%3DSiracusa%26aufirst%3DM.%2BA.%26aulast%3DMateria%26aufirst%3DL.%26aulast%3DBuccioni%26aufirst%3DM.%26aulast%3DMarucci%26aufirst%3DG.%26aulast%3DMinneman%26aufirst%3DK.%2BP.%26atitle%3DHigh%2520affinity%2520ligands%2520and%2520potent%2520antagonists%2520for%2520the%2520alpha1D-adrenergic%2520receptor.%2520Novel%25203%252C8-disubstituted%2520%255B1%255Dbenzothieno%255B3%252C2-d%255Dpyrimidine%2520derivatives%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2014%26volume%3D83%26spage%3D419%26epage%3D432%26doi%3D10.1016%2Fj.ejmech.2014.06.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cartier, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zentz, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labia, R.</span></span> <span> </span><span class="NLM_article-title">Atypical oxidation reaction by thionyl chloride: easy two-step synthesis of N-Alkyl-1,4-dithiines</span>. <i>Synth. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">3591</span>– <span class="NLM_lpage">3597</span>, <span class="refDoi"> DOI: 10.1080/00397910600943600</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1080%2F00397910600943600" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmslSisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2006&pages=3591-3597&author=A.+Vallaauthor=D.+Cartierauthor=F.+Zentzauthor=R.+Labia&title=Atypical+oxidation+reaction+by+thionyl+chloride%3A+easy+two-step+synthesis+of+N-Alkyl-1%2C4-dithiines&doi=10.1080%2F00397910600943600"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Atypical oxidation reaction by thionyl chloride. Easy two-step synthesis of N-alkyl-1,4-dithiines</span></div><div class="casAuthors">Valla, Alain; Cartier, Dominique; Zentz, Frederic; Labia, Roger</div><div class="citationInfo"><span class="NLM_cas:title">Synthetic Communications</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3591-3597</span>CODEN:
                <span class="NLM_cas:coden">SYNCAV</span>;
        ISSN:<span class="NLM_cas:issn">0039-7911</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">Easy two-step synthesis of a series of dithiines was performed from succinic anhydride via cyclization of the corresponding 4-(alkylamino)-4-oxobutanoic acids (succinamic acids).  The reaction, carried out in polar aprotic solvents, gave 4,8-dithiine-indacene-1,3,5,7-tetraones (diimides) via 3,7-bis-4,8-dithia-indacene-1,5-diones (diisoimides), which could be isolated.  Surprisingly, in this reaction, thionyl chloride appeared as an oxidant, and this process seemed to be useful for the syntheses of S-contg. heterocyclic compds. such as 1,4-dithiins.  A mechanistic pathway was considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpI0tJ8CMCcg7Vg90H21EOLACvtfcHk0lhJyAXp4INijA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmslSisw%253D%253D&md5=27ab4d2f5dd78d6f1e93f189af1513e6</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1080%2F00397910600943600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00397910600943600%26sid%3Dliteratum%253Aachs%26aulast%3DValla%26aufirst%3DA.%26aulast%3DCartier%26aufirst%3DD.%26aulast%3DZentz%26aufirst%3DF.%26aulast%3DLabia%26aufirst%3DR.%26atitle%3DAtypical%2520oxidation%2520reaction%2520by%2520thionyl%2520chloride%253A%2520easy%2520two-step%2520synthesis%2520of%2520N-Alkyl-1%252C4-dithiines%26jtitle%3DSynth.%2520Commun.%26date%3D2006%26volume%3D36%26spage%3D3591%26epage%3D3597%26doi%3D10.1080%2F00397910600943600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arulsamy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohle, D. S.</span></span> <span> </span><span class="NLM_article-title">Nucleophilic addition of hydroxylamine, methoxylamine, and hydrazine to malononitrileoxime</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1139</span>– <span class="NLM_lpage">1143</span>, <span class="refDoi"> DOI: 10.1021/jo991614t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo991614t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD3cXms1Wlsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2000&pages=1139-1143&author=N.+Arulsamyauthor=D.+S.+Bohle&title=Nucleophilic+addition+of+hydroxylamine%2C+methoxylamine%2C+and+hydrazine+to+malononitrileoxime&doi=10.1021%2Fjo991614t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleophilic addition of hydroxylamine, methoxylamine, and hydrazine to malononitrile oxime</span></div><div class="casAuthors">Arulsamy, Navamoney; Bohle, D. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1139-1143</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The chem. of malononitrile oxime, HONC(CN)2, with respect to nucleophilic addn. to ammonia, methylamine, hydroxylamine, methoxylamine, and hydrazine is reported.  Whereas the poorly nucleophilic ammonia and methylamine do not react, hydroxylamine, methoxylamine, and hydrazine add to the nitrile groups of the oxime, yielding the corresponding amidoximes and amidrazones.  Depending on the stoichiometry of the reactions, hydroxylamine and hydrazine add to one or both of the nitrile groups; methoxylamine adds to only one of the nitrile groups.  Three of the products, namely, cyanoacetamidoxime, 3-amino-2,3-hydroxyiminopropionitrile monohydrate, and 3,5-diaminopyrazolone-4-oxime monohydrochloride monohydrate, are characterized by single-crystal X-ray diffraction data.  All of the products exhibit exothermic decompn. properties with heats of decompn. in the range of 500-1500 kJ mol-1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS2ucQK5Fc77Vg90H21EOLACvtfcHk0lhJyAXp4INijA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXms1Wlsg%253D%253D&md5=4328f83e63ef876c81fe8ff254baf813</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fjo991614t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo991614t%26sid%3Dliteratum%253Aachs%26aulast%3DArulsamy%26aufirst%3DN.%26aulast%3DBohle%26aufirst%3DD.%2BS.%26atitle%3DNucleophilic%2520addition%2520of%2520hydroxylamine%252C%2520methoxylamine%252C%2520and%2520hydrazine%2520to%2520malononitrileoxime%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2000%26volume%3D65%26spage%3D1139%26epage%3D1143%26doi%3D10.1021%2Fjo991614t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vörös, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mucsi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baán, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timári, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermecz, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizsey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finta, Z.</span></span> <span> </span><span class="NLM_article-title">An experimental and theoretical study of reaction mechanisms between nitriles and hydroxylamine</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">8036</span>– <span class="NLM_lpage">8047</span>, <span class="refDoi"> DOI: 10.1039/C4OB00854E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1039%2FC4OB00854E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=25185027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1KhtLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=8036-8047&author=A.+V%C3%B6r%C3%B6sauthor=Z.+Mucsiauthor=Z.+Ba%C3%A1nauthor=G.+Tim%C3%A1riauthor=I.+Hermeczauthor=P.+Mizseyauthor=Z.+Finta&title=An+experimental+and+theoretical+study+of+reaction+mechanisms+between+nitriles+and+hydroxylamine&doi=10.1039%2FC4OB00854E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">An experimental and theoretical study of reaction mechanisms between nitriles and hydroxylamine</span></div><div class="casAuthors">Voros, Attila; Mucsi, Zoltan; Baan, Zoltan; Timari, Geza; Hermecz, Istvan; Mizsey, Peter; Finta, Zoltan</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">8036-8047</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The industrially relevant reaction between nitriles and hydroxylamine yielding amidoximes was studied in different mol. solvents and in ionic liqs.  In industry, this procedure is carried out on the ton scale in alc. solns. and the above transformation produces a significant amt. of unexpected amide byproduct, depending on the nature of the nitrile, which can cause further anal. and purifn. issues.  Although there were earlier attempts to propose mechanisms for this transformation, the real reaction pathway is still under discussion.  A new detailed reaction mechanistic explanation, based on theor. and exptl. proof, is given to augment the former mechanisms, which allowed us to find a more efficient, side-product free procedure.  Interpreting the theor. results obtained, it was shown that the application of specific imidazolium, phosphonium and quaternary ammonium based ionic liqs. could decrease simultaneously the reaction time while eliminating the amide side-product, leading to the targeted product selectively.  This robust and economic procedure now affords a fast, selective amide free synthesis of amidoximes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ0NtTmLTvULVg90H21EOLACvtfcHk0lgHPjceglei0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1KhtLfO&md5=27181cd022238675fb7728fc6bd84002</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1039%2FC4OB00854E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4OB00854E%26sid%3Dliteratum%253Aachs%26aulast%3DV%25C3%25B6r%25C3%25B6s%26aufirst%3DA.%26aulast%3DMucsi%26aufirst%3DZ.%26aulast%3DBa%25C3%25A1n%26aufirst%3DZ.%26aulast%3DTim%25C3%25A1ri%26aufirst%3DG.%26aulast%3DHermecz%26aufirst%3DI.%26aulast%3DMizsey%26aufirst%3DP.%26aulast%3DFinta%26aufirst%3DZ.%26atitle%3DAn%2520experimental%2520and%2520theoretical%2520study%2520of%2520reaction%2520mechanisms%2520between%2520nitriles%2520and%2520hydroxylamine%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2014%26volume%3D12%26spage%3D8036%26epage%3D8047%26doi%3D10.1039%2FC4OB00854E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luciani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marie-Claire, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruffet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaumont, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roques, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fournie-Zaluski, M. C.</span></span> <span> </span><span class="NLM_article-title">Characterization of Glu350 as a critical residue involved in the N-terminal amine binding site of aminopeptidase N (EC 3.4.11.2): insights into its mechanism of action</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">686</span>– <span class="NLM_lpage">692</span>, <span class="refDoi"> DOI: 10.1021/bi971705p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi971705p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaK2sXotVKis7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1998&pages=686-692&author=N.+Lucianiauthor=C.+Marie-Claireauthor=E.+Ruffetauthor=A.+Beaumontauthor=B.+P.+Roquesauthor=M.+C.+Fournie-Zaluski&title=Characterization+of+Glu350+as+a+critical+residue+involved+in+the+N-terminal+amine+binding+site+of+aminopeptidase+N+%28EC+3.4.11.2%29%3A+insights+into+its+mechanism+of+action&doi=10.1021%2Fbi971705p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of Glu350 as a Critical Residue Involved in the N-Terminal Amine Binding Site of Aminopeptidase N (EC 3.4.11.2): Insights into Its Mechanism of Action</span></div><div class="casAuthors">Luciani, Nathalie; Marie-Claire, Cynthia; Ruffet, Emmanuel; Beaumont, Ann; Roques, Bernard P.; Fournie-Zaluski, Marie-Claude</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">686-692</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The mol. components ensuring the strict exopeptidase action of aminopeptidase N (APN) and related zinc aminopeptidases of the M1 family have not yet been clearly established.  The specific recognition of the N-terminal amino acid of the substrates by the enzymes has been proposed to involve either the complexation of the free amino group by the catalytic zinc ion or an interaction with an anionic binding site, which could be constituted by an aspartate or glutamate residue.  To investigate the existence of such an ionic binding site, site-directed mutagenesis expts. have been performed on acidic residues of pig APN.  Given that aminopeptidases of the M1 family are likely to have a common mechanism of action, only strictly conserved residues were mutated.  As compared to the wild-type enzyme, the mutation D220E led only to slight modifications in the kinetic parameters of the enzyme and in the Ki values of various inhibitors, indicating that this residue is not critically involved in the hydrolytic mechanism.  In contrast, the mutations E350Q and E350D induced a large decrease in enzyme activity, essentially due to modifications in kcat, whereas the E350A mutation led to an almost completely inactive enzyme.  Moreover, among the inhibitors tested, only those acting as transition state analogs showed significant increases in their Ki values.  These data are in favor of E350 belonging to the "anionic binding site" in APN.  A mechanism of action, derived from that of thermolysin, is proposed for these aminopeptidases, which explains the importance of E350 in transition state formation, rather than in the Michaelis complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6gCDhJijok7Vg90H21EOLACvtfcHk0lgHPjceglei0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXotVKis7c%253D&md5=d02d8f5fb796fda649d314a23cd58d11</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fbi971705p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi971705p%26sid%3Dliteratum%253Aachs%26aulast%3DLuciani%26aufirst%3DN.%26aulast%3DMarie-Claire%26aufirst%3DC.%26aulast%3DRuffet%26aufirst%3DE.%26aulast%3DBeaumont%26aufirst%3DA.%26aulast%3DRoques%26aufirst%3DB.%2BP.%26aulast%3DFournie-Zaluski%26aufirst%3DM.%2BC.%26atitle%3DCharacterization%2520of%2520Glu350%2520as%2520a%2520critical%2520residue%2520involved%2520in%2520the%2520N-terminal%2520amine%2520binding%2520site%2520of%2520aminopeptidase%2520N%2520%2528EC%25203.4.11.2%2529%253A%2520insights%2520into%2520its%2520mechanism%2520of%2520action%26jtitle%3DBiochemistry%26date%3D1998%26volume%3D37%26spage%3D686%26epage%3D692%26doi%3D10.1021%2Fbi971705p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Graham, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smiley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nairn, R.</span></span> <span> </span><span class="NLM_article-title">Characteristics of a human cell line transformed by DNA from human adenovirus type 5</span>. <i>J. Gen. Virol.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">59</span>– <span class="NLM_lpage">74</span>, <span class="refDoi"> DOI: 10.1099/0022-1317-36-1-59</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1099%2F0022-1317-36-1-59" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=886304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A280%3ADyaE2s3ivFGmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1977&pages=59-74&author=F.+L.+Grahamauthor=J.+Smileyauthor=W.+C.+Russellauthor=R.+Nairn&title=Characteristics+of+a+human+cell+line+transformed+by+DNA+from+human+adenovirus+type+5&doi=10.1099%2F0022-1317-36-1-59"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Characteristics of a human cell line transformed by DNA from human adenovirus type 5</span></div><div class="casAuthors">Graham F L; Smiley J; Russell W C; Nairn R</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of general virology</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">59-74</span>
        ISSN:<span class="NLM_cas:issn">0022-1317</span>.
    </div><div class="casAbstract">Human embryonic kidney cells have been transformed by exposing cells to sheared fragments of adenovirus type 5 DNA.  The transformed cells (designated 293 cells) exhibited many of the characteristics of transformation including the elaboration of a virus-specific tumour antigen.  Analysis of the polypeptides synthesized in the 293 cells by labelling with 35S-methionine and SDS PAGE showed a variable pattern of synthesis, different in a number of respects from that seen in otheruman cells.  On labelling the surface of cells by lactoperoxidase catalysed radio-iodination, the absence of a labelled polypeptide analogous to the 250 K (LETS) glycoprotein was noted.  Hybridization of labelled cellular RNA with restriction fragments of adenovirus type 5 DNA indicated transcription of a portion of the adenovirus genome at the conventional left hand end.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSm6_NOgLpQVh0LsK7uFuY7fW6udTcc2ebCOZiOogvPXbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaE2s3ivFGmtQ%253D%253D&md5=f1dc59399d0bbb5d89889cc5573fcdd8</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1099%2F0022-1317-36-1-59&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252F0022-1317-36-1-59%26sid%3Dliteratum%253Aachs%26aulast%3DGraham%26aufirst%3DF.%2BL.%26aulast%3DSmiley%26aufirst%3DJ.%26aulast%3DRussell%26aufirst%3DW.%2BC.%26aulast%3DNairn%26aufirst%3DR.%26atitle%3DCharacteristics%2520of%2520a%2520human%2520cell%2520line%2520transformed%2520by%2520DNA%2520from%2520human%2520adenovirus%2520type%25205%26jtitle%3DJ.%2520Gen.%2520Virol.%26date%3D1977%26volume%3D36%26spage%3D59%26epage%3D74%26doi%3D10.1099%2F0022-1317-36-1-59" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, J. F.</span></span> <span> </span><span class="NLM_article-title">Kinetics of the reversible inhibition of enzyme-catalysed reactions by tight-binding inhibitors</span>. <i>Biochim. Biophys. Acta, Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1969</span>,  <span class="NLM_volume"><i>185</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">286</span>, <span class="refDoi"> DOI: 10.1016/0005-2744(69)90420-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2F0005-2744%2869%2990420-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=4980133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaF1MXkvVOqsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=185&publication_year=1969&pages=269-286&author=J.+F.+Morrison&title=Kinetics+of+the+reversible+inhibition+of+enzyme-catalysed+reactions+by+tight-binding+inhibitors&doi=10.1016%2F0005-2744%2869%2990420-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics of the reversible inhibition of enzyme-catalyzed reactions by tight-binding inhibitors</span></div><div class="casAuthors">Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Enzymology</span>
        (<span class="NLM_cas:date">1969</span>),
    <span class="NLM_cas:volume">185</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-86</span>CODEN:
                <span class="NLM_cas:coden">BBEZAD</span>;
        ISSN:<span class="NLM_cas:issn">0924-1086</span>.
    </div><div class="casAbstract">A theoretical kinetic study has been undertaken to det. the kinetic characteristics of enzymic reactions that are reversibly inhibited by a substrate analog. at concns. comparable to that of the enzyme.  A general rate equation, that allows for the combination of such a tight-binding inhibitor with one or more enzyme forms of any reaction mechanism, has been derived and shown to be quadratic with sq. and linear terms in υ which represents the true initial steady-state velocity.  From plots of υ against total enzyme concn. at fixed total inhibitor concn., the concn. of enzyme catalytic sites can be detd. irrespective of the nature of the inhibition and the reaction mechanism.  However, plots of υ (or I/υ) against total inhibitor concn. can be used for this purpose only if the dissocn. const. assocd. with the inhibitor is virtually zero.  Particular initial rate equations have been obtained for the cases in which the tight-binding inhibitor gives rise to competitive, noncompetitive, and uncompetitive inhibition with respect to one of the substrates of a bireactant mechanism.  From these equations it has been deduced that, in the presence of inhibitor, plots of I/υ against the reciprocals of the substrate concns. yield families of concave-down, nonrectangular hyperbolas.  The difficulties assocd. with the detn. of inhibition consts. for tight-binding inhibitors are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpj4s_N2iV3CLVg90H21EOLACvtfcHk0lifKqomzMSI6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF1MXkvVOqsbo%253D&md5=74d467477385488dfbb053358534ada6</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2F0005-2744%2869%2990420-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0005-2744%252869%252990420-3%26sid%3Dliteratum%253Aachs%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DKinetics%2520of%2520the%2520reversible%2520inhibition%2520of%2520enzyme-catalysed%2520reactions%2520by%2520tight-binding%2520inhibitors%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Enzymol.%26date%3D1969%26volume%3D185%26spage%3D269%26epage%3D286%26doi%3D10.1016%2F0005-2744%2869%2990420-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, J. F.</span></span> <span> </span><span class="NLM_article-title">The kinetics of reversible tight-binding inhibition</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">437</span>– <span class="NLM_lpage">467</span>, <span class="refDoi"> DOI: 10.1016/0076-6879(79)63019-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2F0076-6879%2879%2963019-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=502865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaL3cXovF2ntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=1979&pages=437-467&author=J.+W.+Williamsauthor=J.+F.+Morrison&title=The+kinetics+of+reversible+tight-binding+inhibition&doi=10.1016%2F0076-6879%2879%2963019-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">The kinetics of reversible tight-binding inhibition</span></div><div class="casAuthors">Williams, Jeffrey W.; Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1979</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">Enzyme Kinet. Mech., Part A</span>),
    <span class="NLM_cas:pages">437-67</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    </div><div class="casAbstract">In addn. to a general discussion of the kinetics of reversible tight-binding inhibitors, which exert their effect at a concn. comparable to that of the enzyme, and for which steady-state theory is used, exptl. results are given of the tight-binding inhibition of creatine kinase by Eu3+ (actually Eu-ADP) under steady-state conditions.  The inhibition of creatine kinase by fixed concns. of Eu3+ in the presence of varying concns. of Mg-ADP indicated that the inhibition is competitive, suggesting that Eu-ADP acts as a structural analog of Mg-ADP.  The inhibition was noncompetitive with respect to creatine phosphate.  The dissocn. consts. for binding of Eu-ADP to the free enzyme and the enzyme-phosphocreatine complex were 1.5 and 2.2 nM, resp., as detd. from Ki values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2CwV3voXt27Vg90H21EOLACvtfcHk0lifKqomzMSI6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXovF2ntw%253D%253D&md5=a770f91b51661c9502d0aca44a24e1c7</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2F0076-6879%2879%2963019-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0076-6879%252879%252963019-7%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DJ.%2BW.%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DThe%2520kinetics%2520of%2520reversible%2520tight-binding%2520inhibition%26jtitle%3DMethods%2520Enzymol.%26date%3D1979%26volume%3D63%26spage%3D437%26epage%3D467%26doi%3D10.1016%2F0076-6879%2879%2963019-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bevan, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, R. S.</span></span> <span> </span><span class="NLM_article-title">A high-throughput screening method for the determination of aqueous drug solubility using laser nephelometry in microtiter plates</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">1781</span>– <span class="NLM_lpage">1787</span>, <span class="refDoi"> DOI: 10.1021/ac9912247</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac9912247" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhsFyhsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2000&pages=1781-1787&author=C.+D.+Bevanauthor=R.+S.+Lloyd&title=A+high-throughput+screening+method+for+the+determination+of+aqueous+drug+solubility+using+laser+nephelometry+in+microtiter+plates&doi=10.1021%2Fac9912247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">A High-Throughput Screening Method for the Determination of Aqueous Drug Solubility Using Laser Nephelometry in Microtiter Plates</span></div><div class="casAuthors">Bevan, Christopher D.; Lloyd, Richard S.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1781-1787</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The detn. of aq. soly. in a high-throughput screening environment is invaluable in the selection of the most promising potential drug candidates.  We describe a fast method based on laser nephelometry that can det. the soly. of potential drug candidates (usually from combinatorial chem.) supplied as DMSO solns. in 96-well plates.  In the sample, the percent DMSO is kept const. allowing direct comparison of results.  The nephelometric method has been shown to produce results equiv. to those produced by an HPLC method and to be largely unaffected by colored solns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7jLYnIgg-_bVg90H21EOLACvtfcHk0lifKqomzMSI6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhsFyhsb4%253D&md5=5023317d334eafd7a60860e3692c477d</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fac9912247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac9912247%26sid%3Dliteratum%253Aachs%26aulast%3DBevan%26aufirst%3DC.%2BD.%26aulast%3DLloyd%26aufirst%3DR.%2BS.%26atitle%3DA%2520high-throughput%2520screening%2520method%2520for%2520the%2520determination%2520of%2520aqueous%2520drug%2520solubility%2520using%2520laser%2520nephelometry%2520in%2520microtiter%2520plates%26jtitle%3DAnal.%2520Chem.%26date%3D2000%26volume%3D72%26spage%3D1781%26epage%3D1787%26doi%3D10.1021%2Fac9912247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shalaeva, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tupper, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, F.</span></span> <span> </span><span class="NLM_article-title">ElogD(oct): a tool for lipophilicity determination in drug discovery. 2. Basic and neutral compounds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">2490</span>– <span class="NLM_lpage">2497</span>, <span class="refDoi"> DOI: 10.1021/jm0100990</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0100990" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD3MXksFOmtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=2490-2497&author=F.+Lombardoauthor=M.+Y.+Shalaevaauthor=K.+A.+Tupperauthor=F.+Gao&title=ElogD%28oct%29%3A+a+tool+for+lipophilicity+determination+in+drug+discovery.+2.+Basic+and+neutral+compounds&doi=10.1021%2Fjm0100990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">ElogDoct: A Tool for Lipophilicity Determination in Drug Discovery. 2. Basic and Neutral Compounds</span></div><div class="casAuthors">Lombardo, Franco; Shalaeva, Marina Y.; Tupper, Karl A.; Gao, Feng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2490-2497</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present an RP-HPLC method for the detn. of the octanol-water distribution coeffs. at pH 7.4, as log Doct7.4 values, for neutral and basic drugs, which combines ease of operation with high accuracy.  The method is shown to work for a training set of 90 mols. comprised largely of drugs, and it was also applied to a test set of 10 proprietary compds.  This work expands the applicability of the method presented in our earlier report, for the detn. of logPoct for neutral compds. (J. Med. Chem. 2000, 43, 2922-2928), and it offers the same general features but widens the scope.  Generally, the method (i) is compd. sparing (≤1 mL of a 50-100 μg/mL soln. needed), (ii) is insensitive to concn. and phase ratio effects obsd. in some shake-flask detns., (iii) is amenable to rapid detns. (≤20 min on av.), (iv) is insensitive to impurities, (v) possesses a wide lipophilicity range (>7 log Doct7.4 units), and (vi) offers a good accuracy, (vii) an excellent reproducibility, (viii) and an excellent potential for automation.  To the best of our knowledge, a similar performance, on a set of noncongeneric drugs, has not been previously reported.  We refer to the value generated via this method as ElogDoct.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrynjvQ2-0sibVg90H21EOLACvtfcHk0lifKqomzMSI6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXksFOmtbc%253D&md5=e390cfeacbb1cab5660909eccfeb7eb2</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fjm0100990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0100990%26sid%3Dliteratum%253Aachs%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DShalaeva%26aufirst%3DM.%2BY.%26aulast%3DTupper%26aufirst%3DK.%2BA.%26aulast%3DGao%26aufirst%3DF.%26atitle%3DElogD%2528oct%2529%253A%2520a%2520tool%2520for%2520lipophilicity%2520determination%2520in%2520drug%2520discovery.%25202.%2520Basic%2520and%2520neutral%2520compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D2490%26epage%3D2497%26doi%3D10.1021%2Fjm0100990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Schrödinger</span>.  <i>The PyMOL Molecular Graphics System</i>, version 1.8; <span class="NLM_publisher-name">Schrödinger, LLC</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=Schr%C3%B6dinger&title=The+PyMOL+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Development and testing of a general amber force field</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1157</span>– <span class="NLM_lpage">1174</span>, <span class="refDoi"> DOI: 10.1002/jcc.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+a+general+amber+force+field&doi=10.1002%2Fjcc.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0ljqcjtnKFWceg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520a%2520general%2520amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174%26doi%3D10.1002%2Fjcc.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jakalian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bush, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jack, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, C. I.</span></span> <span> </span><span class="NLM_article-title">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: I. Method</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">132</span>– <span class="NLM_lpage">146</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1096-987X(20000130)21:2<132::AID-JCC5>3.0.CO;2-P</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD3cXkt1SgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2000&pages=132-146&author=A.+Jakalianauthor=B.+L.+Bushauthor=D.+B.+Jackauthor=C.+I.+Bayly&title=Fast%2C+efficient+generation+of+high-quality+atomic+charges.+AM1-BCC+model%3A+I.+Method&doi=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: I. Method</span></div><div class="casAuthors">Jakalian, Araz; Bush, Bruce L.; Jack, David B.; Bayly, Christopher I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">132-146</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The AM1-BCC method quickly and efficiently generates high-quality at. charges for use in condensed-phase simulations.  The underlying features of the electron distribution including formal charge and delocalization are first captured by AM1 at. charges for the individual mol.  Bond charge corrections (BCCs), which have been parameterized against the HF/6-31G* electrostatic potential (ESP) of a training set of compds. contg. relevant functional groups, are then added using a formalism identical to the consensus BCI (bond charge increment) approach.  As a proof of the concept, we fit BCCs simultaneously to 45 compds. including O-, N-, and S-contg. functionalities, aroms., and heteroaroms., using only 41 BCC parameters.  AM1-BCC yields charge sets of comparable quality to HF/6-31G* ESP-derived charges in a fraction of the time while reducing instabilities in the at. charges compared to direct ESP-fit methods.  We then apply the BCC parameters to a small "test set" consisting of aspirin, D-glucose, and eryodictyol; the AM1-BCC model again provides at. charges of quality comparable with HF/6-31G* RESP charges, as judged by an increase of only 0.01 to 0.02 at. units in the root-mean-square (RMS) error in ESP.  Based on these encouraging results, we intend to parameterize the AM1-BCC model to provide a consistent charge model for any org. or biol. mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruxZWGx7IgLbVg90H21EOLACvtfcHk0ljqcjtnKFWceg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXkt1SgsQ%253D%253D&md5=aa4774739d3491dbc8dd95ed1948d856</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%252820000130%252921%253A2%253C132%253A%253AAID-JCC5%253E3.0.CO%253B2-P%26sid%3Dliteratum%253Aachs%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DBush%26aufirst%3DB.%2BL.%26aulast%3DJack%26aufirst%3DD.%2BB.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26atitle%3DFast%252C%2520efficient%2520generation%2520of%2520high-quality%2520atomic%2520charges.%2520AM1-BCC%2520model%253A%2520I.%2520Method%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2000%26volume%3D21%26spage%3D132%26epage%3D146%26doi%3D10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jakalian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jack, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, C. I.</span></span> <span> </span><span class="NLM_article-title">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. Parameterization and validation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1623</span>– <span class="NLM_lpage">1641</span>, <span class="refDoi"> DOI: 10.1002/jcc.10128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1002%2Fjcc.10128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=12395429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2002&pages=1623-1641&author=A.+Jakalianauthor=D.+B.+Jackauthor=C.+I.+Bayly&title=Fast%2C+efficient+generation+of+high-quality+atomic+charges.+AM1-BCC+model%3A+II.+Parameterization+and+validation&doi=10.1002%2Fjcc.10128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. parameterization and validation</span></div><div class="casAuthors">Jakalian, Araz; Jack, David B.; Bayly, Christopher I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1623-1641</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We present the first global parameterization and validation of a novel charge model, called AM1-BCC, which quickly and efficiently generates high-quality at. charges for computer simulations of org. mols. in polar media.  The goal of the charge model is to produce at. charges that emulate the HF/6-31G* electrostatic potential (ESP) of a mol.  Underlying electronic structure features, including formal charge and electron delocalization, are first captured by AM1 population charges; simple additive bond charge corrections (BCCs) are then applied to these AM1 at. charges to produce the AM1-BCC charges.  The parameterization of BCCs was carried out by fitting to the HF/6-31G* ESP of a training set of >2700 mols.  Most org. functional groups and their combinations were sampled, as well as an extensive variety of cyclic and fused bicyclic heteroaryl systems.  The resulting BCC parameters allow the AM1-BCC charging scheme to handle virtually all types of org. compds. listed in The Merck Index and the NCI Database.  Validation of the model was done through comparisons of hydrogen-bonded dimer energies and relative free energies of solvation using AM1-BCC charges in conjunction with the 1994 Cornell et al. forcefield for AMBER.  Homo-dimer and hetero-dimer hydrogen-bond energies of a diverse set of org. mols. were reproduced to within 0.95 kcal/mol RMS deviation from the ab initio values, and for DNA dimers the energies were within 0.9 kcal/mol RMS deviation from ab initio values.  The calcd. relative free energies of solvation for a diverse set of monofunctional isosteres were reproduced to within 0.69 kcal/mol of expt.  In all these validation tests, AMBER with the AM1-BCC charge model maintained a correlation coeff. above 0.96.  Thus, the parameters presented here for use with the AM1-BCC method present a fast, accurate, and robust alternative to HF/6-31G* ESP-fit charges for general use with the AMBER force field in computer simulations involving org. small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn9DkJ_gMzA7Vg90H21EOLACvtfcHk0ljqcjtnKFWceg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D&md5=3f2b738617bb17b2898f7ac4d751d7ec</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10128%26sid%3Dliteratum%253Aachs%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DJack%26aufirst%3DD.%2BB.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26atitle%3DFast%252C%2520efficient%2520generation%2520of%2520high-quality%2520atomic%2520charges.%2520AM1-BCC%2520model%253A%2520II.%2520Parameterization%2520and%2520validation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2002%26volume%3D23%26spage%3D1623%26epage%3D1641%26doi%3D10.1002%2Fjcc.10128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fiser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Šali, A.</span></span> <span> </span><span class="NLM_article-title">Modeller: generation and refinement of homology-based protein structure models</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>374</i></span>,  <span class="NLM_fpage">461</span>– <span class="NLM_lpage">491</span>, <span class="refDoi"> DOI: 10.1016/S0076-6879(03)74020-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2FS0076-6879%2803%2974020-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=14696385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotlyntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=374&publication_year=2003&pages=461-491&author=A.+Fiserauthor=A.+%C5%A0ali&title=Modeller%3A+generation+and+refinement+of+homology-based+protein+structure+models&doi=10.1016%2FS0076-6879%2803%2974020-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">MODELLER: generation and refinement of homology-based protein structure models</span></div><div class="casAuthors">Fiser, Andras; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">374</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part D</span>),
    <span class="NLM_cas:pages">461-491</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Comparative modeling usually consists of five steps: search for related protein structures, selection of one or more templates, target-template alignment, model building, and model evaluation.  Generic considerations in all five steps of comparative modeling are described.  The considerations in practice by discussing three applications of the program MODELLER to specific modeling problems are also illustrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL_0iXCSdXLrVg90H21EOLACvtfcHk0lhj5t5Qf7mOWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotlyntA%253D%253D&md5=5b21fff95e9f7158501051f96f7beba3</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2803%2974020-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252803%252974020-8%26sid%3Dliteratum%253Aachs%26aulast%3DFiser%26aufirst%3DA.%26aulast%3D%25C5%25A0ali%26aufirst%3DA.%26atitle%3DModeller%253A%2520generation%2520and%2520refinement%2520of%2520homology-based%2520protein%2520structure%2520models%26jtitle%3DMethods%2520Enzymol.%26date%3D2003%26volume%3D374%26spage%3D461%26epage%3D491%26doi%3D10.1016%2FS0076-6879%2803%2974020-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gasteiger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsili, M.</span></span> <span> </span><span class="NLM_article-title">Iterative partial equalization of orbital electronegativity—a rapid access to atomic charges</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">1980</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">3219</span>– <span class="NLM_lpage">3228</span>, <span class="refDoi"> DOI: 10.1016/0040-4020(80)80168-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2F0040-4020%2880%2980168-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaL3MXhslCjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1980&pages=3219-3228&author=J.+Gasteigerauthor=M.+Marsili&title=Iterative+partial+equalization+of+orbital+electronegativity%E2%80%94a+rapid+access+to+atomic+charges&doi=10.1016%2F0040-4020%2880%2980168-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Iterative partial equalization of orbital electronegativity: a rapid access to atomic charges</span></div><div class="casAuthors">Gasteiger, Johann; Marsili, Mario</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3219-22</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    </div><div class="casAbstract">A method is presented for the rapid calcn. of at. charges in σ-bonded and nonconjugated π-systems (alkyl halides and oxoalkanes, resp.), considering only the connectivities of the atoms.  Atoms are characterized by their orbital electronegativities.  Partial equalization of orbital electronegativity is obtained by an iterative procedure.  Excellent correlations of the at. charges with core-electron binding energies and with acidity consts. are obsd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKA3wil1SiWrVg90H21EOLACvtfcHk0lhj5t5Qf7mOWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXhslCjtbs%253D&md5=90c580857e6001482bee92450278fe9f</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2F0040-4020%2880%2980168-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0040-4020%252880%252980168-2%26sid%3Dliteratum%253Aachs%26aulast%3DGasteiger%26aufirst%3DJ.%26aulast%3DMarsili%26aufirst%3DM.%26atitle%3DIterative%2520partial%2520equalization%2520of%2520orbital%2520electronegativity%25E2%2580%2594a%2520rapid%2520access%2520to%2520atomic%2520charges%26jtitle%3DTetrahedron%26date%3D1980%26volume%3D36%26spage%3D3219%26epage%3D3228%26doi%3D10.1016%2F0040-4020%2880%2980168-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekhar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madura, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Impey, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M. L.</span></span> <span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">926</span>– <span class="NLM_lpage">935</span>, <span class="refDoi"> DOI: 10.1063/1.445869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+W.+Impeyauthor=M.+L.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water&doi=10.1063%2F1.445869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0lhj5t5Qf7mOWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%2BW.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935%26doi%3D10.1063%2F1.445869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheatham, T. E.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, K. M.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onufriev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmerling, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, R. J.</span></span> <span> </span><span class="NLM_article-title">The Amber biomolecular simulation programs</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1668</span>– <span class="NLM_lpage">1688</span>, <span class="refDoi"> DOI: 10.1002/jcc.20290</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1002%2Fjcc.20290" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=16200636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1668-1688&author=D.+A.+Caseauthor=T.+E.+Cheathamauthor=T.+Dardenauthor=H.+Gohlkeauthor=R.+Luoauthor=K.+M.+Merzauthor=A.+Onufrievauthor=C.+Simmerlingauthor=B.+Wangauthor=R.+J.+Woods&title=The+Amber+biomolecular+simulation+programs&doi=10.1002%2Fjcc.20290"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">The amber biomolecular simulation programs</span></div><div class="casAuthors">Case, David A.; Cheatham, Thomas E., III; Darden, Tom; Gohlke, Holger; Luo, Ray; Merz, Kenneth M., Jr.; Onufriev, Alexey; Simmerling, Carlos; Wang, Bing; Woods, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1668-1688</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The authors describe the development, current features, and some directions for future development of the Amber package of computer programs.  This package evolved from a program that was constructed in the late 1970s to do Assisted Model Building with Energy Refinement, and now contains a group of programs embodying a no. of powerful tools of modern computational chem., focused on mol. dynamics and free energy calcns. of proteins, nucleic acids, and carbohydrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre0tZP_Umb7rVg90H21EOLACvtfcHk0lgze6KyoozhfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbM&md5=93be29ff894bab96c783d24e9886c7d0</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20290%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DR.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26aulast%3DOnufriev%26aufirst%3DA.%26aulast%3DSimmerling%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWoods%26aufirst%3DR.%2BJ.%26atitle%3DThe%2520Amber%2520biomolecular%2520simulation%2520programs%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1668%26epage%3D1688%26doi%3D10.1002%2Fjcc.20290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gumbart, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tajkhorshid, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villa, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chipot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skeel, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kale, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulten, K.</span></span> <span> </span><span class="NLM_article-title">Scalable molecular dynamics with NAMD</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1781</span>– <span class="NLM_lpage">1802</span>, <span class="refDoi"> DOI: 10.1002/jcc.20289</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1002%2Fjcc.20289" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=16222654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1781-1802&author=J.+C.+Phillipsauthor=R.+Braunauthor=W.+Wangauthor=J.+Gumbartauthor=E.+Tajkhorshidauthor=E.+Villaauthor=C.+Chipotauthor=R.+D.+Skeelauthor=L.+Kaleauthor=K.+Schulten&title=Scalable+molecular+dynamics+with+NAMD&doi=10.1002%2Fjcc.20289"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Scalable molecular dynamics with NAMD</span></div><div class="casAuthors">Phillips, James C.; Braun, Rosemary; Wang, Wei; Gumbart, James; Tajkhorshid, Emad; Villa, Elizabeth; Chipot, Christophe; Skeel, Robert D.; Kale, Laxmikant; Schulten, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1781-1802</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">NAMD is a parallel mol. dynamics code designed for high-performance simulation of large biomol. systems.  NAMD scales to hundreds of processors on high-end parallel platforms, as well as tens of processors on low-cost commodity clusters, and also runs on individual desktop and laptop computers.  NAMD works with AMBER and CHARMM potential functions, parameters, and file formats.  This article, directed to novices as well as experts, first introduces concepts and methods used in the NAMD program, describing the classical mol. dynamics force field, equations of motion, and integration methods along with the efficient electrostatics evaluation algorithms employed and temp. and pressure controls used.  Features for steering the simulation across barriers and for calcg. both alchem. and conformational free energy differences are presented.  The motivations for and a roadmap to the internal design of NAMD, implemented in C++ and based on Charm++ parallel objects, are outlined.  The factors affecting the serial and parallel performance of a simulation are discussed.  Finally, typical NAMD use is illustrated with representative applications to a small, a medium, and a large biomol. system, highlighting particular features of NAMD, for example, the Tcl scripting language.  The article also provides a list of the key features of NAMD and discusses the benefits of combining NAMD with the mol. graphics/sequence anal. software VMD and the grid computing/collab. software BioCoRE.  NAMD is distributed free of charge with source code at www.ks.uiuc.edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7fq_yG00T1rVg90H21EOLACvtfcHk0lgze6KyoozhfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbJ&md5=189051128443b547f4300a1b8fb0e034</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20289&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20289%26sid%3Dliteratum%253Aachs%26aulast%3DPhillips%26aufirst%3DJ.%2BC.%26aulast%3DBraun%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DGumbart%26aufirst%3DJ.%26aulast%3DTajkhorshid%26aufirst%3DE.%26aulast%3DVilla%26aufirst%3DE.%26aulast%3DChipot%26aufirst%3DC.%26aulast%3DSkeel%26aufirst%3DR.%2BD.%26aulast%3DKale%26aufirst%3DL.%26aulast%3DSchulten%26aufirst%3DK.%26atitle%3DScalable%2520molecular%2520dynamics%2520with%2520NAMD%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1781%26epage%3D1802%26doi%3D10.1002%2Fjcc.20289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryckaert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccotti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span> <span> </span><span class="NLM_article-title">Numerical integration of the cartesian equations of motion of a system with constraints: molecular dynamics of n-alkanes</span>. <i>J. Comput. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/0021-9991(77)90098-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2F0021-9991%2877%2990098-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=327-341&author=J.-P.+Ryckaertauthor=G.+Ciccottiauthor=H.+J.+C.+Berendsen&title=Numerical+integration+of+the+cartesian+equations+of+motion+of+a+system+with+constraints%3A+molecular+dynamics+of+n-alkanes&doi=10.1016%2F0021-9991%2877%2990098-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Numerical integration of the Cartesian equations of motion of a system with constraints:  molecular dynamics of n-alkanes</span></div><div class="casAuthors">Ryckaert, Jean Paul; Ciccotti, Giovanni; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-41</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    </div><div class="casAbstract">A numerical algorithm integrating the 3N Cartesian equation of motion of a system of N points subject to holonomic constraints is applied to mol. dynamics simulation of a liq. of 64 butane mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPlhG1U3Uw7Vg90H21EOLACvtfcHk0lgze6KyoozhfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D&md5=b4aecddfde149117813a5ea4f5353ce2</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990098-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990098-5%26sid%3Dliteratum%253Aachs%26aulast%3DRyckaert%26aufirst%3DJ.-P.%26aulast%3DCiccotti%26aufirst%3DG.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DNumerical%2520integration%2520of%2520the%2520cartesian%2520equations%2520of%2520motion%2520of%2520a%2520system%2520with%2520constraints%253A%2520molecular%2520dynamics%2520of%2520n-alkanes%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D327%26epage%3D341%26doi%3D10.1016%2F0021-9991%2877%2990098-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kawasaki, K.</span></span> <span> </span><span class="NLM_article-title">Simple derivations of generalized linear and nonlinear Langevin equations</span>. <i>J. Phys. A: Math., Nucl. Gen.</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1289</span>, <span class="refDoi"> DOI: 10.1088/0305-4470/6/9/004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1088%2F0305-4470%2F6%2F9%2F004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1973&pages=1289&author=K.+Kawasaki&title=Simple+derivations+of+generalized+linear+and+nonlinear+Langevin+equations&doi=10.1088%2F0305-4470%2F6%2F9%2F004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1088%2F0305-4470%2F6%2F9%2F004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1088%252F0305-4470%252F6%252F9%252F004%26sid%3Dliteratum%253Aachs%26aulast%3DKawasaki%26aufirst%3DK.%26atitle%3DSimple%2520derivations%2520of%2520generalized%2520linear%2520and%2520nonlinear%2520Langevin%2520equations%26jtitle%3DJ.%2520Phys.%2520A%253A%2520Math.%252C%2520Nucl.%2520Gen.%26date%3D1973%26volume%3D6%26spage%3D1289%26doi%3D10.1088%2F0305-4470%2F6%2F9%2F004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Postma, J. P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van
Gunsteren, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiNola, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haak, J. R.</span></span> <span> </span><span class="NLM_article-title">Molecular dynamics with coupling to an external bath</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">3684</span>– <span class="NLM_lpage">3690</span>, <span class="refDoi"> DOI: 10.1063/1.448118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1063%2F1.448118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaL2cXmtlGksbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=1984&pages=3684-3690&author=H.+J.+C.+Berendsenauthor=J.+P.+M.+Postmaauthor=W.+F.+van%0AGunsterenauthor=A.+DiNolaauthor=J.+R.+Haak&title=Molecular+dynamics+with+coupling+to+an+external+bath&doi=10.1063%2F1.448118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics with coupling to an external bath</span></div><div class="casAuthors">Berendsen, H. J. C.; Postma, J. P. M.; Van Gunsteren, W. F.; DiNola, A.; Haak, J. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3684-90</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">In mol. dynamics (MD) simulations, the need often arises to maintain such parameters as temp. or pressure rather than energy and vol., or to impose gradients for studying transport properties in nonequil. MD.  A method is described to realize coupling to an external bath with const. temp. or pressure with adjustable time consts. for the coupling.  The method is easily extendable to other variables and to gradients, and can be applied also to polyat. mols. involving internal constraints.  The influence of coupling time consts. on dynamical variables is evaluated.  A leap-frog algorithm is presented for the general case involving constraints with coupling to both a const. temp. and a const. pressure bath.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoecrpUZWkh37Vg90H21EOLACvtfcHk0lhjT3pEdoqY8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXmtlGksbY%253D&md5=5510dc00297d63b91ee3a7a4ae5aacb1</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1063%2F1.448118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.448118%26sid%3Dliteratum%253Aachs%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DPostma%26aufirst%3DJ.%2BP.%2BM.%26aulast%3Dvan%2BGunsteren%26aufirst%3DW.%2BF.%26aulast%3DDiNola%26aufirst%3DA.%26aulast%3DHaak%26aufirst%3DJ.%2BR.%26atitle%3DMolecular%2520dynamics%2520with%2520coupling%2520to%2520an%2520external%2520bath%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1984%26volume%3D81%26spage%3D3684%26epage%3D3690%26doi%3D10.1063%2F1.448118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Essmann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perera, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkowitz, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L. G.</span></span> <span> </span><span class="NLM_article-title">A smooth particle mesh Ewald method</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">8577</span>– <span class="NLM_lpage">8593</span>, <span class="refDoi"> DOI: 10.1063/1.470117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1063%2F1.470117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1995&pages=8577-8593&author=U.+Essmannauthor=L.+Pereraauthor=M.+L.+Berkowitzauthor=T.+Dardenauthor=H.+Leeauthor=L.+G.+Pedersen&title=A+smooth+particle+mesh+Ewald+method&doi=10.1063%2F1.470117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">A smooth particle mesh Ewald method</span></div><div class="casAuthors">Essmann, Ulrich; Perera, Lalith; Berkowitz, Max L.; Darden, Tom; Lee, Hsing; Pedersen, Lee G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8577-93</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The previously developed particle mesh Ewald method is reformulated in terms of efficient B-spline interpolation of the structure factors.  This reformulation allows a natural extension of the method to potentials of the form 1/rp with p ≥ 1.  Furthermore, efficient calcn. of the virial tensor follows.  Use of B-splines in the place of Lagrange interpolation leads to analytic gradients as well as a significant improvement in the accuracy.  The authors demonstrate that arbitrary accuracy can be achieved, independent of system size N, at a cost that scales as N log(N).  For biomol. systems with many thousands of atoms and this method permits the use of Ewald summation at a computational cost comparable to that of a simple truncation method of 10 Å or less.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_Bp1XlSY32LVg90H21EOLACvtfcHk0lhjT3pEdoqY8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D&md5=092a679dd3bee08da28df41e302383a7</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1063%2F1.470117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.470117%26sid%3Dliteratum%253Aachs%26aulast%3DEssmann%26aufirst%3DU.%26aulast%3DPerera%26aufirst%3DL.%26aulast%3DBerkowitz%26aufirst%3DM.%2BL.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DPedersen%26aufirst%3DL.%2BG.%26atitle%3DA%2520smooth%2520particle%2520mesh%2520Ewald%2520method%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1995%26volume%3D103%26spage%3D8577%26epage%3D8593%26doi%3D10.1063%2F1.470117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 11 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Lucas G. Viviani, Erika Piccirillo, Henning Ulrich, <span class="NLM_string-name hlFld-ContribAuthor">Antonia T.-do Amaral</span>. </span><span class="cited-content_cbyCitation_article-title">Virtual Screening Approach for the Identification of Hydroxamic Acids as Novel Human Ecto-5′-Nucleotidase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (2)
                                     , 621-630. <a href="https://doi.org/10.1021/acs.jcim.9b00884" title="DOI URL">https://doi.org/10.1021/acs.jcim.9b00884</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.9b00884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.9b00884%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DVirtual%252BScreening%252BApproach%252Bfor%252Bthe%252BIdentification%252Bof%252BHydroxamic%252BAcids%252Bas%252BNovel%252BHuman%252BEcto-5%2525E2%252580%2525B2-Nucleotidase%252BInhibitors%26aulast%3DViviani%26aufirst%3DLucas%2BG.%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D30092019%26date%3D26112019%26date%3D12112019%26volume%3D60%26issue%3D2%26spage%3D621%26epage%3D630" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Zachary Maben, Richa Arya, Digamber Rane, W. Frank An, Shailesh Metkar, Marc Hickey, Samantha Bender, Akbar Ali, Tina T. Nguyen, Irini Evnouchidou, Roger Schilling, Efstratios Stratikos, Jennifer Golden, <span class="NLM_string-name hlFld-ContribAuthor">Lawrence J. Stern</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Selective Inhibitors of Endoplasmic Reticulum Aminopeptidase 1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (1)
                                     , 103-121. <a href="https://doi.org/10.1021/acs.jmedchem.9b00293" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00293</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00293%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BSelective%252BInhibitors%252Bof%252BEndoplasmic%252BReticulum%252BAminopeptidase%252B1%26aulast%3DMaben%26aufirst%3DZachary%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D14022019%26date%3D30122019%26date%3D16122019%26volume%3D63%26issue%3D1%26spage%3D103%26epage%3D121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rency  Mathew</span>, <span class="hlFld-ContribAuthor ">Juliane  Wunderlich</span>, <span class="hlFld-ContribAuthor ">Karine  Thivierge</span>, <span class="hlFld-ContribAuthor ">Krystyna  Cwiklinski</span>, <span class="hlFld-ContribAuthor ">Claire  Dumont</span>, <span class="hlFld-ContribAuthor ">Leann  Tilley</span>, <span class="hlFld-ContribAuthor ">Petra  Rohrbach</span>, <span class="hlFld-ContribAuthor ">John P.  Dalton</span>. </span><span class="cited-content_cbyCitation_article-title">Biochemical and cellular characterisation of the Plasmodium falciparum M1 alanyl aminopeptidase (PfM1AAP) and M17 leucyl aminopeptidase (PfM17LAP). </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2021,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-021-82499-4" title="DOI URL">https://doi.org/10.1038/s41598-021-82499-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-021-82499-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-021-82499-4%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DBiochemical%252Band%252Bcellular%252Bcharacterisation%252Bof%252Bthe%252BPlasmodium%252Bfalciparum%252BM1%252Balanyl%252Baminopeptidase%252B%252528PfM1AAP%252529%252Band%252BM17%252Bleucyl%252Baminopeptidase%252B%252528PfM17LAP%252529%26aulast%3DMathew%26aufirst%3DRency%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Birgit  Wilding</span>, <span class="hlFld-ContribAuthor ">A. Elisa  Pasqua</span>, <span class="hlFld-ContribAuthor ">Nicola  E. A. Chessum</span>, <span class="hlFld-ContribAuthor ">Olivier A.  Pierrat</span>, <span class="hlFld-ContribAuthor ">Tamas  Hahner</span>, <span class="hlFld-ContribAuthor ">Kathy  Tomlin</span>, <span class="hlFld-ContribAuthor ">Erald  Shehu</span>, <span class="hlFld-ContribAuthor ">Rosemary  Burke</span>, <span class="hlFld-ContribAuthor ">G. Meirion  Richards</span>, <span class="hlFld-ContribAuthor ">Bradleigh  Whitton</span>, <span class="hlFld-ContribAuthor ">Esther N.  Arwert</span>, <span class="hlFld-ContribAuthor ">Arjun  Thapaliya</span>, <span class="hlFld-ContribAuthor ">Ramya  Salimraj</span>, <span class="hlFld-ContribAuthor ">Rob  van Montfort</span>, <span class="hlFld-ContribAuthor ">Agi  Skawinska</span>, <span class="hlFld-ContribAuthor ">Angela  Hayes</span>, <span class="hlFld-ContribAuthor ">Florence  Raynaud</span>, <span class="hlFld-ContribAuthor ">Rajesh  Chopra</span>, <span class="hlFld-ContribAuthor ">Keith  Jones</span>, <span class="hlFld-ContribAuthor ">Gary  Newton</span>, <span class="hlFld-ContribAuthor ">Matthew D.  Cheeseman</span>. </span><span class="cited-content_cbyCitation_article-title">Investigating the phosphinic acid tripeptide mimetic DG013A as a tool compound inhibitor of the M1-aminopeptidase ERAP1. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>42 </em>, 128050. <a href="https://doi.org/10.1016/j.bmcl.2021.128050" title="DOI URL">https://doi.org/10.1016/j.bmcl.2021.128050</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2021.128050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2021.128050%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DInvestigating%252Bthe%252Bphosphinic%252Bacid%252Btripeptide%252Bmimetic%252BDG013A%252Bas%252Ba%252Btool%252Bcompound%252Binhibitor%252Bof%252Bthe%252BM1-aminopeptidase%252BERAP1%26aulast%3DWilding%26aufirst%3DBirgit%26date%3D2021%26volume%3D42%26spage%3D128050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Cheng  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiao-Qing  Ye</span>, <span class="hlFld-ContribAuthor ">Wenting  Du</span>, <span class="hlFld-ContribAuthor ">Shenxin  Zeng</span>, <span class="hlFld-ContribAuthor ">Jian-Hong  Xu</span>, <span class="hlFld-ContribAuthor ">Hong  Yin</span>. </span><span class="cited-content_cbyCitation_article-title">An efficient and practical aerobic oxidation of benzylic methylenes by recyclable
              N
              -hydroxyimide. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2021,</strong> <em>11 </em>
                                    (5)
                                     , 3003-3011. <a href="https://doi.org/10.1039/D0RA10475B" title="DOI URL">https://doi.org/10.1039/D0RA10475B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0RA10475B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0RA10475B%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DAn%252Befficient%252Band%252Bpractical%252Baerobic%252Boxidation%252Bof%252Bbenzylic%252Bmethylenes%252Bby%252Brecyclable%252BN%252B-hydroxyimide%26aulast%3DWang%26aufirst%3DJian%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D5%26spage%3D3003%26epage%3D3011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jisook  Lee</span>, <span class="hlFld-ContribAuthor ">Nyssa  Drinkwater</span>, <span class="hlFld-ContribAuthor ">Sheena  McGowan</span>, <span class="hlFld-ContribAuthor ">Peter  Scammells</span>. </span><span class="cited-content_cbyCitation_article-title">A Structure−Activity Relationship Study of Novel Hydroxamic Acid Inhibitors around the S1 Subsite of Human Aminopeptidase N. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>16 </em>
                                    (1)
                                     , 234-249. <a href="https://doi.org/10.1002/cmdc.202000527" title="DOI URL">https://doi.org/10.1002/cmdc.202000527</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000527%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DA%252BStructure%2525E2%252588%252592Activity%252BRelationship%252BStudy%252Bof%252BNovel%252BHydroxamic%252BAcid%252BInhibitors%252Baround%252Bthe%252BS1%252BSubsite%252Bof%252BHuman%252BAminopeptidase%252BN%26aulast%3DLee%26aufirst%3DJisook%26date%3D2021%26date%3D2020%26volume%3D16%26issue%3D1%26spage%3D234%26epage%3D249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenhua  Liu</span>, <span class="hlFld-ContribAuthor ">Yuchen  Liang</span>, <span class="hlFld-ContribAuthor ">Xiaoyong  Si</span>. </span><span class="cited-content_cbyCitation_article-title">Hydroxamic acid hybrids as the potential anticancer agents: An Overview. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>205 </em>, 112679. <a href="https://doi.org/10.1016/j.ejmech.2020.112679" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112679</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112679&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112679%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHydroxamic%252Bacid%252Bhybrids%252Bas%252Bthe%252Bpotential%252Banticancer%252Bagents%25253A%252BAn%252BOverview%26aulast%3DLiu%26aufirst%3DWenhua%26date%3D2020%26volume%3D205%26spage%3D112679" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nicholas  Barlow</span>, <span class="hlFld-ContribAuthor ">Philip E.  Thompson</span>. </span><span class="cited-content_cbyCitation_article-title">IRAP Inhibitors: M1-Aminopeptidase Family Inspiration. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2020,</strong> <em>11 </em><a href="https://doi.org/10.3389/fphar.2020.585930" title="DOI URL">https://doi.org/10.3389/fphar.2020.585930</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2020.585930&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2020.585930%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DIRAP%252BInhibitors%25253A%252BM1-Aminopeptidase%252BFamily%252BInspiration%26aulast%3DBarlow%26aufirst%3DNicholas%26date%3D2020%26date%3D2020%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Weronika  Wanat</span>, <span class="hlFld-ContribAuthor ">Michał  Talma</span>, <span class="hlFld-ContribAuthor ">Błażej  Dziuk</span>, <span class="hlFld-ContribAuthor ">Paweł  Kafarski</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and Inhibitory Studies of Phosphonic Acid Analogues of Homophenylalanine and Phenylalanine towards Alanyl Aminopeptidases. </span><span class="cited-content_cbyCitation_journal-name">Biomolecules</span><span> <strong>2020,</strong> <em>10 </em>
                                    (9)
                                     , 1319. <a href="https://doi.org/10.3390/biom10091319" title="DOI URL">https://doi.org/10.3390/biom10091319</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/biom10091319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fbiom10091319%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomolecules%26atitle%3DSynthesis%252Band%252BInhibitory%252BStudies%252Bof%252BPhosphonic%252BAcid%252BAnalogues%252Bof%252BHomophenylalanine%252Band%252BPhenylalanine%252Btowards%252BAlanyl%252BAminopeptidases%26aulast%3DWanat%26aufirst%3DWeronika%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D9%26spage%3D1319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mengying  Zhai</span>, <span class="hlFld-ContribAuthor ">Zixia  Yang</span>, <span class="hlFld-ContribAuthor ">Chenrui  Zhang</span>, <span class="hlFld-ContribAuthor ">Jinping  Li</span>, <span class="hlFld-ContribAuthor ">Jing  Jia</span>, <span class="hlFld-ContribAuthor ">Lingyi  Zhou</span>, <span class="hlFld-ContribAuthor ">Rong  Lu</span>, <span class="hlFld-ContribAuthor ">Zhi  Yao</span>, <span class="hlFld-ContribAuthor ">Zheng  Fu</span>. </span><span class="cited-content_cbyCitation_article-title">APN-mediated phosphorylation of BCKDK promotes hepatocellular carcinoma metastasis and proliferation via the ERK signaling pathway. </span><span class="cited-content_cbyCitation_journal-name">Cell Death & Disease</span><span> <strong>2020,</strong> <em>11 </em>
                                    (5)
                                     <a href="https://doi.org/10.1038/s41419-020-2610-1" title="DOI URL">https://doi.org/10.1038/s41419-020-2610-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41419-020-2610-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41419-020-2610-1%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Death%2520%2526%2520Disease%26atitle%3DAPN-mediated%252Bphosphorylation%252Bof%252BBCKDK%252Bpromotes%252Bhepatocellular%252Bcarcinoma%252Bmetastasis%252Band%252Bproliferation%252Bvia%252Bthe%252BERK%252Bsignaling%252Bpathway%26aulast%3DZhai%26aufirst%3DMengying%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Weronika  Wanat</span>, <span class="hlFld-ContribAuthor ">Michał  Talma</span>, <span class="hlFld-ContribAuthor ">Błażej  Dziuk</span>, <span class="hlFld-ContribAuthor ">Jean-Luc  Pirat</span>, <span class="hlFld-ContribAuthor ">Paweł  Kafarski</span>. </span><span class="cited-content_cbyCitation_article-title">Phosphonic Acid Analogs of Fluorophenylalanines as Inhibitors of Human and Porcine Aminopeptidases N: Validation of the Importance of the Substitution of the Aromatic Ring. </span><span class="cited-content_cbyCitation_journal-name">Biomolecules</span><span> <strong>2020,</strong> <em>10 </em>
                                    (4)
                                     , 579. <a href="https://doi.org/10.3390/biom10040579" title="DOI URL">https://doi.org/10.3390/biom10040579</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/biom10040579&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fbiom10040579%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomolecules%26atitle%3DPhosphonic%252BAcid%252BAnalogs%252Bof%252BFluorophenylalanines%252Bas%252BInhibitors%252Bof%252BHuman%252Band%252BPorcine%252BAminopeptidases%252BN%25253A%252BValidation%252Bof%252Bthe%252BImportance%252Bof%252Bthe%252BSubstitution%252Bof%252Bthe%252BAromatic%252BRing%26aulast%3DWanat%26aufirst%3DWeronika%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D4%26spage%3D579" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/medium/jm-2019-007578_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00757&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/medium/jm-2019-007578_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0001.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structure of <i>N</i>-(2-(hydroxyamino)-2-oxo-1-[3′,4′,5′-trifluoro(1,1′-biphenyl)-4-yl]ethyl)pivalamide (<b>1</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00757&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/medium/jm-2019-007578_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) X-ray crystal structure of <b>1</b> bound to <i>Pf</i>A-M1 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZX4">4ZX4</a>). (B) Predicted binding pose of <b>1</b> bound to APN obtained from docking (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYQ">4FYQ</a>). (C) Molecular dynamics (MD) simulation of <b>1</b> docked to APN showed that the trifluorophenyl group formed stable interactions in the S1 pocket. The ligand is shown in magenta, and residues of <i>Pf</i>A-M1 and APN are colored in blue and dark green, respectively. Interactions between the ligand and proteins are depicted as black dashed lines. The zinc is represented by the gray sphere.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00757&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/medium/jm-2019-007578_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structures of targeted hydroxamic acid analogues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00757&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/medium/jm-2019-007578_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0007.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Hydroxamic Acid Inhibitors<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00757&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 3,4,5-trifluorophenylboronic acid, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, 1 M Na<sub>2</sub>CO<sub>3</sub>, tetrahydrofuran (THF), reflux, 2 h; (b) SeO<sub>2</sub>, anhyd. pyridine, reflux, 24 h; (c) (i) benzylamine, Na(OAc)<sub>3</sub>BH, dichloroethane (DCE), room temperature (rt), 24 h; (ii) concd H<sub>2</sub>SO<sub>4</sub>, MeOH, reflux, 24 h; (iii) H<sub>2</sub>, 10% Pd/C, cat. HCl MeOH, rt, 24 h; (d) (i) carbamoylbenzoic acid, (COCl)<sub>2</sub>, dimethylformamide (DMF), dichloromethane (DCM), rt, 1 h; (ii) <b>4</b>, <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA), DCM, rt, 30 min; (e) carboxylic acid, <i>O</i>-(1<i>H</i>-6-chlorobenzotriazol-l-yl)-1,1,3,3-tetramethyluronium hexafluorophosphate (HCTU), DIPEA, DMF, DCM, rt, 24 h; (f) carboxylic acid, 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide (EDCI), 4-dimethylamonipyridine (DMAP), DCM, rt, 24 h; (g) NH<sub>2</sub>OH·HCl, 5 M KOH in anhyd. MeOH, rt; (h) 20% trifluoroacetyl (TFA) in DCM, rt; (i) 1 M BBr<sub>3</sub> in DCM, −78 °C to rt, 2–24 h.</p></p></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/medium/jm-2019-007578_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0008.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Benzylamine Analogue <b>9</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00757&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) benzaldehyde, Na(OAc)<sub>3</sub>BH, anhyd. DCE, rt, 24 h; (b) NH<sub>2</sub>OH·HCl, 5 M KOH in anhyd. MeOH, rt; (c) (i) 1 M BBr<sub>3</sub> in DCM, −78 °C to rt, 2–24 h; (ii) 1 M HCl, MeOH, rt, 24 h.</p></p></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/medium/jm-2019-007578_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Three conformations of compound <b>6ad</b> (A–C) observed during MD simulation. The carbon atoms of the flexible loop are colored in light pink, and other residues in the binding site are wheat-colored. Interactions between compound <b>6ad</b> (carbon atoms in white) and APN are shown in black dashed lines. (D) Overlay of two predicted binding poses of compound <b>6ae</b>. Residues around compound <b>6ae</b> (carbon atoms in magenta) are wheat-colored (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYQ">4FYQ</a>). Overlaid Arg<sup>442</sup> residues are shown in green. Interactions between the compound and APN are depicted in black dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00757&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/medium/jm-2019-007578_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Comparison of the binding interactions occurred with tight-binding inhibitor <b>6ad</b> (left) and the weaker inhibitor <b>6ag</b> (right) (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYQ">4FYQ</a>). The carbon atoms of the flexible loop are colored in light pink, and other residues in the binding site are wheat-colored. Compounds <b>6ad</b> and <b>6ag</b> are colored in white and green, respectively. Interactions between the ligands and APN are shown in black dashed lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00757&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/medium/jm-2019-007578_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0006.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Hydroxamic acid–zinc complex in the active site of APN. APN residues are wheat-colored, and the ligand is shown in magenta. Interactions between the ligand and the catalytic triad and Glu<sup>389</sup> are illustrated in black dashed lines (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYQ">4FYQ</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-15/acs.jmedchem.9b00757/20190801/images/large/jm-2019-007578_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00757&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i131">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15755" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15755" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 80 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maroux, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louvard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baratti, J.</span></span> <span> </span><span class="NLM_article-title">The aminopeptidase from hog intestinal brush border</span>. <i>Biochim. Biophys. Acta, Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>321</i></span>,  <span class="NLM_fpage">282</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1016/0005-2744(73)90083-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2F0005-2744%2873%2990083-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=4750768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaE2cXksFSmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=1973&pages=282-295&author=S.+Marouxauthor=D.+Louvardauthor=J.+Baratti&title=The+aminopeptidase+from+hog+intestinal+brush+border&doi=10.1016%2F0005-2744%2873%2990083-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Aminopeptidase from hog intestinal brush border</span></div><div class="casAuthors">Maroux, S.; Louvard, D.; Baratti, J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Enzymology</span>
        (<span class="NLM_cas:date">1973</span>),
    <span class="NLM_cas:volume">321</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">282-95</span>CODEN:
                <span class="NLM_cas:coden">BBEZAD</span>;
        ISSN:<span class="NLM_cas:issn">0924-1086</span>.
    </div><div class="casAbstract">The aminopeptidase bound to the brush border membrane of porcine jejunum and ileum was purified.  After treatment of the mucosa with Triton X-100, the enzyme was still attached to very small membrane fragments (the aminopeptidase complex) from which it was liberated by trypsin.  The free enzyme could be further purified to homogeneity by using classical techniques.  The high-molecular-weight complex contained, besides aminopeptidase, at least 1 other protein inactive on the substrates of this enzyme.  The free aminopeptidase had a molecular weight of ∼280,000 and contained 23% sugars and 2 atoms of Zn per mole.  Gel electrophoresis in Na dodecyl sulfate showed 4 bands.  One, however, is probably an artifact arising from trypsin treatment.  The other 3 suggest the existence in aminopeptidase of 3 subunits with molecular weights of 130,000, 97,000, and 49,000 and which were also obtained from the complex.  The purified aminopeptidase accounts for all, or almost all, of the peptidase activity of the brush border membrane.  It also accounts for all of the arylamidase activity, and a small but apparently significant substrate, ∼ half the tripeptidase activity, and a small but apparently significant part of the dipeptidase activity of the whole jejunal and ileal mucosa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqkqw_YjJkBLVg90H21EOLACvtfcHk0ljBIKGMkAd-UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2cXksFSmsQ%253D%253D&md5=aab1fece3802aaa127064adcfa44392b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2F0005-2744%2873%2990083-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0005-2744%252873%252990083-1%26sid%3Dliteratum%253Aachs%26aulast%3DMaroux%26aufirst%3DS.%26aulast%3DLouvard%26aufirst%3DD.%26aulast%3DBaratti%26aufirst%3DJ.%26atitle%3DThe%2520aminopeptidase%2520from%2520hog%2520intestinal%2520brush%2520border%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Enzymol.%26date%3D1973%26volume%3D321%26spage%3D282%26epage%3D295%26doi%3D10.1016%2F0005-2744%2873%2990083-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dixon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaklamanis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turley, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickson, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leek, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gatter, K. C.</span></span> <span> </span><span class="NLM_article-title">Expression of aminopeptidase-n (CD 13) in normal tissues and malignant neoplasms of epithelial and lymphoid origin</span>. <i>J. Clin. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">47</span>, <span class="refDoi"> DOI: 10.1136/jcp.47.1.43</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1136%2Fjcp.47.1.43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=7907609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A280%3ADyaK2c7nvFGhtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=1994&pages=43-47&author=J.+Dixonauthor=L.+Kaklamanisauthor=H.+Turleyauthor=I.+D.+Hicksonauthor=R.+D.+Leekauthor=A.+L.+Harrisauthor=K.+C.+Gatter&title=Expression+of+aminopeptidase-n+%28CD+13%29+in+normal+tissues+and+malignant+neoplasms+of+epithelial+and+lymphoid+origin&doi=10.1136%2Fjcp.47.1.43"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of aminopeptidase-n (CD 13) in normal tissues and malignant neoplasms of epithelial and lymphoid origin</span></div><div class="casAuthors">Dixon J; Kaklamanis L; Turley H; Hickson I D; Leek R D; Harris A L; Gatter K C</div><div class="citationInfo"><span class="NLM_cas:title">Journal of clinical pathology</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-7</span>
        ISSN:<span class="NLM_cas:issn">0021-9746</span>.
    </div><div class="casAbstract">AIMS:  To provide a detailed knowledge of the distribution of the CD13 molecule, also known as the protease aminopeptidase-N, on both normal tissues and malignant neoplasms of epithelial and lymphoid origin.  METHODS:  CD13 antigen was examined by immunocytochemistry, using a recently produced antibody (VS5E) alongside a commercially available anti-CD13 monoclonal antibody.  The VS5E recognising CD13 was produced by immunising a doxorubicin resistant breast cancer cell line (MCF-7-ADr).  A striking feature of this antibody was that it stained the doxorubicin resistant cells but not the parental cell line.  Both antibodies were tested on a broad range of normal tissues and three common types of epithelial malignancy (colon n = 28, lung n = 30, breast n = 35), and 12 cases of Hodgkin's and 52 of non-Hodgkin's lymphomas.  RESULTS:  CD13 was expressed on many tissue and cell types outside the haematopoietic system.  In particular it was present on breast epithelium and in 20% (seven of 35) of breast carcinomas, but absent in normal and neoplastic colonic and bronchial tissues and lymphomas.  CONCLUSIONS:  This study provides not only detailed information about the expression of the CD13 antigen, but also raises the important possibility that CD13 expression may correlate with drug resistance in breast carcinomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRHCFYsz4T-0ax48Rj5GnGofW6udTcc2eY8vV0wI6BeL7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK2c7nvFGhtQ%253D%253D&md5=6cd79912e08bdf02227d88d22b78a781</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1136%2Fjcp.47.1.43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fjcp.47.1.43%26sid%3Dliteratum%253Aachs%26aulast%3DDixon%26aufirst%3DJ.%26aulast%3DKaklamanis%26aufirst%3DL.%26aulast%3DTurley%26aufirst%3DH.%26aulast%3DHickson%26aufirst%3DI.%2BD.%26aulast%3DLeek%26aufirst%3DR.%2BD.%26aulast%3DHarris%26aufirst%3DA.%2BL.%26aulast%3DGatter%26aufirst%3DK.%2BC.%26atitle%3DExpression%2520of%2520aminopeptidase-n%2520%2528CD%252013%2529%2520in%2520normal%2520tissues%2520and%2520malignant%2520neoplasms%2520of%2520epithelial%2520and%2520lymphoid%2520origin%26jtitle%3DJ.%2520Clin.%2520Pathol.%26date%3D1994%26volume%3D47%26spage%3D43%26epage%3D47%26doi%3D10.1136%2Fjcp.47.1.43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mina-Osorio, P.</span></span> <span> </span><span class="NLM_article-title">The moonlighting enzyme CD13: old and new functions to target</span>. <i>Trends Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">371</span>, <span class="refDoi"> DOI: 10.1016/j.molmed.2008.06.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.molmed.2008.06.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=18603472" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD1cXpsVCltrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=361-371&author=P.+Mina-Osorio&title=The+moonlighting+enzyme+CD13%3A+old+and+new+functions+to+target&doi=10.1016%2Fj.molmed.2008.06.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The moonlighting enzyme CD13: old and new functions to target</span></div><div class="casAuthors">Mina-Osorio, Paola</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Molecular Medicine</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">361-371</span>CODEN:
                <span class="NLM_cas:coden">TMMRCY</span>;
        ISSN:<span class="NLM_cas:issn">1471-4914</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Aminopeptidase N (CD13) is a widely expressed ectoenzyme with functions that do not always depend on its enzymic activity: an aspect that has been overlooked.  Numerous CD13-targeting tools have been developed in the last few years.  Several of them are already undergoing clin. trials, and there are promising reports on the effectiveness of others in animal models of disease.  However, their efficacy might be obscured by their effects on unrecognized functions of CD13, resulting in unexpected complications.  The purpose of this review is (i) to discuss the various functions ascribed to CD13 and the possible mechanisms behind them and (ii) to consider some of the questions that need to be answered to achieve a better understanding of the biol. relevance of these functions, a more precise interpretation of the results obtained after their manipulation and a more rational design of CD13-targeting agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVuM5Zcq3wFrVg90H21EOLACvtfcHk0ljBIKGMkAd-UA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXpsVCltrc%253D&md5=10447fdd2c56d40b650bd904363f5fc5</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.molmed.2008.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molmed.2008.06.003%26sid%3Dliteratum%253Aachs%26aulast%3DMina-Osorio%26aufirst%3DP.%26atitle%3DThe%2520moonlighting%2520enzyme%2520CD13%253A%2520old%2520and%2520new%2520functions%2520to%2520target%26jtitle%3DTrends%2520Mol.%2520Med.%26date%3D2008%26volume%3D14%26spage%3D361%26epage%3D371%26doi%3D10.1016%2Fj.molmed.2008.06.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wong, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rini, J. M.</span></span> <span> </span><span class="NLM_article-title">The X-ray crystal structure of human aminopeptidase N reveals a novel dimer and the basis for peptide processing</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">36804</span>– <span class="NLM_lpage">36813</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.398842</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1074%2Fjbc.M112.398842" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=22932899" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFOqsrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=36804-36813&author=A.+H.+Wongauthor=D.+Zhouauthor=J.+M.+Rini&title=The+X-ray+crystal+structure+of+human+aminopeptidase+N+reveals+a+novel+dimer+and+the+basis+for+peptide+processing&doi=10.1074%2Fjbc.M112.398842"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The X-ray Crystal Structure of Human Aminopeptidase N Reveals a Novel Dimer and the Basis for Peptide Processing</span></div><div class="casAuthors">Wong, Alan H. M.; Zhou, Dongxia; Rini, James M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">36804-36813</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Human aminopeptidase N (hAPN/hCD13) is a dimeric membrane protein and a member of the M1 family of zinc metallopeptidases.  Within the rennin-angiotensin system, its enzymic activity is responsible for processing peptide hormones angiotensin III and IV.  In addn., hAPN is also involved in cell adhesion, endocytosis, and signal transduction and it is an important target for cancer therapy.  Reported here are the high resoln. x-ray crystal structures of the dimeric ectodomain of hAPN and its complexes with angiotensin IV and the peptidomimetic inhibitors, amastatin and bestatin.  Each monomer of the dimer is found in what has been termed the closed form in other M1 enzymes and each monomer is characterized by an internal cavity surrounding the catalytic site as well as a unique substrate/inhibitor-dependent loop ordering, which in the case of the bestatin complex suggests a new route to inhibitor design.  The hAPN structure provides the first example of a dimeric M1 family member and the obsd. structural features, in conjunction with a model for the open form, provide novel insights into the mechanism of peptide processing and signal transduction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1AkeQE7nfNLVg90H21EOLACvtfcHk0liSB0mGYiAiug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFOqsrnF&md5=7b4950107a47e98f51b22d02695b8363</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.398842&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.398842%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DA.%2BH.%26aulast%3DZhou%26aufirst%3DD.%26aulast%3DRini%26aufirst%3DJ.%2BM.%26atitle%3DThe%2520X-ray%2520crystal%2520structure%2520of%2520human%2520aminopeptidase%2520N%2520reveals%2520a%2520novel%2520dimer%2520and%2520the%2520basis%2520for%2520peptide%2520processing%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D36804%26epage%3D36813%26doi%3D10.1074%2Fjbc.M112.398842" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fournie-Zaluski, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chauvel, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roques, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corvol, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llorens-Cortes, C.</span></span> <span> </span><span class="NLM_article-title">Identification of metabolic pathways of brain angiotensin II and III using specific aminopeptidase inhibitors: predominant role of angiotensin III in the control of vasopressin release</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">11968</span>– <span class="NLM_lpage">11973</span>, <span class="refDoi"> DOI: 10.1073/pnas.93.21.11968</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1073%2Fpnas.93.21.11968" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=8876246" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaK28XmtlarsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1996&pages=11968-11973&author=S.+Ziniauthor=M.+C.+Fournie-Zaluskiauthor=E.+Chauvelauthor=B.+P.+Roquesauthor=P.+Corvolauthor=C.+Llorens-Cortes&title=Identification+of+metabolic+pathways+of+brain+angiotensin+II+and+III+using+specific+aminopeptidase+inhibitors%3A+predominant+role+of+angiotensin+III+in+the+control+of+vasopressin+release&doi=10.1073%2Fpnas.93.21.11968"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of metabolic pathways of brain angiotensin II and III using specific aminopeptidase inhibitors: Predominant role of angiotensin III in the control of vasopressin release</span></div><div class="casAuthors">Zini, Sylvie; Fournie-Zaluski, Marie-Claude; Chauvel, Eric; Roques, Bernard P.; Corvol, Pierre; Llorens-Cortex, Catherine</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11968-11973</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Angiotensin (Ang) II and Ang III are two peptide effectors of the brain renin-angiotensin system that participate in the control of blood pressure and increase water consumption and vasopressin release.  To delineate the resp. roles of these peptides in the regulation of vasopressin secretion, their metabolic pathways and their effects on vasopressin release were identified in vivo.  For this purpose, we used recently developed selective inhibitors of aminopeptidase A (APA) and aminopeptidase N (APN), two enzymes that are believed to be responsible for the N-terminal cleavage of Ang II and Ang III, resp.  Mice received [3H]Ang II intracerebroventricularly (i.c.v.) in the presence or absence of the APA inhibitor, EC 33 (3-amino-4-thio-Bu sulfonate) or the APN inhibitor, EC 27 (2-amino-pentan-1,5-dithiol).  [3H]Ang II and [3H]Ang III levels were evaluated from hypothalamus homogenates by HPLC.  EC 33 increased the half-life of [3H]Ang II 2.6-fold and completely blocked the formation of [3H]Ang III, whereas EC 27 increased the half-life of [3H]Ang III 2.3-fold.  In addn., the effects of EC 33 and EC 27 on Ang-induced vasopressin release were studied in mice.  Ang II was injected i.c.v. in the presence or absence of EC 33, and plasma vasopressin levels were estd. by RIA.  While vasopressin levels were increased 2-fold by Ang II (5 ng), EC 33 inhibited Ang II-induced vasopressin release in a dose-dependent manner.  In contrast, EC 27 injected alone increased in a dose-dependent manner vasopressin levels.  The EC 27-induced vasopressin release was completely blocked by the coadministration of the Ang receptor antagonist [Sar1-Ala8]Ang II.  Thus, APA and APN are involved in vivo in the metab. of brain Ang II and Ang III, resp., and the action of Ang II on vasopressin release depends upon the prior conversion of Ang II to Ang III.  This shows that Ang III behaves as one of the main effector peptides of the brain renin-angiotensin system in the control of vasopressin release.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpOvvF5qOP0bVg90H21EOLACvtfcHk0liSB0mGYiAiug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XmtlarsrY%253D&md5=0526bbe74b15a9984199e2163c3a6702</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1073%2Fpnas.93.21.11968&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.93.21.11968%26sid%3Dliteratum%253Aachs%26aulast%3DZini%26aufirst%3DS.%26aulast%3DFournie-Zaluski%26aufirst%3DM.%2BC.%26aulast%3DChauvel%26aufirst%3DE.%26aulast%3DRoques%26aufirst%3DB.%2BP.%26aulast%3DCorvol%26aufirst%3DP.%26aulast%3DLlorens-Cortes%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520metabolic%2520pathways%2520of%2520brain%2520angiotensin%2520II%2520and%2520III%2520using%2520specific%2520aminopeptidase%2520inhibitors%253A%2520predominant%2520role%2520of%2520angiotensin%2520III%2520in%2520the%2520control%2520of%2520vasopressin%2520release%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1996%26volume%3D93%26spage%3D11968%26epage%3D11973%26doi%3D10.1073%2Fpnas.93.21.11968" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giros, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gros, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solhonne, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartz, J. C.</span></span> <span> </span><span class="NLM_article-title">Characterization of aminopeptidases responsible for inactivating endogenous (Met5)enkephalin in brain slices using peptidase inhibitors and anti-aminopeptidase M antibodies</span>. <i>Mol. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">281</span>– <span class="NLM_lpage">287</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=2869404" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaL28XhvVaitbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1986&pages=281-287&author=B.+Girosauthor=C.+Grosauthor=B.+Solhonneauthor=J.+C.+Schwartz&title=Characterization+of+aminopeptidases+responsible+for+inactivating+endogenous+%28Met5%29enkephalin+in+brain+slices+using+peptidase+inhibitors+and+anti-aminopeptidase+M+antibodies"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of aminopeptidases responsible for inactivating endogenous (Met5)enkephalin in brain slices using peptidase inhibitors and anti-aminopeptidase M antibodies</span></div><div class="casAuthors">Giros, Bruno; Gros, Claude; Solhonne, Brigitte; Schwartz, Jean Charles</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmacology</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">281-7</span>CODEN:
                <span class="NLM_cas:coden">MOPMA3</span>;
        ISSN:<span class="NLM_cas:issn">0026-895X</span>.
    </div><div class="casAbstract">In addn. to enkephalinase (EC 3.4.24.11), 2 enkephalin-hydrolyzing aminopeptidases recently identified in cerebral membranes [aminopeptidase M (EC 3.4.11.2) and a puromycin-sensitive aminopeptidase (also designated MII or aminoenkephalinase)] are potentially involved in endogenous enkephalin inactivation.  Their participation in the hydrolysis of the endogenous (Met5)enkephalin released by depolarization of slices from rat globus pallidus was assessed by using 3 inhibitory agents: bestatin, puromycin, and anti-aminopeptidase M antibodies.  The selectivity and potency of these agents were first detn. by evaluating their IC50 (concn. for 50% max. inhibition) values for inhibition of [3H](Met5)enkephalin hydrolysis by increasingly complex prepns. comprising semipurified aminopeptidases, pallidal membranes, and pallidal slices.  Bestatin was a fairly potent inhibitor but lacked selectivity, as there was only a 3-fold difference between its IC50 values for the 2 aminopeptidases, and it displayed restricted diffusion and degrdn. in the slice prepn.  Puromycin discriminated well between the 2 aminopeptidases (30-fold difference in IC50 values) and did not show any apparent restricted diffusion in the slice prepn.  Anti-aminopeptidase M antibodies were highly discriminant (>300-fold difference in IC50 values for the 2 aminopeptidases) but displayed restricted diffusion.  Anal. of the concn.-protection curves of the 3 agents for recovery of the (Met5)enkephalin released from pallidal slices in the presence of the enkephalinase inhibitor thiorphan indicated that both aminopeptidases participated in enkephalin degrdn. but that the role of aminopeptidase M was largely predominant, in contrast with its low relative activity in the prepn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouT8HLGNgJjrVg90H21EOLACvtfcHk0liSB0mGYiAiug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28XhvVaitbg%253D&md5=f65e26f08374b1474c519c768613683e</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGiros%26aufirst%3DB.%26aulast%3DGros%26aufirst%3DC.%26aulast%3DSolhonne%26aufirst%3DB.%26aulast%3DSchwartz%26aufirst%3DJ.%2BC.%26atitle%3DCharacterization%2520of%2520aminopeptidases%2520responsible%2520for%2520inactivating%2520endogenous%2520%2528Met5%2529enkephalin%2520in%2520brain%2520slices%2520using%2520peptidase%2520inhibitors%2520and%2520anti-aminopeptidase%2520M%2520antibodies%26jtitle%3DMol.%2520Pharmacol.%26date%3D1986%26volume%3D29%26spage%3D281%26epage%3D287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheinberg, D. A.</span></span> <span> </span><span class="NLM_article-title">Substance P and bradykinin are natural inhibitors of CD13/aminopeptidase N</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>208</i></span>,  <span class="NLM_fpage">664</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1006/bbrc.1995.1390</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1006%2Fbbrc.1995.1390" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=7535053" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaK2MXksVKhsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=208&publication_year=1995&pages=664-674&author=Y.+Xuauthor=D.+Wellnerauthor=D.+A.+Scheinberg&title=Substance+P+and+bradykinin+are+natural+inhibitors+of+CD13%2Faminopeptidase+N&doi=10.1006%2Fbbrc.1995.1390"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Substance P and bradykinin are natural inhibitors of CD13/aminopeptidase N</span></div><div class="casAuthors">Xu, Yang; Wellner, Daniel; Scheinberg, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">208</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">664-74</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Aminopeptidase N (EC 3.4.11.2) is an important enzyme that is involved in the degrdn. of regulatory peptides including enkephalins.  The authors report here that purified and native membrane-bound aminopeptidase N will sequentially and completely hydrolyze both Leu-enkephalin and Met-enkephalin from the amino terminus.  Both purified pig aminopeptidase N and the enzyme of live HL60 cells displayed similar Km values for enkephalin.  The naturally occurring neuropeptides substance P and bradykinin, and the morphine agonist, morphiceptin, were not hydrolyzed by aminopeptidase N and each inhibited the enzymic activity.  Each of these peptides contains a proline at the second residue.  The Ki values for substance P (0.44 μM), bradykinin (9.4 μM), and morphiceptin (169 μM) were obtained with the enzyme on live HL60 cells.  The values for the purified enzyme from pig were similar.  The potent inhibition of aminopeptidase N by substance P and bradykinin suggests that these peptides may be natural inhibitors of the enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKLpukdjBcGLVg90H21EOLACvtfcHk0ljfHdBDpiN55A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXksVKhsbw%253D&md5=67009d9bb902fd3a97c57f9128373e14</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1006%2Fbbrc.1995.1390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fbbrc.1995.1390%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DY.%26aulast%3DWellner%26aufirst%3DD.%26aulast%3DScheinberg%26aufirst%3DD.%2BA.%26atitle%3DSubstance%2520P%2520and%2520bradykinin%2520are%2520natural%2520inhibitors%2520of%2520CD13%252Faminopeptidase%2520N%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D1995%26volume%3D208%26spage%3D664%26epage%3D674%26doi%3D10.1006%2Fbbrc.1995.1390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bosch, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smits, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haagmans, B. L.</span></span> <span> </span><span class="NLM_article-title">Membrane ectopeptidases targeted by human coronaviruses</span>. <i>Curr. Opin. Virol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1016/j.coviro.2014.03.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.coviro.2014.03.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=24762977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVarsLjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=55-60&author=B.+J.+Boschauthor=S.+L.+Smitsauthor=B.+L.+Haagmans&title=Membrane+ectopeptidases+targeted+by+human+coronaviruses&doi=10.1016%2Fj.coviro.2014.03.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Membrane ectopeptidases targeted by human coronaviruses</span></div><div class="casAuthors">Bosch, Berend Jan; Smits, Saskia L.; Haagmans, Bart L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Virology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">55-60</span>CODEN:
                <span class="NLM_cas:coden">COVUAF</span>;
        ISSN:<span class="NLM_cas:issn">1879-6257</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">A review.  Six coronaviruses, including the recently identified Middle East respiratory syndrome coronavirus, are known to target the human respiratory tract causing mild to severe disease.  Their interaction with receptors expressed on cells located in the respiratory tract is an essential first step in the infection.  Thus far three membrane ectopeptidases, dipeptidyl peptidase 4 (DPP4), angiotensin-converting enzyme 2 (ACE2) and aminopeptidase N (APN), have been identified as entry receptors for four human-infecting coronaviruses.  Although the catalytic activity of the ACE2, APN, and DPP4 peptidases is not required for virus entry, co-expression of other host proteases allows efficient viral entry.  In addn., evolutionary conservation of these receptors may permit interspecies transmissions.  Because of the physiol. function of these peptidase systems, pathogenic host responses may be potentially amplified and cause acute respiratory distress.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohFxBacvTPabVg90H21EOLACvtfcHk0ljfHdBDpiN55A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVarsLjL&md5=b86845672b1f34fddab209e22e199bce</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.coviro.2014.03.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coviro.2014.03.011%26sid%3Dliteratum%253Aachs%26aulast%3DBosch%26aufirst%3DB.%2BJ.%26aulast%3DSmits%26aufirst%3DS.%2BL.%26aulast%3DHaagmans%26aufirst%3DB.%2BL.%26atitle%3DMembrane%2520ectopeptidases%2520targeted%2520by%2520human%2520coronaviruses%26jtitle%3DCurr.%2520Opin.%2520Virol.%26date%3D2014%26volume%3D6%26spage%3D55%26epage%3D60%26doi%3D10.1016%2Fj.coviro.2014.03.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegyi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maile, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heister, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagemann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siddell, S. G.</span></span> <span> </span><span class="NLM_article-title">Molecular analysis of the coronavirus-receptor function of aminopeptidase N</span>. <i>Adv. Exp. Med. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>440</i></span>,  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">67</span>, <span class="refDoi"> DOI: 10.1007/978-1-4615-5331-1_8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1007%2F978-1-4615-5331-1_8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=9782265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaK1cXmvFKgu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=440&publication_year=1998&pages=61-67&author=A.+F.+Kolbauthor=A.+Hegyiauthor=J.+Maileauthor=A.+Heisterauthor=M.+Hagemannauthor=S.+G.+Siddell&title=Molecular+analysis+of+the+coronavirus-receptor+function+of+aminopeptidase+N&doi=10.1007%2F978-1-4615-5331-1_8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular analysis of the coronavirus-receptor function of aminopeptidase N</span></div><div class="casAuthors">Kolb, Andreas F.; Hegyi, Annette; Maile, Julia; Heister, Angelien; Hagemann, Margitta; Siddell, Stuart G.</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Experimental Medicine and Biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">440</span>
        (<span class="NLM_cas:issue">Coronaviruses and Arteriviruses</span>),
    <span class="NLM_cas:pages">61-67</span>CODEN:
                <span class="NLM_cas:coden">AEMBAP</span>;
        ISSN:<span class="NLM_cas:issn">0065-2598</span>.
    
            (<span class="NLM_cas:orgname">Plenum Publishing Corp.</span>)
        </div><div class="casAbstract">Aminopeptidase N (APN) is a major cell surface for coronaviruses of the serogroup I.  By using chimeric APN proteins assembled from human, porcine and feline APN the authors have identified determinants which are critically involved in the coronavirus-APN interaction.  The results indicate that human coronavirus 229E (HCV 229E) is distinct from the other serogroup I coronaviruses in that determinants located within the N-terminal parts of the human and feline APN proteins mediate the infection of HCV 229E, whereas determinants located within the C-terminal parts of porcine, feline and canine APN mediate the infection of transmissible gastro-enteritis virus (TGEV), feline infectious peritonitis virus (FIPV) and canine coronavirus (CCV), resp.  A further anal. of the mapped amino acid segments by site directed mutagenesis revealed that a short stretch of 8 amino acids in the hAPN protein plays a decisive role in mediating HCV 229E reception.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKoYqOIhFVh7Vg90H21EOLACvtfcHk0ljfHdBDpiN55A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmvFKgu78%253D&md5=088b2ac1d1445e968d14097778819963</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2F978-1-4615-5331-1_8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-4615-5331-1_8%26sid%3Dliteratum%253Aachs%26aulast%3DKolb%26aufirst%3DA.%2BF.%26aulast%3DHegyi%26aufirst%3DA.%26aulast%3DMaile%26aufirst%3DJ.%26aulast%3DHeister%26aufirst%3DA.%26aulast%3DHagemann%26aufirst%3DM.%26aulast%3DSiddell%26aufirst%3DS.%2BG.%26atitle%3DMolecular%2520analysis%2520of%2520the%2520coronavirus-receptor%2520function%2520of%2520aminopeptidase%2520N%26jtitle%3DAdv.%2520Exp.%2520Med.%2520Biol.%26date%3D1998%26volume%3D440%26spage%3D61%26epage%3D67%26doi%3D10.1007%2F978-1-4615-5331-1_8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yeager, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashmun, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardellichio, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Look, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, K. V.</span></span> <span> </span><span class="NLM_article-title">Human aminopeptidase N is a receptor for human coronavirus 229E</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>357</i></span>,  <span class="NLM_fpage">420</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1038/357420a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1038%2F357420a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1350662" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaK38Xkt1CjtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=357&publication_year=1992&pages=420-422&author=C.+L.+Yeagerauthor=R.+A.+Ashmunauthor=R.+K.+Williamsauthor=C.+B.+Cardellichioauthor=L.+H.+Shapiroauthor=A.+T.+Lookauthor=K.+V.+Holmes&title=Human+aminopeptidase+N+is+a+receptor+for+human+coronavirus+229E&doi=10.1038%2F357420a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Human aminopeptidase N is a receptor for human coronavirus 229E</span></div><div class="casAuthors">Yeager, Curtis L.; Ashmun, Richard A.; Williams, Richard K.; Cardellichio, Christine B.; Shapiro, Linda H.; Look, A. Thomas; Holmes, Kathryn V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">357</span>
        (<span class="NLM_cas:issue">6377</span>),
    <span class="NLM_cas:pages">420-2</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Transmissible gastroenteritis virus (TGEV) replicates selectively in the differentiated enterocytes covering the villi of the small intestine.  In order to investigate the mol. determinants of the infection, the surface mol. used by the virus for binding and entry into host cells was studied.  Aminopeptidase N, an ectoenzyme abundantly expressed at the apical membrane of the enterocyte serves as a receptor for TGEV.  Monoclonal antibodies were selected for their ability to block infection by TGEV of porcine cell lines.  They recognized a brush border membrane protein of Mr 150K, which was identified as aminopeptidase N by N-terminal sequencing.  Two lines of evidence supported the view that the peptidase itself acts as a receptor.  First, virions bound specifically to aminopeptidase N that was purified to homogeneity.  Second, recombinant expression of aminopeptidase N conferred infectivity by TGEV to an otherwise non-permissive cell line.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphfMWdeAI-0rVg90H21EOLACvtfcHk0li9eI2l0k0pRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xkt1CjtL4%253D&md5=e911c3bd8d7b89cc84e41beca7151b7a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2F357420a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F357420a0%26sid%3Dliteratum%253Aachs%26aulast%3DYeager%26aufirst%3DC.%2BL.%26aulast%3DAshmun%26aufirst%3DR.%2BA.%26aulast%3DWilliams%26aufirst%3DR.%2BK.%26aulast%3DCardellichio%26aufirst%3DC.%2BB.%26aulast%3DShapiro%26aufirst%3DL.%2BH.%26aulast%3DLook%26aufirst%3DA.%2BT.%26aulast%3DHolmes%26aufirst%3DK.%2BV.%26atitle%3DHuman%2520aminopeptidase%2520N%2520is%2520a%2520receptor%2520for%2520human%2520coronavirus%2520229E%26jtitle%3DNature%26date%3D1992%26volume%3D357%26spage%3D420%26epage%3D422%26doi%3D10.1038%2F357420a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mina-Osorio, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, E.</span></span> <span> </span><span class="NLM_article-title">Aminopeptidase N (CD13) functionally interacts with FcgammaRs in human monocytes</span>. <i>J. Leukocyte Biol.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">1008</span>– <span class="NLM_lpage">1017</span>, <span class="refDoi"> DOI: 10.1189/jlb.1204714</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1189%2Fjlb.1204714" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=15758076" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltValurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2005&pages=1008-1017&author=P.+Mina-Osorioauthor=E.+Ortega&title=Aminopeptidase+N+%28CD13%29+functionally+interacts+with+FcgammaRs+in+human+monocytes&doi=10.1189%2Fjlb.1204714"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Aminopeptidase N (CD13) functionally interacts with FcγRs in human monocytes</span></div><div class="casAuthors">Mina-Osorio, Paola; Ortega, Enrique</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Leukocyte Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1008-1017</span>CODEN:
                <span class="NLM_cas:coden">JLBIE7</span>;
        ISSN:<span class="NLM_cas:issn">0741-5400</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Aminopeptidase N (E.C. 3.4.11.2) is a membrane-bound metalloproteinase expressed in many tissues.  Although its cytoplasmic portion has only eight amino acids, crosslinking of CD13 by monoclonal antibodies (mAb) has been shown to trigger intracellular signaling.  A functional assocn. between CD13 and receptors for IgG (FcγRs) has been proposed.  In this work, we evaluated possible functional interactions between CD13 and FcγRs in human peripheral blood monocytes and in U-937 promonocytic cells.  Our results show that during FcγR-mediated phagocytosis, CD13 redistributes to the phagocytic cup and is internalized into the phagosomes.  Moreover, modified erythrocytes that interact with the monocytic cell membrane through FcγRI and CD13 are ingested simultaneously, more efficiently than those that interact through the FcγRI only.  Also, co-crosslinking of CD13 with FcγRI by specific mAbs increases the level and duration of Syk phosphorylation induced by FcγRI crosslinking.  Finally, FcγRI and CD13 colocalize in zones of cellular polarization and coredistribute after aggregation of either of them.  These results demonstrate that CD13 and FcγRI can functionally interact on the monocytic cell membrane and suggest that CD13 may act as a signal regulator of FcγR function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8zhizZl_ECbVg90H21EOLACvtfcHk0li9eI2l0k0pRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltValurc%253D&md5=8fac3c179e79f9a914cc6058b57d6482</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1189%2Fjlb.1204714&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1189%252Fjlb.1204714%26sid%3Dliteratum%253Aachs%26aulast%3DMina-Osorio%26aufirst%3DP.%26aulast%3DOrtega%26aufirst%3DE.%26atitle%3DAminopeptidase%2520N%2520%2528CD13%2529%2520functionally%2520interacts%2520with%2520FcgammaRs%2520in%2520human%2520monocytes%26jtitle%3DJ.%2520Leukocyte%2520Biol.%26date%3D2005%26volume%3D77%26spage%3D1008%26epage%3D1017%26doi%3D10.1189%2Fjlb.1204714" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pasqualini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koivunen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kain, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lahdenranta, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stryhn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashmun, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arap, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruoslahti, E.</span></span> <span> </span><span class="NLM_article-title">Aminopeptidase N is a receptor for tumor-homing peptides and a target for inhibiting angiogenesis</span>. <i>Cancer. Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">722</span>– <span class="NLM_lpage">727</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10676659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD3cXht1Gmtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2000&pages=722-727&author=R.+Pasqualiniauthor=E.+Koivunenauthor=R.+Kainauthor=J.+Lahdenrantaauthor=M.+Sakamotoauthor=A.+Stryhnauthor=R.+A.+Ashmunauthor=L.+H.+Shapiroauthor=W.+Arapauthor=E.+Ruoslahti&title=Aminopeptidase+N+is+a+receptor+for+tumor-homing+peptides+and+a+target+for+inhibiting+angiogenesis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Aminopeptidase N is a receptor for tumor-homing peptides and a target for inhibiting angiogenesis</span></div><div class="casAuthors">Pasqualini, Renata; Koivunen, Erkki; Kain, Renate; Lahdenranta, Johanna; Sakamoto, Michiie; Stryhn, Anette; Ashmun, Richard A.; Shapiro, Linda H.; Arap, Wadih; Ruoslahti, Erkki</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">722-727</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">AACR Subscription Office</span>)
        </div><div class="casAbstract">Phage that display a surface peptide with the NGR sequence motif home selectively to tumor vasculature in vivo.  A drug coupled to an NGR peptide has more potent antitumor effects than the free drug.  The authors show here that the receptor for the NGR peptides in tumor vasculature is aminopeptidase N (APN; also called CD13).  NGR phage specifically bound to immunocaptured APN and to cells engineered to express APN on their surface.  Antibodies against APN inhibited in vivo tumor homing by the NGR phage.  Immunohistochem. staining showed that APN expression is up-regulated in endothelial cells within mouse and human tumors.  In another tissue that undergoes angiogenesis, corpus luteum, blood vessels also expressed APN, but APN was not detected in blood vessels of various other normal tissues stained under the same conditions.  APN antagonists specifically inhibited angiogenesis in chorioallantoic membranes and in the retina and suppressed tumor growth.  Thus, APN is involved in angiogenesis and can serve as a target for delivering drugs into tumors and for inhibiting angiogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobSSRsFaZLubVg90H21EOLACvtfcHk0li9eI2l0k0pRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXht1Gmtbc%253D&md5=b5731d5e35221aeef1a2191e040486d1</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPasqualini%26aufirst%3DR.%26aulast%3DKoivunen%26aufirst%3DE.%26aulast%3DKain%26aufirst%3DR.%26aulast%3DLahdenranta%26aufirst%3DJ.%26aulast%3DSakamoto%26aufirst%3DM.%26aulast%3DStryhn%26aufirst%3DA.%26aulast%3DAshmun%26aufirst%3DR.%2BA.%26aulast%3DShapiro%26aufirst%3DL.%2BH.%26aulast%3DArap%26aufirst%3DW.%26aulast%3DRuoslahti%26aufirst%3DE.%26atitle%3DAminopeptidase%2520N%2520is%2520a%2520receptor%2520for%2520tumor-homing%2520peptides%2520and%2520a%2520target%2520for%2520inhibiting%2520angiogenesis%26jtitle%3DCancer.%2520Res.%26date%3D2000%26volume%3D60%26spage%3D722%26epage%3D727" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mina-Osorio, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortega, E.</span></span> <span> </span><span class="NLM_article-title">CD13 in cell adhesion: aminopeptidase N (CD13) mediates homotypic aggregation of monocytic cells</span>. <i>J. Leukocyte Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">719</span>– <span class="NLM_lpage">730</span>, <span class="refDoi"> DOI: 10.1189/jlb.0705425</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1189%2Fjlb.0705425" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=16415167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD28XjsFCmurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2006&pages=719-730&author=P.+Mina-Osorioauthor=L.+H.+Shapiroauthor=E.+Ortega&title=CD13+in+cell+adhesion%3A+aminopeptidase+N+%28CD13%29+mediates+homotypic+aggregation+of+monocytic+cells&doi=10.1189%2Fjlb.0705425"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">CD13 in cell adhesion: aminopeptidase N (CD13) mediates homotypic aggregation of monocytic cells</span></div><div class="casAuthors">Mina-Osorio, Paola; Shapiro, Linda H.; Ortega, Enrique</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Leukocyte Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">719-730</span>CODEN:
                <span class="NLM_cas:coden">JLBIE7</span>;
        ISSN:<span class="NLM_cas:issn">0741-5400</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Homotypic aggregation (HA) of cells plays key roles in physiol. and pathol. processes, such as embryogenesis, immune responses, angiogenesis, tumor cell invasion, and metastasis.  Aminopeptidase N (CD13) has been implicated in most of these phenomena, although its participation has been attributed to its enzymic activity, while its role as an adhesion mol. has been almost unexplored.  Here, we show that certain anti-CD13 monoclonal antibodies induce HA of monocytic U-937 cells, independently of their effect on enzymic activity.  The phenomenon is related to binding to a specific site on the CD13 mol. and is independent of integrins.  It is abrogated by low temp., by the glycolysis inhibitor 2-deoxyglucose, and by inhibitors of tyrosine and mitogen-activated protein kinases.  The inhibitor of microtubule polymn. colchicine has a synergistic effect on CD13-mediated aggregation, suggesting an inhibitory role of microtubules in this process.  Finally, during HA, CD13 actively redistributes to the zones of cell-cell contact, as detd. by live cell imaging studies, demonstrating a direct role of CD13 in the adhesion phenomenon.  Together, these data show for the first time the participation of CD13 in monocytic cell adhesion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrvQUEMsZK9rVg90H21EOLACvtfcHk0li9eI2l0k0pRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjsFCmurY%253D&md5=adf38b17294b6cf1eca0b7de7e81e97e</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1189%2Fjlb.0705425&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1189%252Fjlb.0705425%26sid%3Dliteratum%253Aachs%26aulast%3DMina-Osorio%26aufirst%3DP.%26aulast%3DShapiro%26aufirst%3DL.%2BH.%26aulast%3DOrtega%26aufirst%3DE.%26atitle%3DCD13%2520in%2520cell%2520adhesion%253A%2520aminopeptidase%2520N%2520%2528CD13%2529%2520mediates%2520homotypic%2520aggregation%2520of%2520monocytic%2520cells%26jtitle%3DJ.%2520Leukocyte%2520Biol.%26date%3D2006%26volume%3D79%26spage%3D719%26epage%3D730%26doi%3D10.1189%2Fjlb.0705425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span> <span> </span><span class="NLM_article-title">Structural basis for multifunctional roles of mammalian aminopeptidase N</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">17966</span>– <span class="NLM_lpage">17971</span>, <span class="refDoi"> DOI: 10.1073/pnas.1210123109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1073%2Fpnas.1210123109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=23071329" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=17966-17971&author=L.+Chenauthor=Y.+L.+Linauthor=G.+Pengauthor=F.+Li&title=Structural+basis+for+multifunctional+roles+of+mammalian+aminopeptidase+N&doi=10.1073%2Fpnas.1210123109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for multifunctional roles of mammalian aminopeptidase N</span></div><div class="casAuthors">Chen, Lang; Lin, Yi-Lun; Peng, Guiqing; Li, Fang</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">17966-17971, S17966/1-S17966/6</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Mammalian aminopeptidase N (APN) plays multifunctional roles in many physiol. processes, including peptide metab., cell motility and adhesion, and coronavirus entry.  Here we detd. crystal structures of porcine APN at 1.85 Å resoln. and its complexes with a peptide substrate and a variety of inhibitors.  APN is a cell surface-anchored and seahorse-shaped zinc-aminopeptidase that forms head-to-head dimers.  Captured in a catalytically active state, these structures of APN illustrate a detailed catalytic mechanism for its aminopeptidase activity.  The active site and peptide-binding channel of APN reside in cavities with wide openings, allowing easy access to peptides.  The cavities can potentially open up further to bind the exposed N terminus of proteins.  The active site anchors the N-terminal neutral residue of peptides/proteins, and the peptide-binding channel binds the remainder of the peptides/proteins in a sequence-independent fashion.  APN also provides an exposed outer surface for coronavirus binding, without its physiol. functions being affected.  These structural features enable APN to function ubiquitously in peptide metab., interact with other proteins to mediate cell motility and adhesion, and serve as a coronavirus receptor.  This study elucidates multifunctional roles of APN and can guide therapeutic efforts to treat APN-related diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7DJL9z8NePrVg90H21EOLACvtfcHk0ljKVk3SzmCAww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvE&md5=630e3553442e5c38718e68488f37aee8</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1210123109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1210123109%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DY.%2BL.%26aulast%3DPeng%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DF.%26atitle%3DStructural%2520basis%2520for%2520multifunctional%2520roles%2520of%2520mammalian%2520aminopeptidase%2520N%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D17966%26epage%3D17971%26doi%3D10.1073%2Fpnas.1210123109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bogyo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salvesen, G. S.</span></span> <span> </span><span class="NLM_article-title">Aminopeptidase fingerprints, an integrated approach for identification of good substrates and optimal inhibitors</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">3310</span>– <span class="NLM_lpage">3318</span>, <span class="refDoi"> DOI: 10.1074/jbc.M109.060418</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1074%2Fjbc.M109.060418" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=19948737" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVektw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=3310-3318&author=M.+Dragauthor=M.+Bogyoauthor=J.+A.+Ellmanauthor=G.+S.+Salvesen&title=Aminopeptidase+fingerprints%2C+an+integrated+approach+for+identification+of+good+substrates+and+optimal+inhibitors&doi=10.1074%2Fjbc.M109.060418"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Aminopeptidase Fingerprints, an Integrated Approach for Identification of Good Substrates and Optimal Inhibitors</span></div><div class="casAuthors">Drag, Marcin; Bogyo, Matthew; Ellman, Jonathan A.; Salvesen, Guy S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">3310-3318</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Aminopeptidases process the N-terminal amino acids of target substrates by sequential cleavage of one residue at a time.  They are found in all cell compartments of prokaryotes and eukaryotes, being implicated in the major proteolytic events of cell survival, defense, growth, and development.  We present a new approach for the fast and reliable evaluation of the substrate specificity of individual aminopeptidases.  Using solid phase chem. with the 7-amino-4-carbamoylmethylcoumarin fluorophore, we have synthesized a library of 61 individual natural and unnatural amino acid substrates, chosen to cover a broad spectrum of the possible interactions in the S1 pocket of this type of protease.  As proof of concept, we detd. the substrate specificity of human, pig, and rat orthologs of aminopeptidase N (CD13), a highly conserved cell surface protease that inactivates enkephalins and other bioactive peptides.  Our data reveal a large and hydrophobic character for the S1 pocket of aminopeptidase N that is conserved with aminopeptidase Ns.  Our approach, which can be applied in principle to all aminopeptidases, yields useful information for the design of specific inhibitors, and more importantly, reveals a relation between the kinetics of substrate hydrolysis and the kinetics of enzyme inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT41yqFNTF3bVg90H21EOLACvtfcHk0ljKVk3SzmCAww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVektw%253D%253D&md5=eb4f0c442d396c5f290dd6aabdb64898</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.060418&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.060418%26sid%3Dliteratum%253Aachs%26aulast%3DDrag%26aufirst%3DM.%26aulast%3DBogyo%26aufirst%3DM.%26aulast%3DEllman%26aufirst%3DJ.%2BA.%26aulast%3DSalvesen%26aufirst%3DG.%2BS.%26atitle%3DAminopeptidase%2520fingerprints%252C%2520an%2520integrated%2520approach%2520for%2520identification%2520of%2520good%2520substrates%2520and%2520optimal%2520inhibitors%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D3310%26epage%3D3318%26doi%3D10.1074%2Fjbc.M109.060418" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGowan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowther, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stack, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golding, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skinner-Adams, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trenholme, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teuscher, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donnelly, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mucha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kafarski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degori, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckle, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardiner, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whisstock, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalton, J. P.</span></span> <span> </span><span class="NLM_article-title">Structural basis for the inhibition of the essential <i>Plasmodium falciparum</i> M1 neutral aminopeptidase</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">2537</span>– <span class="NLM_lpage">2542</span>, <span class="refDoi"> DOI: 10.1073/pnas.0807398106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1073%2Fpnas.0807398106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=19196988" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD1MXislahsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=2537-2542&author=S.+McGowanauthor=C.+J.+Porterauthor=J.+Lowtherauthor=C.+M.+Stackauthor=S.+J.+Goldingauthor=T.+S.+Skinner-Adamsauthor=K.+R.+Trenholmeauthor=F.+Teuscherauthor=S.+M.+Donnellyauthor=J.+Grembeckaauthor=A.+Muchaauthor=P.+Kafarskiauthor=R.+Degoriauthor=A.+M.+Buckleauthor=D.+L.+Gardinerauthor=J.+C.+Whisstockauthor=J.+P.+Dalton&title=Structural+basis+for+the+inhibition+of+the+essential+Plasmodium+falciparum+M1+neutral+aminopeptidase&doi=10.1073%2Fpnas.0807398106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for the inhibition of the essential Plasmodium falciparum M1 neutral aminopeptidase</span></div><div class="casAuthors">McGowan, Sheena; Porter, Corrine J.; Lowther, Jonathan; Stack, Colin M.; Golding, Sarah J.; Skinner-Adams, Tina S.; Trenholme, Katharine R.; Teuscher, Franka; Donnelly, Sheila M.; Grembecka, Jolanta; Mucha, Artur; Kafarski, Pawel; DeGori, Ross; Buckle, Ashley M.; Gardiner, Donald L.; Whisstock, James C.; Dalton, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2537-2542</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Plasmodium falciparum parasites are responsible for the major global disease malaria, which results in >2 million deaths each year.  With the rise of drug-resistant malarial parasites, novel drug targets and lead compds. are urgently required for the development of new therapeutic strategies.  Here, we address this important problem by targeting the malarial neutral aminopeptidases that are involved in the terminal stages of Hb digestion and essential for the provision of amino acids used for parasite growth and development within the erythrocyte.  We characterize the structure and substrate specificity of one such aminopeptidase, PfA-M1, a validated drug target.  The X-ray crystal structure of PfA-M1 alone and in complex with the generic inhibitor, bestatin, and a phosphinate dipeptide analog with potent in vitro and in vivo antimalarial activity, hPheP[CH2]Phe, reveals features within the protease active site that are crit. to its function as an aminopeptidase and can be exploited for drug development.  These results set the groundwork for the development of antimalarial therapeutics that target the neutral aminopeptidases of the parasite.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-6_ZNvWMUP7Vg90H21EOLACvtfcHk0ljKVk3SzmCAww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXislahsrY%253D&md5=23bc5efa86832c3fbc10e900f42f3216</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0807398106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0807398106%26sid%3Dliteratum%253Aachs%26aulast%3DMcGowan%26aufirst%3DS.%26aulast%3DPorter%26aufirst%3DC.%2BJ.%26aulast%3DLowther%26aufirst%3DJ.%26aulast%3DStack%26aufirst%3DC.%2BM.%26aulast%3DGolding%26aufirst%3DS.%2BJ.%26aulast%3DSkinner-Adams%26aufirst%3DT.%2BS.%26aulast%3DTrenholme%26aufirst%3DK.%2BR.%26aulast%3DTeuscher%26aufirst%3DF.%26aulast%3DDonnelly%26aufirst%3DS.%2BM.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DMucha%26aufirst%3DA.%26aulast%3DKafarski%26aufirst%3DP.%26aulast%3DDegori%26aufirst%3DR.%26aulast%3DBuckle%26aufirst%3DA.%2BM.%26aulast%3DGardiner%26aufirst%3DD.%2BL.%26aulast%3DWhisstock%26aufirst%3DJ.%2BC.%26aulast%3DDalton%26aufirst%3DJ.%2BP.%26atitle%3DStructural%2520basis%2520for%2520the%2520inhibition%2520of%2520the%2520essential%2520Plasmodium%2520falciparum%2520M1%2520neutral%2520aminopeptidase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2009%26volume%3D106%26spage%3D2537%26epage%3D2542%26doi%3D10.1073%2Fpnas.0807398106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hitzerd, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verbrugge, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ossenkoppele, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jansen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, G. J.</span></span> <span> </span><span class="NLM_article-title">Positioning of aminopeptidase inhibitors in next generation cancer therapy</span>. <i>Amino Acids</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">793</span>– <span class="NLM_lpage">808</span>, <span class="refDoi"> DOI: 10.1007/s00726-013-1648-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1007%2Fs00726-013-1648-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=24385243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVOltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2014&pages=793-808&author=S.+M.+Hitzerdauthor=S.+E.+Verbruggeauthor=G.+Ossenkoppeleauthor=G.+Jansenauthor=G.+J.+Peters&title=Positioning+of+aminopeptidase+inhibitors+in+next+generation+cancer+therapy&doi=10.1007%2Fs00726-013-1648-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Positioning of aminopeptidase inhibitors in next generation cancer therapy</span></div><div class="casAuthors">Hitzerd, Sarina M.; Verbrugge, Sue Ellen; Ossenkoppele, Gert; Jansen, Gerrit; Peters, Godefridus J.</div><div class="citationInfo"><span class="NLM_cas:title">Amino Acids</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">793-808</span>CODEN:
                <span class="NLM_cas:coden">AACIE6</span>;
        ISSN:<span class="NLM_cas:issn">0939-4451</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag GmbH</span>)
        </div><div class="casAbstract">A review.  Aminopeptidases represent a class of (zinc) metalloenzymes that catalyze the cleavage of amino acids nearby the N-terminus of polypeptides, resulting in hydrolysis of peptide bonds.  Aminopeptidases operate downstream of the ubiquitin-proteasome pathway and are implicated in the final step of intracellular protein degrdn. either by trimming proteasome-generated peptides for antigen presentation or full hydrolysis into free amino acids for recycling in renewed protein synthesis.  This review focuses on the function and subcellular location of five key aminopeptidases (aminopeptidase N, leucine aminopeptidase, puromycin-sensitive aminopeptidase, leukotriene A4 hydrolase and endoplasmic reticulum aminopeptidase 1/2) and their assocn. with different diseases, in particular cancer and their current position as target for therapeutic intervention by aminopeptidase inhibitors.  Historically, bestatin was the first prototypical aminopeptidase inhibitor that entered the clinic 35 years ago and is still used for the treatment of lung cancer.  More recently, new generation aminopeptidase inhibitors became available, including the aminopeptidase inhibitor prodrug tosedostat, which is currently tested in phase II clin. trials for acute myeloid leukemia.  Beyond bestatin and tosedostat, medicinal chem. has emerged with addnl. series of potential aminopeptidases inhibitors which are still in an early phase of (pre)clin. investigations.  The expanded knowledge of the unique mechanism of action of aminopeptidases has revived interest in aminopeptidase inhibitors for drug combination regimens in anti-cancer treatment.  In this context, this review will discuss relevant features and mechanisms of action of aminopeptidases and will also elaborate on factors contributing to aminopeptidase inhibitor efficacy and/or loss of efficacy due to drug resistance-related phenomena.  Together, a growing body of data point to aminopeptidase inhibitors as attractive tools for combination chemotherapy, hence their implementation may be a step forward in a new era of personalized treatment of cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrABQrQu0W5GrVg90H21EOLACvtfcHk0lh3qBKFZT72Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVOltQ%253D%253D&md5=c17cc3a379f6a21b0126d672f3680dc6</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1007%2Fs00726-013-1648-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00726-013-1648-0%26sid%3Dliteratum%253Aachs%26aulast%3DHitzerd%26aufirst%3DS.%2BM.%26aulast%3DVerbrugge%26aufirst%3DS.%2BE.%26aulast%3DOssenkoppele%26aufirst%3DG.%26aulast%3DJansen%26aufirst%3DG.%26aulast%3DPeters%26aufirst%3DG.%2BJ.%26atitle%3DPositioning%2520of%2520aminopeptidase%2520inhibitors%2520in%2520next%2520generation%2520cancer%2520therapy%26jtitle%3DAmino%2520Acids%26date%3D2014%26volume%3D46%26spage%3D793%26epage%3D808%26doi%3D10.1007%2Fs00726-013-1648-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bauvois, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dauzonne, D.</span></span> <span> </span><span class="NLM_article-title">Aminopeptidase-N/CD13 (EC 3.4.11.2) inhibitors: chemistry, biological evaluations, and therapeutic prospects</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">88</span>– <span class="NLM_lpage">130</span>, <span class="refDoi"> DOI: 10.1002/med.20044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1002%2Fmed.20044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=16216010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD28Xmt1CitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=88-130&author=B.+Bauvoisauthor=D.+Dauzonne&title=Aminopeptidase-N%2FCD13+%28EC+3.4.11.2%29+inhibitors%3A+chemistry%2C+biological+evaluations%2C+and+therapeutic+prospects&doi=10.1002%2Fmed.20044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Aminopeptidase-N/CD13 (EC 3.4.11.2) inhibitors: chemistry, biological evaluations, and therapeutic prospects</span></div><div class="casAuthors">Bauvois, Brigitte; Dauzonne, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-130</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Aminopeptidase N (APN)/CD13 (EC 3.4.11.2) is a transmembrane protease present in a wide variety of human tissues and cell types (endothelial, epithelial, fibroblast, leukocyte).  APN/CD13 expression is dysregulated in inflammatory diseases and in cancers (solid and hematol. tumors).  APN/CD13 serves as a receptor for coronaviruses.  Natural and synthetic inhibitors of APN activity have been characterized.  These inhibitors have revealed that APN is able to modulate bioactive peptide responses (pain management, vasopressin release) and to influence immune functions and major biol. events (cell proliferation, secretion, invasion, angiogenesis).  Therefore, inhibition of APN/CD13 may lead to the development of anti-cancer and anti-inflammatory drugs.  This review provides an update on the biol. and pharmacol. profiles of known natural and synthetic APN inhibitors.  Current status on their potential use as therapeutic agents is discussed with regard to toxicity and specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7XAstbaJj6bVg90H21EOLACvtfcHk0lh3qBKFZT72Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xmt1CitQ%253D%253D&md5=b1377159539c0e5cdaeb5e5d736554e0</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fmed.20044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.20044%26sid%3Dliteratum%253Aachs%26aulast%3DBauvois%26aufirst%3DB.%26aulast%3DDauzonne%26aufirst%3DD.%26atitle%3DAminopeptidase-N%252FCD13%2520%2528EC%25203.4.11.2%2529%2520inhibitors%253A%2520chemistry%252C%2520biological%2520evaluations%252C%2520and%2520therapeutic%2520prospects%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2006%26volume%3D26%26spage%3D88%26epage%3D130%26doi%3D10.1002%2Fmed.20044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wickström, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nygren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gullbo, J.</span></span> <span> </span><span class="NLM_article-title">Aminopeptidase N (CD13) as a target for cancer chemotherapy</span>. <i>Cancer Sci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">501</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1111/j.1349-7006.2010.01826.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1111%2Fj.1349-7006.2010.01826.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=21205077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A280%3ADC%252BC3M3gtFKkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2011&pages=501-508&author=M.+Wickstr%C3%B6mauthor=R.+Larssonauthor=P.+Nygrenauthor=J.+Gullbo&title=Aminopeptidase+N+%28CD13%29+as+a+target+for+cancer+chemotherapy&doi=10.1111%2Fj.1349-7006.2010.01826.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Aminopeptidase N (CD13) as a target for cancer chemotherapy</span></div><div class="casAuthors">Wickstrom Malin; Larsson Rolf; Nygren Peter; Gullbo Joachim</div><div class="citationInfo"><span class="NLM_cas:title">Cancer science</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">501-8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The enzyme aminopeptidase N (APN, also known as CD13) is a Zn(2+) dependent membrane-bound ectopeptidase that degrades preferentially proteins and peptides with a N-terminal neutral amino acid.  Aminopeptidase N has been associated with the growth of different human cancers and suggested as a suitable target for anti-cancerous therapy.  Different approaches have been used to develop new drugs directed to this target, including enzyme inhibitors as well as APN-targeted carrier constructs.  This review discusses the prevalence and possible function of APN in malignant diseases, mainly solid tumors, as well as its "drugability" evaluated in preclinical in vivo models, and also provides a brief overview of current clinical trials focused on APN.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR-AFmpp47W5PEeosoOnpOLfW6udTcc2eb611lSlyg_nLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3M3gtFKkuw%253D%253D&md5=907f8f3efe78ae795d81408d68986b44</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1111%2Fj.1349-7006.2010.01826.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1349-7006.2010.01826.x%26sid%3Dliteratum%253Aachs%26aulast%3DWickstr%25C3%25B6m%26aufirst%3DM.%26aulast%3DLarsson%26aufirst%3DR.%26aulast%3DNygren%26aufirst%3DP.%26aulast%3DGullbo%26aufirst%3DJ.%26atitle%3DAminopeptidase%2520N%2520%2528CD13%2529%2520as%2520a%2520target%2520for%2520cancer%2520chemotherapy%26jtitle%3DCancer%2520Sci.%26date%3D2011%26volume%3D102%26spage%3D501%26epage%3D508%26doi%3D10.1111%2Fj.1349-7006.2010.01826.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amin, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, T.</span></span> <span> </span><span class="NLM_article-title">Design of aminopeptidase N inhibitors as anti-cancer agents</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6468</span>– <span class="NLM_lpage">6490</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00782</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00782" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A280%3ADC%252BC1Mjgt1Wjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6468-6490&author=S.+A.+Aminauthor=N.+Adhikariauthor=T.+Jha&title=Design+of+aminopeptidase+N+inhibitors+as+anti-cancer+agents&doi=10.1021%2Facs.jmedchem.7b00782"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Design of Aminopeptidase N Inhibitors as Anti-cancer Agents</span></div><div class="casAuthors">Amin Sk Abdul; Adhikari Nilanjan; Jha Tarun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6468-6490</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Aminopeptidase N (APN) is an important metalloenzyme.  It regulates multivariate cellular functions by different mechanisms such as enzymatic cleavage of peptides.  This may play a role in endocytosis and regulate signal transduction.  APN, a member of the M1 zinc metallopeptidase family, plays crucial roles in a variety of functions such as migration and invasion, and angiogenesis and metastasis of tumor cells.  Therefore, APN inhibitors may be useful for the treatment of cancer.  In this Perspective, structure-activity relationships of APN inhibitors are discussed to get an idea of possible lead candidates.  APN inhibitors should possess an aryl hydrophobic function along with a zinc binding group attached to the hydrophobic group(s) to achieve high potency.  This and other design aspects of APN inhibitors are discussed in this Perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRteeQg7iu8x6qcneEtCi92fW6udTcc2eYdB-EPI3h_5rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mjgt1Wjug%253D%253D&md5=9b277fdc4171daeee7560bf0154046c9</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00782%26sid%3Dliteratum%253Aachs%26aulast%3DAmin%26aufirst%3DS.%2BA.%26aulast%3DAdhikari%26aufirst%3DN.%26aulast%3DJha%26aufirst%3DT.%26atitle%3DDesign%2520of%2520aminopeptidase%2520N%2520inhibitors%2520as%2520anti-cancer%2520agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6468%26epage%3D6490%26doi%3D10.1021%2Facs.jmedchem.7b00782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ashmun, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Look, A. T.</span></span> <span> </span><span class="NLM_article-title">Metalloprotease activity of CD13/aminopeptidase N on the surface of human myeloid cells</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">469</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1967220" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaK3cXpsF2luw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=1990&pages=462-469&author=R.+A.+Ashmunauthor=A.+T.+Look&title=Metalloprotease+activity+of+CD13%2Faminopeptidase+N+on+the+surface+of+human+myeloid+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Metalloprotease activity of CD13/aminopeptidase N on the surface of human myeloid cells</span></div><div class="casAuthors">Ashmun, Richard A.; Look, A. Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-9</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    </div><div class="casAbstract">It was previously found that the myeloid cell surface glycoprotein CD13 (gp150) is identical to aminopeptidase N (EC 3.4.11.2), a widely distributed membrane-bound, Zn-dependent metallogprotease with an extracellular enzymic domain that cleaves N-terminal amino acid residues from oligopeptides.  As a 1st step toward defining the function of this mol. on myeloid cells, we the cell surface-assocd. N-terminal peptidase was assessed activity by sensitive spectrophotometric measurements of the cleavage of p-nitroanilide amino acid derivs.  Aminopeptidase activity detected on the surface of normal and malignant hematopoietic cells coincided with the level of cell surface CD13 expression as measured by flow cytometry.  The enzyme was specifically inhibited by the Zn-binding metalloprotease inhibitors, bestatin, 1,10-phenanthroline, or 2.2'-dipyridyl, but was not affected by several inhibitors of other classes of proteases.  Aminopeptidase activity was demonstrated for CD13 mols. specifically immunopptd. from the surface of CD13-pos. cells and was blocked by the metalloprotease inhibitor 1,10-phenanthroline.  Moreover, cell surface aminopeptidase activity was partially inhibited when viable cells were incubated with 2 of a panel of 11 monoclonal antibodies (MoAbs) known to be specific for extracellular epitopes of human CD13.  This inhibition was apparent in the absence of detectable downmodulation of CD13 mols. from the cell surface, suggesting that these MoAbs either phys. interfere with substrate binding or alter the Zn-coordinating properties of aminopeptidase N mols.  Aminopeptidase N could play an important role in modulating signals generated by peptides at the surface of myeloid cells, either by removing key N-terminal residues from active peptides or by converting inactive peptides to active forms.  The inhibitory antibodies used in this study should prove useful in delineating the physiol. roles of CD13/aminopeptidase N on normal and malignant myeloid cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlZp8__KZ8-7Vg90H21EOLACvtfcHk0ljiymCpo6bNSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXpsF2luw%253D%253D&md5=932fb4f057da474e269fc2955bd106de</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAshmun%26aufirst%3DR.%2BA.%26aulast%3DLook%26aufirst%3DA.%2BT.%26atitle%3DMetalloprotease%2520activity%2520of%2520CD13%252Faminopeptidase%2520N%2520on%2520the%2520surface%2520of%2520human%2520myeloid%2520cells%26jtitle%3DBlood%26date%3D1990%26volume%3D75%26spage%3D462%26epage%3D469" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hashida, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takabayashi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adachi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kondo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohno, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaoka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyake, M.</span></span> <span> </span><span class="NLM_article-title">Aminopeptidase N is involved in cell motility and angiogenesis: its clinical significance in human colon cancer</span>. <i>Gastroenterology</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>122</i></span>,  <span class="NLM_fpage">376</span>– <span class="NLM_lpage">386</span>, <span class="refDoi"> DOI: 10.1053/gast.2002.31095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1053%2Fgast.2002.31095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=11832452" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD38XhsVGns7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2002&pages=376-386&author=H.+Hashidaauthor=A.+Takabayashiauthor=M.+Kanaiauthor=M.+Adachiauthor=K.+Kondoauthor=N.+Kohnoauthor=Y.+Yamaokaauthor=M.+Miyake&title=Aminopeptidase+N+is+involved+in+cell+motility+and+angiogenesis%3A+its+clinical+significance+in+human+colon+cancer&doi=10.1053%2Fgast.2002.31095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Aminopeptidase N is involved in cell motility and angiogenesis: its clinical significance in human colon cancer</span></div><div class="casAuthors">Hashida, Hiroki; Takabayashi, Arimichi; Kanai, Michiyuki; Adachi, Masashi; Kondo, Keiichi; Kohno, Nobuoki; Yamaoka, Yoshio; Miyake, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Gastroenterology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">376-386</span>CODEN:
                <span class="NLM_cas:coden">GASTAB</span>;
        ISSN:<span class="NLM_cas:issn">0016-5085</span>.
    
            (<span class="NLM_cas:orgname">W. B. Saunders Co.</span>)
        </div><div class="casAbstract">Background & Aims: The mol. basis of cell motility is highly complex and is controlled by a no. of mol. systems, whereas angiogenesis is an important biol. component of tumor progression.  The aims of this study were to investigate the possible involvement of proteins at the cell surface in controlling cell motility and angiogenesis, and to identify the cell surface mols. involved in gastrointestinal tumors.  Methods: We addressed these issues using functional monoclonal antibodies, which inhibit cell motility, endothelial cell migration, and tube formation.  Furthermore, we investigated the relationship between this antigen and colon cancer, and showed the prognostic significance in human colon cancer.  Results: We established a murine monoclonal antibody MH8-11, which inhibits cell motility and in vitro angiogenesis.  This epitope was a 165-kilodalton protein, and the sequencing anal. revealed that it was almost identical to aminopeptidase N (APN)/cluster of differentiation (CD) 13.  APN/CD13 expression was assocd. with tumor status (P = 0.025).  The disease-free and overall survival rate for patients with pos. APN/CD13 expression tumors was significantly lower than that for patients with neg. APN/CD13 expression tumors (P = 0.014, 0.033, resp.).  Among 47 node-pos. patients, the survival rate of patients with neg. APN/CD13 expression was better than that of those with pos. APN/CD13 expression.  Conclusions: Our data suggest that APN/CD13 is involved in cell motility and angiogenesis, and APN/CD13 expression may be a useful indicator of a poor prognosis for node-pos. patients with colon cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhHjwhNpLFZrVg90H21EOLACvtfcHk0ljiymCpo6bNSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhsVGns7s%253D&md5=0ee1dfdc5e46097c8193b604d93d11e0</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1053%2Fgast.2002.31095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1053%252Fgast.2002.31095%26sid%3Dliteratum%253Aachs%26aulast%3DHashida%26aufirst%3DH.%26aulast%3DTakabayashi%26aufirst%3DA.%26aulast%3DKanai%26aufirst%3DM.%26aulast%3DAdachi%26aufirst%3DM.%26aulast%3DKondo%26aufirst%3DK.%26aulast%3DKohno%26aufirst%3DN.%26aulast%3DYamaoka%26aufirst%3DY.%26aulast%3DMiyake%26aufirst%3DM.%26atitle%3DAminopeptidase%2520N%2520is%2520involved%2520in%2520cell%2520motility%2520and%2520angiogenesis%253A%2520its%2520clinical%2520significance%2520in%2520human%2520colon%2520cancer%26jtitle%3DGastroenterology%26date%3D2002%26volume%3D122%26spage%3D376%26epage%3D386%26doi%3D10.1053%2Fgast.2002.31095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitaichi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komoto, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaganoi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyake, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamasaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kondo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imamura, M.</span></span> <span> </span><span class="NLM_article-title">Clinicopathological significance of <tep-common:author-query>AQ7: Please provide a DOI number for ref 23 or indicate if one doesn&amp;#x2019;t exist.</tep-common:author-query>aminopeptidase N/CD13 expression in human gastric carcinoma</span>. <i>Hepatogastroenterology</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.1016/j.cellbi.2008.01.130</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.cellbi.2008.01.130" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=17419227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A280%3ADC%252BD2s3hsFWltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2007&pages=36-40&author=J.+Kawamuraauthor=Y.+Shimadaauthor=H.+Kitaichiauthor=I.+Komotoauthor=Y.+Hashimotoauthor=J.+Kaganoiauthor=M.+Miyakeauthor=S.+Yamasakiauthor=K.+Kondoauthor=M.+Imamura&title=Clinicopathological+significance+of+%3Ctep-common%3Aauthor-query%3EAQ7%3A+Please+provide+a+DOI+number+for+ref+23+or+indicate+if+one+doesn%26amp%3B%23x2019%3Bt+exist.%3C%2Ftep-common%3Aauthor-query%3Eaminopeptidase+N%2FCD13+expression+in+human+gastric+carcinoma&doi=10.1016%2Fj.cellbi.2008.01.130"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Clinicopathological significance of aminopeptidase N/CD13 expression in human gastric carcinoma</span></div><div class="casAuthors">Kawamura Junichiro; Shimada Yutaka; Kitaichi Hidenori; Komoto Izumi; Hashimoto Yosuhe; Kaganoi Junichi; Miyake Masayuki; Yamasaki Seiji; Kondo Kan; Imamura Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Hepato-gastroenterology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">73</span>),
    <span class="NLM_cas:pages">36-40</span>
        ISSN:<span class="NLM_cas:issn">0172-6390</span>.
    </div><div class="casAbstract">BACKGROUND/AIMS:  Aminopeptidase N (APN)/CD13 is a transmembrane ectoenzyme occurring in a wide variety of cells.  Recently, APN/CD13 has been reported to be involved in tumor invasion and metastasis.  However, precise functions in tumor cells, and its role in gastric carcinoma remain unclear.  METHODOLOGY:  To evaluate the role of APN/CD13 in gastric carcinoma, we conducted immunohistochemical staining for APN/CD13 in 121 gastric carcinoma specimens, using anti-APN monoclonal antibody.  The relationship between APN/CD13 expression and various prognostic factors of gastric carcinoma were investigated.  RESULTS:  Of the 121 patients with gastric carcinoma studied, 48 were strongly positive, 36 were weakly positive, and 37 were negative.  Overall survival rate of the patients with negative APN/CD13 expression was significantly lower than that of the patients with positive APN/CD13 expression.  APN/CD13 expression was negatively associated with lymph node metastasis.  Multivariate analysis showed APN/CD13 expression to be a significant prognostic factor.  CONCLUSIONS:  Decreased expression of APN/CD13 was associated with a poor prognosis.  Hence, our results demonstrate that the immunohistochemical detection of APN/CD13 could provide useful information as one of the prognostic factors in gastric cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR_WCHzzJ7alTjk2McQZ6XmfW6udTcc2eYdB-EPI3h_5rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s3hsFWltA%253D%253D&md5=0ca2651b5c05608b8d76991b421b5f04</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.cellbi.2008.01.130&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cellbi.2008.01.130%26sid%3Dliteratum%253Aachs%26aulast%3DKawamura%26aufirst%3DJ.%26aulast%3DShimada%26aufirst%3DY.%26aulast%3DKitaichi%26aufirst%3DH.%26aulast%3DKomoto%26aufirst%3DI.%26aulast%3DHashimoto%26aufirst%3DY.%26aulast%3DKaganoi%26aufirst%3DJ.%26aulast%3DMiyake%26aufirst%3DM.%26aulast%3DYamasaki%26aufirst%3DS.%26aulast%3DKondo%26aufirst%3DK.%26aulast%3DImamura%26aufirst%3DM.%26atitle%3DClinicopathological%2520significance%2520of%2520%253Ctep-common%253Aauthor-query%253EAQ7%253A%2520Please%2520provide%2520a%2520DOI%2520number%2520for%2520ref%252023%2520or%2520indicate%2520if%2520one%2520doesn%2526amp%253B%2523x2019%253Bt%2520exist.%253C%252Ftep-common%253Aauthor-query%253Eaminopeptidase%2520N%252FCD13%2520expression%2520in%2520human%2520gastric%2520carcinoma%26jtitle%3DHepatogastroenterology%26date%3D2007%26volume%3D54%26spage%3D36%26epage%3D40%26doi%3D10.1016%2Fj.cellbi.2008.01.130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span> <span> </span><span class="NLM_article-title">Possible contribution of aminopeptidase N (APN/CD13) to migration and invasion of human osteosarcoma cell lines</span>. <i>Int. J. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">2475</span>– <span class="NLM_lpage">2485</span>, <span class="refDoi"> DOI: 10.3892/ijo.2014.2664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.3892%2Fijo.2014.2664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=25340499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVamt7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2014&pages=2475-2485&author=W.+Liangauthor=B.+Gaoauthor=G.+Xuauthor=D.+Wengauthor=M.+Xieauthor=Y.+Qian&title=Possible+contribution+of+aminopeptidase+N+%28APN%2FCD13%29+to+migration+and+invasion+of+human+osteosarcoma+cell+lines&doi=10.3892%2Fijo.2014.2664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Possible contribution of aminopeptidase N (APN/CD13) to migration and invasion of human osteosarcoma cell lines</span></div><div class="casAuthors">Liang, Wenqing; Gao, Bo; Xu, Guojian; Weng, Dong; Xie, Minghua; Qian, Yu</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2475-2485</span>CODEN:
                <span class="NLM_cas:coden">IJONES</span>;
        ISSN:<span class="NLM_cas:issn">1019-6439</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Osteosarcoma is the most common primary malignancy of the bone.  Aminopeptidase N (APN/CD13), a Zn+2-dependent ectopeptidase localized on the cell surface, is widely considered to influence the invasion mechanism.  This study explores the potential involvement of APN in migration and invasion of human osteosarcoma cells in vitro using inhibitors and activators of APN.  Cells treated with APN inhibitor bestatin displayed decreased migration and invasion in a Boyden chamber Transwell assay.  Western blotting revealed reduced levels of mitogen-activated protein kinase (MAPK) and phosphatidylinositol 3-kinase (PI3K) pathway proteins, reduced phosphorylation of p38, ERK1/2 and JNK and decreased levels of NF-κB.  Bestatin treatment also lowered APN, matrix metalloproteinase (MMP)-2 and -9 enzymic activity and their mRNA expression.  Reduced MMP-2 and -9 protein levels were also obsd.  By comparison, cells treated with cytokine interleukin-6 (IL-6), a stimulator of APN, displayed increased migration and invasion.  Western blotting revealed increased levels of MAPK and PI3K pathway proteins, phosphorylated p38, ERK1/2 and JNK, and NF-κB.  IL-6 treatment also increased APN and MMP-2 and -9 enzymic activity.  An increase of APN, MMP-2 and -9 mRNA levels, and MMP-2 and -9 protein levels was also obsd.  Together these expts. reveal potential enzymic and signalling roles for APN in osteosarcoma and establish a starting point for an in-depth anal. of the role of APN in regulating invasiveness.  A deeper knowledge about the regulatory mechanisms of APN may contribute to the development of anti-metastatic therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprlLnunSv0_LVg90H21EOLACvtfcHk0li1APP9fRIsTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVamt7vJ&md5=50a91f5dc8f5c9c8bfd3bb1fad9f4a23</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.3892%2Fijo.2014.2664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijo.2014.2664%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DW.%26aulast%3DGao%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DWeng%26aufirst%3DD.%26aulast%3DXie%26aufirst%3DM.%26aulast%3DQian%26aufirst%3DY.%26atitle%3DPossible%2520contribution%2520of%2520aminopeptidase%2520N%2520%2528APN%252FCD13%2529%2520to%2520migration%2520and%2520invasion%2520of%2520human%2520osteosarcoma%2520cell%2520lines%26jtitle%3DInt.%2520J.%2520Oncol.%26date%3D2014%26volume%3D45%26spage%3D2475%26epage%3D2485%26doi%3D10.3892%2Fijo.2014.2664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nohara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujiwara, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakuragi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yanagihara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwanuma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kajiyama, Y.</span></span> <span> </span><span class="NLM_article-title">Aminopeptidase N (APN/CD13) as a target molecule for scirrhous gastric cancer</span>. <i>Clin. Res. Hepatol. Gastroenterol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">494</span>– <span class="NLM_lpage">503</span>, <span class="refDoi"> DOI: 10.1016/j.clinre.2015.11.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.clinre.2015.11.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=26774363" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVenurfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2016&pages=494-503&author=S.+Noharaauthor=K.+Katoauthor=D.+Fujiwaraauthor=N.+Sakuragiauthor=K.+Yanagiharaauthor=Y.+Iwanumaauthor=Y.+Kajiyama&title=Aminopeptidase+N+%28APN%2FCD13%29+as+a+target+molecule+for+scirrhous+gastric+cancer&doi=10.1016%2Fj.clinre.2015.11.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Aminopeptidase N (APN/CD13) as a target molecule for scirrhous gastric cancer</span></div><div class="casAuthors">Nohara, Shigeo; Kato, Kazunori; Fujiwara, Daisuke; Sakuragi, Naoya; Yanagihara, Kazuyoshi; Iwanuma, Yoshimi; Kajiyama, Yoshiaki</div><div class="citationInfo"><span class="NLM_cas:title">Clinics and Research in Hepatology and Gastroenterology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">494-503</span>CODEN:
                <span class="NLM_cas:coden">CRHGAQ</span>;
        ISSN:<span class="NLM_cas:issn">2210-7401</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Scirrhous gastric cancer is assocd. with peritoneal dissemination and advanced lymph node metastasis from an early stage, and the prognosis is still poor.  In this study, we aimed to analyze candidate mols. for targeted therapy of scirrhous gastric cancer.  We searched for mols./metabolic activity that might be predominantly expressed in a subpopulation of scirrhous gastric cancer cells and might function as cancer stem cell markers.For this purpose, we investigated the expression of various cell surface markers and of aldehyde dehydrogenase (ALDH) activity.  These analyses showed that the scirrhous gastric cancer cell lines HSC-58 and HSC-44PE heterogeneously expressed CD13, while CD44, CDCP1, EpCAM and ABCG2 were expressed uniformly.  Moreover, 10% of the total HSC-58 cell population expressed ALDH enzyme activity.  A subpopulation of cells strongly pos. for ALDH also expressed high levels of CD13, both of which are known as cancer stem cell markers.  HSC-58 cells expressing high levels of CD13 showed lower sensitivity to a cancer drug cisplatin than cells with low levels of CD13.  In contrast, CD13-high subpopulation of HSC-58 was more sensitive to an aminopeptidase N inhibitor bestatin.  In terms of antibody-drug therapy, anti-CD13-immunotoxin was highly cytotoxic towards HSC-58 cells and was more cytotoxic than anti-EpCAM-immunotoxin.These data suggest that CD13 is a suitable cell surface candidate for targeted antibody-drug therapy of scirrhous gastric cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolezJYMlPnvLVg90H21EOLACvtfcHk0li1APP9fRIsTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVenurfK&md5=bcb17ef6086f2fad9a3fa7b3996435b0</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.clinre.2015.11.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.clinre.2015.11.003%26sid%3Dliteratum%253Aachs%26aulast%3DNohara%26aufirst%3DS.%26aulast%3DKato%26aufirst%3DK.%26aulast%3DFujiwara%26aufirst%3DD.%26aulast%3DSakuragi%26aufirst%3DN.%26aulast%3DYanagihara%26aufirst%3DK.%26aulast%3DIwanuma%26aufirst%3DY.%26aulast%3DKajiyama%26aufirst%3DY.%26atitle%3DAminopeptidase%2520N%2520%2528APN%252FCD13%2529%2520as%2520a%2520target%2520molecule%2520for%2520scirrhous%2520gastric%2520cancer%26jtitle%3DClin.%2520Res.%2520Hepatol.%2520Gastroenterol.%26date%3D2016%26volume%3D40%26spage%3D494%26epage%3D503%26doi%3D10.1016%2Fj.clinre.2015.11.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brand, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stucke-Ring, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schliemann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrach, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorlich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillejan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller-Tidow, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardelmann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiewrodt, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berdel, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwoppe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, W.</span></span> <span> </span><span class="NLM_article-title">Potential therapeutic impact of CD13 expression in non-small cell lung cancer</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0177146</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0177146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1371%2Fjournal.pone.0177146" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=28604784" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1agsrzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=L.+H.+Schmidtauthor=C.+Brandauthor=J.+Stucke-Ringauthor=C.+Schliemannauthor=T.+Kesslerauthor=S.+Harrachauthor=M.+Mohrauthor=D.+Gorlichauthor=A.+Marraauthor=L.+Hillejanauthor=C.+Muller-Tidowauthor=G.+Lenzauthor=E.+Wardelmannauthor=R.+Wiewrodtauthor=W.+E.+Berdelauthor=C.+Schwoppeauthor=W.+Hartmann&title=Potential+therapeutic+impact+of+CD13+expression+in+non-small+cell+lung+cancer&doi=10.1371%2Fjournal.pone.0177146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Potential therapeutic impact of CD13 expression in non-small cell lung cancer</span></div><div class="casAuthors">Schmidt, Lars Henning; Brand, Caroline; Stucke-Ring, Janine; Schliemann, Christoph; Kessler, Torsten; Harrach, Saliha; Mohr, Michael; Goerlich, Dennis; Marra, Alessandro; Hillejan, Ludger; Mueller-Tidow, Carsten; Lenz, Georg; Wardelmann, Eva; Wiewrodt, Rainer; Berdel, Wolfgang E.; Schwoeppe, Christian; Hartmann, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e0177146/1-e0177146/16</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Background: Aminopeptidase N (CD13) is a zinc-binding protease that has functional effects on both cancerogenesis and tumor angiogenesis.  Since CD13 is an antigen suitable for mol. targeted therapies (e.g. tTF-NGR induced tumor vascular infarction), we evaluated its impact in NSCLC patients, and tested the effects of the CD13-targeted fusion protein tTF-NGR (truncated tissue factor (tTF) contg. the NGR motif: asparagine-glycine-arginine) in vivo in nude mice.  Methods: Expression of both CD13 and CD31 was studied in 270 NSCLC patients by immunohistochem.  Clin. correlations and prognostic effects of the expression profiles were analyzed using univariate and multivariate analyses.  In addn., a microarray-based anal. on the basis of the KM plotter database was performed.  The in vivo effects of the CD13-targeted fusion protein tTF-NGR on tumor growth were tested in CD1 nude mice carrying A549 lung carcinoma xenotransplants.  Results: CD13 expression in tumor endothelial and vessel assocd. stromal cells was found in 15% of the investigated samples, while expression in tumor cells was obsd. in 7%.  Although no significant prognostic impact was obsd. in the full NSCLC study cohort, both univariate and multivariate models identified vascular CD13 protein expression to correlate with poor overall survival in stage III and pN2+ NSCLC patients.  Microarray-based mRNA anal. for either adenocarcinomas or squamous cell carcinomas did not reveal any significant effect.  However, the anal. of CD13 mRNA expression for all lung cancer histologies demonstrated a pos. prognostic effect.  In vivo, systemic application of CD13-targeted tissue factor tTF-NGR significantly reduced CD13+ A549 tumor growth in nude mice.  Conclusions Our results contribute a data basis for prioritizing clin. testing of tTF-NGR and other antitumor mols. targeted by NGR-peptides in NSCLC.  Because: CD13 expression in NSCLC tissues was found only in a specific subset of NSCLC patients, rigorous pre-therapeutic testing will help to select patients for these studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLBFxsuIJ6lLVg90H21EOLACvtfcHk0li1APP9fRIsTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1agsrzP&md5=41ed1bce9f3f7ed37b54617f97242acd</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0177146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0177146%26sid%3Dliteratum%253Aachs%26aulast%3DSchmidt%26aufirst%3DL.%2BH.%26aulast%3DBrand%26aufirst%3DC.%26aulast%3DStucke-Ring%26aufirst%3DJ.%26aulast%3DSchliemann%26aufirst%3DC.%26aulast%3DKessler%26aufirst%3DT.%26aulast%3DHarrach%26aufirst%3DS.%26aulast%3DMohr%26aufirst%3DM.%26aulast%3DGorlich%26aufirst%3DD.%26aulast%3DMarra%26aufirst%3DA.%26aulast%3DHillejan%26aufirst%3DL.%26aulast%3DMuller-Tidow%26aufirst%3DC.%26aulast%3DLenz%26aufirst%3DG.%26aulast%3DWardelmann%26aufirst%3DE.%26aulast%3DWiewrodt%26aufirst%3DR.%26aulast%3DBerdel%26aufirst%3DW.%2BE.%26aulast%3DSchwoppe%26aufirst%3DC.%26aulast%3DHartmann%26aufirst%3DW.%26atitle%3DPotential%2520therapeutic%2520impact%2520of%2520CD13%2520expression%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0177146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tokuhara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishida, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Higashiyama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyake, M.</span></span> <span> </span><span class="NLM_article-title">Clinical significance of aminopeptidase N in non-small cell lung cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">3971</span>– <span class="NLM_lpage">3978</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-06-0338</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1158%2F1078-0432.CCR-06-0338" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=16818694" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsValtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2006&pages=3971-3978&author=T.+Tokuharaauthor=N.+Hattoriauthor=H.+Ishidaauthor=T.+Hiraiauthor=M.+Higashiyamaauthor=K.+Kodamaauthor=M.+Miyake&title=Clinical+significance+of+aminopeptidase+N+in+non-small+cell+lung+cancer&doi=10.1158%2F1078-0432.CCR-06-0338"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical Significance of Aminopeptidase N in Non-Small Cell Lung Cancer</span></div><div class="casAuthors">Tokuhara, Takahiro; Hattori, Noboru; Ishida, Hisao; Hirai, Tatsuya; Higashiyama, Masahiko; Kodama, Ken; Miyake, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3971-3978</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: The aim of our study is to investigate the mechanism of metastasis, to detect novel metastasis-assocd. mols., and to evaluate the mols. from the point of view of clin. application.  A monoclonal antibody MH8-11, which we established, recognizes a glycoprotein that is identical to aminopeptidase N (APN/CD13).  APN/CD13 degrades the extracellular matrix, while it is also involved in cell motility and improves angiogenesis.  Exptl. Design: We investigated the expression of APN/CD13 in 194 cases of non-small cell lung cancer (NSCLC) by immunohistochem. analyses and reverse transcription-PCR assay to det. the significance of this prognostic factor; 95 tumors were stage I, 36 were stage II, 39 were stage IIIA, and 24 were stage IIIB.  Moreover, we investigated that the relationship between the expression of APN/CD13 and angiogenesis and prognosis for patients with NSCLC.  Results: We found a correlation between the expression of APN/CD13 and angiogenesis (r = 0.659; P < 0.0001).  In the 194 patients with NSCLC, we found 68 patients to be APN/CD13+ and 126 patients to be APN/CD13-.  The 5-yr survival rate in patients with APN/CD13+ tumors was significantly lower than in those whose tumors had neg. APN/CD13 (48.3% vs. 67.1%; P = 0.0001).  Conclusion: Our data suggest the expression of APN/CD13 for patients with NSCLC to be assocd. with a poor prognosis and angiogenesis.  This is the first study to show the relationship between the expression of APN/CD13 and the prognosis of patients with NSCLC.  The inhibition of APN/CD13 may be an effective new mol. target therapy for patients with NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8_LPwESR3ALVg90H21EOLACvtfcHk0lhLpIisEVeL8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsValtbo%253D&md5=a797d87b167bbc2630d3f9269e1b4bf6</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-06-0338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-06-0338%26sid%3Dliteratum%253Aachs%26aulast%3DTokuhara%26aufirst%3DT.%26aulast%3DHattori%26aufirst%3DN.%26aulast%3DIshida%26aufirst%3DH.%26aulast%3DHirai%26aufirst%3DT.%26aulast%3DHigashiyama%26aufirst%3DM.%26aulast%3DKodama%26aufirst%3DK.%26aulast%3DMiyake%26aufirst%3DM.%26atitle%3DClinical%2520significance%2520of%2520aminopeptidase%2520N%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2006%26volume%3D12%26spage%3D3971%26epage%3D3978%26doi%3D10.1158%2F1078-0432.CCR-06-0338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valastyan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Tumor metastasis: molecular insights and evolving paradigms</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">275</span>– <span class="NLM_lpage">292</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.09.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.cell.2011.09.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=22000009" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlaksb3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2011&pages=275-292&author=S.+Valastyanauthor=R.+A.+Weinberg&title=Tumor+metastasis%3A+molecular+insights+and+evolving+paradigms&doi=10.1016%2Fj.cell.2011.09.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Tumor Metastasis: Molecular Insights and Evolving Paradigms</span></div><div class="casAuthors">Valastyan, Scott; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">275-292</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Metastases represent the end products of a multistep cell-biol. process termed the invasion-metastasis cascade, which involves dissemination of cancer cells to anatomically distant organ sites and their subsequent adaptation to foreign tissue microenvironments.  Each of these events is driven by the acquisition of genetic and/or epigenetic alterations within tumor cells and the co-option of nonneoplastic stromal cells, which together endow incipient metastatic cells with traits needed to generate macroscopic metastases.  Recent advances provide provocative insights into these cell-biol. and mol. changes, which have implications regarding the steps of the invasion-metastasis cascade that appear amenable to therapeutic targeting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVqOPVKwPqtbVg90H21EOLACvtfcHk0lhLpIisEVeL8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlaksb3N&md5=fa62d15c6b61bf90cfcdc1d51bbe98b9</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.09.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.09.024%26sid%3Dliteratum%253Aachs%26aulast%3DValastyan%26aufirst%3DS.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DTumor%2520metastasis%253A%2520molecular%2520insights%2520and%2520evolving%2520paradigms%26jtitle%3DCell%26date%3D2011%26volume%3D147%26spage%3D275%26epage%3D292%26doi%3D10.1016%2Fj.cell.2011.09.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span> <span> </span><span class="NLM_article-title">Development of synthetic aminopeptidase N/CD13 inhibitors to overcome cancer metastasis and angiogenesis</span>. <i>ACS. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">959</span>– <span class="NLM_lpage">964</span>, <span class="refDoi"> DOI: 10.1021/ml3000758</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml3000758" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2ls7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=959-964&author=L.+Suauthor=J.+Caoauthor=Y.+Jiaauthor=X.+Zhangauthor=H.+Fangauthor=W.+Xu&title=Development+of+synthetic+aminopeptidase+N%2FCD13+inhibitors+to+overcome+cancer+metastasis+and+angiogenesis&doi=10.1021%2Fml3000758"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Development of synthetic aminopeptidase N/CD13 inhibitors to overcome cancer metastasis and angiogenesis</span></div><div class="casAuthors">Su, Li; Cao, Jiangying; Jia, Yuping; Zhang, Xiaonan; Fang, Hao; Xu, Wenfang</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">959-964</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cancer metastasis is a major barrier to its treatment and an important cause of patient death.  Antimetastatic agents hold promise for patients with advanced metastatic tumors.  Aminopeptidase N/CD13 (APN) is being pursued by many as an important target against cancer metastasis and angiogenesis, but there are few reports on the in vivo evaluation of synthetic APN inhibitors.  Herein, a series of compds. targeting APN were synthesized and evaluated for their antimetastasis and antiangiogenesis potency both in vitro and in vivo.  Excitingly, compds. (I) (R1 = 1-naphthyl, CH2-3-OMeC6H4, CH2-1-naphthyl), with the most potent APN inhibitory activities, displayed significant antimetastasis and antiangiogenesis effects in vitro and in vivo, suggesting that those synthetic APN inhibitors have the potential to overcome cancer metastasis and angiogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMkj2KpECZfrVg90H21EOLACvtfcHk0lhLpIisEVeL8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2ls7vL&md5=342276e4b1621b1afb642bbcf0ae2464</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fml3000758&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml3000758%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DJ.%26aulast%3DJia%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DFang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DW.%26atitle%3DDevelopment%2520of%2520synthetic%2520aminopeptidase%2520N%252FCD13%2520inhibitors%2520to%2520overcome%2520cancer%2520metastasis%2520and%2520angiogenesis%26jtitle%3DACS.%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D959%26epage%3D964%26doi%3D10.1021%2Fml3000758" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wirtz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konstantopoulos, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Searson, P. C.</span></span> <span> </span><span class="NLM_article-title">The physics of cancer: the role of physical interactions and mechanical forces in metastasis</span>. <i>Nat. Rev. Cancer.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">512</span>– <span class="NLM_lpage">522</span>, <span class="refDoi"> DOI: 10.1038/nrc3080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1038%2Fnrc3080" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=21701513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvFSqtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=512-522&author=D.+Wirtzauthor=K.+Konstantopoulosauthor=P.+C.+Searson&title=The+physics+of+cancer%3A+the+role+of+physical+interactions+and+mechanical+forces+in+metastasis&doi=10.1038%2Fnrc3080"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">The physics of cancer: the role of physical interactions and mechanical forces in metastasis</span></div><div class="casAuthors">Wirtz, Denis; Konstantopoulos, Konstantinos; Searson, Peter C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">512-522</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Metastasis is a complex, multistep process responsible for >90% of cancer-related deaths.  In addn. to genetic and external environmental factors, the phys. interactions of cancer cells with their microenvironment, as well as their modulation by mech. forces, are key determinants of the metastatic process.  We reconstruct the metastatic process and describe the importance of key phys. and mech. processes at each step of the cascade.  The emerging insight into these phys. interactions may help to solve some long-standing questions in disease progression and may lead to new approaches to developing cancer diagnostics and therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRh9IWK-3RjLVg90H21EOLACvtfcHk0lg4_WYFiYH9-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvFSqtb0%253D&md5=a87cc9cd6d6383a52bb37932cfcb10a7</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnrc3080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3080%26sid%3Dliteratum%253Aachs%26aulast%3DWirtz%26aufirst%3DD.%26aulast%3DKonstantopoulos%26aufirst%3DK.%26aulast%3DSearson%26aufirst%3DP.%2BC.%26atitle%3DThe%2520physics%2520of%2520cancer%253A%2520the%2520role%2520of%2520physical%2520interactions%2520and%2520mechanical%2520forces%2520in%2520metastasis%26jtitle%3DNat.%2520Rev.%2520Cancer.%26date%3D2011%26volume%3D11%26spage%3D512%26epage%3D522%26doi%3D10.1038%2Fnrc3080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaffer, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">A perspective on cancer cell metastasis</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>331</i></span>,  <span class="NLM_fpage">1559</span>– <span class="NLM_lpage">1564</span>, <span class="refDoi"> DOI: 10.1126/science.1203543</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1126%2Fscience.1203543" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=21436443" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjs1Cgtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=331&publication_year=2011&pages=1559-1564&author=C.+L.+Chafferauthor=R.+A.+Weinberg&title=A+perspective+on+cancer+cell+metastasis&doi=10.1126%2Fscience.1203543"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A Perspective on Cancer Cell Metastasis</span></div><div class="casAuthors">Chaffer, Christine L.; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">331</span>
        (<span class="NLM_cas:issue">6024</span>),
    <span class="NLM_cas:pages">1559-1564</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Metastasis causes most cancer deaths, yet this process remains one of the most enigmatic aspects of the disease.  Building on new mechanistic insights emerging from recent research, we offer our perspective on the metastatic process and reflect on possible paths of future exploration.  We suggest that metastasis can be portrayed as a 2-phase process: The first phase involves the phys. translocation of a cancer cell to a distant organ, whereas the second encompasses the ability of the cancer cell to develop into a metastatic lesion at that distant site.  Although much remains to be learned about the second phase, we feel that an understanding of the first phase is now within sight, due in part to a better understanding of how cancer cell behavior can be modified by a cell-biol. program called the epithelial-to-mesenchymal transition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS1rLIOdukebVg90H21EOLACvtfcHk0lg4_WYFiYH9-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjs1Cgtbo%253D&md5=a361a1380ba9a72d974189059ebd9210</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1126%2Fscience.1203543&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1203543%26sid%3Dliteratum%253Aachs%26aulast%3DChaffer%26aufirst%3DC.%2BL.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DA%2520perspective%2520on%2520cancer%2520cell%2520metastasis%26jtitle%3DScience%26date%3D2011%26volume%3D331%26spage%3D1559%26epage%3D1564%26doi%3D10.1126%2Fscience.1203543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saiki, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoneda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakajima, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuruo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azuma, I.</span></span> <span> </span><span class="NLM_article-title">Role of aminopeptidase N (CD13) in tumor-cell invasion and extracellular matrix degradation</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">137</span>– <span class="NLM_lpage">143</span>, <span class="refDoi"> DOI: 10.1002/ijc.2910540122</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1002%2Fijc.2910540122" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=8097496" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaK3sXlsFOhs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1993&pages=137-143&author=I.+Saikiauthor=H.+Fujiiauthor=J.+Yonedaauthor=F.+Abeauthor=M.+Nakajimaauthor=T.+Tsuruoauthor=I.+Azuma&title=Role+of+aminopeptidase+N+%28CD13%29+in+tumor-cell+invasion+and+extracellular+matrix+degradation&doi=10.1002%2Fijc.2910540122"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Role of aminopeptidase N (CD13) in tumor-cell invasion and extracellular matrix degradation</span></div><div class="casAuthors">Saiki, Ikuo; Fujii, Hideji; Yoneda, Junya; Abe, Tuminori; Nakajima, Motowo; Tsuruo, Takashi; Azuma, Ichiro</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">137-43</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    </div><div class="casAbstract">The effect of monoclonal antibodies (MAbs) specific for aminopeptidase N/CD13 on the invasion of human metastatic tumor cells into reconstituted basement membrane (Matrigel) was studied.  The invasion of human metastatic tumor cells (SN12M renal-cell carcinoma, HT 1080 fibrosarcoma and A375M melanoma) into Matrigel-coated filters was inhibited by an anti-CD13 MAb, WM15, in a concn.-dependent manner.  However, this MAb did not have any effect on tumor-cell adhesion and migration to the extracellular matrixes, which may be involved in tumor-cell invasion.  MAb WM15 inhibited the degrdn. of type-IV collagen by tumor cells in a concn.-dependent manner.  Also, WM15 inhibited hydrolyzing activities towards substrates of aminopeptidases in 3 different tumor cells.  Since a previous study indicated that bestatin, an aminopeptidase inhibitor, was able to inhibit tumor-cell invasion, as well as aminopeptidase activities of murine and human metastatic tumor cells, cell-surface aminopeptidase N/CD13 may be partly involved in the activation mechanism for type-IV collagenolysis to achieve tumor-cell invasion, and anti-CD13 MAb WM15 may inhibit tumor-cell invasion through a mechanism involving its inhibitory action on the aminopeptidase N in tumor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogwayYWiR2P7Vg90H21EOLACvtfcHk0lg4_WYFiYH9-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXlsFOhs7w%253D&md5=36333c86bcefb529543bce7af4132376</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fijc.2910540122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.2910540122%26sid%3Dliteratum%253Aachs%26aulast%3DSaiki%26aufirst%3DI.%26aulast%3DFujii%26aufirst%3DH.%26aulast%3DYoneda%26aufirst%3DJ.%26aulast%3DAbe%26aufirst%3DF.%26aulast%3DNakajima%26aufirst%3DM.%26aulast%3DTsuruo%26aufirst%3DT.%26aulast%3DAzuma%26aufirst%3DI.%26atitle%3DRole%2520of%2520aminopeptidase%2520N%2520%2528CD13%2529%2520in%2520tumor-cell%2520invasion%2520and%2520extracellular%2520matrix%2520degradation%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D1993%26volume%3D54%26spage%3D137%26epage%3D143%26doi%3D10.1002%2Fijc.2910540122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoneda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saiki, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kojima, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azuma, I.</span></span> <span> </span><span class="NLM_article-title">Inhibition of tumor invasion and extracellular matrix degradation by ubenimex (bestatin)</span>. <i>Clin. Exp. Metastasis</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">59</span>, <span class="refDoi"> DOI: 10.1007/BF00163576</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1007%2FBF00163576" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1733647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaK38XitVOnsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1992&pages=49-59&author=J.+Yonedaauthor=I.+Saikiauthor=H.+Fujiiauthor=F.+Abeauthor=Y.+Kojimaauthor=I.+Azuma&title=Inhibition+of+tumor+invasion+and+extracellular+matrix+degradation+by+ubenimex+%28bestatin%29&doi=10.1007%2FBF00163576"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of tumor invasion and extracellular matrix degradation by ubenimex (bestatin)</span></div><div class="casAuthors">Yoneda, Junya; Saiki, Ikuo; Fujii, Hideji; Abe, Fuminori; Kojima, Yutaka; Azuma, Ichiro</div><div class="citationInfo"><span class="NLM_cas:title">Clinical & Experimental Metastasis</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">49-59</span>CODEN:
                <span class="NLM_cas:coden">CEXMD2</span>;
        ISSN:<span class="NLM_cas:issn">0262-0898</span>.
    </div><div class="casAbstract">The authors have investigated the effect of the immunomodulator ubenimex (bestatin) on the enzymic degrdn. of the extracellular matrix by human renal cell carcinoma SN12M cells during the invasive process.  The invasion of SN12M cells into reconstituted basement membrane (Matrigel) was inhibited by the presence of bestatin in a concn.-dependent manner.  However, bestatin did not have any effect on tumor cell adhesion and migration to the extracellular matrixes which may be involved in tumor cell invasion.  Bestatin inhibited the degrdn. of type IV collagen by tumor cells, but not by tumor-conditioned medium (TCM), in a concn.-dependent manner.  The authors also found that bestatin inhibited hydrolyzing activities towards substrates of aminopeptidases in SN12M cells.  Since bestatin was found to inhibit aminopeptidase activity, the inhibition of tumor invasion by bestatin is likely to be assocd. with its action as an enzyme inhibitor.  Bestatin only slightly inhibited tumor cell plasmin activity, which can lead to the conversion of the latent collagenase to the active form, but this slight effect was not significant.  The zymog. of TCM from SN12M cells showed that the treatment of tumor cells with bestatin resulted in the disappearance of the 689 kDa type IV collagenase-enzyme level (active form) and slight redn. of the 72 kDa type IV collagenase-enzyme level (latent form).  These results indicated that bestatin may inhibit tumor cell invasion through a mechanism involving its inhibitory action on aminopeptidases in tumor cells, suggesting that the aminopeptidase may partly be assocd. with the conversion of a latent form of type IV procollagenase to an active form or the secretion of the collagenases from tumor cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooBzxijM_O1rVg90H21EOLACvtfcHk0lhwLi8Dw1FO2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XitVOnsbk%253D&md5=b6239f33e3bcf9169498553a53225a0a</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1007%2FBF00163576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00163576%26sid%3Dliteratum%253Aachs%26aulast%3DYoneda%26aufirst%3DJ.%26aulast%3DSaiki%26aufirst%3DI.%26aulast%3DFujii%26aufirst%3DH.%26aulast%3DAbe%26aufirst%3DF.%26aulast%3DKojima%26aufirst%3DY.%26aulast%3DAzuma%26aufirst%3DI.%26atitle%3DInhibition%2520of%2520tumor%2520invasion%2520and%2520extracellular%2520matrix%2520degradation%2520by%2520ubenimex%2520%2528bestatin%2529%26jtitle%3DClin.%2520Exp.%2520Metastasis%26date%3D1992%26volume%3D10%26spage%3D49%26epage%3D59%26doi%3D10.1007%2FBF00163576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mucha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalton, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kafarski, P.</span></span> <span> </span><span class="NLM_article-title">Metallo-aminopeptidase inhibitors</span>. <i>Biochimie</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">1509</span>– <span class="NLM_lpage">1529</span>, <span class="refDoi"> DOI: 10.1016/j.biochi.2010.04.026</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.biochi.2010.04.026" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=20457213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtl2isLrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2010&pages=1509-1529&author=A.+Muchaauthor=M.+Dragauthor=J.+P.+Daltonauthor=P.+Kafarski&title=Metallo-aminopeptidase+inhibitors&doi=10.1016%2Fj.biochi.2010.04.026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Metallo-aminopeptidase inhibitors</span></div><div class="casAuthors">Mucha, Artur; Drag, Marcin; Dalton, John P.; Kafarski, Pawel</div><div class="citationInfo"><span class="NLM_cas:title">Biochimie</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1509-1529</span>CODEN:
                <span class="NLM_cas:coden">BICMBE</span>;
        ISSN:<span class="NLM_cas:issn">0300-9084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Aminopeptidases are enzymes that selectively hydrolyze an amino acid residue from the N-terminus of proteins and peptides.  They are important for the proper functioning of prokaryotic and eukaryotic cells, but very often are central players in the devastating human diseases like cancer, malaria and diabetes.  The largest aminopeptidase group include enzymes contg. metal ion(s) in their active centers, which often dets. the type of inhibitors that are the most suitable for them.  Effective ligands mostly bind in a non-covalent mode by forming complexes with the metal ion(s).  Here, we present several approaches for the design of inhibitors for metallo-aminopeptidases.  The optimized structures should be considered as potential leads in the drug discovery process against endogenous and infectious diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGGCPbxscFkrVg90H21EOLACvtfcHk0lhwLi8Dw1FO2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtl2isLrF&md5=5b9bb175448ac8e3058f1e4ad8f5654c</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.biochi.2010.04.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biochi.2010.04.026%26sid%3Dliteratum%253Aachs%26aulast%3DMucha%26aufirst%3DA.%26aulast%3DDrag%26aufirst%3DM.%26aulast%3DDalton%26aufirst%3DJ.%2BP.%26aulast%3DKafarski%26aufirst%3DP.%26atitle%3DMetallo-aminopeptidase%2520inhibitors%26jtitle%3DBiochimie%26date%3D2010%26volume%3D92%26spage%3D1509%26epage%3D1529%26doi%3D10.1016%2Fj.biochi.2010.04.026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span> <span> </span><span class="NLM_article-title">Recent advance in aminopeptidase N (APN/CD13) inhibitor research</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">5011</span>– <span class="NLM_lpage">5021</span>, <span class="refDoi"> DOI: 10.2174/092986711797535155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.2174%2F092986711797535155" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=22050749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVOrtLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=5011-5021&author=X.+Zhangauthor=H.+Fangauthor=J.+Zhangauthor=Y.+Yuanauthor=W.+Xu&title=Recent+advance+in+aminopeptidase+N+%28APN%2FCD13%29+inhibitor+research&doi=10.2174%2F092986711797535155"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advance in aminopeptidase N (APN/CD13) inhibitor research</span></div><div class="casAuthors">Zhang, X.; Fang, H.; Zhang, J.; Yuan, Y.; Xu, W.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">5011-5021</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  APN is an important zinc dependent metallo-exopeptidase, which can degrade the extracellular matrix, and plays an important role in tumor invasion, metastasis and angiogenesis, it has been considered as a suitable target for anti-cancer drug design.  Recently, research of structure-based APN inhibitors is becoming an interesting field, and many APN inhibitors have been reported.  Here we review a series of APN inhibitors and the recent progress in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8lQD3shghIrVg90H21EOLACvtfcHk0lhwLi8Dw1FO2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVOrtLrP&md5=76c3d93c00979eda9f2ccf67c9b31145</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.2174%2F092986711797535155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986711797535155%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DFang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DW.%26atitle%3DRecent%2520advance%2520in%2520aminopeptidase%2520N%2520%2528APN%252FCD13%2529%2520inhibitor%2520research%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2011%26volume%3D18%26spage%3D5011%26epage%3D5021%26doi%3D10.2174%2F092986711797535155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scornik, O. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botbol, V.</span></span> <span> </span><span class="NLM_article-title">Bestatin as an experimental tool in mammals</span>. <i>Curr. Drug Metab.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">85</span>, <span class="refDoi"> DOI: 10.2174/1389200013338748</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.2174%2F1389200013338748" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=11465152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD3MXisFWjurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2001&pages=67-85&author=O.+A.+Scornikauthor=V.+Botbol&title=Bestatin+as+an+experimental+tool+in+mammals&doi=10.2174%2F1389200013338748"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Bestatin as an experimental tool in mammals</span></div><div class="casAuthors">Scornik, O. A.; Botbol, V.</div><div class="citationInfo"><span class="NLM_cas:title">Current Drug Metabolism</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-85</span>CODEN:
                <span class="NLM_cas:coden">CDMUBU</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review with 163 refs.  Bestatin, an antibiotic of microbial origin, is a potent inhibitor of some, but not all aminopeptidases.  It can be administered, with low toxicity, to cultured cells, intact animals and humans.  It has become a useful tool in elucidating the physiol. role of some mammalian exopeptidases in the regulation of the immune system, in the growth of tumors and their invasion of surrounding tissues, and in the degrdn. of cellular proteins.  Bestatin-sensitive enzymes play important roles in the digestion and absorption of peptides in the brush border of the intestine and the kidney, in the reproductive system, and in the metab. of opioid peptides and leukotrienes.  Aminopeptidase N emerges as the major target for the effects of bestatin on the immune system and some of its effects on tumor growth and the endometrium.  It is also the major bestatin-sensitive enzyme involved in the degrdn. of oligopeptides on the surface of intestine and kidney brush borders, and the inactivation of enkephalins in the brain.  Bestatin-sensitive cytosolic exopeptidases are important in the degrdn. to amino acids of di- and tripeptides generated in most cells by cellular protein degrdn., as well as those absorbed through the brush border of intestine and kidney.  Inhibition of one of these exopeptidases, cytosol alanine aminopeptidase, results in apoptosis.  Bestatin-sensitive cystinyl aminopeptidase is abundant in placenta.  Two bestatin-sensitive enzymes, aminopeptidase B and nardilysin, are particularly abundant in late spermatids.  Finally bestatin-sensitive LTA4 hydrolase generates the potent chemotactic agent, LTB4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6UOKu-QtgUrVg90H21EOLACvtfcHk0lhwLi8Dw1FO2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXisFWjurY%253D&md5=02986f26a62a737ffb23b19b5e38109f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.2174%2F1389200013338748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1389200013338748%26sid%3Dliteratum%253Aachs%26aulast%3DScornik%26aufirst%3DO.%2BA.%26aulast%3DBotbol%26aufirst%3DV.%26atitle%3DBestatin%2520as%2520an%2520experimental%2520tool%2520in%2520mammals%26jtitle%3DCurr.%2520Drug%2520Metab.%26date%3D2001%26volume%3D2%26spage%3D67%26epage%3D85%26doi%3D10.2174%2F1389200013338748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Umezawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aoyagi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeuchi, T.</span></span> <span> </span><span class="NLM_article-title">Bestatin, an inhibitor of aminopeptidase B, produced by actinomycetes</span>. <i>J. Antibiot.</i> <span class="NLM_year" style="font-weight: bold;">1976</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">97</span>– <span class="NLM_lpage">99</span>, <span class="refDoi"> DOI: 10.7164/antibiotics.29.97</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.7164%2Fantibiotics.29.97" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=931798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaE28XktlWmt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1976&pages=97-99&author=H.+Umezawaauthor=T.+Aoyagiauthor=H.+Sudaauthor=M.+Hamadaauthor=T.+Takeuchi&title=Bestatin%2C+an+inhibitor+of+aminopeptidase+B%2C+produced+by+actinomycetes&doi=10.7164%2Fantibiotics.29.97"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Bestatin, an inhibitor of aminopeptidase B, produced by actinomycetes</span></div><div class="casAuthors">Umezawa, Hamao; Aoyagi, Takaaki; Suda, Hiroyuki; Hamada, Masa; Takeuchi, Tomio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antibiotics</span>
        (<span class="NLM_cas:date">1976</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">97-9</span>CODEN:
                <span class="NLM_cas:coden">JANTAJ</span>;
        ISSN:<span class="NLM_cas:issn">0021-8820</span>.
    </div><div class="casAbstract">Bestatin [58970-76-6], a specific inhibitor of aminopeptidase B (EC 3.4.11.6) [9073-92-1], was produced by Streptomyces olivoreticuli strain MD976-C7.  A typical medium contained maltose 1.5, yeast ext. 0.3, N-Z amine 1, and NaCl 0.3% (pH 7.4).  Max. prodn. was attained in 2-3 days of shaking at 27°, and prodn. was maintained for 2-6 days thereafter.  Bestatin was extd. from the culture filtrate by adsorption on activated C or Amberlite-XAD or by extn. with an org. solvent.  Bestatin also inhibited leucine aminopeptidase (EC 3.4.11.1) [9001-61-0].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6vWkQ-jPexbVg90H21EOLACvtfcHk0liU4eDBaztygA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XktlWmt7g%253D&md5=eefb781bea66a640c0967769a24ea231</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.7164%2Fantibiotics.29.97&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7164%252Fantibiotics.29.97%26sid%3Dliteratum%253Aachs%26aulast%3DUmezawa%26aufirst%3DH.%26aulast%3DAoyagi%26aufirst%3DT.%26aulast%3DSuda%26aufirst%3DH.%26aulast%3DHamada%26aufirst%3DM.%26aulast%3DTakeuchi%26aufirst%3DT.%26atitle%3DBestatin%252C%2520an%2520inhibitor%2520of%2520aminopeptidase%2520B%252C%2520produced%2520by%2520actinomycetes%26jtitle%3DJ.%2520Antibiot.%26date%3D1976%26volume%3D29%26spage%3D97%26epage%3D99%26doi%3D10.7164%2Fantibiotics.29.97" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ichinose, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Genka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koike, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kato, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iioka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakuma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, M.</span></span> <span> </span><span class="NLM_article-title">Randomized double-blind placebo-controlled trial of bestatin in patients with resected stage I squamous-cell lung carcinoma</span>. <i>JNCI, J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>95</i></span>,  <span class="NLM_fpage">605</span>– <span class="NLM_lpage">610</span>, <span class="refDoi"> DOI: 10.1093/jnci/95.8.605</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1093%2Fjnci%2F95.8.605" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=12697853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjtFWqsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2003&pages=605-610&author=Y.+Ichinoseauthor=K.+Genkaauthor=T.+Koikeauthor=H.+Katoauthor=Y.+Watanabeauthor=T.+Moriauthor=S.+Iiokaauthor=A.+Sakumaauthor=M.+Ohta&title=Randomized+double-blind+placebo-controlled+trial+of+bestatin+in+patients+with+resected+stage+I+squamous-cell+lung+carcinoma&doi=10.1093%2Fjnci%2F95.8.605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Randomized double-blind placebo-controlled trial of bestatin in patients with resected stage I squamous-cell lung carcinoma</span></div><div class="casAuthors">Ichinose, Yukito; Genka, Keiichiro; Koike, Teruaki; Kato, Harubumi; Watanabe, Yoh; Mori, Takashi; Iioka, Sogo; Sakuma, Akira; Ohta, Mitsuo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">95</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">605-610</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Background: Bestatin is a potent aminopeptidase inhibitor that has immunostimulant and antitumor activity.  We conducted a prospective randomized, double-blind, placebo-controlled trial to det. whether postoperative adjuvant treatment with bestatin could prolong the survival of patients with completely resected stage I squamous-cell lung carcinoma.  Methods: Patients with confirmed, resected stage I squamous-cell lung carcinoma were randomly assigned to receive either bestatin (30 mg) or placebo daily by mouth for 2 yr.  We assessed whether bestatin treatment was assocd. with overall survival and 5-yr cancer-free survival and assessed its safety.  All statistical tests were two-sided.  Results: From July 8, 1992, through Mar. 30, 1995, 402 patients were entered in the study, 202 in the bestatin group and 198 in the placebo group.  The median follow-up for surviving patients was 76 mo (range = 58-92 mo).  The 5-yr overall survival was 81% in the bestatin group and 74% in the placebo group for a difference of 7% (95% confidence interval [CI] = -1.4% to 15.0%).  The 5-yr cancer-free survival was 71% in the bestatin group and 62% in the placebo group for a difference of 9% (95% CI = -0.7% to 17.8%).  Overall survival (P = .033, log-rank test) and cancer-free survival (P = .017, log-rank test) were statistically significantly different by Kaplan-Meier anal.  Few adverse events were obsd. in either group.  Conclusions: Survival was statistically significantly better for patients with completely resected stage I squamous-cell lung carcinoma who were treated with bestatin as a postoperative adjuvant therapy than for those who received a placebo.  This result requires confirmation in other phase III trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKPQhXgUTESbVg90H21EOLACvtfcHk0liU4eDBaztygA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjtFWqsb4%253D&md5=f794262982689c02a29e138a93d4e44f</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2F95.8.605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252F95.8.605%26sid%3Dliteratum%253Aachs%26aulast%3DIchinose%26aufirst%3DY.%26aulast%3DGenka%26aufirst%3DK.%26aulast%3DKoike%26aufirst%3DT.%26aulast%3DKato%26aufirst%3DH.%26aulast%3DWatanabe%26aufirst%3DY.%26aulast%3DMori%26aufirst%3DT.%26aulast%3DIioka%26aufirst%3DS.%26aulast%3DSakuma%26aufirst%3DA.%26aulast%3DOhta%26aufirst%3DM.%26atitle%3DRandomized%2520double-blind%2520placebo-controlled%2520trial%2520of%2520bestatin%2520in%2520patients%2520with%2520resected%2520stage%2520I%2520squamous-cell%2520lung%2520carcinoma%26jtitle%3DJNCI%252C%2520J.%2520Natl.%2520Cancer%2520Inst.%26date%3D2003%26volume%3D95%26spage%3D605%26epage%3D610%26doi%3D10.1093%2Fjnci%2F95.8.605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ota, K.</span></span> <span> </span><span class="NLM_article-title">Review of ubenimex (Bestatin): clinical research</span>. <i>Biomed. Pharmacother.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1016/0753-3322(91)90123-B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2F0753-3322%2891%2990123-B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1912370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A280%3ADyaK38%252Fgtlalsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=1991&pages=55-60&author=K.+Ota&title=Review+of+ubenimex+%28Bestatin%29%3A+clinical+research&doi=10.1016%2F0753-3322%2891%2990123-B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Review of ubenimex (Bestatin): clinical research</span></div><div class="casAuthors">Ota K</div><div class="citationInfo"><span class="NLM_cas:title">Biomedicine & pharmacotherapy = Biomedecine & pharmacotherapie</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">55-60</span>
        ISSN:<span class="NLM_cas:issn">0753-3322</span>.
    </div><div class="casAbstract">An immunomodulating agent, ubenimex (Bestatin) has low toxicity even after long-term oral administration and brings about significant modifications in immunological response.  In a cooperative randomized controlled study of Bestatin immunotherapy for adult acute nonlymphocytic leukemia, prolongation of remission duration and survival was achieved with Bestatin immunotherapy combined with remission maintenance chemotherapy.  The significant prolongation of remission duration and survival was seen in the Bestatin group, especially in the elderly patients.  Randomized controlled studies of Bestatin immunotherapy were performed in solid tumors including malignant melanoma, carcinoma of the lung, stomach, bladder, head and neck and esophagus, and therapeutic benefits regarding disease free-interval or survival were observed in certain types of the above-mentioned cancers; however, Bestatin immunotherapy for these cancers should be further investigated in large-scale controlled studies to confirm its activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQgjoNzGIjBCcScOqQTSkOffW6udTcc2ebDT6-EzbNs_7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK38%252Fgtlalsw%253D%253D&md5=17a5625acdeaf93d830fd935bd3475ee</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2F0753-3322%2891%2990123-B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0753-3322%252891%252990123-B%26sid%3Dliteratum%253Aachs%26aulast%3DOta%26aufirst%3DK.%26atitle%3DReview%2520of%2520ubenimex%2520%2528Bestatin%2529%253A%2520clinical%2520research%26jtitle%3DBiomed.%2520Pharmacother.%26date%3D1991%26volume%3D45%26spage%3D55%26epage%3D60%26doi%3D10.1016%2F0753-3322%2891%2990123-B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ota, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurita, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masaoka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uzuka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogawa, N.</span></span> <span> </span><span class="NLM_article-title">Immunotherapy with bestatin for acute nonlymphocytic leukemia in adults</span>. <i>Cancer Immunol. Immunother.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1007/BF00205548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1007%2FBF00205548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=3533256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A280%3ADyaL2s%252FivVWjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1986&pages=5-10&author=K.+Otaauthor=S.+Kuritaauthor=K.+Yamadaauthor=T.+Masaokaauthor=Y.+Uzukaauthor=N.+Ogawa&title=Immunotherapy+with+bestatin+for+acute+nonlymphocytic+leukemia+in+adults&doi=10.1007%2FBF00205548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Immunotherapy with bestatin for acute nonlymphocytic leukemia in adults</span></div><div class="casAuthors">Ota K; Kurita S; Yamada K; Masaoka T; Uzuka Y; Ogawa N</div><div class="citationInfo"><span class="NLM_cas:title">Cancer immunology, immunotherapy : CII</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-10</span>
        ISSN:<span class="NLM_cas:issn">0340-7004</span>.
    </div><div class="casAbstract">In a cooperative randomized control study of immunotherapy with bestatin in combination with chemotherapy in adults with acute nonlymphocytic leukemia (ANLL), 101 patients (48 in the bestatin group and 53 in the control group) out of 115 patients registered were evaluated as eligible.  The bestatin group achieved a statistically significant prolongation of survival compared with the control group in overall ANLL and acute myelogenous leukemia.  In the analysis of patient age, the bestatin group achieved a statistically significant prolongation of both the remission duration and survival in patients aged 50 to 65 years, while the differences were not significant in the 15 to 49 age group.  The bestatin group tended to achieve a higher rate of reinduction of remission in patients who had recurrence of leukemia.  Side effects developed in only 5 (9.6%) of 52 patients treated with bestatin.  None of these side effects were particularly serious in nature.  It is concluded that bestatin is useful for prolongation of survival of adult patients with ANLL, making for a longer remission duration especially in elderly patients and with few side effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRj-k_x41rUPOqrn-c4Of3KfW6udTcc2ebDT6-EzbNs_7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL2s%252FivVWjug%253D%253D&md5=23e6deab11cc762aa7df680ac674cdde</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1007%2FBF00205548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00205548%26sid%3Dliteratum%253Aachs%26aulast%3DOta%26aufirst%3DK.%26aulast%3DKurita%26aufirst%3DS.%26aulast%3DYamada%26aufirst%3DK.%26aulast%3DMasaoka%26aufirst%3DT.%26aulast%3DUzuka%26aufirst%3DY.%26aulast%3DOgawa%26aufirst%3DN.%26atitle%3DImmunotherapy%2520with%2520bestatin%2520for%2520acute%2520nonlymphocytic%2520leukemia%2520in%2520adults%26jtitle%3DCancer%2520Immunol.%2520Immunother.%26date%3D1986%26volume%3D23%26spage%3D5%26epage%3D10%26doi%3D10.1007%2FBF00205548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Urabe, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mutoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizoguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takaku, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogawa, N.</span></span> <span> </span><span class="NLM_article-title">Ubenimex in the treatment of acute nonlymphocytic leukemia in adults</span>. <i>Ann. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1007/BF01788128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1007%2FBF01788128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=8347731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A280%3ADyaK3szltVSntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=1993&pages=63-66&author=A.+Urabeauthor=Y.+Mutohauthor=H.+Mizoguchiauthor=F.+Takakuauthor=N.+Ogawa&title=Ubenimex+in+the+treatment+of+acute+nonlymphocytic+leukemia+in+adults&doi=10.1007%2FBF01788128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Ubenimex in the treatment of acute nonlymphocytic leukemia in adults</span></div><div class="casAuthors">Urabe A; Mutoh Y; Mizoguchi H; Takaku F; Ogawa N</div><div class="citationInfo"><span class="NLM_cas:title">Annals of hematology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">63-6</span>
        ISSN:<span class="NLM_cas:issn">0939-5555</span>.
    </div><div class="casAbstract">A multi-institutional randomized study for the evaluation of ubenimex (Bestatin) in the treatment of adult acute nonlymphocytic leukemia was performed.  One hundred and ninety-five patients were registered from February 1988 to December 1990.  Patients who had reached complete remission by one or two courses of remission induction chemotherapy were divided into the ubenimex group or the control group by randomization.  Patients of the ubenimex group started to receive 30 mg ubenimex orally once a day when maintenance therapy began and continued as long as possible.  Remission duration and survival were analyzed based on the data as of August 31, 1991.  Remission duration of the ubenimex group was superior to that of the control group (generalized Wilcoxon test: p = 0.0338).  Fifty percent remission duration was 508 days in the ubenimex group and 386 days in the control group.  There has been no statistical difference in survival between the two groups as yet.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR-t0sTByak7WK_SFaaT9INfW6udTcc2eZml1edkjRgvLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK3szltVSntQ%253D%253D&md5=4a8545655f9819fdbaef9b3bbd012d9c</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1007%2FBF01788128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF01788128%26sid%3Dliteratum%253Aachs%26aulast%3DUrabe%26aufirst%3DA.%26aulast%3DMutoh%26aufirst%3DY.%26aulast%3DMizoguchi%26aufirst%3DH.%26aulast%3DTakaku%26aufirst%3DF.%26aulast%3DOgawa%26aufirst%3DN.%26atitle%3DUbenimex%2520in%2520the%2520treatment%2520of%2520acute%2520nonlymphocytic%2520leukemia%2520in%2520adults%26jtitle%3DAnn.%2520Hematol.%26date%3D1993%26volume%3D67%26spage%3D63%26epage%3D66%26doi%3D10.1007%2FBF01788128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krige, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Needham, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bawden, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flores, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farmer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miles, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callaghan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, V. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirwin-Jones, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legris, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Box, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laber, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Odedra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eccles, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bone, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayscough, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drummond, A. H.</span></span> <span> </span><span class="NLM_article-title">CHR-2797: an antiproliferative aminopeptidase inhibitor that leads to amino acid deprivation in human leukemic cells</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">6669</span>– <span class="NLM_lpage">6679</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-07-6627</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1158%2F0008-5472.CAN-07-6627" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=18701491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD1cXpslKms70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=6669-6679&author=D.+Krigeauthor=L.+A.+Needhamauthor=L.+J.+Bawdenauthor=N.+Floresauthor=H.+Farmerauthor=L.+E.+Milesauthor=E.+Stoneauthor=J.+Callaghanauthor=S.+Chandlerauthor=V.+L.+Clarkauthor=P.+Kirwin-Jonesauthor=V.+Legrisauthor=J.+Owenauthor=T.+Patelauthor=S.+Woodauthor=G.+Boxauthor=D.+Laberauthor=R.+Odedraauthor=A.+Wrightauthor=L.+M.+Woodauthor=S.+A.+Ecclesauthor=E.+A.+Boneauthor=A.+Ayscoughauthor=A.+H.+Drummond&title=CHR-2797%3A+an+antiproliferative+aminopeptidase+inhibitor+that+leads+to+amino+acid+deprivation+in+human+leukemic+cells&doi=10.1158%2F0008-5472.CAN-07-6627"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">CHR-2797: An Antiproliferative Aminopeptidase Inhibitor that Leads to Amino Acid Deprivation in Human Leukemic Cells</span></div><div class="casAuthors">Krige, David; Needham, Lindsey A.; Bawden, Lindsay J.; Flores, Nicolas; Farmer, Hannah; Miles, Lauren E. C.; Stone, Erica; Callaghan, Juliana; Chandler, Stephen; Clark, Vanessa L.; Kirwin-Jones, Patricia; Legris, Valerie; Owen, Jo; Patel, Thakor; Wood, Steve; Box, Gary; Laber, David; Odedra, Rajesh; Wright, Annette; Wood, L. Michael; Eccles, Suzanne A.; Bone, Elisabeth A.; Ayscough, Andrew; Drummond, Alan H.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6669-6679</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">CHR-2797 is a novel metalloenzyme inhibitor that is converted into a pharmacol. active acid product (CHR-79888) inside cells.  CHR-79888 is a potent inhibitor of a no. of intracellular aminopeptidases, including leucine aminopeptidase.  CHR-2797 exerts antiproliferative effects against a range of tumor cell lines in vitro and in vivo and shows selectivity for transformed over nontransformed cells.  Its antiproliferative effects are at least 300 times more potent than the prototypical aminopeptidase inhibitor, bestatin.  However, the mechanism by which inhibition of these enzymes leads to proliferative changes is not understood.  Gene expression microarrays were used to profile changes in mRNA expression levels in the human promyelocytic leukemia cell line HL-60 treated with CHR-2797.  This anal. showed that CHR-2797 treatment induced a transcriptional response indicative of amino acid depletion, the amino acid deprivation response, which involves up-regulation of amino acid synthetic genes, transporters, and tRNA synthetases.  These changes were confirmed in other leukemic cell lines sensitive to the antiproliferative effects of CHR-2797.  Furthermore, CHR-2797 treatment inhibited phosphorylation of mTOR substrates and reduced protein synthesis in HL-60 cells, both also indicative of amino acid depletion.  Treatment with CHR-2797 led to an increase in the concn. of intracellular small peptides, the substrates of aminopeptidases.  It is suggested that aminopeptidase inhibitors, such as CHR-2797 and bestatin, deplete sensitive tumor cells of amino acids by blocking protein recycling, and this generates an antiproliferative effect.  CHR-2797 is orally bioavailable and currently undergoing phase II clin. investigation in the treatment of myeloid leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmWE9oaA_yI7Vg90H21EOLACvtfcHk0ljxTYh5i2Oeww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXpslKms70%253D&md5=e116fdd3ec63512bb64c45001ece3351</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-07-6627&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-07-6627%26sid%3Dliteratum%253Aachs%26aulast%3DKrige%26aufirst%3DD.%26aulast%3DNeedham%26aufirst%3DL.%2BA.%26aulast%3DBawden%26aufirst%3DL.%2BJ.%26aulast%3DFlores%26aufirst%3DN.%26aulast%3DFarmer%26aufirst%3DH.%26aulast%3DMiles%26aufirst%3DL.%2BE.%26aulast%3DStone%26aufirst%3DE.%26aulast%3DCallaghan%26aufirst%3DJ.%26aulast%3DChandler%26aufirst%3DS.%26aulast%3DClark%26aufirst%3DV.%2BL.%26aulast%3DKirwin-Jones%26aufirst%3DP.%26aulast%3DLegris%26aufirst%3DV.%26aulast%3DOwen%26aufirst%3DJ.%26aulast%3DPatel%26aufirst%3DT.%26aulast%3DWood%26aufirst%3DS.%26aulast%3DBox%26aufirst%3DG.%26aulast%3DLaber%26aufirst%3DD.%26aulast%3DOdedra%26aufirst%3DR.%26aulast%3DWright%26aufirst%3DA.%26aulast%3DWood%26aufirst%3DL.%2BM.%26aulast%3DEccles%26aufirst%3DS.%2BA.%26aulast%3DBone%26aufirst%3DE.%2BA.%26aulast%3DAyscough%26aufirst%3DA.%26aulast%3DDrummond%26aufirst%3DA.%2BH.%26atitle%3DCHR-2797%253A%2520an%2520antiproliferative%2520aminopeptidase%2520inhibitor%2520that%2520leads%2520to%2520amino%2520acid%2520deprivation%2520in%2520human%2520leukemic%2520cells%26jtitle%3DCancer%2520Res.%26date%3D2008%26volume%3D68%26spage%3D6669%26epage%3D6679%26doi%3D10.1158%2F0008-5472.CAN-07-6627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lowenberg, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ossenkoppele, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burnett, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zachee, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duhrsen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dierickx, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller-Tidow, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonneveld, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krug, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bone, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flores, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hooftman, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zweegman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, F.</span></span> <span> </span><span class="NLM_article-title">Phase I/II clinical study of Tosedostat, an inhibitor of aminopeptidases, in patients with acute myeloid leukemia and myelodysplasia</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">4333</span>– <span class="NLM_lpage">4338</span>, <span class="refDoi"> DOI: 10.1200/JCO.2009.27.6295</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1200%2FJCO.2009.27.6295" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=20733120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A280%3ADC%252BC3cflvFOntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2010&pages=4333-4338&author=B.+Lowenbergauthor=G.+Morganauthor=G.+J.+Ossenkoppeleauthor=A.+K.+Burnettauthor=P.+Zacheeauthor=U.+Duhrsenauthor=D.+Dierickxauthor=C.+Muller-Tidowauthor=P.+Sonneveldauthor=U.+Krugauthor=E.+Boneauthor=N.+Floresauthor=A.+F.+Richardsonauthor=L.+Hooftmanauthor=C.+Jenkinsauthor=S.+Zweegmanauthor=F.+Davies&title=Phase+I%2FII+clinical+study+of+Tosedostat%2C+an+inhibitor+of+aminopeptidases%2C+in+patients+with+acute+myeloid+leukemia+and+myelodysplasia&doi=10.1200%2FJCO.2009.27.6295"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Phase I/II clinical study of Tosedostat, an inhibitor of aminopeptidases, in patients with acute myeloid leukemia and myelodysplasia</span></div><div class="casAuthors">Lowenberg Bob; Morgan Gareth; Ossenkoppele Gert J; Burnett Alan K; Zachee Pierre; Duhrsen Ulrich; Dierickx Daan; Muller-Tidow Carsten; Sonneveld Pieter; Krug Utz; Bone Elisabeth; Flores Nicolas; Richardson Alison F; Hooftman Leon; Jenkins Chris; Zweegman Sonja; Davies Faith</div><div class="citationInfo"><span class="NLM_cas:title">Journal of clinical oncology : official journal of the American Society of Clinical Oncology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">4333-8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">PURPOSE:  To identify the maximum-tolerated dose (MTD) and to evaluate the antileukemic activity of tosedostat (formerly CHR-2797), an orally bioavailable aminopeptidase inhibitor.  PATIENTS AND METHODS:  In phase I, the MTD of once daily oral doses of tosedostat in hematologic malignancies was defined.  In phase II, the therapeutic activity of the maximum-acceptable dose (MAD) of tosedostat was evaluated in elderly and/or relapsing patients with acute myeloid leukemia (AML) or myelodysplastic syndrome.  RESULTS:  In phase I, 16 patients were treated in four cohorts with tosedostat (60 mg to 180 mg) for 28 days.  Three patients reported dose-limiting toxicities: two with reversible thrombocytopenia (> 75% reduction in platelet count) at 180 mg (MTD) and one with a Common Toxicity Criteria (CTC) grade 3 ALT elevation at 130 mg (MAD).  In phase II, 41 patients were treated with 130 mg tosedostat.  In phases I and II, the most common severe (CTC grades 3 to 5) adverse event was a reduction in the platelet count.  Of the 51 AML patients in this study, seven reached complete marrow response (< 5% marrow blasts), with three achieving complete remission, and a further seven patients reaching a partial marrow response (between 5% and 15% marrow blasts).  The overall response rate was therefore 27%.  All responders were age > 60 years, and 79% had either relapsed or refractory AML.  CONCLUSION:  This phase I/II study demonstrates that oral once daily dosing with 130 mg tosedostat is well tolerated and has significant antileukemic activity.  The favorable risk-benefit profile suggests that further clinical trials are warranted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSC4BwIVFbbZpPq5LRczm8ffW6udTcc2eYRq1xjtFlAk7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cflvFOntQ%253D%253D&md5=f1209aab57f7448877dba45b96d4c4b1</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1200%2FJCO.2009.27.6295&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2009.27.6295%26sid%3Dliteratum%253Aachs%26aulast%3DLowenberg%26aufirst%3DB.%26aulast%3DMorgan%26aufirst%3DG.%26aulast%3DOssenkoppele%26aufirst%3DG.%2BJ.%26aulast%3DBurnett%26aufirst%3DA.%2BK.%26aulast%3DZachee%26aufirst%3DP.%26aulast%3DDuhrsen%26aufirst%3DU.%26aulast%3DDierickx%26aufirst%3DD.%26aulast%3DMuller-Tidow%26aufirst%3DC.%26aulast%3DSonneveld%26aufirst%3DP.%26aulast%3DKrug%26aufirst%3DU.%26aulast%3DBone%26aufirst%3DE.%26aulast%3DFlores%26aufirst%3DN.%26aulast%3DRichardson%26aufirst%3DA.%2BF.%26aulast%3DHooftman%26aufirst%3DL.%26aulast%3DJenkins%26aufirst%3DC.%26aulast%3DZweegman%26aufirst%3DS.%26aulast%3DDavies%26aufirst%3DF.%26atitle%3DPhase%2520I%252FII%2520clinical%2520study%2520of%2520Tosedostat%252C%2520an%2520inhibitor%2520of%2520aminopeptidases%252C%2520in%2520patients%2520with%2520acute%2520myeloid%2520leukemia%2520and%2520myelodysplasia%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2010%26volume%3D28%26spage%3D4333%26epage%3D4338%26doi%3D10.1200%2FJCO.2009.27.6295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drinkwater, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vinh, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamert, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggeri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holleran, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loganathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paiardini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charman, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avery, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGowan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scammells, P. J.</span></span> <span> </span><span class="NLM_article-title">Potent dual inhibitors of <i>Plasmodium falciparum</i> M1 and M17 aminopeptidases through optimization of S1 pocket interactions</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">64</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.01.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.ejmech.2016.01.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=26807544" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhtl2mur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2016&pages=43-64&author=N.+Drinkwaterauthor=N.+B.+Vinhauthor=S.+N.+Mistryauthor=R.+S.+Bamertauthor=C.+Ruggeriauthor=J.+P.+Holleranauthor=S.+Loganathanauthor=A.+Paiardiniauthor=S.+A.+Charmanauthor=A.+K.+Powellauthor=V.+M.+Averyauthor=S.+McGowanauthor=P.+J.+Scammells&title=Potent+dual+inhibitors+of+Plasmodium+falciparum+M1+and+M17+aminopeptidases+through+optimization+of+S1+pocket+interactions&doi=10.1016%2Fj.ejmech.2016.01.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Potent dual inhibitors of Plasmodium falciparum M1 and M17 aminopeptidases through optimization of S1 pocket interactions</span></div><div class="casAuthors">Drinkwater, Nyssa; Vinh, Natalie B.; Mistry, Shailesh N.; Bamert, Rebecca S.; Ruggeri, Chiara; Holleran, John P.; Loganathan, Sasdekumar; Paiardini, Alessandro; Charman, Susan A.; Powell, Andrew K.; Avery, Vicky M.; McGowan, Sheena; Scammells, Peter J.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43-64</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Malaria remains a global health problem, and though international efforts for treatment and eradication have made some headway, the emergence of drug-resistant parasites threatens this progress.  Antimalarial therapeutics acting via novel mechanisms are urgently required.  Plasmodium falciparum M1 and M17 are neutral aminopeptidases which are essential for parasite growth and development.  Previous work in our group has identified inhibitors capable of dual inhibition of PfA-M1 and PfA-M17, and revealed further regions within the protease S1 pockets that could be exploited in the development of ligands with improved inhibitory activity.  Herein, we report the structure-based design and synthesis of novel hydroxamic acid analogs that are capable of potent inhibition of both PfA-M1 and PfA-M17.  Furthermore, the developed compds. potently inhibit Pf growth in culture, including the multi-drug resistant strain Dd2.  The ongoing development of dual PfA-M1/PfA-M17 inhibitors continues to be an attractive strategy for the design of novel antimalarial therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWA9kTGB140LVg90H21EOLACvtfcHk0lh6rek4yIbRZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhtl2mur8%253D&md5=0ca2a60568b2469364c5a75070d67da2</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.01.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.01.015%26sid%3Dliteratum%253Aachs%26aulast%3DDrinkwater%26aufirst%3DN.%26aulast%3DVinh%26aufirst%3DN.%2BB.%26aulast%3DMistry%26aufirst%3DS.%2BN.%26aulast%3DBamert%26aufirst%3DR.%2BS.%26aulast%3DRuggeri%26aufirst%3DC.%26aulast%3DHolleran%26aufirst%3DJ.%2BP.%26aulast%3DLoganathan%26aufirst%3DS.%26aulast%3DPaiardini%26aufirst%3DA.%26aulast%3DCharman%26aufirst%3DS.%2BA.%26aulast%3DPowell%26aufirst%3DA.%2BK.%26aulast%3DAvery%26aufirst%3DV.%2BM.%26aulast%3DMcGowan%26aufirst%3DS.%26aulast%3DScammells%26aufirst%3DP.%2BJ.%26atitle%3DPotent%2520dual%2520inhibitors%2520of%2520Plasmodium%2520falciparum%2520M1%2520and%2520M17%2520aminopeptidases%2520through%2520optimization%2520of%2520S1%2520pocket%2520interactions%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D110%26spage%3D43%26epage%3D64%26doi%3D10.1016%2Fj.ejmech.2016.01.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kannan
Sivaraman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paiardini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sienczyk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggeri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oellig, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalton, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scammells, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGowan, S.</span></span> <span> </span><span class="NLM_article-title">Synthesis and structure-activity relationships of phosphonic arginine mimetics as inhibitors of the M1 and M17 aminopeptidases from <i>Plasmodium falciparum</i></span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">5213</span>– <span class="NLM_lpage">5217</span>, <span class="refDoi"> DOI: 10.1021/jm4005972</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4005972" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A280%3ADC%252BC3snns1yjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=5213-5217&author=K.+Kannan%0ASivaramanauthor=A.+Paiardiniauthor=M.+Sienczykauthor=C.+Ruggeriauthor=C.+A.+Oelligauthor=J.+P.+Daltonauthor=P.+J.+Scammellsauthor=M.+Dragauthor=S.+McGowan&title=Synthesis+and+structure-activity+relationships+of+phosphonic+arginine+mimetics+as+inhibitors+of+the+M1+and+M17+aminopeptidases+from+Plasmodium+falciparum&doi=10.1021%2Fjm4005972"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and structure-activity relationships of phosphonic arginine mimetics as inhibitors of the M1 and M17 aminopeptidases from Plasmodium falciparum</span></div><div class="casAuthors">Kannan Sivaraman Komagal; Paiardini Alessandro; Sienczyk Marcin; Ruggeri Chiara; Oellig Christine A; Dalton John P; Scammells Peter J; Drag Marcin; McGowan Sheena</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5213-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The malaria parasite Plasmodium falciparum employs two metallo-aminopeptidases, PfA-M1 and PfA-M17, which are essential for parasite survival.  Compounds that inhibit the activity of either enzyme represent leads for the development of new antimalarial drugs.  Here we report the synthesis and structure-activity relationships of a small library of phosphonic acid arginine mimetics that probe the S1 pocket of both enzymes and map the necessary interactions that would be important for a dual inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTyoiyTPibjo4IRAdBLKY41fW6udTcc2eYRq1xjtFlAk7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3snns1yjsw%253D%253D&md5=7a25fc0da999cc98324351269f8df212</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm4005972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4005972%26sid%3Dliteratum%253Aachs%26aulast%3DKannan%2BSivaraman%26aufirst%3DK.%26aulast%3DPaiardini%26aufirst%3DA.%26aulast%3DSienczyk%26aufirst%3DM.%26aulast%3DRuggeri%26aufirst%3DC.%26aulast%3DOellig%26aufirst%3DC.%2BA.%26aulast%3DDalton%26aufirst%3DJ.%2BP.%26aulast%3DScammells%26aufirst%3DP.%2BJ.%26aulast%3DDrag%26aufirst%3DM.%26aulast%3DMcGowan%26aufirst%3DS.%26atitle%3DSynthesis%2520and%2520structure-activity%2520relationships%2520of%2520phosphonic%2520arginine%2520mimetics%2520as%2520inhibitors%2520of%2520the%2520M1%2520and%2520M17%2520aminopeptidases%2520from%2520Plasmodium%2520falciparum%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D5213%26epage%3D5217%26doi%3D10.1021%2Fjm4005972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mistry, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drinkwater, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggeri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivaraman, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loganathan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drag, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paiardini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avery, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scammells, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGowan, S.</span></span> <span> </span><span class="NLM_article-title">Two-pronged attack: dual inhibition of <i>Plasmodium falciparum</i> M1 and M17 metalloaminopeptidases by a novel series of hydroxamic acid-based inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">9168</span>– <span class="NLM_lpage">9183</span>, <span class="refDoi"> DOI: 10.1021/jm501323a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501323a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslahs77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9168-9183&author=S.+N.+Mistryauthor=N.+Drinkwaterauthor=C.+Ruggeriauthor=K.+K.+Sivaramanauthor=S.+Loganathanauthor=S.+Fletcherauthor=M.+Dragauthor=A.+Paiardiniauthor=V.+M.+Averyauthor=P.+J.+Scammellsauthor=S.+McGowan&title=Two-pronged+attack%3A+dual+inhibition+of+Plasmodium+falciparum+M1+and+M17+metalloaminopeptidases+by+a+novel+series+of+hydroxamic+acid-based+inhibitors&doi=10.1021%2Fjm501323a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Two-Pronged Attack: Dual Inhibition of Plasmodium falciparum M1 and M17 Metalloaminopeptidases by a Novel Series of Hydroxamic Acid-Based Inhibitors</span></div><div class="casAuthors">Mistry, Shailesh N.; Drinkwater, Nyssa; Ruggeri, Chiara; Sivaraman, Komagal Kannan; Loganathan, Sasdekumar; Fletcher, Sabine; Drag, Marcin; Paiardini, Alessandro; Avery, Vicky M.; Scammells, Peter J.; McGowan, Sheena</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9168-9183</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Plasmodium parasites, the causative agents of malaria, have developed resistance to most of the current antimalarial therapies, including artemisinin combination therapies which are widely described as the last line of defense.  Antimalarial agents with a novel mode of action are urgently required.  Two Plasmodium falciparum aminopeptidases, PfA-M1 and PfA-M17, play crucial roles in the erythrocytic stage of infection and have been validated as potential antimalarial targets.  Using compd.-bound crystal structures of both enzymes, the authors have used a structure-guided approach to develop a novel series of inhibitors capable of potent inhibition of both PfA-M1 and PfA-M17 activity and parasite growth in culture.  Herein the authors describe the design, synthesis, and evaluation of a series of hydroxamic acid-based inhibitors I [R1 = PhCO, 2-FC6H4CO, 3-H2NC6H4CO, etc.; R2 = HO, HONH] and demonstrate the compds. to be exciting new leads for the development of novel antimalarial therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTJUvJd2YOGLVg90H21EOLACvtfcHk0lh_fTtYlgKyiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslahs77E&md5=6b6fe308bc3b434d686268e65014d9c1</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fjm501323a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501323a%26sid%3Dliteratum%253Aachs%26aulast%3DMistry%26aufirst%3DS.%2BN.%26aulast%3DDrinkwater%26aufirst%3DN.%26aulast%3DRuggeri%26aufirst%3DC.%26aulast%3DSivaraman%26aufirst%3DK.%2BK.%26aulast%3DLoganathan%26aufirst%3DS.%26aulast%3DFletcher%26aufirst%3DS.%26aulast%3DDrag%26aufirst%3DM.%26aulast%3DPaiardini%26aufirst%3DA.%26aulast%3DAvery%26aufirst%3DV.%2BM.%26aulast%3DScammells%26aufirst%3DP.%2BJ.%26aulast%3DMcGowan%26aufirst%3DS.%26atitle%3DTwo-pronged%2520attack%253A%2520dual%2520inhibition%2520of%2520Plasmodium%2520falciparum%2520M1%2520and%2520M17%2520metalloaminopeptidases%2520by%2520a%2520novel%2520series%2520of%2520hydroxamic%2520acid-based%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9168%26epage%3D9183%26doi%3D10.1021%2Fjm501323a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, K. M.,  Jr.</span></span> <span> </span><span class="NLM_article-title">Metal ion modeling using classical mechanics</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">1564</span>– <span class="NLM_lpage">1686</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00440</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00440" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvVSg" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2017&pages=1564-1686&author=P.+Liauthor=K.+M.+Merz&title=Metal+ion+modeling+using+classical+mechanics&doi=10.1021%2Facs.chemrev.6b00440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Metal Ion Modeling Using Classical Mechanics</span></div><div class="casAuthors">Li, Pengfei; Merz, Kenneth M., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1564-1686</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Metal ions play significant roles in numerous fields including chem., geochem., biochem. and materials science.  With computational tools increasingly becoming important in chem. research, methods have emerged to effectively face the challenge of modeling metal ions in the gas, aq. and solid phases.  Herein we review both quantum and classical modeling strategies for metal ion contg. systems that have been developed over the past few decades.  This review focuses on classical metal ion modeling based on unpolarized models (including the nonbonded, bonded, cationic dummy atom, and combined models), polarizable models (e.g., the fluctuating charge, Drude oscillator, and the induced dipole models), the angular overlap model, and valence bond based models.  Quantum mech. studies of metal ion contg. systems at the semiempirical, ab initio and d. functional levels of theory are reviewed as well with a particular focus on how these methods inform classical modeling efforts.  Finally, conclusions and future prospects and directions are offered that will further enhance the classical modeling of metal ion contg. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5FykOPt7hi7Vg90H21EOLACvtfcHk0lh_fTtYlgKyiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvVSg&md5=1cd2a84bd580b3b4e3493bfdd4bc4da1</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00440%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DP.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26atitle%3DMetal%2520ion%2520modeling%2520using%2520classical%2520mechanics%26jtitle%3DChem.%2520Rev.%26date%3D2017%26volume%3D117%26spage%3D1564%26epage%3D1686%26doi%3D10.1021%2Facs.chemrev.6b00440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porebski, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kass, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckle, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGowan, S.</span></span> <span> </span><span class="NLM_article-title">Generation of AMBER force field parameters for zinc centres of M1 and M17 family aminopeptidases</span>. <i>J. Biomol. Struct. Dyn.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_fpage">2595</span>– <span class="NLM_lpage">2604</span>, <span class="refDoi"> DOI: 10.1080/07391102.2017.1364669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1080%2F07391102.2017.1364669" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=28782419" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVans7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=2595-2604&author=W.+Yangauthor=B.+T.+Rileyauthor=X.+Leiauthor=B.+T.+Porebskiauthor=I.+Kassauthor=A.+M.+Buckleauthor=S.+McGowan&title=Generation+of+AMBER+force+field+parameters+for+zinc+centres+of+M1+and+M17+family+aminopeptidases&doi=10.1080%2F07391102.2017.1364669"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Generation of AMBER force field parameters for zinc centres of M1 and M17 family aminopeptidases</span></div><div class="casAuthors">Yang, Wei; Riley, Blake T.; Lei, Xiangyun; Porebski, Benjamin T.; Kass, Itamar; Buckle, Ashley M.; McGowan, Sheena</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Structure and Dynamics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2595-2604</span>CODEN:
                <span class="NLM_cas:coden">JBSDD6</span>;
        ISSN:<span class="NLM_cas:issn">0739-1102</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">The M1 and M17 aminopeptidases are metallo-exopeptidases that rely on the presence of divalent cations, usually zinc, in their active site for proteolytic activity.  They are from sep. protease superfamilies, however, members often have overlapping substrate specificity.  Inhibitors of one or both enzymes can be used to modulate hypertension, reduce proliferation of certain types of cancers and control malaria parasites.  Current inhibitors act to chelate the zinc ions in the active site, locking the enzymes in an inactive transition state.  We were interested in using a computational approach to understand the structure and dynamics of the M1 and M17 aminopeptidases, however, the presence of the essential metal ions in the proteases presents a challenge to classical mol. dynamics (MD) simulation.  The zinc amber force field does not contain applicable descriptions of the zinc coordination environment present in either of these two protease families.  To provide tools for the study of these two enzymes, we have used the metal center parameter builder to generate new hybrid bonded/nonbonded force field (FF) parameters to correctly describe the active site architecture for each enzyme.  The new parameters were evaluated by fitting the normal mode frequencies of mol. mechanics to the quantum mechanics frequencies and validated by performing short MD simulations.  The new FF parameters now enable more accurate and reliable MD simulations for any member of the M1 or M17 aminopeptidase superfamilies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql0MSrgIHsmLVg90H21EOLACvtfcHk0lh_fTtYlgKyiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVans7fP&md5=b47e57600f925e5bf8e9b7053743c7cc</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1080%2F07391102.2017.1364669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F07391102.2017.1364669%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DW.%26aulast%3DRiley%26aufirst%3DB.%2BT.%26aulast%3DLei%26aufirst%3DX.%26aulast%3DPorebski%26aufirst%3DB.%2BT.%26aulast%3DKass%26aufirst%3DI.%26aulast%3DBuckle%26aufirst%3DA.%2BM.%26aulast%3DMcGowan%26aufirst%3DS.%26atitle%3DGeneration%2520of%2520AMBER%2520force%2520field%2520parameters%2520for%2520zinc%2520centres%2520of%2520M1%2520and%2520M17%2520family%2520aminopeptidases%26jtitle%3DJ.%2520Biomol.%2520Struct.%2520Dyn.%26date%3D2018%26spage%3D2595%26epage%3D2604%26doi%3D10.1080%2F07391102.2017.1364669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porebski, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kass, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckle, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGowan, S.</span></span> <span> </span><span class="NLM_article-title">Mapping the pathway and dynamics of bestatin inhibition of the <i>Plasmodium falciparum</i> M1 aminopeptidase, PfA-M1</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2504</span>– <span class="NLM_lpage">2513</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201800563</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1002%2Fcmdc.201800563" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=30318749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFejt7nO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=2504-2513&author=W.+Yangauthor=B.+Rileyauthor=X.+Leiauthor=B.+T.+Porebskiauthor=I.+Kassauthor=A.+M.+Buckleauthor=S.+McGowan&title=Mapping+the+pathway+and+dynamics+of+bestatin+inhibition+of+the+Plasmodium+falciparum+M1+aminopeptidase%2C+PfA-M1&doi=10.1002%2Fcmdc.201800563"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Mapping the Pathway and Dynamics of Bestatin Inhibition of the Plasmodium falciparum M1 Aminopeptidase PfA-M1</span></div><div class="casAuthors">Yang, Wei; Riley, Blake T.; Lei, Xiangyun; Porebski, Benjamin T.; Kass, Itamar; Buckle, Ashley M.; McGowan, Sheena</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2504-2513</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The M1 metallo-aminopeptidase from Plasmodium falciparum, PfA-M1, is an attractive drug target for the design of new antimalarials.  Bestatin, a broad-spectrum metalloprotease inhibitor, is a moderate inhibitor of PfA-M1, and has been used to provide structure-activity relationships to inform drug design.  The crystal structure of PfA-M1 with bestatin bound within its active site has been detd.; however, dynamics of the inhibitor and the assocn. or dissocn. pathway have yet to be characterized.  Here we present an all-atom mol. dynamics study where we have generated a hidden Markov state model from 2.3 μs of mol. dynamics simulation.  Our hidden Markov state model identifies five macrostates that clearly show the events involved in bestatin dissocn. from the PfA-M1 active site.  The results show for the first time that bestatin can escape the substrate specificity pockets of the enzyme, primarily due to weak interactions within the pockets.  Our approach identifies relevant conformational sampling of the inhibitor inside the enzyme and the protein dynamics that could be exploited to produce potent and selective inhibitors that can differentiate between similar members of the M1 aminopeptidase superfamily.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpM9tv3inDxhrVg90H21EOLACvtfcHk0lh_fTtYlgKyiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFejt7nO&md5=b9c9757cb142807f3dc986123f71bbd9</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201800563&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201800563%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DW.%26aulast%3DRiley%26aufirst%3DB.%26aulast%3DLei%26aufirst%3DX.%26aulast%3DPorebski%26aufirst%3DB.%2BT.%26aulast%3DKass%26aufirst%3DI.%26aulast%3DBuckle%26aufirst%3DA.%2BM.%26aulast%3DMcGowan%26aufirst%3DS.%26atitle%3DMapping%2520the%2520pathway%2520and%2520dynamics%2520of%2520bestatin%2520inhibition%2520of%2520the%2520Plasmodium%2520falciparum%2520M1%2520aminopeptidase%252C%2520PfA-M1%26jtitle%3DChemMedChem%26date%3D2018%26volume%3D13%26spage%3D2504%26epage%3D2513%26doi%3D10.1002%2Fcmdc.201800563" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beebe, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilius, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merta, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soni, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beutel, B. A.</span></span> <span> </span><span class="NLM_article-title">Synthesis and SAR evaluation of oxadiazolopyrazines as selective Haemophilus influenzae antibacterial agents</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">3133</span>– <span class="NLM_lpage">3136</span>, <span class="refDoi"> DOI: 10.1016/S0960-894X(03)00727-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2FS0960-894X%2803%2900727-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=12951079" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvV2lsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2003&pages=3133-3136&author=X.+Beebeauthor=A.+M.+Niliusauthor=P.+J.+Mertaauthor=N.+B.+Soniauthor=M.+H.+Buiauthor=R.+Wagnerauthor=B.+A.+Beutel&title=Synthesis+and+SAR+evaluation+of+oxadiazolopyrazines+as+selective+Haemophilus+influenzae+antibacterial+agents&doi=10.1016%2FS0960-894X%2803%2900727-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and SAR evaluation of oxadiazolopyrazines as selective Haemophilus influenzae antibacterial agents</span></div><div class="casAuthors">Beebe, Xenia; Nilius, Angela M.; Merta, Philip J.; Soni, Niru B.; Bui, Mai H.; Wagner, Rolf; Beutel, Bruce A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3133-3136</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">The parallel synthesis and antibacterial activity of 5-hydroxy-1,2,5-oxadiazolo[3,4-b]pyrazines, e.g., I, is reported.  The compds. were synthesized by condensing diaminofurazan (II) with α-keto acids to give a variety of aryl-substituted analogs.  Halogenated Ph groups at C-6 give rise to the greatest Haemophilus influenzae antibacterial activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXyJKZ9hzxF7Vg90H21EOLACvtfcHk0lhiDhfJueZclA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvV2lsLk%253D&md5=a64b56982c86d0e7a16897346c203a03</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2FS0960-894X%2803%2900727-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-894X%252803%252900727-3%26sid%3Dliteratum%253Aachs%26aulast%3DBeebe%26aufirst%3DX.%26aulast%3DNilius%26aufirst%3DA.%2BM.%26aulast%3DMerta%26aufirst%3DP.%2BJ.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DWagner%26aufirst%3DR.%26aulast%3DBeutel%26aufirst%3DB.%2BA.%26atitle%3DSynthesis%2520and%2520SAR%2520evaluation%2520of%2520oxadiazolopyrazines%2520as%2520selective%2520Haemophilus%2520influenzae%2520antibacterial%2520agents%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2003%26volume%3D13%26spage%3D3133%26epage%3D3136%26doi%3D10.1016%2FS0960-894X%2803%2900727-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adediran, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabaret, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohier, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakselman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, R. F.</span></span> <span> </span><span class="NLM_article-title">Substituted aryl malonamates as new serine beta-lactamase substrates: structure-activity studies</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">282</span>– <span class="NLM_lpage">291</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2009.10.056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.bmc.2009.10.056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=19932622" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFykurvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=282-291&author=S.+A.+Adediranauthor=D.+Cabaretauthor=J.+F.+Lohierauthor=M.+Wakselmanauthor=R.+F.+Pratt&title=Substituted+aryl+malonamates+as+new+serine+beta-lactamase+substrates%3A+structure-activity+studies&doi=10.1016%2Fj.bmc.2009.10.056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Substituted aryl malonamates as new serine β-lactamase substrates: Structure-activity studies</span></div><div class="casAuthors">Adediran, S. A.; Cabaret, D.; Lohier, J.-F.; Wakselman, M.; Pratt, R. F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">282-291</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of substituted aryl malonamates have been prepd.  These compds. are analogs of aryl phenaceturates where the amido side chain has been replaced by a retro-amide.  Like the phenaceturates, these compds. are substrates of typical class A and class C β-lactamases, particularly of the latter, and of sol. DD-peptidases.  The effect of substituents α to the ester carbonyl group on turnover by these enzymes is similar to that in the phenaceturates.  On the other hand, N-alkylation of the side chain amide of malonamates, but not of phenaceturates, retains the susceptibility of the compds. to hydrolysis by β-lactamases.  This reactivity is not enhanced, however, by bridging the amide nitrogen and Cα atoms.  A phosphonate analog of the malonamates was found to be an irreversible inhibitor of the β-lactamases.  These results, therefore, provide further evidence for the covalent access of compds. bearing retro-amide side chains to the active sites of β-lactam-recognizing enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMbLiZVPEKNbVg90H21EOLACvtfcHk0lhiDhfJueZclA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFykurvK&md5=82f87374778b0963de10dec3d4d0bcdc</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2009.10.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2009.10.056%26sid%3Dliteratum%253Aachs%26aulast%3DAdediran%26aufirst%3DS.%2BA.%26aulast%3DCabaret%26aufirst%3DD.%26aulast%3DLohier%26aufirst%3DJ.%2BF.%26aulast%3DWakselman%26aufirst%3DM.%26aulast%3DPratt%26aufirst%3DR.%2BF.%26atitle%3DSubstituted%2520aryl%2520malonamates%2520as%2520new%2520serine%2520beta-lactamase%2520substrates%253A%2520structure-activity%2520studies%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2010%26volume%3D18%26spage%3D282%26epage%3D291%26doi%3D10.1016%2Fj.bmc.2009.10.056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Im, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. C.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationships of heteroaromatic esters as human rhinovirus 3C protease inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">3632</span>– <span class="NLM_lpage">3636</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2009.04.114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.bmcl.2009.04.114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=19464175" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmvFGisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=3632-3636&author=I.+Imauthor=E.+S.+Leeauthor=S.+J.+Choiauthor=J.+Y.+Leeauthor=Y.+C.+Kim&title=Structure-activity+relationships+of+heteroaromatic+esters+as+human+rhinovirus+3C+protease+inhibitors&doi=10.1016%2Fj.bmcl.2009.04.114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity relationships of heteroaromatic esters as human rhinovirus 3C protease inhibitors</span></div><div class="casAuthors">Im, Isak; Lee, Eui Seung; Choi, Soo Jeong; Lee, Ju-Yeon; Kim, Yong-Chul</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3632-3636</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Human rhinovirus 3C protease (HRV 3Cpro) is known to be a promising target for development of therapeutic agents against the common cold because of the importance of the protease in viral replication as well as its expression in a large no. of serotypes.  To explore non-peptidic inhibitors of HRV 3Cpro, a series of novel heteroarom. esters was synthesized and evaluated for inhibitory activity against HRV 3Cpro, to det. the structure-activity relationships.  The most potent inhibitor, 7, with a 5-bromopyridinyl group, had an IC50 value of 80 nM.  In addn., the binding mode of a novel analog, 19 (I), with the 4-hydroxyquinolinone moiety, was explored by mol. docking, suggesting a new interaction in the S1 pocket.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4DsOHx3pVs7Vg90H21EOLACvtfcHk0lhiDhfJueZclA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmvFGisbk%253D&md5=60df4e967a2ee30e11661a0990e62825</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.04.114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.04.114%26sid%3Dliteratum%253Aachs%26aulast%3DIm%26aufirst%3DI.%26aulast%3DLee%26aufirst%3DE.%2BS.%26aulast%3DChoi%26aufirst%3DS.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DKim%26aufirst%3DY.%2BC.%26atitle%3DStructure-activity%2520relationships%2520of%2520heteroaromatic%2520esters%2520as%2520human%2520rhinovirus%25203C%2520protease%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D3632%26epage%3D3636%26doi%3D10.1016%2Fj.bmcl.2009.04.114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kam, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhee, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Back, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Na, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choo, H. Y.</span></span> <span> </span><span class="NLM_article-title">Synthesis and bradykinin inhibitory activity of novel non-peptide compounds, and evaluation of in vivo analgesic activity</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2327</span>– <span class="NLM_lpage">2336</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2010.01.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.bmc.2010.01.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=20189403" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtVCrtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=2327-2336&author=Y.+L.+Kamauthor=H.+K.+Rheeauthor=H.+J.+Kimauthor=S.+K.+Backauthor=H.+S.+Naauthor=H.+Y.+Choo&title=Synthesis+and+bradykinin+inhibitory+activity+of+novel+non-peptide+compounds%2C+and+evaluation+of+in+vivo+analgesic+activity&doi=10.1016%2Fj.bmc.2010.01.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and bradykinin inhibitory activity of novel non-peptide compounds, and evaluation of in vivo analgesic activity</span></div><div class="casAuthors">Kam, Yoo Lim; Rhee, Hee-Kyung; Kim, Hwa-Jung; Back, Seung Keun; Na, Heung Sik; Choo, Hea-Young Park</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2327-2336</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of novel non-peptide diamide compds. was synthesized and evaluated as antibradykinin agents by utilizing guinea-pig ileum smooth muscle.  Among the final compds., (Z)-4-(4-(bis(4-fluorophenyl)methyl)piperazin-1-yl)-4-oxo-N-(4-phenylbutan-2-yl)but-2-enamide I showed most favorable bradykinin inhibitory activity and demonstrated analgesic efficacies in the rat models of inflammatory and neuropathic pain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzrCYHX2555bVg90H21EOLACvtfcHk0lg3Xt6ErHEc_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtVCrtrc%253D&md5=1b64a06ed2b8a5108d00fa82388b94ee</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2010.01.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2010.01.050%26sid%3Dliteratum%253Aachs%26aulast%3DKam%26aufirst%3DY.%2BL.%26aulast%3DRhee%26aufirst%3DH.%2BK.%26aulast%3DKim%26aufirst%3DH.%2BJ.%26aulast%3DBack%26aufirst%3DS.%2BK.%26aulast%3DNa%26aufirst%3DH.%2BS.%26aulast%3DChoo%26aufirst%3DH.%2BY.%26atitle%3DSynthesis%2520and%2520bradykinin%2520inhibitory%2520activity%2520of%2520novel%2520non-peptide%2520compounds%252C%2520and%2520evaluation%2520of%2520in%2520vivo%2520analgesic%2520activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2010%26volume%3D18%26spage%3D2327%26epage%3D2336%26doi%3D10.1016%2Fj.bmc.2010.01.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williams, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zammit, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shackleford, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morizzi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krum, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, D. J.</span></span> <span> </span><span class="NLM_article-title">3′,4′-Bis-difluoromethoxycinnamoylanthranilate (FT061): an orally-active antifibrotic agent that reduces albuminuria in a rat model of progressive diabetic nephropathy</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">6868</span>– <span class="NLM_lpage">6873</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.09.100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.bmcl.2013.09.100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=24169234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1yqurrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=6868-6873&author=S.+J.+Williamsauthor=S.+C.+Zammitauthor=A.+J.+Coxauthor=D.+M.+Shacklefordauthor=J.+Morizziauthor=Y.+Zhangauthor=A.+K.+Powellauthor=R.+E.+Gilbertauthor=H.+Krumauthor=D.+J.+Kelly&title=3%E2%80%B2%2C4%E2%80%B2-Bis-difluoromethoxycinnamoylanthranilate+%28FT061%29%3A+an+orally-active+antifibrotic+agent+that+reduces+albuminuria+in+a+rat+model+of+progressive+diabetic+nephropathy&doi=10.1016%2Fj.bmcl.2013.09.100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">3',4'-Bis-difluoromethoxycinnamoylanthranilate (FT061): An orally-active antifibrotic agent that reduces albuminuria in a rat model of progressive diabetic nephropathy</span></div><div class="casAuthors">Williams, Spencer J.; Zammit, Steven C.; Cox, Alison J.; Shackleford, David M.; Morizzi, Julia; Zhang, Yuan; Powell, Andrew K.; Gilbert, Richard E.; Krum, Henry; Kelly, Darren J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">6868-6873</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cinnamoylanthranilates including tranilast have been identified as promising antifibrotics that can reduce fibrosis occurring in the kidney during diabetes, thereby delaying and/or preventing kidney dysfunction.  Structure-activity relationships aimed at improving potency and metabolic stability have led to the discovery of FT061.  This compd., which bears a bis-difluoromethoxy catechol, attenuates TGF-β-stimulated prodn. of collagen in cultured renal mesangial cells (approx 50% at 3 μM).  When dosed orally at 20 mg/kg to male Sprague Dawley rats, FT061 exhibited a high bioavailability (73%), Cmax of 200 μM and Tmax of 150 min, and a half-life of 5.4 h.  FT061 reduced albuminuria when orally dosed in rats at 200 mg kg/day in a late intervention study of a rat model of progressive diabetic nephropathy.  The prepn. of cinnamoylanthranilates is also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGog6CM_Mf_lprVg90H21EOLACvtfcHk0lg3Xt6ErHEc_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1yqurrN&md5=2f74b5facaf9c7c186e18ce7bac5dfbb</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.09.100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.09.100%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DS.%2BJ.%26aulast%3DZammit%26aufirst%3DS.%2BC.%26aulast%3DCox%26aufirst%3DA.%2BJ.%26aulast%3DShackleford%26aufirst%3DD.%2BM.%26aulast%3DMorizzi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DPowell%26aufirst%3DA.%2BK.%26aulast%3DGilbert%26aufirst%3DR.%2BE.%26aulast%3DKrum%26aufirst%3DH.%26aulast%3DKelly%26aufirst%3DD.%2BJ.%26atitle%3D3%25E2%2580%25B2%252C4%25E2%2580%25B2-Bis-difluoromethoxycinnamoylanthranilate%2520%2528FT061%2529%253A%2520an%2520orally-active%2520antifibrotic%2520agent%2520that%2520reduces%2520albuminuria%2520in%2520a%2520rat%2520model%2520of%2520progressive%2520diabetic%2520nephropathy%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D6868%26epage%3D6873%26doi%3D10.1016%2Fj.bmcl.2013.09.100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Butters, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill-Cousins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kariuki, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ooi, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wordingham, S. V.</span></span> <span> </span><span class="NLM_article-title">Synthesis and stereochemical determination of batzelladine C methyl ester</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">5001</span>– <span class="NLM_lpage">5009</span>, <span class="refDoi"> DOI: 10.1039/b914744f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1039%2Fb914744f" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=19907792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVSgtbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2009&pages=5001-5009&author=M.+Buttersauthor=C.+D.+Daviesauthor=M.+C.+Elliottauthor=J.+Hill-Cousinsauthor=B.+M.+Kariukiauthor=L.+L.+Ooiauthor=J.+L.+Woodauthor=S.+V.+Wordingham&title=Synthesis+and+stereochemical+determination+of+batzelladine+C+methyl+ester&doi=10.1039%2Fb914744f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and stereochemical determination of batzelladine C methyl ester</span></div><div class="casAuthors">Butters, Michael; Davies, Christopher D.; Elliott, Mark C.; Hill-Cousins, Joseph; Kariuki, Benson M.; Ooi, Li-ling; Wood, John L.; Wordingham, Stuart V.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5001-5009</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Batzelladine C (I) is a tricyclic guanidine alkaloid of unknown stereochem. at one center as well as unknown abs. stereochem.  The two possible diastereoisomers of the Me ester corresponding to this compd. have been synthesized, permitting the relative and abs. stereochem. of this compd. to be assigned.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA38AGKZAuULVg90H21EOLACvtfcHk0lg3Xt6ErHEc_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVSgtbbI&md5=292546c0516a3ed7053d859e9322a90d</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1039%2Fb914744f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb914744f%26sid%3Dliteratum%253Aachs%26aulast%3DButters%26aufirst%3DM.%26aulast%3DDavies%26aufirst%3DC.%2BD.%26aulast%3DElliott%26aufirst%3DM.%2BC.%26aulast%3DHill-Cousins%26aufirst%3DJ.%26aulast%3DKariuki%26aufirst%3DB.%2BM.%26aulast%3DOoi%26aufirst%3DL.%2BL.%26aulast%3DWood%26aufirst%3DJ.%2BL.%26aulast%3DWordingham%26aufirst%3DS.%2BV.%26atitle%3DSynthesis%2520and%2520stereochemical%2520determination%2520of%2520batzelladine%2520C%2520methyl%2520ester%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2009%26volume%3D7%26spage%3D5001%26epage%3D5009%26doi%3D10.1039%2Fb914744f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, H. J.</span></span> <span> </span><span class="NLM_article-title">Rational design of an indolebutanoic acid derivative as a novel aldose reductase inhibitor based on docking and 3D QSAR studies of phenethylamine derivatives</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">5619</span>– <span class="NLM_lpage">5627</span>, <span class="refDoi"> DOI: 10.1021/jm0205346</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0205346" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD3sXptlOhsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=5619-5627&author=W.+S.+Sunauthor=Y.+S.+Parkauthor=J.+Yooauthor=K.+D.+Parkauthor=S.+H.+Kimauthor=J.+H.+Kimauthor=H.+J.+Park&title=Rational+design+of+an+indolebutanoic+acid+derivative+as+a+novel+aldose+reductase+inhibitor+based+on+docking+and+3D+QSAR+studies+of+phenethylamine+derivatives&doi=10.1021%2Fjm0205346"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design of an Indolebutanoic Acid Derivative as a Novel Aldose Reductase Inhibitor Based on Docking and 3D QSAR Studies of Phenethylamine Derivatives</span></div><div class="casAuthors">Sun, Won Suck; Park, Yoon Sun; Yoo, Jakyung; Park, Ki Duk; Kim, Sung Han; Kim, Jung-Han; Park, Hyun-Ju</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">5619-5627</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of 45 phenethylamine derivs. were synthesized and evaluated for their inhibitory activity against pig kidney aldose reductase (ALR2, EC 1.1.1.21).  Their IC50 values ranged from 400 μM to 24 μM.  The binding modes of compds. at the active site of ALR2 were examd. using flexible docking.  The results indicated that phenethylamine derivs. nicely fit into the active pocket of ALR2 by forming various hydrogen bonding and hydrophobic interactions.  3D-QSAR anal. was also conducted using FlexX-docked alignment of the compds.  The best prediction was obtained by CoMSIA combined with hydrophobic and hydrogen bond donor/acceptor field (q2 = 0.557, r2 = 0.934).  A new deriv., 4-oxo-4-(4-hydroxyindole)butanoic acid, was designed, taking into account the CoMSIA field and the binding mode derived by FlexX docking.  This rationally designed compd. exhibits an ALR2 inhibition with an IC50 value of 7.4 μM, which compares favorably to that of a well-known ALR2 inhibitor, tolrestat (IC50 = 16 μM) and represents a potency approx. 240-fold higher than that of an original phenethylamine lead compd., YUA001.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvwAtyhojGd7Vg90H21EOLACvtfcHk0liPwZ6DynvgPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXptlOhsro%253D&md5=1439f168344cc235b2cb3225cacdc9ca</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fjm0205346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0205346%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DW.%2BS.%26aulast%3DPark%26aufirst%3DY.%2BS.%26aulast%3DYoo%26aufirst%3DJ.%26aulast%3DPark%26aufirst%3DK.%2BD.%26aulast%3DKim%26aufirst%3DS.%2BH.%26aulast%3DKim%26aufirst%3DJ.%2BH.%26aulast%3DPark%26aufirst%3DH.%2BJ.%26atitle%3DRational%2520design%2520of%2520an%2520indolebutanoic%2520acid%2520derivative%2520as%2520a%2520novel%2520aldose%2520reductase%2520inhibitor%2520based%2520on%2520docking%2520and%25203D%2520QSAR%2520studies%2520of%2520phenethylamine%2520derivatives%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D5619%26epage%3D5627%26doi%3D10.1021%2Fjm0205346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sadovoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">K, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golubeva, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sviridova, L.</span></span> <span> </span><span class="NLM_article-title">Regioselective synthesis of 1-alkyl-5-(indol-3-yl- and -2-yl)pyrrolidin-2-ones from available reagents</span>. <i>Chem. Heterocycl. Compd.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1215</span>– <span class="NLM_lpage">1223</span>, <span class="refDoi"> DOI: 10.1007/s10593-011-0655-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1007%2Fs10593-011-0655-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisVajsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2011&pages=1215-1223&author=A.+Sadovoyauthor=A.+Kauthor=G.+Golubevaauthor=L.+Sviridova&title=Regioselective+synthesis+of+1-alkyl-5-%28indol-3-yl-+and+-2-yl%29pyrrolidin-2-ones+from+available+reagents&doi=10.1007%2Fs10593-011-0655-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Regioselective synthesis of 1-alkyl-5-(indol-3-yl- and -2-yl)pyrrolidin-2-ones from available reagents</span></div><div class="casAuthors">Sadovoy, A. V.; Kovrov, A. E.; Golubeva, G. A.; Sviridova, L. A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry of Heterocyclic Compounds (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1215-1223</span>CODEN:
                <span class="NLM_cas:coden">CHCCAL</span>;
        ISSN:<span class="NLM_cas:issn">0009-3122</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">The reaction under mild conditions of 1-alkyl-5-hydroxypyrrolidin-2-ones with different indoles having a free 3 position leads exclusively to 1-alkyl-5-(indol-3-yl)pyrrolidin-2-ones but if position 3 is occupied to 1-alkyl-5-(indol-2-yl)pyrrolidin-2-ones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqd1MPCXnyqxrVg90H21EOLACvtfcHk0liPwZ6DynvgPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisVajsbo%253D&md5=87ab5cb35d35413ca21abbbe57d62aa3</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1007%2Fs10593-011-0655-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10593-011-0655-x%26sid%3Dliteratum%253Aachs%26aulast%3DSadovoy%26aufirst%3DA.%26aulast%3DK%26aufirst%3DA.%26aulast%3DGolubeva%26aufirst%3DG.%26aulast%3DSviridova%26aufirst%3DL.%26atitle%3DRegioselective%2520synthesis%2520of%25201-alkyl-5-%2528indol-3-yl-%2520and%2520-2-yl%2529pyrrolidin-2-ones%2520from%2520available%2520reagents%26jtitle%3DChem.%2520Heterocycl.%2520Compd.%26date%3D2011%26volume%3D46%26spage%3D1215%26epage%3D1223%26doi%3D10.1007%2Fs10593-011-0655-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romeo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salerno, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittala, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modica, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siracusa, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Materia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buccioni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marucci, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minneman, K. P.</span></span> <span> </span><span class="NLM_article-title">High affinity ligands and potent antagonists for the alpha1D-adrenergic receptor. Novel 3,8-disubstituted [1]benzothieno[3,2-d]pyrimidine derivatives</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">419</span>– <span class="NLM_lpage">432</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2014.06.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2Fj.ejmech.2014.06.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=24992070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1aks7vI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2014&pages=419-432&author=G.+Romeoauthor=L.+Salernoauthor=V.+Pittalaauthor=M.+N.+Modicaauthor=M.+A.+Siracusaauthor=L.+Materiaauthor=M.+Buccioniauthor=G.+Marucciauthor=K.+P.+Minneman&title=High+affinity+ligands+and+potent+antagonists+for+the+alpha1D-adrenergic+receptor.+Novel+3%2C8-disubstituted+%5B1%5Dbenzothieno%5B3%2C2-d%5Dpyrimidine+derivatives&doi=10.1016%2Fj.ejmech.2014.06.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">High affinity ligands and potent antagonists for the α1D-adrenergic receptor. Novel 3,8-disubstituted [1]benzothieno[3,2-d]pyrimidine derivatives</span></div><div class="casAuthors">Romeo, Giuseppe; Salerno, Loredana; Pittala, Valeria; Modica, Maria N.; Siracusa, Maria A.; Materia, Luisa; Buccioni, Michela; Marucci, Gabriella; Minneman, Kenneth P.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">419-432</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A new series of high affinity ligands and antagonists for the α1D-adrenergic receptor (AR) has been discovered.  New mols. present a [1]benzothieno[3,2-d]pyrimidin-2,4(1H,3H)-dione or a [1]benzothieno[3,2-d]pyrimidin-4(3H)-one scaffold and bear a 2-[4-(2-methoxyphenyl)piperazin-1-yl]ethyl moiety in the 3-position and various amide substituents in the 8-position.  In binding assays at the three human cloned α1A-, α1B-, and α1D-AR subtypes, they showed high affinity values, particularly for the α1D-AR subtype.  Compd. 22 (RX18), N1-methyl-N5-[3-[2-[4-(2-methoxyphenyl)piperazin-1-yl]ethyl]-2,4-dioxo-1,2,3,4-tetrahydro[1]benzothieno[3,2-d]pyrimidin-8-yl]-N1-(phenylmethyl)pentanediamide, was the most interesting in the series displaying very high affinity (pKi = 10.25) and potent antagonism (pKb = 9.15) when tested in a functional assay at the α1D-AR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB0qdtVlxDw7Vg90H21EOLACvtfcHk0liPwZ6DynvgPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1aks7vI&md5=9bf0c8a151b4f424fa07515b1ea5a158</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2014.06.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2014.06.057%26sid%3Dliteratum%253Aachs%26aulast%3DRomeo%26aufirst%3DG.%26aulast%3DSalerno%26aufirst%3DL.%26aulast%3DPittala%26aufirst%3DV.%26aulast%3DModica%26aufirst%3DM.%2BN.%26aulast%3DSiracusa%26aufirst%3DM.%2BA.%26aulast%3DMateria%26aufirst%3DL.%26aulast%3DBuccioni%26aufirst%3DM.%26aulast%3DMarucci%26aufirst%3DG.%26aulast%3DMinneman%26aufirst%3DK.%2BP.%26atitle%3DHigh%2520affinity%2520ligands%2520and%2520potent%2520antagonists%2520for%2520the%2520alpha1D-adrenergic%2520receptor.%2520Novel%25203%252C8-disubstituted%2520%255B1%255Dbenzothieno%255B3%252C2-d%255Dpyrimidine%2520derivatives%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2014%26volume%3D83%26spage%3D419%26epage%3D432%26doi%3D10.1016%2Fj.ejmech.2014.06.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cartier, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zentz, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labia, R.</span></span> <span> </span><span class="NLM_article-title">Atypical oxidation reaction by thionyl chloride: easy two-step synthesis of N-Alkyl-1,4-dithiines</span>. <i>Synth. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">3591</span>– <span class="NLM_lpage">3597</span>, <span class="refDoi"> DOI: 10.1080/00397910600943600</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1080%2F00397910600943600" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmslSisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2006&pages=3591-3597&author=A.+Vallaauthor=D.+Cartierauthor=F.+Zentzauthor=R.+Labia&title=Atypical+oxidation+reaction+by+thionyl+chloride%3A+easy+two-step+synthesis+of+N-Alkyl-1%2C4-dithiines&doi=10.1080%2F00397910600943600"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Atypical oxidation reaction by thionyl chloride. Easy two-step synthesis of N-alkyl-1,4-dithiines</span></div><div class="casAuthors">Valla, Alain; Cartier, Dominique; Zentz, Frederic; Labia, Roger</div><div class="citationInfo"><span class="NLM_cas:title">Synthetic Communications</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3591-3597</span>CODEN:
                <span class="NLM_cas:coden">SYNCAV</span>;
        ISSN:<span class="NLM_cas:issn">0039-7911</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">Easy two-step synthesis of a series of dithiines was performed from succinic anhydride via cyclization of the corresponding 4-(alkylamino)-4-oxobutanoic acids (succinamic acids).  The reaction, carried out in polar aprotic solvents, gave 4,8-dithiine-indacene-1,3,5,7-tetraones (diimides) via 3,7-bis-4,8-dithia-indacene-1,5-diones (diisoimides), which could be isolated.  Surprisingly, in this reaction, thionyl chloride appeared as an oxidant, and this process seemed to be useful for the syntheses of S-contg. heterocyclic compds. such as 1,4-dithiins.  A mechanistic pathway was considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpI0tJ8CMCcg7Vg90H21EOLACvtfcHk0liBn1V9QKQ5TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmslSisw%253D%253D&md5=27ab4d2f5dd78d6f1e93f189af1513e6</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1080%2F00397910600943600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00397910600943600%26sid%3Dliteratum%253Aachs%26aulast%3DValla%26aufirst%3DA.%26aulast%3DCartier%26aufirst%3DD.%26aulast%3DZentz%26aufirst%3DF.%26aulast%3DLabia%26aufirst%3DR.%26atitle%3DAtypical%2520oxidation%2520reaction%2520by%2520thionyl%2520chloride%253A%2520easy%2520two-step%2520synthesis%2520of%2520N-Alkyl-1%252C4-dithiines%26jtitle%3DSynth.%2520Commun.%26date%3D2006%26volume%3D36%26spage%3D3591%26epage%3D3597%26doi%3D10.1080%2F00397910600943600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arulsamy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohle, D. S.</span></span> <span> </span><span class="NLM_article-title">Nucleophilic addition of hydroxylamine, methoxylamine, and hydrazine to malononitrileoxime</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1139</span>– <span class="NLM_lpage">1143</span>, <span class="refDoi"> DOI: 10.1021/jo991614t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo991614t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD3cXms1Wlsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2000&pages=1139-1143&author=N.+Arulsamyauthor=D.+S.+Bohle&title=Nucleophilic+addition+of+hydroxylamine%2C+methoxylamine%2C+and+hydrazine+to+malononitrileoxime&doi=10.1021%2Fjo991614t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Nucleophilic addition of hydroxylamine, methoxylamine, and hydrazine to malononitrile oxime</span></div><div class="casAuthors">Arulsamy, Navamoney; Bohle, D. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1139-1143</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The chem. of malononitrile oxime, HONC(CN)2, with respect to nucleophilic addn. to ammonia, methylamine, hydroxylamine, methoxylamine, and hydrazine is reported.  Whereas the poorly nucleophilic ammonia and methylamine do not react, hydroxylamine, methoxylamine, and hydrazine add to the nitrile groups of the oxime, yielding the corresponding amidoximes and amidrazones.  Depending on the stoichiometry of the reactions, hydroxylamine and hydrazine add to one or both of the nitrile groups; methoxylamine adds to only one of the nitrile groups.  Three of the products, namely, cyanoacetamidoxime, 3-amino-2,3-hydroxyiminopropionitrile monohydrate, and 3,5-diaminopyrazolone-4-oxime monohydrochloride monohydrate, are characterized by single-crystal X-ray diffraction data.  All of the products exhibit exothermic decompn. properties with heats of decompn. in the range of 500-1500 kJ mol-1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS2ucQK5Fc77Vg90H21EOLACvtfcHk0liBn1V9QKQ5TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXms1Wlsg%253D%253D&md5=4328f83e63ef876c81fe8ff254baf813</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fjo991614t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo991614t%26sid%3Dliteratum%253Aachs%26aulast%3DArulsamy%26aufirst%3DN.%26aulast%3DBohle%26aufirst%3DD.%2BS.%26atitle%3DNucleophilic%2520addition%2520of%2520hydroxylamine%252C%2520methoxylamine%252C%2520and%2520hydrazine%2520to%2520malononitrileoxime%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2000%26volume%3D65%26spage%3D1139%26epage%3D1143%26doi%3D10.1021%2Fjo991614t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vörös, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mucsi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baán, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timári, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermecz, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizsey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finta, Z.</span></span> <span> </span><span class="NLM_article-title">An experimental and theoretical study of reaction mechanisms between nitriles and hydroxylamine</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">8036</span>– <span class="NLM_lpage">8047</span>, <span class="refDoi"> DOI: 10.1039/C4OB00854E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1039%2FC4OB00854E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=25185027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1KhtLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=8036-8047&author=A.+V%C3%B6r%C3%B6sauthor=Z.+Mucsiauthor=Z.+Ba%C3%A1nauthor=G.+Tim%C3%A1riauthor=I.+Hermeczauthor=P.+Mizseyauthor=Z.+Finta&title=An+experimental+and+theoretical+study+of+reaction+mechanisms+between+nitriles+and+hydroxylamine&doi=10.1039%2FC4OB00854E"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">An experimental and theoretical study of reaction mechanisms between nitriles and hydroxylamine</span></div><div class="casAuthors">Voros, Attila; Mucsi, Zoltan; Baan, Zoltan; Timari, Geza; Hermecz, Istvan; Mizsey, Peter; Finta, Zoltan</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">8036-8047</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The industrially relevant reaction between nitriles and hydroxylamine yielding amidoximes was studied in different mol. solvents and in ionic liqs.  In industry, this procedure is carried out on the ton scale in alc. solns. and the above transformation produces a significant amt. of unexpected amide byproduct, depending on the nature of the nitrile, which can cause further anal. and purifn. issues.  Although there were earlier attempts to propose mechanisms for this transformation, the real reaction pathway is still under discussion.  A new detailed reaction mechanistic explanation, based on theor. and exptl. proof, is given to augment the former mechanisms, which allowed us to find a more efficient, side-product free procedure.  Interpreting the theor. results obtained, it was shown that the application of specific imidazolium, phosphonium and quaternary ammonium based ionic liqs. could decrease simultaneously the reaction time while eliminating the amide side-product, leading to the targeted product selectively.  This robust and economic procedure now affords a fast, selective amide free synthesis of amidoximes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ0NtTmLTvULVg90H21EOLACvtfcHk0liBn1V9QKQ5TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1KhtLfO&md5=27181cd022238675fb7728fc6bd84002</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1039%2FC4OB00854E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4OB00854E%26sid%3Dliteratum%253Aachs%26aulast%3DV%25C3%25B6r%25C3%25B6s%26aufirst%3DA.%26aulast%3DMucsi%26aufirst%3DZ.%26aulast%3DBa%25C3%25A1n%26aufirst%3DZ.%26aulast%3DTim%25C3%25A1ri%26aufirst%3DG.%26aulast%3DHermecz%26aufirst%3DI.%26aulast%3DMizsey%26aufirst%3DP.%26aulast%3DFinta%26aufirst%3DZ.%26atitle%3DAn%2520experimental%2520and%2520theoretical%2520study%2520of%2520reaction%2520mechanisms%2520between%2520nitriles%2520and%2520hydroxylamine%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2014%26volume%3D12%26spage%3D8036%26epage%3D8047%26doi%3D10.1039%2FC4OB00854E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luciani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marie-Claire, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruffet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaumont, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roques, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fournie-Zaluski, M. C.</span></span> <span> </span><span class="NLM_article-title">Characterization of Glu350 as a critical residue involved in the N-terminal amine binding site of aminopeptidase N (EC 3.4.11.2): insights into its mechanism of action</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">686</span>– <span class="NLM_lpage">692</span>, <span class="refDoi"> DOI: 10.1021/bi971705p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi971705p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaK2sXotVKis7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1998&pages=686-692&author=N.+Lucianiauthor=C.+Marie-Claireauthor=E.+Ruffetauthor=A.+Beaumontauthor=B.+P.+Roquesauthor=M.+C.+Fournie-Zaluski&title=Characterization+of+Glu350+as+a+critical+residue+involved+in+the+N-terminal+amine+binding+site+of+aminopeptidase+N+%28EC+3.4.11.2%29%3A+insights+into+its+mechanism+of+action&doi=10.1021%2Fbi971705p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of Glu350 as a Critical Residue Involved in the N-Terminal Amine Binding Site of Aminopeptidase N (EC 3.4.11.2): Insights into Its Mechanism of Action</span></div><div class="casAuthors">Luciani, Nathalie; Marie-Claire, Cynthia; Ruffet, Emmanuel; Beaumont, Ann; Roques, Bernard P.; Fournie-Zaluski, Marie-Claude</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">686-692</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The mol. components ensuring the strict exopeptidase action of aminopeptidase N (APN) and related zinc aminopeptidases of the M1 family have not yet been clearly established.  The specific recognition of the N-terminal amino acid of the substrates by the enzymes has been proposed to involve either the complexation of the free amino group by the catalytic zinc ion or an interaction with an anionic binding site, which could be constituted by an aspartate or glutamate residue.  To investigate the existence of such an ionic binding site, site-directed mutagenesis expts. have been performed on acidic residues of pig APN.  Given that aminopeptidases of the M1 family are likely to have a common mechanism of action, only strictly conserved residues were mutated.  As compared to the wild-type enzyme, the mutation D220E led only to slight modifications in the kinetic parameters of the enzyme and in the Ki values of various inhibitors, indicating that this residue is not critically involved in the hydrolytic mechanism.  In contrast, the mutations E350Q and E350D induced a large decrease in enzyme activity, essentially due to modifications in kcat, whereas the E350A mutation led to an almost completely inactive enzyme.  Moreover, among the inhibitors tested, only those acting as transition state analogs showed significant increases in their Ki values.  These data are in favor of E350 belonging to the "anionic binding site" in APN.  A mechanism of action, derived from that of thermolysin, is proposed for these aminopeptidases, which explains the importance of E350 in transition state formation, rather than in the Michaelis complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6gCDhJijok7Vg90H21EOLACvtfcHk0lio-b840lmLjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXotVKis7c%253D&md5=d02d8f5fb796fda649d314a23cd58d11</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fbi971705p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi971705p%26sid%3Dliteratum%253Aachs%26aulast%3DLuciani%26aufirst%3DN.%26aulast%3DMarie-Claire%26aufirst%3DC.%26aulast%3DRuffet%26aufirst%3DE.%26aulast%3DBeaumont%26aufirst%3DA.%26aulast%3DRoques%26aufirst%3DB.%2BP.%26aulast%3DFournie-Zaluski%26aufirst%3DM.%2BC.%26atitle%3DCharacterization%2520of%2520Glu350%2520as%2520a%2520critical%2520residue%2520involved%2520in%2520the%2520N-terminal%2520amine%2520binding%2520site%2520of%2520aminopeptidase%2520N%2520%2528EC%25203.4.11.2%2529%253A%2520insights%2520into%2520its%2520mechanism%2520of%2520action%26jtitle%3DBiochemistry%26date%3D1998%26volume%3D37%26spage%3D686%26epage%3D692%26doi%3D10.1021%2Fbi971705p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Graham, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smiley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nairn, R.</span></span> <span> </span><span class="NLM_article-title">Characteristics of a human cell line transformed by DNA from human adenovirus type 5</span>. <i>J. Gen. Virol.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">59</span>– <span class="NLM_lpage">74</span>, <span class="refDoi"> DOI: 10.1099/0022-1317-36-1-59</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1099%2F0022-1317-36-1-59" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=886304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A280%3ADyaE2s3ivFGmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1977&pages=59-74&author=F.+L.+Grahamauthor=J.+Smileyauthor=W.+C.+Russellauthor=R.+Nairn&title=Characteristics+of+a+human+cell+line+transformed+by+DNA+from+human+adenovirus+type+5&doi=10.1099%2F0022-1317-36-1-59"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Characteristics of a human cell line transformed by DNA from human adenovirus type 5</span></div><div class="casAuthors">Graham F L; Smiley J; Russell W C; Nairn R</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of general virology</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">59-74</span>
        ISSN:<span class="NLM_cas:issn">0022-1317</span>.
    </div><div class="casAbstract">Human embryonic kidney cells have been transformed by exposing cells to sheared fragments of adenovirus type 5 DNA.  The transformed cells (designated 293 cells) exhibited many of the characteristics of transformation including the elaboration of a virus-specific tumour antigen.  Analysis of the polypeptides synthesized in the 293 cells by labelling with 35S-methionine and SDS PAGE showed a variable pattern of synthesis, different in a number of respects from that seen in otheruman cells.  On labelling the surface of cells by lactoperoxidase catalysed radio-iodination, the absence of a labelled polypeptide analogous to the 250 K (LETS) glycoprotein was noted.  Hybridization of labelled cellular RNA with restriction fragments of adenovirus type 5 DNA indicated transcription of a portion of the adenovirus genome at the conventional left hand end.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSm6_NOgLpQVh0LsK7uFuY7fW6udTcc2eZ0AB8jlsSA7rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaE2s3ivFGmtQ%253D%253D&md5=f1dc59399d0bbb5d89889cc5573fcdd8</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1099%2F0022-1317-36-1-59&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252F0022-1317-36-1-59%26sid%3Dliteratum%253Aachs%26aulast%3DGraham%26aufirst%3DF.%2BL.%26aulast%3DSmiley%26aufirst%3DJ.%26aulast%3DRussell%26aufirst%3DW.%2BC.%26aulast%3DNairn%26aufirst%3DR.%26atitle%3DCharacteristics%2520of%2520a%2520human%2520cell%2520line%2520transformed%2520by%2520DNA%2520from%2520human%2520adenovirus%2520type%25205%26jtitle%3DJ.%2520Gen.%2520Virol.%26date%3D1977%26volume%3D36%26spage%3D59%26epage%3D74%26doi%3D10.1099%2F0022-1317-36-1-59" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, J. F.</span></span> <span> </span><span class="NLM_article-title">Kinetics of the reversible inhibition of enzyme-catalysed reactions by tight-binding inhibitors</span>. <i>Biochim. Biophys. Acta, Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1969</span>,  <span class="NLM_volume"><i>185</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">286</span>, <span class="refDoi"> DOI: 10.1016/0005-2744(69)90420-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2F0005-2744%2869%2990420-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=4980133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaF1MXkvVOqsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=185&publication_year=1969&pages=269-286&author=J.+F.+Morrison&title=Kinetics+of+the+reversible+inhibition+of+enzyme-catalysed+reactions+by+tight-binding+inhibitors&doi=10.1016%2F0005-2744%2869%2990420-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics of the reversible inhibition of enzyme-catalyzed reactions by tight-binding inhibitors</span></div><div class="casAuthors">Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Enzymology</span>
        (<span class="NLM_cas:date">1969</span>),
    <span class="NLM_cas:volume">185</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">269-86</span>CODEN:
                <span class="NLM_cas:coden">BBEZAD</span>;
        ISSN:<span class="NLM_cas:issn">0924-1086</span>.
    </div><div class="casAbstract">A theoretical kinetic study has been undertaken to det. the kinetic characteristics of enzymic reactions that are reversibly inhibited by a substrate analog. at concns. comparable to that of the enzyme.  A general rate equation, that allows for the combination of such a tight-binding inhibitor with one or more enzyme forms of any reaction mechanism, has been derived and shown to be quadratic with sq. and linear terms in υ which represents the true initial steady-state velocity.  From plots of υ against total enzyme concn. at fixed total inhibitor concn., the concn. of enzyme catalytic sites can be detd. irrespective of the nature of the inhibition and the reaction mechanism.  However, plots of υ (or I/υ) against total inhibitor concn. can be used for this purpose only if the dissocn. const. assocd. with the inhibitor is virtually zero.  Particular initial rate equations have been obtained for the cases in which the tight-binding inhibitor gives rise to competitive, noncompetitive, and uncompetitive inhibition with respect to one of the substrates of a bireactant mechanism.  From these equations it has been deduced that, in the presence of inhibitor, plots of I/υ against the reciprocals of the substrate concns. yield families of concave-down, nonrectangular hyperbolas.  The difficulties assocd. with the detn. of inhibition consts. for tight-binding inhibitors are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpj4s_N2iV3CLVg90H21EOLACvtfcHk0lgROln0JIXR_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF1MXkvVOqsbo%253D&md5=74d467477385488dfbb053358534ada6</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2F0005-2744%2869%2990420-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0005-2744%252869%252990420-3%26sid%3Dliteratum%253Aachs%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DKinetics%2520of%2520the%2520reversible%2520inhibition%2520of%2520enzyme-catalysed%2520reactions%2520by%2520tight-binding%2520inhibitors%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Enzymol.%26date%3D1969%26volume%3D185%26spage%3D269%26epage%3D286%26doi%3D10.1016%2F0005-2744%2869%2990420-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Williams, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, J. F.</span></span> <span> </span><span class="NLM_article-title">The kinetics of reversible tight-binding inhibition</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">437</span>– <span class="NLM_lpage">467</span>, <span class="refDoi"> DOI: 10.1016/0076-6879(79)63019-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2F0076-6879%2879%2963019-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=502865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaL3cXovF2ntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=1979&pages=437-467&author=J.+W.+Williamsauthor=J.+F.+Morrison&title=The+kinetics+of+reversible+tight-binding+inhibition&doi=10.1016%2F0076-6879%2879%2963019-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">The kinetics of reversible tight-binding inhibition</span></div><div class="casAuthors">Williams, Jeffrey W.; Morrison, John F.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">1979</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">Enzyme Kinet. Mech., Part A</span>),
    <span class="NLM_cas:pages">437-67</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    </div><div class="casAbstract">In addn. to a general discussion of the kinetics of reversible tight-binding inhibitors, which exert their effect at a concn. comparable to that of the enzyme, and for which steady-state theory is used, exptl. results are given of the tight-binding inhibition of creatine kinase by Eu3+ (actually Eu-ADP) under steady-state conditions.  The inhibition of creatine kinase by fixed concns. of Eu3+ in the presence of varying concns. of Mg-ADP indicated that the inhibition is competitive, suggesting that Eu-ADP acts as a structural analog of Mg-ADP.  The inhibition was noncompetitive with respect to creatine phosphate.  The dissocn. consts. for binding of Eu-ADP to the free enzyme and the enzyme-phosphocreatine complex were 1.5 and 2.2 nM, resp., as detd. from Ki values.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2CwV3voXt27Vg90H21EOLACvtfcHk0ljmU5nJ1ysoUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXovF2ntw%253D%253D&md5=a770f91b51661c9502d0aca44a24e1c7</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2F0076-6879%2879%2963019-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0076-6879%252879%252963019-7%26sid%3Dliteratum%253Aachs%26aulast%3DWilliams%26aufirst%3DJ.%2BW.%26aulast%3DMorrison%26aufirst%3DJ.%2BF.%26atitle%3DThe%2520kinetics%2520of%2520reversible%2520tight-binding%2520inhibition%26jtitle%3DMethods%2520Enzymol.%26date%3D1979%26volume%3D63%26spage%3D437%26epage%3D467%26doi%3D10.1016%2F0076-6879%2879%2963019-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bevan, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, R. S.</span></span> <span> </span><span class="NLM_article-title">A high-throughput screening method for the determination of aqueous drug solubility using laser nephelometry in microtiter plates</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">1781</span>– <span class="NLM_lpage">1787</span>, <span class="refDoi"> DOI: 10.1021/ac9912247</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac9912247" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhsFyhsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2000&pages=1781-1787&author=C.+D.+Bevanauthor=R.+S.+Lloyd&title=A+high-throughput+screening+method+for+the+determination+of+aqueous+drug+solubility+using+laser+nephelometry+in+microtiter+plates&doi=10.1021%2Fac9912247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">A High-Throughput Screening Method for the Determination of Aqueous Drug Solubility Using Laser Nephelometry in Microtiter Plates</span></div><div class="casAuthors">Bevan, Christopher D.; Lloyd, Richard S.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1781-1787</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The detn. of aq. soly. in a high-throughput screening environment is invaluable in the selection of the most promising potential drug candidates.  We describe a fast method based on laser nephelometry that can det. the soly. of potential drug candidates (usually from combinatorial chem.) supplied as DMSO solns. in 96-well plates.  In the sample, the percent DMSO is kept const. allowing direct comparison of results.  The nephelometric method has been shown to produce results equiv. to those produced by an HPLC method and to be largely unaffected by colored solns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7jLYnIgg-_bVg90H21EOLACvtfcHk0ljmU5nJ1ysoUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhsFyhsb4%253D&md5=5023317d334eafd7a60860e3692c477d</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fac9912247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac9912247%26sid%3Dliteratum%253Aachs%26aulast%3DBevan%26aufirst%3DC.%2BD.%26aulast%3DLloyd%26aufirst%3DR.%2BS.%26atitle%3DA%2520high-throughput%2520screening%2520method%2520for%2520the%2520determination%2520of%2520aqueous%2520drug%2520solubility%2520using%2520laser%2520nephelometry%2520in%2520microtiter%2520plates%26jtitle%3DAnal.%2520Chem.%26date%3D2000%26volume%3D72%26spage%3D1781%26epage%3D1787%26doi%3D10.1021%2Fac9912247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shalaeva, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tupper, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, F.</span></span> <span> </span><span class="NLM_article-title">ElogD(oct): a tool for lipophilicity determination in drug discovery. 2. Basic and neutral compounds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">2490</span>– <span class="NLM_lpage">2497</span>, <span class="refDoi"> DOI: 10.1021/jm0100990</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0100990" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD3MXksFOmtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=2490-2497&author=F.+Lombardoauthor=M.+Y.+Shalaevaauthor=K.+A.+Tupperauthor=F.+Gao&title=ElogD%28oct%29%3A+a+tool+for+lipophilicity+determination+in+drug+discovery.+2.+Basic+and+neutral+compounds&doi=10.1021%2Fjm0100990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">ElogDoct: A Tool for Lipophilicity Determination in Drug Discovery. 2. Basic and Neutral Compounds</span></div><div class="casAuthors">Lombardo, Franco; Shalaeva, Marina Y.; Tupper, Karl A.; Gao, Feng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2490-2497</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present an RP-HPLC method for the detn. of the octanol-water distribution coeffs. at pH 7.4, as log Doct7.4 values, for neutral and basic drugs, which combines ease of operation with high accuracy.  The method is shown to work for a training set of 90 mols. comprised largely of drugs, and it was also applied to a test set of 10 proprietary compds.  This work expands the applicability of the method presented in our earlier report, for the detn. of logPoct for neutral compds. (J. Med. Chem. 2000, 43, 2922-2928), and it offers the same general features but widens the scope.  Generally, the method (i) is compd. sparing (≤1 mL of a 50-100 μg/mL soln. needed), (ii) is insensitive to concn. and phase ratio effects obsd. in some shake-flask detns., (iii) is amenable to rapid detns. (≤20 min on av.), (iv) is insensitive to impurities, (v) possesses a wide lipophilicity range (>7 log Doct7.4 units), and (vi) offers a good accuracy, (vii) an excellent reproducibility, (viii) and an excellent potential for automation.  To the best of our knowledge, a similar performance, on a set of noncongeneric drugs, has not been previously reported.  We refer to the value generated via this method as ElogDoct.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrynjvQ2-0sibVg90H21EOLACvtfcHk0ljmU5nJ1ysoUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXksFOmtbc%253D&md5=e390cfeacbb1cab5660909eccfeb7eb2</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fjm0100990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0100990%26sid%3Dliteratum%253Aachs%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DShalaeva%26aufirst%3DM.%2BY.%26aulast%3DTupper%26aufirst%3DK.%2BA.%26aulast%3DGao%26aufirst%3DF.%26atitle%3DElogD%2528oct%2529%253A%2520a%2520tool%2520for%2520lipophilicity%2520determination%2520in%2520drug%2520discovery.%25202.%2520Basic%2520and%2520neutral%2520compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D2490%26epage%3D2497%26doi%3D10.1021%2Fjm0100990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Schrödinger</span>.  <i>The PyMOL Molecular Graphics System</i>, version 1.8; <span class="NLM_publisher-name">Schrödinger, LLC</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=Schr%C3%B6dinger&title=The+PyMOL+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolf, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Development and testing of a general amber force field</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1157</span>– <span class="NLM_lpage">1174</span>, <span class="refDoi"> DOI: 10.1002/jcc.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+a+general+amber+force+field&doi=10.1002%2Fjcc.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0ljmU5nJ1ysoUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520a%2520general%2520amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174%26doi%3D10.1002%2Fjcc.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jakalian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bush, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jack, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, C. I.</span></span> <span> </span><span class="NLM_article-title">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: I. Method</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">132</span>– <span class="NLM_lpage">146</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1096-987X(20000130)21:2<132::AID-JCC5>3.0.CO;2-P</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD3cXkt1SgsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2000&pages=132-146&author=A.+Jakalianauthor=B.+L.+Bushauthor=D.+B.+Jackauthor=C.+I.+Bayly&title=Fast%2C+efficient+generation+of+high-quality+atomic+charges.+AM1-BCC+model%3A+I.+Method&doi=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: I. Method</span></div><div class="casAuthors">Jakalian, Araz; Bush, Bruce L.; Jack, David B.; Bayly, Christopher I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">132-146</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The AM1-BCC method quickly and efficiently generates high-quality at. charges for use in condensed-phase simulations.  The underlying features of the electron distribution including formal charge and delocalization are first captured by AM1 at. charges for the individual mol.  Bond charge corrections (BCCs), which have been parameterized against the HF/6-31G* electrostatic potential (ESP) of a training set of compds. contg. relevant functional groups, are then added using a formalism identical to the consensus BCI (bond charge increment) approach.  As a proof of the concept, we fit BCCs simultaneously to 45 compds. including O-, N-, and S-contg. functionalities, aroms., and heteroaroms., using only 41 BCC parameters.  AM1-BCC yields charge sets of comparable quality to HF/6-31G* ESP-derived charges in a fraction of the time while reducing instabilities in the at. charges compared to direct ESP-fit methods.  We then apply the BCC parameters to a small "test set" consisting of aspirin, D-glucose, and eryodictyol; the AM1-BCC model again provides at. charges of quality comparable with HF/6-31G* RESP charges, as judged by an increase of only 0.01 to 0.02 at. units in the root-mean-square (RMS) error in ESP.  Based on these encouraging results, we intend to parameterize the AM1-BCC model to provide a consistent charge model for any org. or biol. mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruxZWGx7IgLbVg90H21EOLACvtfcHk0lhbKGiV8E8WFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXkt1SgsQ%253D%253D&md5=aa4774739d3491dbc8dd95ed1948d856</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%252820000130%252921%253A2%253C132%253A%253AAID-JCC5%253E3.0.CO%253B2-P%26sid%3Dliteratum%253Aachs%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DBush%26aufirst%3DB.%2BL.%26aulast%3DJack%26aufirst%3DD.%2BB.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26atitle%3DFast%252C%2520efficient%2520generation%2520of%2520high-quality%2520atomic%2520charges.%2520AM1-BCC%2520model%253A%2520I.%2520Method%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2000%26volume%3D21%26spage%3D132%26epage%3D146%26doi%3D10.1002%2F%28SICI%291096-987X%2820000130%2921%3A2%3C132%3A%3AAID-JCC5%3E3.0.CO%3B2-P" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jakalian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jack, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, C. I.</span></span> <span> </span><span class="NLM_article-title">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. Parameterization and validation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1623</span>– <span class="NLM_lpage">1641</span>, <span class="refDoi"> DOI: 10.1002/jcc.10128</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1002%2Fjcc.10128" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=12395429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2002&pages=1623-1641&author=A.+Jakalianauthor=D.+B.+Jackauthor=C.+I.+Bayly&title=Fast%2C+efficient+generation+of+high-quality+atomic+charges.+AM1-BCC+model%3A+II.+Parameterization+and+validation&doi=10.1002%2Fjcc.10128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. parameterization and validation</span></div><div class="casAuthors">Jakalian, Araz; Jack, David B.; Bayly, Christopher I.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1623-1641</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We present the first global parameterization and validation of a novel charge model, called AM1-BCC, which quickly and efficiently generates high-quality at. charges for computer simulations of org. mols. in polar media.  The goal of the charge model is to produce at. charges that emulate the HF/6-31G* electrostatic potential (ESP) of a mol.  Underlying electronic structure features, including formal charge and electron delocalization, are first captured by AM1 population charges; simple additive bond charge corrections (BCCs) are then applied to these AM1 at. charges to produce the AM1-BCC charges.  The parameterization of BCCs was carried out by fitting to the HF/6-31G* ESP of a training set of >2700 mols.  Most org. functional groups and their combinations were sampled, as well as an extensive variety of cyclic and fused bicyclic heteroaryl systems.  The resulting BCC parameters allow the AM1-BCC charging scheme to handle virtually all types of org. compds. listed in The Merck Index and the NCI Database.  Validation of the model was done through comparisons of hydrogen-bonded dimer energies and relative free energies of solvation using AM1-BCC charges in conjunction with the 1994 Cornell et al. forcefield for AMBER.  Homo-dimer and hetero-dimer hydrogen-bond energies of a diverse set of org. mols. were reproduced to within 0.95 kcal/mol RMS deviation from the ab initio values, and for DNA dimers the energies were within 0.9 kcal/mol RMS deviation from ab initio values.  The calcd. relative free energies of solvation for a diverse set of monofunctional isosteres were reproduced to within 0.69 kcal/mol of expt.  In all these validation tests, AMBER with the AM1-BCC charge model maintained a correlation coeff. above 0.96.  Thus, the parameters presented here for use with the AM1-BCC method present a fast, accurate, and robust alternative to HF/6-31G* ESP-fit charges for general use with the AMBER force field in computer simulations involving org. small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpn9DkJ_gMzA7Vg90H21EOLACvtfcHk0lhbKGiV8E8WFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosF2rt74%253D&md5=3f2b738617bb17b2898f7ac4d751d7ec</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1002%2Fjcc.10128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.10128%26sid%3Dliteratum%253Aachs%26aulast%3DJakalian%26aufirst%3DA.%26aulast%3DJack%26aufirst%3DD.%2BB.%26aulast%3DBayly%26aufirst%3DC.%2BI.%26atitle%3DFast%252C%2520efficient%2520generation%2520of%2520high-quality%2520atomic%2520charges.%2520AM1-BCC%2520model%253A%2520II.%2520Parameterization%2520and%2520validation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2002%26volume%3D23%26spage%3D1623%26epage%3D1641%26doi%3D10.1002%2Fjcc.10128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fiser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Šali, A.</span></span> <span> </span><span class="NLM_article-title">Modeller: generation and refinement of homology-based protein structure models</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>374</i></span>,  <span class="NLM_fpage">461</span>– <span class="NLM_lpage">491</span>, <span class="refDoi"> DOI: 10.1016/S0076-6879(03)74020-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2FS0076-6879%2803%2974020-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=14696385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotlyntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=374&publication_year=2003&pages=461-491&author=A.+Fiserauthor=A.+%C5%A0ali&title=Modeller%3A+generation+and+refinement+of+homology-based+protein+structure+models&doi=10.1016%2FS0076-6879%2803%2974020-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">MODELLER: generation and refinement of homology-based protein structure models</span></div><div class="casAuthors">Fiser, Andras; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">374</span>
        (<span class="NLM_cas:issue">Macromolecular Crystallography, Part D</span>),
    <span class="NLM_cas:pages">461-491</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Comparative modeling usually consists of five steps: search for related protein structures, selection of one or more templates, target-template alignment, model building, and model evaluation.  Generic considerations in all five steps of comparative modeling are described.  The considerations in practice by discussing three applications of the program MODELLER to specific modeling problems are also illustrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL_0iXCSdXLrVg90H21EOLACvtfcHk0lhbKGiV8E8WFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotlyntA%253D%253D&md5=5b21fff95e9f7158501051f96f7beba3</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2803%2974020-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0076-6879%252803%252974020-8%26sid%3Dliteratum%253Aachs%26aulast%3DFiser%26aufirst%3DA.%26aulast%3D%25C5%25A0ali%26aufirst%3DA.%26atitle%3DModeller%253A%2520generation%2520and%2520refinement%2520of%2520homology-based%2520protein%2520structure%2520models%26jtitle%3DMethods%2520Enzymol.%26date%3D2003%26volume%3D374%26spage%3D461%26epage%3D491%26doi%3D10.1016%2FS0076-6879%2803%2974020-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gasteiger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsili, M.</span></span> <span> </span><span class="NLM_article-title">Iterative partial equalization of orbital electronegativity—a rapid access to atomic charges</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">1980</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">3219</span>– <span class="NLM_lpage">3228</span>, <span class="refDoi"> DOI: 10.1016/0040-4020(80)80168-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2F0040-4020%2880%2980168-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaL3MXhslCjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1980&pages=3219-3228&author=J.+Gasteigerauthor=M.+Marsili&title=Iterative+partial+equalization+of+orbital+electronegativity%E2%80%94a+rapid+access+to+atomic+charges&doi=10.1016%2F0040-4020%2880%2980168-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Iterative partial equalization of orbital electronegativity: a rapid access to atomic charges</span></div><div class="casAuthors">Gasteiger, Johann; Marsili, Mario</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">3219-22</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    </div><div class="casAbstract">A method is presented for the rapid calcn. of at. charges in σ-bonded and nonconjugated π-systems (alkyl halides and oxoalkanes, resp.), considering only the connectivities of the atoms.  Atoms are characterized by their orbital electronegativities.  Partial equalization of orbital electronegativity is obtained by an iterative procedure.  Excellent correlations of the at. charges with core-electron binding energies and with acidity consts. are obsd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKA3wil1SiWrVg90H21EOLACvtfcHk0lgD0yMRUS1N2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3MXhslCjtbs%253D&md5=90c580857e6001482bee92450278fe9f</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2F0040-4020%2880%2980168-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0040-4020%252880%252980168-2%26sid%3Dliteratum%253Aachs%26aulast%3DGasteiger%26aufirst%3DJ.%26aulast%3DMarsili%26aufirst%3DM.%26atitle%3DIterative%2520partial%2520equalization%2520of%2520orbital%2520electronegativity%25E2%2580%2594a%2520rapid%2520access%2520to%2520atomic%2520charges%26jtitle%3DTetrahedron%26date%3D1980%26volume%3D36%26spage%3D3219%26epage%3D3228%26doi%3D10.1016%2F0040-4020%2880%2980168-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekhar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madura, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Impey, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M. L.</span></span> <span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">926</span>– <span class="NLM_lpage">935</span>, <span class="refDoi"> DOI: 10.1063/1.445869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+W.+Impeyauthor=M.+L.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water&doi=10.1063%2F1.445869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0lgD0yMRUS1N2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%2BW.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935%26doi%3D10.1063%2F1.445869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheatham, T. E.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merz, K. M.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Onufriev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simmerling, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, R. J.</span></span> <span> </span><span class="NLM_article-title">The Amber biomolecular simulation programs</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1668</span>– <span class="NLM_lpage">1688</span>, <span class="refDoi"> DOI: 10.1002/jcc.20290</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1002%2Fjcc.20290" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=16200636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1668-1688&author=D.+A.+Caseauthor=T.+E.+Cheathamauthor=T.+Dardenauthor=H.+Gohlkeauthor=R.+Luoauthor=K.+M.+Merzauthor=A.+Onufrievauthor=C.+Simmerlingauthor=B.+Wangauthor=R.+J.+Woods&title=The+Amber+biomolecular+simulation+programs&doi=10.1002%2Fjcc.20290"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">The amber biomolecular simulation programs</span></div><div class="casAuthors">Case, David A.; Cheatham, Thomas E., III; Darden, Tom; Gohlke, Holger; Luo, Ray; Merz, Kenneth M., Jr.; Onufriev, Alexey; Simmerling, Carlos; Wang, Bing; Woods, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1668-1688</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The authors describe the development, current features, and some directions for future development of the Amber package of computer programs.  This package evolved from a program that was constructed in the late 1970s to do Assisted Model Building with Energy Refinement, and now contains a group of programs embodying a no. of powerful tools of modern computational chem., focused on mol. dynamics and free energy calcns. of proteins, nucleic acids, and carbohydrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre0tZP_Umb7rVg90H21EOLACvtfcHk0lgD0yMRUS1N2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbM&md5=93be29ff894bab96c783d24e9886c7d0</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20290%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DGohlke%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DR.%26aulast%3DMerz%26aufirst%3DK.%2BM.%26aulast%3DOnufriev%26aufirst%3DA.%26aulast%3DSimmerling%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWoods%26aufirst%3DR.%2BJ.%26atitle%3DThe%2520Amber%2520biomolecular%2520simulation%2520programs%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1668%26epage%3D1688%26doi%3D10.1002%2Fjcc.20290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gumbart, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tajkhorshid, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villa, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chipot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skeel, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kale, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulten, K.</span></span> <span> </span><span class="NLM_article-title">Scalable molecular dynamics with NAMD</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1781</span>– <span class="NLM_lpage">1802</span>, <span class="refDoi"> DOI: 10.1002/jcc.20289</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1002%2Fjcc.20289" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=16222654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2005&pages=1781-1802&author=J.+C.+Phillipsauthor=R.+Braunauthor=W.+Wangauthor=J.+Gumbartauthor=E.+Tajkhorshidauthor=E.+Villaauthor=C.+Chipotauthor=R.+D.+Skeelauthor=L.+Kaleauthor=K.+Schulten&title=Scalable+molecular+dynamics+with+NAMD&doi=10.1002%2Fjcc.20289"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Scalable molecular dynamics with NAMD</span></div><div class="casAuthors">Phillips, James C.; Braun, Rosemary; Wang, Wei; Gumbart, James; Tajkhorshid, Emad; Villa, Elizabeth; Chipot, Christophe; Skeel, Robert D.; Kale, Laxmikant; Schulten, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">1781-1802</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">NAMD is a parallel mol. dynamics code designed for high-performance simulation of large biomol. systems.  NAMD scales to hundreds of processors on high-end parallel platforms, as well as tens of processors on low-cost commodity clusters, and also runs on individual desktop and laptop computers.  NAMD works with AMBER and CHARMM potential functions, parameters, and file formats.  This article, directed to novices as well as experts, first introduces concepts and methods used in the NAMD program, describing the classical mol. dynamics force field, equations of motion, and integration methods along with the efficient electrostatics evaluation algorithms employed and temp. and pressure controls used.  Features for steering the simulation across barriers and for calcg. both alchem. and conformational free energy differences are presented.  The motivations for and a roadmap to the internal design of NAMD, implemented in C++ and based on Charm++ parallel objects, are outlined.  The factors affecting the serial and parallel performance of a simulation are discussed.  Finally, typical NAMD use is illustrated with representative applications to a small, a medium, and a large biomol. system, highlighting particular features of NAMD, for example, the Tcl scripting language.  The article also provides a list of the key features of NAMD and discusses the benefits of combining NAMD with the mol. graphics/sequence anal. software VMD and the grid computing/collab. software BioCoRE.  NAMD is distributed free of charge with source code at www.ks.uiuc.edu.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7fq_yG00T1rVg90H21EOLACvtfcHk0lg6JdID7l7Rmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1SlsbbJ&md5=189051128443b547f4300a1b8fb0e034</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20289&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20289%26sid%3Dliteratum%253Aachs%26aulast%3DPhillips%26aufirst%3DJ.%2BC.%26aulast%3DBraun%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DGumbart%26aufirst%3DJ.%26aulast%3DTajkhorshid%26aufirst%3DE.%26aulast%3DVilla%26aufirst%3DE.%26aulast%3DChipot%26aufirst%3DC.%26aulast%3DSkeel%26aufirst%3DR.%2BD.%26aulast%3DKale%26aufirst%3DL.%26aulast%3DSchulten%26aufirst%3DK.%26atitle%3DScalable%2520molecular%2520dynamics%2520with%2520NAMD%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2005%26volume%3D26%26spage%3D1781%26epage%3D1802%26doi%3D10.1002%2Fjcc.20289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryckaert, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccotti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span> <span> </span><span class="NLM_article-title">Numerical integration of the cartesian equations of motion of a system with constraints: molecular dynamics of n-alkanes</span>. <i>J. Comput. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/0021-9991(77)90098-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1016%2F0021-9991%2877%2990098-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=327-341&author=J.-P.+Ryckaertauthor=G.+Ciccottiauthor=H.+J.+C.+Berendsen&title=Numerical+integration+of+the+cartesian+equations+of+motion+of+a+system+with+constraints%3A+molecular+dynamics+of+n-alkanes&doi=10.1016%2F0021-9991%2877%2990098-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Numerical integration of the Cartesian equations of motion of a system with constraints:  molecular dynamics of n-alkanes</span></div><div class="casAuthors">Ryckaert, Jean Paul; Ciccotti, Giovanni; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-41</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    </div><div class="casAbstract">A numerical algorithm integrating the 3N Cartesian equation of motion of a system of N points subject to holonomic constraints is applied to mol. dynamics simulation of a liq. of 64 butane mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPlhG1U3Uw7Vg90H21EOLACvtfcHk0lg6JdID7l7Rmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D&md5=b4aecddfde149117813a5ea4f5353ce2</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990098-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990098-5%26sid%3Dliteratum%253Aachs%26aulast%3DRyckaert%26aufirst%3DJ.-P.%26aulast%3DCiccotti%26aufirst%3DG.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DNumerical%2520integration%2520of%2520the%2520cartesian%2520equations%2520of%2520motion%2520of%2520a%2520system%2520with%2520constraints%253A%2520molecular%2520dynamics%2520of%2520n-alkanes%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D327%26epage%3D341%26doi%3D10.1016%2F0021-9991%2877%2990098-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kawasaki, K.</span></span> <span> </span><span class="NLM_article-title">Simple derivations of generalized linear and nonlinear Langevin equations</span>. <i>J. Phys. A: Math., Nucl. Gen.</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1289</span>, <span class="refDoi"> DOI: 10.1088/0305-4470/6/9/004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1088%2F0305-4470%2F6%2F9%2F004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1973&pages=1289&author=K.+Kawasaki&title=Simple+derivations+of+generalized+linear+and+nonlinear+Langevin+equations&doi=10.1088%2F0305-4470%2F6%2F9%2F004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1088%2F0305-4470%2F6%2F9%2F004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1088%252F0305-4470%252F6%252F9%252F004%26sid%3Dliteratum%253Aachs%26aulast%3DKawasaki%26aufirst%3DK.%26atitle%3DSimple%2520derivations%2520of%2520generalized%2520linear%2520and%2520nonlinear%2520Langevin%2520equations%26jtitle%3DJ.%2520Phys.%2520A%253A%2520Math.%252C%2520Nucl.%2520Gen.%26date%3D1973%26volume%3D6%26spage%3D1289%26doi%3D10.1088%2F0305-4470%2F6%2F9%2F004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Postma, J. P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van
Gunsteren, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiNola, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haak, J. R.</span></span> <span> </span><span class="NLM_article-title">Molecular dynamics with coupling to an external bath</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">3684</span>– <span class="NLM_lpage">3690</span>, <span class="refDoi"> DOI: 10.1063/1.448118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1063%2F1.448118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaL2cXmtlGksbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=1984&pages=3684-3690&author=H.+J.+C.+Berendsenauthor=J.+P.+M.+Postmaauthor=W.+F.+van%0AGunsterenauthor=A.+DiNolaauthor=J.+R.+Haak&title=Molecular+dynamics+with+coupling+to+an+external+bath&doi=10.1063%2F1.448118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics with coupling to an external bath</span></div><div class="casAuthors">Berendsen, H. J. C.; Postma, J. P. M.; Van Gunsteren, W. F.; DiNola, A.; Haak, J. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3684-90</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">In mol. dynamics (MD) simulations, the need often arises to maintain such parameters as temp. or pressure rather than energy and vol., or to impose gradients for studying transport properties in nonequil. MD.  A method is described to realize coupling to an external bath with const. temp. or pressure with adjustable time consts. for the coupling.  The method is easily extendable to other variables and to gradients, and can be applied also to polyat. mols. involving internal constraints.  The influence of coupling time consts. on dynamical variables is evaluated.  A leap-frog algorithm is presented for the general case involving constraints with coupling to both a const. temp. and a const. pressure bath.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoecrpUZWkh37Vg90H21EOLACvtfcHk0lg6JdID7l7Rmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXmtlGksbY%253D&md5=5510dc00297d63b91ee3a7a4ae5aacb1</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1063%2F1.448118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.448118%26sid%3Dliteratum%253Aachs%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DPostma%26aufirst%3DJ.%2BP.%2BM.%26aulast%3Dvan%2BGunsteren%26aufirst%3DW.%2BF.%26aulast%3DDiNola%26aufirst%3DA.%26aulast%3DHaak%26aufirst%3DJ.%2BR.%26atitle%3DMolecular%2520dynamics%2520with%2520coupling%2520to%2520an%2520external%2520bath%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1984%26volume%3D81%26spage%3D3684%26epage%3D3690%26doi%3D10.1063%2F1.448118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Essmann, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perera, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkowitz, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darden, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, L. G.</span></span> <span> </span><span class="NLM_article-title">A smooth particle mesh Ewald method</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">8577</span>– <span class="NLM_lpage">8593</span>, <span class="refDoi"> DOI: 10.1063/1.470117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=10.1063%2F1.470117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;key=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=1995&pages=8577-8593&author=U.+Essmannauthor=L.+Pereraauthor=M.+L.+Berkowitzauthor=T.+Dardenauthor=H.+Leeauthor=L.+G.+Pedersen&title=A+smooth+particle+mesh+Ewald+method&doi=10.1063%2F1.470117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">A smooth particle mesh Ewald method</span></div><div class="casAuthors">Essmann, Ulrich; Perera, Lalith; Berkowitz, Max L.; Darden, Tom; Lee, Hsing; Pedersen, Lee G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8577-93</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The previously developed particle mesh Ewald method is reformulated in terms of efficient B-spline interpolation of the structure factors.  This reformulation allows a natural extension of the method to potentials of the form 1/rp with p ≥ 1.  Furthermore, efficient calcn. of the virial tensor follows.  Use of B-splines in the place of Lagrange interpolation leads to analytic gradients as well as a significant improvement in the accuracy.  The authors demonstrate that arbitrary accuracy can be achieved, independent of system size N, at a cost that scales as N log(N).  For biomol. systems with many thousands of atoms and this method permits the use of Ewald summation at a computational cost comparable to that of a simple truncation method of 10 Å or less.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_Bp1XlSY32LVg90H21EOLACvtfcHk0lg6JdID7l7Rmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXptlehtrw%253D&md5=092a679dd3bee08da28df41e302383a7</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1063%2F1.470117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.470117%26sid%3Dliteratum%253Aachs%26aulast%3DEssmann%26aufirst%3DU.%26aulast%3DPerera%26aufirst%3DL.%26aulast%3DBerkowitz%26aufirst%3DM.%2BL.%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DPedersen%26aufirst%3DL.%2BG.%26atitle%3DA%2520smooth%2520particle%2520mesh%2520Ewald%2520method%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1995%26volume%3D103%26spage%3D8577%26epage%3D8593%26doi%3D10.1063%2F1.470117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZX4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZX4','PDB','4ZX4'); return false;">PDB: 4ZX4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYQ','PDB','4FYQ'); return false;">PDB: 4FYQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYR','PDB','4FYR'); return false;">PDB: 4FYR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYT" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYT','PDB','4FYT'); return false;">PDB: 4FYT</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYS" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYS','PDB','4FYS'); return false;">PDB: 4FYS</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i125"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00757">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_07340"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00757">10.1021/acs.jmedchem.9b00757</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Synthetic procedures (compounds <b>10</b>–<b>16</b>); MMP enzymatic analysis; RMSD of the Cα and ligands of <b>1</b>, <b>6ad</b>, <b>6ae</b>, and <b>6ag</b> along the molecular modeling trajectory; hydrogen-bond occupancy analysis of <b>1</b>, <b>6ad</b>, <b>6ae</b>, and <b>6ag</b> with APN; binding scores for the docked poses of <b>1</b>, <b>6ad</b>, <b>6ae</b>, and <b>6ag</b> into APN; and HPLC traces of biologically evaluated compounds (<b>6a</b>–<b>6az</b> and <b>9</b>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_002.csv">CSV</a>)</p></li><li><p class="inline">PDB file for compound <b>1</b> docking (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_003.pdb">PDB</a>)</p></li><li><p class="inline">MD simulations of APN <b>6ad</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_004.mp4">MP4</a>)</p></li><li><p class="inline">MD simulations of APN <b>6ae</b> ( <a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_005.mp4">MP4</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_001.pdf">jm9b00757_si_001.pdf (4.92 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_002.csv">jm9b00757_si_002.csv (15.36 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_003.pdb">jm9b00757_si_003.pdb (822.47 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_004.mp4">jm9b00757_si_004.mp4 (44.15 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00757/suppl_file/jm9b00757_si_005.mp4">jm9b00757_si_005.mp4 (44.14 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The atomic coordinates for the X-ray crystal structures used as templates for molecular docking are available from the Protein Data Bank (PDB). <i>Pf</i>A-M1-<b>1</b> PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4ZX4">4ZX4</a> and human APN PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FYQ">4FYQ</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00757&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-15%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00757" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00757" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679913c8cfa03d36","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
